# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.3 (2017-03-06) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |(EN)                         |
|collate  |en_US.UTF-8                  |
|tz       |Zulu                         |
|date     |2017-03-15                   |

## Packages

|package        |*  |version    |date       |source                           |
|:--------------|:--|:----------|:----------|:--------------------------------|
|assertthat     |   |0.1        |2013-12-06 |cran (@0.1)                      |
|BH             |   |1.62.0-1   |2016-11-19 |cran (@1.62.0-)                  |
|bindrcpp       |   |0.1        |2016-12-11 |cran (@0.1)                      |
|bit64          |   |0.9-5      |2015-07-05 |cran (@0.9-5)                    |
|covr           |   |2.2.2      |2017-01-05 |cran (@2.2.2)                    |
|DBI            |   |0.6        |2017-03-13 |Github (rstats-db/DBI@76d8d65)   |
|dplyr          |   |0.5.0      |2016-06-24 |CRAN (R 3.3.2)                   |
|dtplyr         |   |0.0.1      |2016-06-27 |cran (@0.0.1)                    |
|ggplot2        |   |2.2.1      |2016-12-30 |cran (@2.2.1)                    |
|glue           |   |0.0.0.9000 |2017-03-14 |Github (tidyverse/glue@099b283)  |
|hms            |   |0.3        |2016-11-22 |cran (@0.3)                      |
|knitr          |   |1.15.1     |2016-11-22 |cran (@1.15.1)                   |
|Lahman         |   |5.0-0      |2016-08-27 |cran (@5.0-0)                    |
|magrittr       |   |1.5        |2014-11-22 |CRAN (R 3.3.1)                   |
|microbenchmark |   |1.4-2.1    |2015-11-25 |cran (@1.4-2.1)                  |
|nycflights13   |   |0.2.2      |2017-01-27 |cran (@0.2.2)                    |
|plogr          |   |0.1-1      |2016-09-24 |cran (@0.1-1)                    |
|R6             |   |2.2.0      |2016-10-05 |cran (@2.2.0)                    |
|Rcpp           |   |0.12.9.4   |2017-03-08 |Github (RcppCore/Rcpp@0566d7c)   |
|rlang          |   |0.0.0.9000 |2017-03-15 |Github (hadley/rlang@7c42c80)    |
|rmarkdown      |   |1.3        |2016-12-21 |cran (@1.3)                      |
|RMySQL         |   |0.10.9     |2016-05-08 |cran (@0.10.9)                   |
|RPostgreSQL    |   |0.4-1      |2016-05-08 |cran (@0.4-1)                    |
|RSQLite        |   |1.1-2      |2017-01-08 |cran (@1.1-2)                    |
|testthat       |   |1.0.2.9000 |2017-02-27 |Github (hadley/testthat@b72a228) |
|tibble         |   |1.2        |2016-08-26 |cran (@1.2)                      |

# Check results

129 packages with problems

|package              |version   | errors| warnings| notes|
|:--------------------|:---------|------:|--------:|-----:|
|ameco                |0.2.6     |      1|        0|     1|
|anametrix            |2.0.0     |      1|        0|     0|
|backtestGraphics     |0.1.6     |      1|        0|     0|
|bayesplot            |1.1.0     |      0|        1|     0|
|bibliometrix         |1.5       |      0|        1|     0|
|bioOED               |0.1.1     |      1|        0|     0|
|blscrapeR            |2.1.2     |      1|        0|     0|
|bossMaps             |0.1.0     |      0|        1|     0|
|broom                |0.4.2     |      1|        1|     0|
|carpenter            |0.2.0     |      2|        1|     0|
|chunked              |0.3       |      2|        0|     0|
|clustrd              |1.1.0     |      1|        0|     0|
|codingMatrices       |0.2.2     |      0|        1|     0|
|cometExactTest       |0.1.3     |      0|        1|     0|
|crawl                |2.0.1     |      0|        1|     1|
|dat                  |0.1.0     |      1|        0|     0|
|ddpcr                |1.6       |      1|        0|     0|
|decoder              |1.1.12    |      0|        1|     0|
|DeLorean             |1.2.4     |      0|        1|     1|
|describer            |0.2.0     |      1|        0|     0|
|DiagrammeR           |0.9.0     |      1|        0|     1|
|doctr                |0.2.0     |      1|        1|     0|
|dotwhisker           |0.2.3     |      1|        1|     0|
|dtplyr               |0.0.1     |      1|        0|     0|
|eechidna             |0.1       |      0|        1|     0|
|efreadr              |0.1.1     |      0|        1|     0|
|emil                 |2.2.6     |      1|        0|     0|
|etl                  |0.3.5     |      1|        0|     0|
|extdplyr             |0.1.4     |      1|        0|     0|
|eyelinker            |0.1       |      0|        1|     0|
|eyetrackingR         |0.1.6     |      2|        0|     0|
|ezsummary            |0.2.1     |      2|        1|     0|
|fitcoach             |1.0       |      1|        0|     0|
|forestmodel          |0.4.0     |      1|        0|     0|
|FRK                  |0.1.1     |      1|        2|     2|
|FSelectorRcpp        |0.1.2     |      1|        0|     2|
|funModeling          |1.6.1     |      0|        1|     0|
|futureheatwaves      |1.0.3     |      0|        1|     0|
|geoknife             |1.5.4     |      0|        1|     0|
|ggCompNet            |0.1.0     |      0|        1|     1|
|ggforce              |0.1.1     |      1|        0|     0|
|ggfortify            |0.4.1     |      2|        0|     1|
|ggraph               |1.0.0     |      1|        0|     1|
|ggvis                |0.4.3     |      2|        0|     0|
|goldi                |1.0.0     |      1|        0|     0|
|grattan              |1.4.1.2   |      1|        1|     1|
|groupdata2           |0.1.0     |      0|        1|     0|
|gutenbergr           |0.1.2     |      1|        0|     0|
|harrietr             |0.2.2     |      1|        0|     0|
|heemod               |0.8.0     |      2|        1|     0|
|highcharter          |0.5.0     |      1|        1|     1|
|HTSSIP               |1.0.3     |      1|        0|     0|
|huxtable             |0.1.1     |      1|        1|     0|
|HydeNet              |0.10.5    |      2|        1|     0|
|IATscores            |0.1-2     |      1|        0|     0|
|incadata             |0.5.3     |      1|        0|     0|
|inferr               |0.1.0     |      0|        1|     0|
|ipft                 |0.2.8     |      0|        1|     0|
|Lahman               |5.0-0     |      0|        1|     1|
|lplyr                |0.1.6     |      1|        1|     0|
|mason                |0.2.5     |      0|        1|     0|
|medicare             |0.2.0     |      0|        1|     0|
|merTools             |0.3.0     |      1|        0|     0|
|metacoder            |0.1.2     |      1|        0|     0|
|metricTester         |1.3.2     |      0|        1|     0|
|Momocs               |1.1.0     |      1|        1|     0|
|mrgsolve             |0.8.2     |      2|        0|     1|
|mtconnectR           |1.0.1     |      2|        1|     0|
|mudata               |0.1       |      1|        0|     0|
|myTAI                |0.5.0     |      1|        1|     1|
|NFP                  |0.99.2    |      1|        0|     0|
|officer              |0.1.1     |      2|        1|     0|
|openair              |2.0-0     |      0|        1|     0|
|padr                 |0.2.1     |      1|        0|     0|
|parsemsf             |0.1.0     |      2|        1|     0|
|photobiologyInOut    |0.4.12    |      1|        0|     0|
|pixiedust            |0.7.4     |      2|        1|     0|
|plotly               |4.5.6     |      2|        0|     0|
|poplite              |0.99.17.3 |      0|        1|     0|
|prophet              |0.1       |      1|        1|     1|
|purrr                |0.2.2     |      1|        0|     0|
|qualvar              |0.1.0     |      0|        1|     0|
|qwraps2              |0.2.4     |      1|        0|     0|
|R6Frame              |0.1.0     |      1|        0|     0|
|radiant.basics       |0.6.0     |      0|        1|     0|
|radiant.data         |0.6.0     |      1|        0|     0|
|radiant.design       |0.6.0     |      0|        1|     0|
|radiant.model        |0.6.0     |      1|        0|     0|
|radiant.multivariate |0.6.0     |      0|        1|     0|
|rattle               |4.1.0     |      0|        1|     3|
|recexcavAAR          |0.3.0     |      0|        1|     1|
|rfishbase            |2.1.1     |      1|        1|     0|
|rmcfs                |1.2.4     |      0|        1|     2|
|rwunderground        |0.1.0     |      1|        0|     0|
|saeSim               |0.8.0     |      2|        0|     0|
|sfc                  |0.1.0     |      0|        1|     0|
|sf                   |0.3-4     |      2|        0|     1|
|shazam               |0.1.4     |      0|        1|     0|
|sjmisc               |2.3.1     |      1|        1|     0|
|sjPlot               |2.3.1     |      1|        1|     0|
|sjstats              |0.9.0     |      1|        0|     1|
|SpaDES               |1.3.1     |      0|        2|     2|
|sparklyr             |0.5.3     |      0|        1|     0|
|sparseHessianFD      |0.3.2     |      0|        1|     0|
|spbabel              |0.4.5     |      1|        0|     0|
|spdplyr              |0.1.2     |      2|        1|     0|
|srvyr                |0.2.0     |      2|        1|     0|
|ss3sim               |0.9.3     |      1|        1|     0|
|stormwindmodel       |0.1.0     |      0|        1|     0|
|stplanr              |0.1.7-3   |      1|        0|     0|
|tadaatoolbox         |0.10.0    |      1|        1|     0|
|texmexseq            |0.3       |      1|        0|     0|
|textmining           |0.0.1     |      0|        1|     0|
|textreuse            |0.1.4     |      2|        1|     0|
|tidyjson             |0.2.1     |      1|        0|     1|
|tidyquant            |0.4.0     |      2|        1|     0|
|tidyr                |0.6.1     |      1|        0|     0|
|tidytext             |0.1.2     |      1|        1|     0|
|treeplyr             |0.1.2     |      1|        0|     0|
|turfR                |0.8-7     |      1|        0|     1|
|unpivotr             |0.1.0     |      1|        1|     0|
|useful               |1.2.1     |      1|        1|     0|
|valr                 |0.1.1     |      1|        0|     0|
|vcfR                 |1.4.0     |      1|        0|     1|
|VWPre                |0.9.5     |      0|        1|     0|
|WHO                  |0.2       |      1|        0|     0|
|wordbankr            |0.2.0     |      0|        1|     0|
|wrswoR               |1.0-1     |      0|        1|     0|
|Zelig                |5.0-17    |      1|        0|     1|

## ameco (0.2.6)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: http://github.com/expersso/ameco/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(ameco)
  > 
  > test_check("ameco")
  1. Failure: Test that current version is still latest version (@tests.R#16) ----
  `last_update` not equal to as.Date("2017-02-13").
  'is.NA' value mismatch: 0 in current 1 in target
  
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Failure: Test that current version is still latest version (@tests.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 15.7Mb
  sub-directories of 1Mb or more:
    data  15.6Mb
```

## anametrix (2.0.0)
Maintainer: Caroline Cochet-Escartin <caroline.escartin@ensighten.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘anametrix-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: authenticate
> ### Title: authenticate
> ### Aliases: authenticate
> 
> ### ** Examples
> 
> username <- "demo.user@demoanametrix.com"
> password <- "password"
> auth     <- authenticate("https://demo.anametrix.com/api", username, password)
Failed to authenticate with Anametrix API 
Error: Connection timed out after 100000 milliseconds 
Error in xmlTreeParse(resp) : object 'resp' not found
Calls: authenticate -> xmlTreeParse
Execution halted
```

## backtestGraphics (0.1.6)
Maintainer: Miller Zijie Zhu <zijie.miller.zhu@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > test_check("backtestGraphics")
  Loading required package: backtestGraphics
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) -------------
  Failed the test for calculating summary statistics not equal to `truth.1`.
  Component "pnl": Component "pnl.drawdown": Component "start": Attributes: < Component "levels": 1 string mismatch >
  Component "pnl": Component "pnl.drawdown": Component "start": 1 string mismatch
  
  
  testthat results ================================================================
  OK: 9 SKIPPED: 0 FAILED: 1
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bayesplot (1.1.0)
Maintainer: Jonah Gabry <jsg2201@columbia.edu>  
Bug reports: https://github.com/stan-dev/bayesplot/issues/

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-14 (./children/SETTINGS-knitr.txt) 
Quitting from lines NA-14 (./children/SETTINGS-knitr.txt) 
Error: processing vignette 'MCMC-diagnostics.Rmd' failed with diagnostics:
object 'params' not found
Execution halted

```

## bibliometrix (1.5)
Maintainer: Massimo Aria <aria@unina.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## bioOED (0.1.1)
Maintainer: Alberto Garre <garre.alberto@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘MEIGOR’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## blscrapeR (2.1.2)
Maintainer: Kris Eberwein <eberwein@knights.ucf.edu>  
Bug reports: https://github.com/keberwein/blscrapeR/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         on.exit(options(expressions = expressions_opt), add = TRUE)
         e$expectation_calls <- frame_calls(11, 2)
         test_error <<- e
         register_expectation(e)
         e$handled <- TRUE
         test_error <<- e
     }, "attempt to select less than one element in integerOneIndex", quote(datalist[[i]] <- mth_vals)) at testthat/test_bls_csv.R:76
  2: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 2 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_bls_csv.R#76) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bossMaps (0.1.0)
Maintainer: Adam M. Wilson <adamw@buffalo.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

... 8 lines ...

    extract

The following object is masked from 'package:dplyr':

    select

Quitting from lines 82-83 (bossMaps.Rmd) 
Error: processing vignette 'bossMaps.Rmd' failed with diagnostics:
replacement has 2 rows, data has 1
Execution halted
```

## broom (0.4.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/tidyverse/broom/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘broom-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cch_tidiers
> ### Title: tidiers for case-cohort data
> ### Aliases: cch_tidiers glance.cch tidy.cch
> 
> ### ** Examples
... 42 lines ...
The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Error in ~seqno : object 'seqno' not found
Calls: %>% ... overscope_eval -> .Call -> tidy -> cch -> ~ -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Terminated

```

## carpenter (0.2.0)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/carpenter/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘carpenter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: carpenter
> ### Title: Build common tables for your research needs!
> ### Aliases: carpenter
> 
> ### ** Examples
> 
> 
> library(magrittr)
> outline_table(iris, 'Species') %>%
+  add_rows(c('Sepal.Length', 'Petal.Length'), stat_meanSD) %>%
+  add_rows('Sepal.Width', stat_medianIQR) %>%
+  renaming('rows', function(x) gsub('Sepal\\.', 'Sepal ', x)) %>%
+  renaming('header', c('Measures', 'Setosa', 'Versicolor', 'Virginica')) %>%
+  build_table(caption = 'A caption for the table')
Warning: 'dplyr::mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: Variable context not set
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  13: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  14: dplyr::mutate_each_(., dplyr::funs(as.character), dplyr::everything())
  15: is_empty(vars) at /home/muelleki/git/R/dplyr/R/colwise.R:279
  16: dplyr::everything() at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/types.R:240
  17: current_vars() at /home/muelleki/git/R/dplyr/R/select-utils.R:129
  18: cur_vars_env$selected %||% stop("Variable context not set", call. = FALSE) at /home/muelleki/git/R/dplyr/R/select-utils.R:46
  19: stop("Variable context not set", call. = FALSE) at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/operators.R:14
  
  testthat results ================================================================
  OK: 20 SKIPPED: 0 FAILED: 1
  1. Error: build_table outputs correct information (@test-build_tables.R#21) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 87-89 (carpenter.Rmd) 
Error: processing vignette 'carpenter.Rmd' failed with diagnostics:
Variable context not set
Execution halted

```

## chunked (0.3)
Maintainer: Edwin de Jonge <edwindjonge@gmail.com>  
Bug reports: https://github.com/edwindj/chunked/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘chunked-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: read_csv_chunkwise
> ### Title: Read chunkwise data from text files
> ### Aliases: read_csv2_chunkwise read_csv_chunkwise read_laf_chunkwise
> ###   read_table_chunkwise
> 
... 7 lines ...
> #
> women_chunked <-
+   read_chunkwise(in_file) %>%  #open chunkwise connection
+   mutate(ratio = weight/height) %>%
+   filter(ratio > 2) %>%
+   select(height, ratio) %>%
+   inner_join(data.frame(height=63:66)) # you can join with data.frames!
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "c('chunkwise', 'tbl')"
Calls: %>% ... eval -> eval -> _fseq -> freduce -> <Anonymous> -> mutate
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  44: unique(c(fname, unlist(lapply(x[-1], all_calls), use.names = FALSE))) at /home/muelleki/git/R/dplyr/R/translate-sql.r:199
  45: unlist(lapply(x[-1], all_calls), use.names = FALSE)
  46: lapply(x[-1], all_calls)
  47: x[-1]
  48: `[.formula`(x, -1)
  
  testthat results ================================================================
  OK: 38 SKIPPED: 0 FAILED: 4
  1. Error: insert_chunkwise_into: can insert modified table into a db (@test-insert-into.R#15) 
  2. Error: mutate(): can add a new column (@test-verbs.R#9) 
  3. Error: mutate(): can add a new column (@test-verbs.R#14) 
  4. Error: write_chunkwise to db works (@test-write.R#29) 
  
  Error: testthat unit tests failed
  Execution halted
```

## clustrd (1.1.0)
Maintainer: Angelos Markos <amarkos@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘clustrd-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cluspca
> ### Title: Joint dimension reduction and clustering of continuous data.
> ### Aliases: cluspca
> 
> ### ** Examples
> 
> data(macro)
> outRKM = cluspca(macro, 3, 2, method = "RKM", rotation = "varimax")
> plot(outRKM, cludesc = TRUE)
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error in bind_rows_(x, .id) : cannot convert object to a data frame
Calls: plot ... clu_means -> rbind -> rbind -> bind_rows -> bind_rows_ -> .Call
Execution halted
```

## codingMatrices (0.2.2)
Maintainer: Bill Venables <Bill.Venables@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'car'

The following object is masked from 'package:dplyr':

    recode

Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'codingMatrices.tex' failed.
BibTeX errors:
The top-level auxiliary file: codingMatrices.aux
I couldn't open style file chicago.bst
---line 19 of file codingMatrices.aux
 : \bibstyle{chicago
 :                  }
I'm skipping whatever remains of this command
I found no style file---while reading file codingMatrices.aux
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## cometExactTest (0.1.3)
Maintainer: Max Leiserson <mdml@cs.brown.edu>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## crawl (2.0.1)
Maintainer: Devin S. Johnson <devin.johnson@noaa.gov>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Attaching package: 'lubridate'

The following object is masked from 'package:base':

    date

Loading required package: zoo
... 8 lines ...

Attaching package: 'xts'

The following objects are masked from 'package:dplyr':

    first, last

Quitting from lines 403-414 (crawl_intro.Rmd) 
Error: processing vignette 'crawl_intro.Rmd' failed with diagnostics:
incompatible size (9701), expecting one (the group size)
Execution halted

checking installed package size ... NOTE
  installed size is  9.3Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   6.3Mb
```

## dat (0.1.0)
Maintainer: Sebastian Warnholz <wahani@gmail.com>  
Bug reports: https://github.com/wahani/dat/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [53s/54s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  2. Failure: make formulas (@test-makeFormulas.R#45) ----------------------------
  `dat3` not identical to mutar(dat, ~1:2, FL(x ~ mean(x))).
  Attributes: < Length mismatch: comparison on first 2 components >
  Attributes: < Component "class": Lengths (1, 3) differ (string compare on first 1) >
  
  
  testthat results ================================================================
  OK: 106 SKIPPED: 0 FAILED: 2
  1. Failure: Package Style (@test-lintr.R#5) 
  2. Failure: make formulas (@test-makeFormulas.R#45) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ddpcr (1.6)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/daattali/ddpcr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ddpcr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ddpcr.Rcheck/00install.out’ for details.
```

## decoder (1.1.12)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.com/cancercentrum/decoder/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Joining, by = "key"
Joining, by = "key"
PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable.
Quitting from lines 102-122 (sjukhus.Rmd) 
Error: processing vignette 'sjukhus.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## DeLorean (1.2.4)
Maintainer: John Reid <john.reid@mrc-bsu.cam.ac.uk>

0 errors | 1 warning  | 1 note 

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.cells:
  function(dl, .filter, number, cells)

filter:
  function(x, filter, method, sides, circular, init)
filter.genes:
  function(dl, .filter, number, genes)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.cells filter.genes
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.cells':
  ‘filter.cells’

S3 methods shown with full name in documentation object 'filter.genes':
  ‘filter.genes’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## describer (0.2.0)
Maintainer: Paul Hendricks <paul.hendricks.2013@owu.edu>  
Bug reports: https://github.com/paulhendricks/describer/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [141s/142s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-window.R:74:8: style: Variable and function names should be all lowercase.
      ), sqlText),
         ^~~~~~~
  tests/testthat/test-window.R:76:12: style: Variable and function names should be all lowercase.
      info = sqlText
             ^~~~~~~
  
  
  testthat results ================================================================
  OK: 7 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test-styling.R#4) 
  
  Error: testthat unit tests failed
  Execution halted
```

## DiagrammeR (0.9.0)
Maintainer: Richard Iannone <riannone@me.com>  
Bug reports: https://github.com/rich-iannone/DiagrammeR/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [187s/187s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(DiagrammeR)
  > 
  > test_check("DiagrammeR")
  1. Error: importing a .graphml file is possible (@test-import_graph.R#5) -------
  subscript out of bounds
  1: import_graph(system.file("extdata", "power_grid.graphml", package = "DiagrammeR")) at testthat/test-import_graph.R:5
  2: create_graph(nodes_df = all_nodes, edges_df = all_edges)
  
  testthat results ================================================================
  OK: 1468 SKIPPED: 0 FAILED: 1
  1. Error: importing a .graphml file is possible (@test-import_graph.R#5) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 10.6Mb
  sub-directories of 1Mb or more:
    extdata       2.4Mb
    htmlwidgets   4.7Mb
    img           2.8Mb
```

## doctr (0.2.0)
Maintainer: Caio Lente <me@ctlente.com>

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  13: eval(expr, envir, enclos)
  14: `_fseq`(`_lhs`)
  15: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  16: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  17: dplyr::mutate(., data = purrr::map(exams[[1]], ~X[[.x]]))
  18: mutate.tbl_df(., data = purrr::map(exams[[1]], ~X[[.x]])) at /home/muelleki/git/R/dplyr/R/manip.r:218
  19: mutate_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:83
  
  testthat results ================================================================
  OK: 26 SKIPPED: 0 FAILED: 2
  1. Error: (unknown) (@test_diagnose.R#7) 
  2. Error: (unknown) (@test_issues.R#13) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading tidyverse: ggplot2
Loading tidyverse: tibble
Loading tidyverse: tidyr
Loading tidyverse: readr
Loading tidyverse: purrr
Loading tidyverse: dplyr
Conflicts with tidy packages ----------------------------------------------
filter(): dplyr, stats
lag():    dplyr, stats
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 43-45 (doctr_diagnose.Rmd) 
Error: processing vignette 'doctr_diagnose.Rmd' failed with diagnostics:
object '.x' not found
Execution halted

```

## dotwhisker (0.2.3)
Maintainer: Yue Hu <yue-hu-1@uiowa.edu>  
Bug reports: https://github.com/fsolt/dotwhisker/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘dotwhisker-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dwplot
> ### Title: Dot-and-Whisker Plots of Regression Results
> ### Aliases: dwplot
> 
> ### ** Examples
... 51 lines ...
 4: rename(., model = am)
 5: function_list[[k]](value)
 6: withVisible(function_list[[k]](value))
 7: freduce(value, `_function_list`)
 8: `_fseq`(`_lhs`)
 9: eval(expr, envir, enclos)
10: eval(quote(`_fseq`(`_lhs`)), env, env)
11: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
12: mtcars %>% group_by(am) %>% do(tidy(lm(mpg ~ wt + cyl + disp,     data = .))) %>% rename(model = am)
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
14: withVisible(eval(expr, envir, enclos))
15: withCallingHandlers(withVisible(eval(expr, envir, enclos)), warning = wHandler,     error = eHandler, message = mHandler)
16: handle(ev <- withCallingHandlers(withVisible(eval(expr, envir,     enclos)), warning = wHandler, error = eHandler, message = mHandler))
17: timing_fn(handle(ev <- withCallingHandlers(withVisible(eval(expr,     envir, enclos)), warning = wHandler, error = eHandler, message = mHandler)))
18: evaluate_call(expr, parsed$src[[i]], envir = envir, enclos = enclos,     debug = debug, last = i == length(out), use_try = stop_on_error !=         2L, keep_warning = keep_warning, keep_message = keep_message,     output_handler = output_handler, include_timing = include_timing)
19: evaluate(code, envir = env, new_device = FALSE, keep_warning = !isFALSE(options$warning),     keep_message = !isFALSE(options$message), stop_on_error = if (options$error &&         options$include) 0L else 2L, output_handler = knit_handlers(options$render,         options))
20: in_dir(input_dir(), evaluate(code, envir = env, new_device = FALSE,     keep_warning = !isFALSE(options$warning), keep_message = !isFALSE(options$message),     stop_on_error = if (options$error && options$include) 0L else 2L,     output_handler = knit_handlers(options$render, options)))
... 8 lines ...
28: knit2html(..., force_v1 = TRUE)
29: (if (grepl("\\.[Rr]md$", file)) knit2html_v1 else if (grepl("\\.[Rr]rst$",     file)) knit2pdf else knit)(file, encoding = encoding, quiet = quiet,     envir = globalenv())
30: vweave(...)
31: engine$weave(file, quiet = quiet, encoding = enc)
32: doTryCatch(return(expr), name, parentenv, handler)
33: tryCatchOne(expr, names, parentenv, handlers[[1L]])
34: tryCatchList(expr, classes, parentenv, handlers)
35: tryCatch({    engine$weave(file, quiet = quiet, encoding = enc)    setwd(startdir)    find_vignette_product(name, by = "weave", engine = engine)}, error = function(e) {    stop(gettextf("processing vignette '%s' failed with diagnostics:\n%s",         file, conditionMessage(e)), domain = NA, call. = FALSE)})
36: buildVignettes(dir = "/home/muelleki/git/R/dplyr/revdep/checks/dotwhisker.Rcheck/vign_test/dotwhisker")
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)
```

## dtplyr (0.0.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/dtplyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 113 SKIPPED: 0 FAILED: 19
  1. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#23) 
  2. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#24) 
  3. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#25) 
  4. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#26) 
  5. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#28) 
  6. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#29) 
  7. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#30) 
  8. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#31) 
  9. Failure: grouped_by uses grouping vars & preserves groups (@test-distinct.R#24) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## eechidna (0.1)
Maintainer: Ben Marwick <benmarwick@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'purrr'

The following objects are masked from 'package:dplyr':
... 8 lines ...

Attaching package: 'scales'

The following object is masked from 'package:purrr':

    discard

Quitting from lines 154-171 (exploring-election-data.Rmd) 
Error: processing vignette 'exploring-election-data.Rmd' failed with diagnostics:
Value of SET_STRING_ELT() must be a 'CHARSXP' not a 'integer'
Execution halted
```

## efreadr (0.1.1)
Maintainer: Marco Bascietto <marco.bascietto@crea.gov.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## emil (2.2.6)
Maintainer: Christofer Backlin <emil@christofer.backlin.se>  
Bug reports: https://github.com/Molmed/emil/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘emil-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: learning_curve
> ### Title: Learning curve analysis
> ### Aliases: learning_curve
> 
> ### ** Examples
... 6 lines ...
15 Mär 19:41    Test set test_fraction 1 of 3 (0.7)
15 Mär 19:41      Evaluating modeling performance...
15 Mär 19:41    Test set test_fraction 2 of 3 (0.5)
15 Mär 19:41      Evaluating modeling performance...
15 Mär 19:41    Test set test_fraction 3 of 3 (0.3)
15 Mär 19:41      Evaluating modeling performance...
> plot(lc)
Error in select.list(., test_fraction = TRUE, fold = TRUE, method = TRUE,  : 
  unused arguments (test_fraction = TRUE, fold = TRUE, method = TRUE, performance = "error")
Calls: plot ... withVisible -> <Anonymous> -> select -> select.list
Execution halted
```

## etl (0.3.5)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/etl/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  trying URL 'http://www.google.com'
  downloaded 10 KB
  
  trying URL 'http://www.nytimes.com'
  Content type 'text/html; charset=utf-8' length 223193 bytes (217 KB)
  ==================================================
  downloaded 217 KB
  
  testthat results ================================================================
  OK: 19 SKIPPED: 0 FAILED: 2
  1. Error: sqlite works (@test-etl.R#9) 
  2. Failure: dplyr works (@test-etl.R#25) 
  
  Error: testthat unit tests failed
  Execution halted
```

## extdplyr (0.1.4)
Maintainer: Yuchen Wang <ycwang0712@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘extdplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: pct_routine
> ### Title: Calculate percentage by group.
> ### Aliases: pct_routine pct_routine_ tally_pct tally_pct_
> 
> ### ** Examples
... 85 lines ...
81   75+     40-79 0-9g/day      2         5
82   75+     40-79    10-19      1         3
83   75+     40-79    20-29      0         3
84   75+     40-79      30+      1         1
85   75+    80-119 0-9g/day      1         1
86   75+    80-119    10-19      1         1
87   75+      120+ 0-9g/day      2         2
88   75+      120+    10-19      1         1
> pct_routine(esoph, agegp, alcgp)
Error: Cannot convert objects of type `logical` to `tidy_quote`
Execution halted
```

## eyelinker (0.1)
Maintainer: Simon Barthelme <simon.barthelme@gipsa-lab.fr>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: eyelinker
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 226-229 (basics.Rmd) 
Error: processing vignette 'basics.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted

```

## eyetrackingR (0.1.6)
Maintainer: Jacob Dink <jacobwdink@gmail.com>  
Bug reports: https://github.com/jwdink/eyetrackingR/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘eyetrackingR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: analyze_time_clusters
> ### Title: Bootstrap analysis of time-clusters.
> ### Aliases: analyze_time_clusters analyze_time_clusters.time_cluster_data
> 
> ### ** Examples
... 16 lines ...
> response_time <- make_time_sequence_data(response_window, time_bin_size = 500, aois = "Animate", 
+                                          predictor_columns = "Sex")
> 
> time_cluster_data <- make_time_cluster_data(data = response_time, predictor_column = "SexM", 
+                          aoi = "Animate", test = "lmer", 
+                          threshold = 1.5, 
+                          formula = LogitAdjusted ~ Sex + (1|Trial) + (1|ParticipantName))
Error in UseMethod("analyze_time_bins") : 
  no applicable method for 'analyze_time_bins' applied to an object of class "data.frame"
Calls: make_time_cluster_data ... make_time_cluster_data.time_sequence_data -> do.call -> <Anonymous>
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [92s/92s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Predictor:	 Target 
  Formula:	 Prop ~ Target 
  Runs of Significant Time Bins: 
  Positive Run 1  ===== 
  	Time:		 15900 - 21000 1. Error: (unknown) (@test_cluster_analysis.R#143) -----------------------------
  no applicable method for 'make_time_cluster_data' applied to an object of class "data.frame"
  1: make_time_cluster_data(response_time_by_ppt, predictor_column = "Sex", test = "boot_splines", 
         within_subj = FALSE, smoother = "smooth.spline", alpha = 0.05) at testthat/test_cluster_analysis.R:143
  
  testthat results ================================================================
  OK: 46 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_cluster_analysis.R#143) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ezsummary (0.2.1)
Maintainer: Hao Zhu <haozhu233@gmail.com>  
Bug reports: https://github.com/haozhu233/ezsummary/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ezsummary-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ezmarkup
> ### Title: Easy way to "markup" a table before it is sent to be displayed
> ### Aliases: ezmarkup
> 
> ### ** Examples
... 10 lines ...
The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> dt <- mtcars %>% group_by(cyl) %>% select(gear, carb) %>% ezsummary_categorical(n=TRUE)
Adding missing grouping variables: `cyl`
> 
> ezmarkup(dt, "...[.(.)]")
Error in cbind_all(x) : cannot convert object to a data frame
Calls: ezmarkup ... ezmarkup_ -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         if (warn) 
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  26: abort_coercion(.x, .to) at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/types.R:422
  27: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/types.R:435
  
  testthat results ================================================================
  OK: 30 SKIPPED: 0 FAILED: 3
  1. Error: ezsummary_quantitative can evaluate grouping info correctly with 1 variable (@test-ezsummary_quantitative.R#19) 
  2. Error: ezsummary_quantitative can work with 2 variables with grouping info (@test-ezsummary_quantitative.R#33) 
  3. Error: ezsummary_quantitative can handle NAs when running with quantile=TRUE (@test-ezsummary_quantitative.R#40) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 110-114 (intro_to_ezsummary_0.2.0.Rmd) 
Error: processing vignette 'intro_to_ezsummary_0.2.0.Rmd' failed with diagnostics:
Cannot convert objects of type `character` to `tidy_quote`
Execution halted

```

## fitcoach (1.0)
Maintainer: Niraj Juneja <njuneja@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) --------------------
  res[1] is not strictly less than 2820. Difference: 29.2
  
  
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) --------------------
  `res` is not strictly less than 2518. Difference: 2.3
  
  
  testthat results ================================================================
  OK: 17 SKIPPED: 0 FAILED: 2
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) 
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) 
  
  Error: testthat unit tests failed
  Execution halted
```

## forestmodel (0.4.0)
Maintainer: Nick Kennedy <r@nick-kennedy.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘forestmodel-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: forest_model
> ### Title: Produce a forest plot based on a regression model
> ### Aliases: forest_model
> 
> ### ** Examples
... 16 lines ...
+   transmute(time,
+             status,
+             Age = age,
+             Sex = factor(sex, labels = c("Male", "Female")),
+             ECOG = factor(lung$ph.ecog),
+             `Meal Cal` = meal.cal)
> 
> print(forest_model(coxph(Surv(time, status) ~ ., pretty_lung)))
Error in cbind_all(x) : incompatible number of rows (2, expecting 1)
Calls: print ... create_term_data -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted
```

## FRK (0.1.1)
Maintainer: Andrew Zammit-Mangion <andrewzm@gmail.com>  
Bug reports: http://github.com/andrewzm/FRK/issues

1 error  | 2 warnings | 2 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [65s/66s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
    |                                                                            
    |                                                                      |   0%
    |                                                                            
    |===================================                                   |  50%
    |                                                                            
    |======================================================================| 100%
  [1] "Maximum EM iterations reached"
  testthat results ================================================================
  OK: 140 SKIPPED: 1 FAILED: 3
  1. Error: sphere_BAUs (@test_BAUs.R#71) 
  2. Error: SRE sphere works (@test_SRE.R#114) 
  3. Error: SRE space-time sphere works (@test_SRE.R#167) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 525-530 (FRK_intro.Rnw) 
Error: processing vignette 'FRK_intro.Rnw' failed with diagnostics:
invalid list argument: all variables should have the same length
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’

Package which this enhances but not available for checking: ‘dggrids’

checking installed package size ... NOTE
  installed size is  7.7Mb
  sub-directories of 1Mb or more:
    data   4.8Mb
    doc    1.6Mb
```

## FSelectorRcpp (0.1.2)
Maintainer: Zygmunt Zawadzki <zygmunt@zstat.pl>  
Bug reports: https://github.com/mi2-warsaw/FSelectorRcpp/issues

1 error  | 0 warnings | 2 notes

```
checking examples ... ERROR
Running examples in ‘FSelectorRcpp-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: feature_search
> ### Title: General Feature Searching Engine
> ### Aliases: feature_search
> 
> ### ** Examples
> 
> 
> # Enable parallelization in examples
>  library(doSNOW) # doSNOW has an option for progress bar
Loading required package: foreach
Loading required package: iterators
Loading required package: snow
>  cl <- makeCluster(2)
Loading required namespace: Rmpi
Error in Rmpi::mpi.comm.spawn(slave = mpitask, slavearg = args, nslaves = count,  : 
  MPI_Comm_spawn is not supported.
Calls: makeCluster -> makeMPIcluster -> <Anonymous>
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘RTCGA.rnaseq’

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   7.3Mb
```

## funModeling (1.6.1)
Maintainer: Pablo Casas <pcasas.biz@gmail.com>  
Bug reports: https://github.com/pablo14/funModeling/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
... 6 lines ...
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## futureheatwaves (1.0.3)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 604-605 (futureheatwaves.Rmd) 
Error: processing vignette 'futureheatwaves.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## geoknife (1.5.4)
Maintainer: Jordan Read <jread@usgs.gov>  
Bug reports: https://github.com/USGS-R/geoknife/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 360-364 (geoknife.Rmd) 
Error: processing vignette 'geoknife.Rmd' failed with diagnostics:
need finite 'xlim' values
Execution halted

```

## ggCompNet (0.1.0)
Maintainer: Sam Tyner <sctyner@iastate.edu>  
Bug reports: https://github.com/sctyner/ggCompNet/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: Removed 8 rows containing missing values (geom_segment).
Quitting from lines 617-629 (examples-from-paper.Rmd) 
Error: processing vignette 'examples-from-paper.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted


checking installed package size ... NOTE
  installed size is  6.4Mb
  sub-directories of 1Mb or more:
    doc   6.1Mb
```

## ggforce (0.1.1)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggforce-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: geom_arc_bar
> ### Title: Arcs and wedges as polygons
> ### Aliases: geom_arc_bar stat_arc_bar stat_pie
> 
> ### ** Examples
... 27 lines ...
> ggplot() + geom_arc_bar(aes(x0=0, y0=0, r0=r0, r=1, amount=amount,
+                             fill=state, explode=focus),
+                         data=pie, stat='pie') +
+   facet_wrap(~type, ncol=1) +
+   coord_fixed() +
+   theme_no_axes() +
+   scale_fill_brewer('', type='qual')
Warning: Ignoring unknown aesthetics: explode
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted
```

## ggfortify (0.4.1)
Maintainer: Masaaki Horikoshi <sinhrks@gmail.com>  
Bug reports: https://github.com/sinhrks/ggfortify/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggfortify-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: gglagplot
> ### Title: Plot time series against lagged versions of themselves
> ### Aliases: gglagplot
> 
> ### ** Examples
> 
> gglagplot(AirPassengers)
Error: dplyr::lag() called with ts object. Do you want stats::lag()?
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’ [48s/51s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  2: dplyr::bind_rows(lapply(seq(1:lags), .lag))
  3: list_or_dots(...) at /home/muelleki/git/R/dplyr/R/bind.r:77
  4: lapply(seq(1:lags), .lag) at /home/muelleki/git/R/dplyr/R/bind.r:120
  5: FUN(X[[i]], ...)
  6: as.vector(lag(ts, k))
  7: lag(ts, k)
  8: stop("dplyr::lag() called with ts object. Do you want stats::lag()?", call. = FALSE) at /home/muelleki/git/R/dplyr/R/lead-lag.R:61
  
  testthat results ================================================================
  OK: 1441 SKIPPED: 9 FAILED: 2
  1. Error: Check autoplot works for list of ggplot (@test-plotlib.R#209) 
  2. Error: gglagplot (@test-tslib.R#103) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  5.7Mb
  sub-directories of 1Mb or more:
    doc   5.0Mb
```

## ggraph (1.0.0)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggraph-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: facet_edges
> ### Title: Create small multiples based on edge attributes
> ### Aliases: facet_edges
> 
> ### ** Examples
... 13 lines ...

> gr <- graph_from_data_frame(highschool)
> 
> ggraph(gr) +
+     geom_edge_link() +
+     geom_node_point() +
+     facet_edges(~year)
Using `nicely` as default layout
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted

checking installed package size ... NOTE
  installed size is  6.1Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    libs   2.0Mb
```

## ggvis (0.4.3)
Maintainer: Winston Chang <winston@rstudio.com>

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggvis-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr-ggvis
> ### Title: Dplyr verbs for ggvis.
> ### Aliases: arrange_.ggvis arrange_.reactive distinct_.ggvis
> ###   distinct_.reactive dplyr-ggvis filter_.ggvis filter_.reactive
> ###   group_by_.ggvis group_by_.reactive groups.ggvis groups.reactive
... 21 lines ...
> base <- mtcars %>% ggvis(~mpg, ~cyl) %>% layer_points()
> base %>% group_by(cyl) %>% summarise(mpg = mean(mpg)) %>%
+   layer_points(fill := "red", size := 100)
> 
> base %>% filter(mpg > 25) %>% layer_points(fill := "red")
> 
> base %>% mutate(cyl = jitter(cyl)) %>% layer_points(fill := "red")
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "ggvis"
Calls: %>% ... eval -> eval -> _fseq -> freduce -> <Anonymous> -> mutate
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
     }) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:21
  25: env$runWith(self, func) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:25
  26: contextFunc() at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:113
  27: ..stacktraceon..(expr) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/reactives.R:1480
  28: prop_value(prop, data()) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/conditions.R:331
  29: prop_value.prop_variable(prop, data())
  30: eval(value(x$value), envir = data, enclos = x$env)
  31: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 443 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test-specs.r#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## goldi (1.0.0)
Maintainer: Christopher B. Cole <chris.c.1221@gmail.com>  
Bug reports: https://github.com/Chris1221/goldi/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: expect_equal(goldi(doc = doc, terms = "empty", output = output, lims = lims, log = log, 
         object = TRUE, reader = "local", term_tdm = TDM.go.df), out) at testthat/test-goldi.R:23
  2: compare(object, expected, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-equality.R:52
  3: goldi(doc = doc, terms = "empty", output = output, lims = lims, log = log, object = TRUE, 
         reader = "local", term_tdm = TDM.go.df) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/compare.R:12
  4: tm_map(doc.corpus, content_transformer(tolower), mc.cores = 1)
  5: tm_map.SimpleCorpus(doc.corpus, content_transformer(tolower), mc.cores = 1) at /tmp/RtmptEsk2b/devtools8d9d1c68fc45/tm/R/transform.R:5
  6: FUN(content(x), ...) at /tmp/RtmptEsk2b/devtools8d9d1c68fc45/tm/R/transform.R:27
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Error: testing main function (@test-goldi.R#23) 
  
  Error: testthat unit tests failed
  Execution halted
```

## grattan (1.4.1.2)
Maintainer: Hugh Parsonage <hugh.parsonage@gmail.com>  
Bug reports: https://github.com/HughParsonage/grattan/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [46s/46s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  13: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  14: filter(., fy_year == "2016-17")
  15: filter.data.frame(., fy_year == "2016-17") at /home/muelleki/git/R/dplyr/R/manip.r:40
  16: as.data.frame(filter(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:62
  17: filter(tbl_df(.data), ...)
  18: filter.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:40
  19: filter_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:61
  
  testthat results ================================================================
  OK: 205 SKIPPED: 35 FAILED: 1
  1. Error: New SAPTO matches old SAPTO for SAPTO (@test_sapto.R#83) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘taxstats’
```

## groupdata2 (0.1.0)
Maintainer: Ludvig Renbo Olsen <r-pkgs@ludvigolsen.dk>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Loading required package: lme4

Attaching package: 'lmerTest'

The following object is masked from 'package:lme4':

... 8 lines ...

Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Quitting from lines 293-301 (introduction_to_groupdata2.Rmd) 
Error: processing vignette 'introduction_to_groupdata2.Rmd' failed with diagnostics:
the table must have a header (column names)
Execution halted
```

## gutenbergr (0.1.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/ropenscilabs/gutenbergr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [5s/13s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  testthat results ================================================================
  OK: 50 SKIPPED: 0 FAILED: 8
  1. Error: gutenberg_get_mirror works (@test-download.R#13) 
  2. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#22) 
  3. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#23) 
  4. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#31) 
  5. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#32) 
  6. Failure: Can download books from a data frame with gutenberg_id column (@test-download.R#43) 
  7. Failure: We can download a file that only has a -8 version (@test-download.R#50) 
  8. Failure: read_zip_url can download and read a zip file (@test-utils.R#7) 
  
  Error: testthat unit tests failed
  Execution halted
```

## harrietr (0.2.2)
Maintainer: Anders Gonçalves da Silva <andersgs@gmail.com>  
Bug reports: https://github.com/andersgs/harrietr/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘ggtree’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## heemod (0.8.0)
Maintainer: Antoine Filipovic-Pierucci <pierucci@gmail.com>  
Bug reports: https://github.com/pierucci/heemod/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘heemod-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: eval_strategy
> ### Title: Evaluate Strategy
> ### Aliases: eval_strategy
> ### Keywords: internal
> 
... 20 lines ...
+   mod,
+   param,
+   init = c(10, 5),
+   cycles = 5,
+   method = "end",
+   inflow = c(0, 0),
+   strategy_name = "A",
+   expand_limit = c(A = 5, B = 5)
+ )
Error: Cannot convert objects of type `double` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [30s/47s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 278 SKIPPED: 0 FAILED: 26
  1. Error: HIV model works (@test_dmhee.R#98) 
  2. Error: HRT model works (@test_dmhee.R#249) 
  3. Error: Identify Frontier Scenarios (@test_efficiency_frontier.R#15) 
  4. Error: Expansion works. (@test_expand.R#203) 
  5. Error: Translating objects to code (@test_get_code.R#42) 
  6. Error: Demographic analysis (@test_heterogeneity.R#37) 
  7. Error: Heterogeneity analysis (@test_heterogeneity.R#115) 
  8. Failure: Functions on matrix objects (@test_matrix.R#119) 
  9. Error: Functions on matrix objects (@test_matrix.R#127) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 98-105 (a-introduction.Rmd) 
Error: processing vignette 'a-introduction.Rmd' failed with diagnostics:
Cannot convert objects of type `double` to `tidy_quote`
Execution halted

```

## highcharter (0.5.0)
Maintainer: Joshua Kunst <jbkunst@gmail.com>  
Bug reports: https://github.com/jbkunst/highcharter/issues

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘highcharter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: hcboxplot
> ### Title: Shortcut to make a boxplot
> ### Aliases: hcboxplot
> 
> ### ** Examples
> 
> hcboxplot(x = iris$Sepal.Length, var = iris$Species, color = "red")
Error in mutate_impl(.data, dots) : 
  incompatible size (5), expecting one (the group size)
Calls: hcboxplot ... transmute.default -> mutate -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Highcharts (www.highcharts.com) is a Highsoft software product which is
not free for commercial and Governmental use
Quitting from lines 45-46 (charting-data-frames.Rmd) 
Error: processing vignette 'charting-data-frames.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is 16.5Mb
  sub-directories of 1Mb or more:
    doc          13.7Mb
    htmlwidgets   1.9Mb
```

## HTSSIP (1.0.3)
Maintainer: Nicholas Youngblut <nyoungb2@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘phyloseq’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## huxtable (0.1.1)
Maintainer: David Hugh-Jones <davidhughjones@gmail.com>  
Bug reports: https://github.com/hughjonesd/huxtable/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         message = handle_message)) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:42
  4: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:130
  5: withVisible(code)
  6: rmarkdown::render("rowheight-multicol-test.Rmd", quiet = TRUE)
  7: convert(output_file, run_citeproc) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/render.R:656
  8: pandoc_convert(utf8_input, pandoc_to, output_format$pandoc$from, output, citeproc, 
         output_format$pandoc$args, !quiet) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/render.R:589
  9: stop("pandoc document conversion failed with error ", result, call. = FALSE) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/pandoc.R:100
  
  testthat results ================================================================
  OK: 9 SKIPPED: 10 FAILED: 1
  1. Error: Row heights do not screw up latex multicol (@test-output.R#44) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 51-102 (design-principles.Rmd) 
Error: processing vignette 'design-principles.Rmd' failed with diagnostics:
argument is of length zero
Execution halted

```

## HydeNet (0.10.5)
Maintainer: Benjamin Nutter <benjamin.nutter@gmail.com>  
Bug reports: https://github.com/nutterb/HydeNet/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘HydeNet-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: HydePosterior
> ### Title: Posterior Distributions of a Decision Network
> ### Aliases: HydePosterior
> 
> ### ** Examples
> 
> data(PE, package="HydeNet")
> Net <- HydeNetwork(~ wells + 
+                      pe | wells + 
+                      d.dimer | pregnant*pe + 
+                      angio | pe + 
+                      treat | d.dimer*angio + 
+                      death | pe*treat,
+                      data = PE) 
>   
>                  
> compiledNet <- compileJagsModel(Net, n.chains=5)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: compileJagsModel ... cpt_workhorse -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [15s/15s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 55 SKIPPED: 0 FAILED: 12
  1. Error: HydeNetwork.list returns expected attributes (@test-HydeNetwork.R#41) 
  2. Error: (unknown) (@test-HydePosterior.R#11) 
  3. Error: (unknown) (@test-bindPosterior.R#12) 
  4. Error: compileJagsModel returns an object of class 'compiledHydeNetwork' (@test-compileJagsModel.R#14) 
  5. Error: cpt.list (@test-cpt.R#12) 
  6. Error: cpt with weights (@test-cpt.R#20) 
  7. Error: (unknown) (@test-print.HydePosterior.R#11) 
  8. Failure: fit the model for dnorm (@test-setNode.R#40) 
  9. Error: (unknown) (@test-setNodeModels.R#14) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: nnet
Quitting from lines 227-233 (DecisionNetworks.Rmd) 
Error: processing vignette 'DecisionNetworks.Rmd' failed with diagnostics:
cannot convert object to a data frame
Execution halted

```

## IATscores (0.1-2)
Maintainer: Giulio Costantini <costantinigiulio@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘nem’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## incadata (0.5.3)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.org/cancercentrum/incadata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(incadata)
  > 
  > test_check("incadata")
  1. Failure: filter (@test-dplyr_methods.R#13) ----------------------------------
  dplyr::filter(testdata, persnr == 198505043334) inherits from `tbl_df/tbl/data.frame` not `incadata`.
  
  
  [1] "ex_data24ee2137bf76.csv2"
  testthat results ================================================================
  OK: 40 SKIPPED: 0 FAILED: 1
  1. Failure: filter (@test-dplyr_methods.R#13) 
  
  Error: testthat unit tests failed
  Execution halted
```

## inferr (0.1.0)
Maintainer: Aravind Hebbali <hebbali.aravind@gmail.com>  
Bug reports: https://github.com/rsquaredacademy/inferr/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
... 14 lines ...
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## ipft (0.2.8)
Maintainer: Emilio Sansano <esansano@uji.es>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
              user system elapsed
ipfProb     13.996  0.032  14.047
ipfEstimate 10.704  0.080  10.793
ipfDist      3.236  0.024   8.752
```

## Lahman (5.0-0)
Maintainer: Chris Dalzell <cdalzell@gmail.com>

0 errors | 1 warning  | 1 note 

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
               user system elapsed
Teams        14.044  0.080  14.169
PitchingPost  5.072  0.004   5.077

checking installed package size ... NOTE
  installed size is  7.6Mb
  sub-directories of 1Mb or more:
    data   7.4Mb
```

## lplyr (0.1.6)
Maintainer: Paul Poncet <paulponcet@yahoo.fr>  
Bug reports: https://github.com/paulponcet/lplyr/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘lplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mutate_.list
> ### Title: Dplyr verbs for lists and pairlists
> ### Aliases: mutate_.list mutate_.pairlist rename_.list rename_.pairlist
> ###   select_.list select_.pairlist transmute_.list transmute_.pairlist
> 
> ### ** Examples
> 
> xs <- list(x1 = 1:3, 
+            x2 = 2:5, 
+            x3 = list("alpha", c("beta", "gamma")))
> 
> # Non-standard evaluation
> mutate(xs, x4 = 4)
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "list"
Calls: mutate
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 7 lines ...

Attaching package: 'lplyr'

The following object is masked from 'package:dplyr':

    pull

Quitting from lines 22-30 (lplyr-vignette.Rmd) 
Error: processing vignette 'lplyr-vignette.Rmd' failed with diagnostics:
no applicable method for 'mutate' applied to an object of class "list"
Execution halted
```

## mason (0.2.5)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/mason/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'dplyr::mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## medicare (0.2.0)
Maintainer: Robert Gambrel <robert.gambrel@gmail.com>  
Bug reports: http://www.github.com/robertgambrel/medicare/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag
... 8 lines ...
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Quitting from lines 197-222 (cost-report-analysis.Rmd) 
Error: processing vignette 'cost-report-analysis.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted
```

## merTools (0.3.0)
Maintainer: Jared E. Knowles <jknowles@gmail.com>  
Bug reports: https://www.github.com/jknowles/merTools

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘merTools-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotREsim
> ### Title: Plot the results of a simulation of the random effects
> ### Aliases: plotREsim
> 
> ### ** Examples
> 
>  require(ggplot2); require(lme4);
Loading required package: ggplot2
>  fm1 <- lmer(Reaction ~ Days + (Days | Subject), sleepstudy)
>  (p1 <- plotREsim(REsim(fm1)))
>  #Plot just the random effects for the Days slope
>  (p2 <- plotREsim(REsim(fm1)[REsim(fm1)$term=="Days",]))
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L)) : 
  replacement has 2 rows, data has 1
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## metacoder (0.1.2)
Maintainer: Zachary Foster <zacharyfoster1989@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  result$taxon_data$taxon_info_2 not equal to c(NA, NA, NA, "9639", "9688").
  target is NULL, current is character
  
  
  testthat results ================================================================
  OK: 155 SKIPPED: 5 FAILED: 5
  1. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#111) 
  2. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#112) 
  3. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#135) 
  4. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#136) 
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  
  Error: testthat unit tests failed
  Execution halted
```

## metricTester (1.3.2)
Maintainer: Eliot Miller <eliot.isaac@gmail.com>  
Bug reports: https://github.com/eliotmiller/metricTester/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## Momocs (1.1.0)
Maintainer: Vincent Bonhomme <bonhomme.vincent@gmail.com>  
Bug reports: https://github.com/vbonhomme/Momocs/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘Momocs-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: transmute
> ### Title: Transmutes (ala dplyr) on Momocs objects
> ### Aliases: transmute
> 
> ### ** Examples
> 
> olea
An Opn object with: 
--------------------
 - $coo: 210 open outlines (99 +/- 4 coordinates)
 - $fac: 4 classifiers:
     'var' (factor 4): Aglan, Cypre, MouBo1, PicMa.
     'domes' (factor 2): cult, wild.
     'view' (factor 2): VD, VL.
     'ind' (factor 30): O1, O10, O11, O12, O13, O14, O15, O16, O17, O18, O19 ... + 19 more.
> transmute(olea, id=factor(1:length(olea)))
Error: evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following object is masked from 'package:base':

    table

method not provided. efourier is used
method not provided. efourier is used
... 8 lines ...
'nb.pts' missing and set to 91
'degree' missing and set to 5
'nb.h' not provided and set to 10 (99% harmonic power)
PC axes 1 to 7 were retained
PC axes 1 to 7 were retained
'nb.h' not provided and set to 10 (99% harmonic power)
PC axes 1 to 7 were retained
no 'fac' provided, returns meanshape
Error: processing vignette 'Momocs_speed_dating.Rmd' failed with diagnostics:
evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted
```

## mrgsolve (0.8.2)
Maintainer: Kyle T Baron  <kyleb@metrumrg.com>  
Bug reports: https://github.com/metrumresearchgroup/mrgsolve/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘mrgsolve-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: data_set
> ### Title: Select and modify a data set for simulation.
> ### Aliases: data_set data_set,mrgmod,data.frame-method
> ###   data_set,mrgmod,ANY-method data_set,mrgmod,missing-method
> 
> ### ** Examples
> 
> 
> mod <- mrgsolve:::house()
> 
> data <- expand.ev(ID=1:3, amt=c(10,20))
> 
> mod %>% data_set(data, ID > 1) %>% mrgsim
Error: Cannot convert objects of type `environment` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [21s/28s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  no applicable method for 'mutate' applied to an object of class "mrgsims"
  1: out %>% mutate(test = 2) at testthat/test-dplyr-generics.R:37
  2: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:6
  3: eval(quote(`_fseq`(`_lhs`)), env, env)
  4: eval(expr, envir, enclos)
  5: `_fseq`(`_lhs`)
  6: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  7: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  8: function_list[[k]](value)
  9: mutate(., test = 2)
  
  DONE ===========================================================================
  Don't worry, you'll get it.
  Error: Test failures
  Execution halted

checking installed package size ... NOTE
  installed size is  6.3Mb
  sub-directories of 1Mb or more:
    libs   5.0Mb
```

## mtconnectR (1.0.1)
Maintainer: Subramanyam Ravishankar <subramanyam@systeminsights.com>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘mtconnectR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: map_gcode_mtc
> ### Title: Create a mapping between simulated and actual data
> ### Aliases: map_gcode_mtc
> 
> ### ** Examples
> 
> data("example_gcode_parsed") # Parsed gcode
> data("example_mtc_device_3") # MTCDevice object of actual log data
> simulated_gcode_data = na.omit(simulate_data_from_gcode(example_gcode_parsed, 
+ start_time = 0, data_res = 0.1, data_type = "HH"))
Error in if (str_detect(current_values$state_motion, "ARC_")) distance_moved = arc_position_change(current_values,  : 
  argument is of length zero
Calls: na.omit ... simulate_data_from_gcode -> update_timestamps -> pfr_time
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [20s/20s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2. Error: (unknown) (@test-simulate_gcode.R#6) ---------------------------------
  argument is of length zero
  1: na.omit(simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, 
         data_type = "HH")) at testthat/test-simulate_gcode.R:6
  2: simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, data_type = "HH")
  3: update_timestamps(current_values, previous_values)
  4: pfr_time(current_values, previous_values, current_values$pfr, current_values$state_motion)
  
  testthat results ================================================================
  OK: 37 SKIPPED: 0 FAILED: 2
  1. Failure: (unknown) (@test-map_gcode_mtc.R#16) 
  2. Error: (unknown) (@test-simulate_gcode.R#6) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 66-74 (simulate_map_gcode.Rmd) 
Error: processing vignette 'simulate_map_gcode.Rmd' failed with diagnostics:
argument is of length zero
Execution halted

```

## mudata (0.1)
Maintainer: Dewey Dunnington <dewey@fishandwhistle.net>  
Bug reports: https://github.com/paleolimbot/mudata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’ [17s/17s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
             return(TRUE)
         }
         else {
             return(FALSE)
         }
     }, kentvillegreenwood$data$flags, reexpanded$data$flags, USE.NAMES = FALSE)
  
  testthat results ================================================================
  OK: 187 SKIPPED: 0 FAILED: 3
  1. Error: expand.tags behaviour in constructor is consistent (@test_mudata.R#75) 
  2. Error: expand/condense functions work properly for data frames (@test_tags.R#10) 
  3. Error: expand/condense tags work properly for mudata objects (@test_tags.R#30) 
  
  Error: testthat unit tests failed
  Execution halted
```

## myTAI (0.5.0)
Maintainer: Hajk-Georg Drost <hgd23@cam.ac.uk>  
Bug reports: https://github.com/HajkD/myTAI/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [56s/56s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: PlotContribution() works properly with PhyloExpressionSet input... (@test-PlotContribution.R#11) 
  PlotContribution(PhyloExpressionSetExample, legendName = "PS") produced warnings.
  
  
  2. Failure: PlotContribution() works properly with DivergenceExpressionSet input... (@test-PlotContribution.R#16) 
  PlotContribution(DivergenceExpressionSetExample, legendName = "DS") produced warnings.
  
  
  testthat results ================================================================
  OK: 197 SKIPPED: 1 FAILED: 2
  1. Failure: PlotContribution() works properly with PhyloExpressionSet input... (@test-PlotContribution.R#11) 
  2. Failure: PlotContribution() works properly with DivergenceExpressionSet input... (@test-PlotContribution.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    data   2.0Mb
    doc    2.3Mb
```

## NFP (0.99.2)
Maintainer: Yang Cao <yiluheihei@gmail.com>  
Bug reports: https://github.com/yiluheihei/NFP/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘KEGGgraph’

Packages suggested but not available for checking: ‘graphite’ ‘NFPdata’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## officer (0.1.1)
Maintainer: David Gohel <david.gohel@ardata.fr>  
Bug reports: https://github.com/davidgohel/officer/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘officer-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: slip_in_img
> ### Title: append an image
> ### Aliases: slip_in_img
> 
> ### ** Examples
> 
> library(magrittr)
> img.file <- file.path( Sys.getenv("R_HOME"), "doc", "html", "logo.jpg" )
> x <- read_docx() %>%
+   body_add_par("R logo: ", style = "Normal") %>%
+   slip_in_img(src = img.file, style = "strong", width = .3, height = .3)
Error: file.exists(src) is not TRUE
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [27s/27s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  9: ph_with_img(., type = "body", src = img.file, height = 1.06, width = 1.39)
  10: external_img(src, width = width, height = height)
  11: stopifnot(file.exists(src))
  12: stop(sprintf(ngettext(length(r), "%s is not TRUE", "%s are not all TRUE"), ch), call. = FALSE, 
         domain = NA)
  
  testthat results ================================================================
  OK: 317 SKIPPED: 0 FAILED: 4
  1. Error: image add  (@test-docx-add.R#68) 
  2. Error: pml fp_border (@test-fp_cell.R#75) 
  3. Error: css fp_border (@test-fp_cell.R#165) 
  4. Error: add img into placeholder (@test-pptx-add.R#67) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 181-191 (powerpoint.Rmd) 
Error: processing vignette 'powerpoint.Rmd' failed with diagnostics:
file.exists(src) is not TRUE
Execution halted

```

## openair (2.0-0)
Maintainer: David Carslaw <david.carslaw@york.ac.uk>  
Bug reports: https://github.com/davidcarslaw/openair/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
          user system elapsed
polarPlot  5.5  0.016   5.522
```

## padr (0.2.1)
Maintainer: Edwin Thoen <edwinthoen@gmail.com>  
Bug reports: https://github.com/EdwinTh/padr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [151s/151s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-window.R:74:8: style: Variable and function names should be all lowercase.
      ), sqlText),
         ^~~~~~~
  tests/testthat/test-window.R:76:12: style: Variable and function names should be all lowercase.
      info = sqlText
             ^~~~~~~
  
  
  testthat results ================================================================
  OK: 192 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test_zzz_lintr.R#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## parsemsf (0.1.0)
Maintainer: Benjamin Jack <benjamin.r.jack@gmail.com>  
Bug reports: https://github.com/benjaminjack/parsemsf/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘parsemsf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: make_area_table
> ### Title: Make a table of peptide areas
> ### Aliases: make_area_table
> 
> ### ** Examples
... 6 lines ...

Traceback:
 1: .Call("dplyr_select_impl", PACKAGE = "dplyr", df, vars)
 2: select_impl(.data, vars)
 3: select.grouped_df(.data, !(!(!dots)))
 4: select_.grouped_df(pep_table, .dots = setNames(old_names, new_names))
 5: select_(pep_table, .dots = setNames(old_names, new_names))
 6: make_pep_table(msf_file, min_conf, prot_regex, collapse)
 7: make_area_table(parsemsf_example("test_db.msf"))
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  34: test_code(NULL, exprs, env)
  35: source_file(path, new.env(parent = env), chdir = TRUE, encoding = encoding,     wrap = wrap)
  36: force(code)
  37: with_reporter(reporter = reporter, start_end_reporter = start_end_reporter,     {        lister$start_file(basename(path))        source_file(path, new.env(parent = env), chdir = TRUE,             encoding = encoding, wrap = wrap)        end_context()    })
  38: FUN(X[[i]], ...)
  39: lapply(paths, test_file, env = env, reporter = current_reporter,     start_end_reporter = FALSE, load_helpers = FALSE, encoding = encoding,     wrap = TRUE)
  40: force(code)
  41: with_reporter(reporter = current_reporter, results <- lapply(paths,     test_file, env = env, reporter = current_reporter, start_end_reporter = FALSE,     load_helpers = FALSE, encoding = encoding, wrap = TRUE))
  42: test_files(paths, reporter = reporter, env = env, encoding = encoding)
  43: test_dir(test_path, reporter = reporter, env = env, filter = filter,     ...)
  44: with_top_env(env, {    test_dir(test_path, reporter = reporter, env = env, filter = filter,         ...)})
  45: run_tests(package, test_path, filter, reporter, ...)
  46: test_check("parsemsf")
  An irrecoverable exception occurred. R is aborting now ...
  Segmentation fault (core dumped)

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
10: withVisible(eval(expr, envir, enclos))
11: withCallingHandlers(withVisible(eval(expr, envir, enclos)), warning = wHandler,     error = eHandler, message = mHandler)
12: handle(ev <- withCallingHandlers(withVisible(eval(expr, envir,     enclos)), warning = wHandler, error = eHandler, message = mHandler))
13: timing_fn(handle(ev <- withCallingHandlers(withVisible(eval(expr,     envir, enclos)), warning = wHandler, error = eHandler, message = mHandler)))
14: evaluate_call(expr, parsed$src[[i]], envir = envir, enclos = enclos,     debug = debug, last = i == length(out), use_try = stop_on_error !=         2L, keep_warning = keep_warning, keep_message = keep_message,     output_handler = output_handler, include_timing = include_timing)
15: evaluate(code, envir = env, new_device = FALSE, keep_warning = !isFALSE(options$warning),     keep_message = !isFALSE(options$message), stop_on_error = if (options$error &&         options$include) 0L else 2L, output_handler = knit_handlers(options$render,         options))
16: in_dir(input_dir(), evaluate(code, envir = env, new_device = FALSE,     keep_warning = !isFALSE(options$warning), keep_message = !isFALSE(options$message),     stop_on_error = if (options$error && options$include) 0L else 2L,     output_handler = knit_handlers(options$render, options)))
... 8 lines ...
24: knit2html(..., force_v1 = TRUE)
25: (if (grepl("\\.[Rr]md$", file)) knit2html_v1 else if (grepl("\\.[Rr]rst$",     file)) knit2pdf else knit)(file, encoding = encoding, quiet = quiet,     envir = globalenv())
26: vweave(...)
27: engine$weave(file, quiet = quiet, encoding = enc)
28: doTryCatch(return(expr), name, parentenv, handler)
29: tryCatchOne(expr, names, parentenv, handlers[[1L]])
30: tryCatchList(expr, classes, parentenv, handlers)
31: tryCatch({    engine$weave(file, quiet = quiet, encoding = enc)    setwd(startdir)    find_vignette_product(name, by = "weave", engine = engine)}, error = function(e) {    stop(gettextf("processing vignette '%s' failed with diagnostics:\n%s",         file, conditionMessage(e)), domain = NA, call. = FALSE)})
32: buildVignettes(dir = "/home/muelleki/git/R/dplyr/revdep/checks/parsemsf.Rcheck/vign_test/parsemsf")
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)
```

## photobiologyInOut (0.4.12)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/photobiologyinout/

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Read 16 items
  2. Failure: SpectraSuite (@test-oo.R#158) --------------------------------------
  as.numeric(getWhenMeasured(ss.spct), tz = "CET") not equal to as.numeric(ymd_hms("2013-05-06 15:13:40", tz = "CET"), tz = "CET").
  1/1 mismatches
  [1] 1.37e+09 - 1.37e+09 == 3450400
  
  
  Read 4 items
  testthat results ================================================================
  OK: 214 SKIPPED: 0 FAILED: 2
  1. Failure: SpectraSuite (@test-oo.R#76) 
  2. Failure: SpectraSuite (@test-oo.R#158) 
  
  Error: testthat unit tests failed
  Execution halted
```

## pixiedust (0.7.4)
Maintainer: Benjamin Nutter <nutter@battelle.org>  
Bug reports: https://github.com/nutterb/pixiedust/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘pixiedust-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: as.data.frame.dust
> ### Title: Convert 'dust' Object to Data Frame
> ### Aliases: as.data.frame.dust as.data.frame.dust_list
> 
> ### ** Examples
> 
> fit <- lm(mpg ~ qsec + factor(am) + wt * factor(gear), data = mtcars)
> Dust <- dust(fit) %>%
+   sprinkle(cols = 2:4, round = 2) %>%
+   sprinkle(cols = 5, fn = quote(pvalString(value))) %>%
+   sprinkle(cols = 3, font_color = "#DA70D6") %>%
+   sprinkle_print_method("html")
Error: Cannot convert objects of type `character` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [16s/16s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 80 SKIPPED: 0 FAILED: 77
  1. Error: as.data.frame.dust (@test-as.data.frame.R#6) 
  2. Error: as.data.frame.dust unsprinkled (@test-as.data.frame.R#17) 
  3. Error: lowercase color (@test-colors.R#8) 
  4. Error: uppercase color (@test-colors.R#15) 
  5. Error: mixed-case color (@test-colors.R#22) 
  6. Error: rgb color (@test-colors.R#29) 
  7. Error: rgba color (@test-colors.R#36) 
  8. Error: rgba color decimal alpha (@test-colors.R#43) 
  9. Error: rgba color decimal alpha preceded by 0 (@test-colors.R#50) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 84-91 (advancedMagic.Rmd) 
Error: processing vignette 'advancedMagic.Rmd' failed with diagnostics:
Cannot convert objects of type `character` to `tidy_quote`
Execution halted

```

## plotly (4.5.6)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: https://github.com/ropensci/plotly/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘plotly-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: add_trace
> ### Title: Add trace(s) to a plotly visualization
> ### Aliases: add_area add_bars add_boxplot add_choropleth add_contour
> ###   add_heatmap add_histogram add_histogram2d add_histogram2dcontour
> ###   add_lines add_markers add_mesh add_paths add_pie add_polygons
... 33 lines ...
+   plot_ly(x = ~long, y = ~lat) %>%
+   add_polygons(hoverinfo = "none") %>%
+   add_markers(text = ~paste(name, "<br />", pop), hoverinfo = "text",
+     data = maps::canada.cities) %>%
+   layout(showlegend = FALSE)
> 
> plot_ly(economics, x = ~date) %>% 
+   add_ribbons(ymin = ~pce - 1e3, ymax = ~pce + 1e3)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: <Anonymous> ... ribbon_dat -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [54s/55s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
     })(dots[[1L]][[3L]], dots[[2L]][[3L]])
  8: train_data(builtData, trace)
  9: ribbon_dat(data)
  10: cbind(x = tmp[["x"]], y = tmp[["ymin"]], others)
  11: cbind(deparse.level, ...)
  12: bind_cols(...) at /home/muelleki/git/R/dplyr/R/bind.r:105
  13: cbind_all(x) at /home/muelleki/git/R/dplyr/R/bind.r:99
  
  testthat results ================================================================
  OK: 812 SKIPPED: 17 FAILED: 2
  1. Error: datetimes are converted to e.g. 2013-01-02 05:00:00 (@test-ggplot-date.R#11) 
  2. Error: Complex example works (@test-plotly.R#190) 
  
  Error: testthat unit tests failed
  Execution halted
```

## poplite (0.99.17.3)
Maintainer: Daniel Bottomly <bottomly@ohsu.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: GenomicRanges
Loading required package: SummarizedExperiment
Loading required package: Biobase
Welcome to Bioconductor

    Vignettes contain introductory material; view with
    'browseVignettes()'. To cite Bioconductor, see
... 8 lines ...

The following object is masked from ‘package:base’:

    tabulate


Error: processing vignette 'poplite.Rnw' failed with diagnostics:
 chunk 15 
Error in c(DataFrameList(META = meta), tbls[unique(tags)]) : 
  could not find symbol "recursive" in environment of the generic function
Execution halted
```

## prophet (0.1)
Maintainer: Sean Taylor <sjt@fb.com>

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [9s/10s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Optimization terminated normally: 
    Convergence detected: absolute parameter change was below tolerance
  3. Failure: fit_with_holidays (@test_prophet.R#188) ----------------------------
  predict(m) threw an error.
  object 'lower' not found
  
  
  testthat results ================================================================
  OK: 32 SKIPPED: 0 FAILED: 3
  1. Failure: fit_predict (@test_prophet.R#18) 
  2. Failure: fit_predict_no_changepoints (@test_prophet.R#30) 
  3. Failure: fit_with_holidays (@test_prophet.R#188) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: Rcpp

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 43-45 (quick_start.Rmd) 
Error: processing vignette 'quick_start.Rmd' failed with diagnostics:
object 'lower' not found
Execution halted


checking installed package size ... NOTE
  installed size is 18.8Mb
  sub-directories of 1Mb or more:
    libs  18.5Mb
```

## purrr (0.2.2)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/purrr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘purrr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/purrr.Rcheck/00install.out’ for details.
```

## qualvar (0.1.0)
Maintainer: Joel Gombin <joel.gombin@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 92-106 (wilcox1973.Rmd) 
Error: processing vignette 'wilcox1973.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## qwraps2 (0.2.4)
Maintainer: Peter DeWitt <dewittpe@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘qwraps2-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: qkmplot
> ### Title: Kaplan-Meier Plot
> ### Aliases: qkmplot qkmplot_bulid_data_frame
> 
> ### ** Examples
> 
> require(survival)
Loading required package: survival
> leukemia.surv <- survival::survfit(survival::Surv(time, status) ~ x, data = survival::aml) 
> survival:::plot.survfit(leukemia.surv, conf.int = TRUE, lty = 2:3, col = 1:2)
> 
> qkmplot(leukemia.surv, conf_int = TRUE) 
Error: Cannot convert objects of type `logical` to `tidy_quote`
Execution halted
```

## R6Frame (0.1.0)
Maintainer: Kristian D. Olsen <kristian@doingit.no>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 128 SKIPPED: 1 FAILED: 16
  1. Error: bind_rows work with R6 data.frame (@test-dplyr_merges.R#10) 
  2. Error: bind_rows work with R6 data.table (@test-dplyr_merges.R#22) 
  3. Error: bind_rows work with R6 tbl_df (@test-dplyr_merges.R#34) 
  4. Error: bind_cols work with R6 data.frame (@test-dplyr_merges.R#47) 
  5. Error: bind_cols work with R6 data.table (@test-dplyr_merges.R#57) 
  6. Error: bind_cols work with R6 tbl_df (@test-dplyr_merges.R#67) 
  7. Error: left_join work with R6 data.frame (@test-dplyr_merges.R#78) 
  8. Error: left_join work with R6 data.table (@test-dplyr_merges.R#89) 
  9. Error: left_join work with R6 tbl_df (@test-dplyr_merges.R#101) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## radiant.basics (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.basics/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 26 lines ...
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.data (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.data/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.data-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: summary.pivotr
> ### Title: Summary method for pivotr
> ### Aliases: summary.pivotr
> 
> ### ** Examples
... 20 lines ...
       cut    n
      Fair  101
      Good  275
 Very Good  677
   Premium  771
     Ideal 1176
     Total 3000

Error in filter_impl(.data, dots) : incorrect length (6), expecting: 1
Calls: %>% ... <Anonymous> -> filter -> filter.tbl_df -> filter_impl -> .Call
Execution halted
```

## radiant.design (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.design/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.model (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.model/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.model-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.model.predict
> ### Title: Plot method for model.predict functions
> ### Aliases: plot.model.predict
> 
> ### ** Examples
... 25 lines ...
See help("Deprecated")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error in rep(yes, length.out = length(ans)) : 
  attempt to replicate an object of type 'closure'
Calls: %>% ... <Anonymous> -> plot -> plot.model.predict -> ifelse
Execution halted
```

## radiant.multivariate (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.multivariate/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 30 lines ...
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## rattle (4.1.0)
Maintainer: Graham Williams <Graham.Williams@togaware.com>

0 errors | 1 warning  | 3 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'rattle.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm2e.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.14 \usepackage
                [^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘pkgDepTools’

checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    data   2.5Mb
    etc    1.9Mb
    po     1.2Mb

checking dependencies in R code ... NOTE

(R:61185): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
```

## recexcavAAR (0.3.0)
Maintainer: Clemens Schmid <clemens@nevrome.de>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: kriging

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 70-71 (recexcavAAR-vignette-1.Rmd) 
Error: processing vignette 'recexcavAAR-vignette-1.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is  5.9Mb
  sub-directories of 1Mb or more:
    doc    2.5Mb
    libs   3.3Mb
```

## rfishbase (2.1.1)
Maintainer: Carl Boettiger <cboettig@ropensci.org>  
Bug reports: https://github.com/ropensci/rfishbase/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         tidy_table(data, server = server)
     }) at /home/muelleki/git/R/dplyr/R/bind.r:120
  5: FUN(X[[i]], ...)
  6: tidy_table(data, server = server)
  7: cbind(x, SpecCode = code)
  8: cbind(deparse.level, ...)
  9: bind_cols(...) at /home/muelleki/git/R/dplyr/R/bind.r:105
  10: cbind_all(x) at /home/muelleki/git/R/dplyr/R/bind.r:99
  
  testthat results ================================================================
  OK: 35 SKIPPED: 31 FAILED: 1
  1. Error: Custom queries give desired result (@test_endpoint.R#10) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: FishBase says that 'Salmo trutta' can also be misapplied to other species
                    but is returning only the best match.  
                    See synonyms('Salmo trutta') for details
Quitting from lines 88-89 (tutorial.Rmd) 
Error: processing vignette 'tutorial.Rmd' failed with diagnostics:
cannot convert object to a data frame
Execution halted

```

## rmcfs (1.2.4)
Maintainer: Michal Draminski <michal.draminski@ipipan.waw.pl>

0 errors | 1 warning  | 2 notes

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.data:
  function(data, mcfs_result, size)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.data
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    java   4.5Mb

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.data':
  ‘filter.data’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## rwunderground (0.1.0)
Maintainer: Alex Shum <alex@ALShum.com>  
Bug reports: https://github.com/alshum/rwunderground/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘rwunderground-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: set_location
> ### Title: Specifies location of request
> ### Aliases: set_location
> 
> ### ** Examples
> 
> set_location(zip_code = "90210")
[1] "90210"
> set_location(territory = "Hawaii", city = "Honolulu")
Error in `[.data.frame`(countrycode::countrycode_data, , c("country.name",  : 
  undefined columns selected
Calls: set_location ... is_valid_territory -> list_countries -> [ -> [.data.frame
Execution halted
```

## saeSim (0.8.0)
Maintainer: Sebastian Warnholz <Sebastian.Warnholz@fu-berlin.de>  
Bug reports: https://github.com/wahani/saeSim/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘saeSim-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: sim_comp_pop
> ### Title: Calculation component
> ### Aliases: sim_comp_agg sim_comp_pop sim_comp_sample
> 
> ### ** Examples
> 
> # Standard behavior
> sim_base() %>% sim_gen_x() %>% sim_comp_N()
Error: Cannot convert objects of type `integer` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  
  
  [[1]]
  [1] 1
  
  
  testthat results ================================================================
  OK: 144 SKIPPED: 0 FAILED: 4
  1. Error: Attributes are preserved (@test-preserve_attributes.R#9) 
  2. Error: Basic functionality (@test-sim_agg.R#11) 
  3. Error: base_add_id (@test-sim_base.R#6) 
  4. Error: sim_comp and comp_var (@test-sim_comp.R#12) 
  
  Error: testthat unit tests failed
  Execution halted
```

## sfc (0.1.0)
Maintainer: Hu Sheng <shenghu@nju.edu.cn>  
Bug reports: https://github.com/ctfysh/sfc/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## sf (0.3-4)
Maintainer: Edzer Pebesma <edzer.pebesma@uni-muenster.de>  
Bug reports: https://github.com/edzer/sfr/issues/

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr
> ### Title: Dplyr verb methods for sf objects
> ### Aliases: dplyr filter_.sf dplyr arrange_.sf dplyr distinct_.sf dplyr
> ###   group_by_.sf dplyr mutate_.sf dplyr transmute_.sf dplyr select_.sf
> ###   dplyr rename_.sf dplyr slice_.sf dplyr summarise_.sf dplyr gather_.sf
... 22 lines ...
Simple feature collection with 100 features and 14 fields
geometry type:  MULTIPOLYGON
dimension:      XY
bbox:           xmin: -84.32385 ymin: 33.88199 xmax: -75.45698 ymax: 36.58965
epsg (SRID):    4267
proj4string:    +proj=longlat +datum=NAD27 +no_defs
> nc %>% filter(AREA > .1) %>% plot()
Error in data.matrix(x) : 
  (list) object cannot be coerced to type 'double'
Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
Execution halted

checking tests ... ERROR
  Running ‘cast.R’
  Comparing ‘cast.Rout’ to ‘cast.Rout.save’ ... OK
  Running ‘crs.R’
  Comparing ‘crs.Rout’ to ‘crs.Rout.save’ ... OK
  Running ‘dist.R’
  Comparing ‘dist.Rout’ to ‘dist.Rout.save’ ... OK
  Running ‘dplyr.R’
Running the tests in ‘tests/dplyr.R’ failed.
Last 13 lines of output:
  
  The following objects are masked from 'package:stats':
  
      filter, lag
  
  The following objects are masked from 'package:base':
  
      intersect, setdiff, setequal, union
  
  > nc = st_read(system.file("shape/nc.shp", package="sf"), quiet = TRUE)
  > nc %>% filter(AREA > .1) %>% plot()
  Error in data.matrix(x) : 
    (list) object cannot be coerced to type 'double'
  Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
  Execution halted

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    3.9Mb
    libs   3.8Mb
```

## shazam (0.1.4)
Maintainer: Jason Vander Heiden <jason.vanderheiden@yale.edu>  
Bug reports: https://bitbucket.org/kleinstein/shazam/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 39 lines ...
                        user system elapsed
createTargetingModel   6.668  0.012   6.675
createTargetingMatrix  6.628  0.048   6.695
plotBaselineSummary    6.120  0.132   6.241
extendMutabilityMatrix 6.008  0.040   6.046
testBaseline           5.756  0.168   5.899
summarizeBaseline      5.560  0.160   5.731
createMutabilityMatrix 5.504  0.012   5.518
observedMutations      5.424  0.040   5.465
getBaselineStats       5.196  0.148   5.351
** found \donttest examples: check also with --run-donttest
```

## sjmisc (2.3.1)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/sjmisc/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjmisc-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: descr
> ### Title: Basic descriptive statistics
> ### Aliases: descr
> 
> ### ** Examples
> 
> data(efc)
> descr(efc, e17age, c160age)
Error in mutate_impl(.data, dots) : object '.x' not found
Calls: descr ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 117-128 (exploringdatasets.Rmd) 
Error: processing vignette 'exploringdatasets.Rmd' failed with diagnostics:
object 'quol_5' not found
Execution halted

```

## sjPlot (2.3.1)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/devel/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjPlot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: sjp.kfold_cv
> ### Title: Plot model fit from k-fold cross-validation
> ### Aliases: sjp.kfold_cv
> 
> ### ** Examples
> 
> library(sjmisc)
> data(efc)
> 
> sjp.kfold_cv(efc, neg_c_7 ~ e42dep + c172code + c12hour)
Error in mutate_impl(.data, dots) : object 'neg_c_7' not found
Calls: sjp.kfold_cv ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    select

The following objects are masked from 'package:stats':

    filter, lag

... 8 lines ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Computing p-values via Kenward-Roger approximation. Use `p.kr = FALSE` if computation takes too long.
Quitting from lines 66-67 (sjtlmer.Rmd) 
Error: processing vignette 'sjtlmer.Rmd' failed with diagnostics:
is.data.frame(df) is not TRUE
Execution halted
```

## sjstats (0.9.0)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/sjstats/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sjstats-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: check_assumptions
> ### Title: Check model assumptions
> ### Aliases: check_assumptions outliers heteroskedastic autocorrelation
> ###   normality multicollin
> 
... 61 lines ...
+   bootstrap(50) %>%
+   mutate(models = lapply(.$strap, function(x) {
+     lm(neg_c_7 ~ e42dep + c12hour + c161sex, data = x)
+   }))
> 
> # for list-variables, argument 'model.column' is the
> # quoted name of the list-variable with fitted models
> tmp %>% heteroskedastic("models")
Error in eval(expr, envir, enclos) : object 'x' not found
Calls: %>% ... .Call -> .f -> <Anonymous> -> ncvTest.lm -> eval -> eval
Execution halted

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘piecewiseSEM’
```

## SpaDES (1.3.1)
Maintainer: Alex M Chubaty <alexander.chubaty@canada.ca>  
Bug reports: https://github.com/PredictiveEcology/SpaDES/issues

0 errors | 2 warnings | 2 notes

```
checking whether package ‘SpaDES’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/SpaDES.Rcheck/00install.out’ for details.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    RFoptions

The following object is masked from 'package:raster':

    atan2

... 8 lines ...

Attaching package: 'grid'

The following object is masked from 'package:SpaDES':

    gpar

Quitting from lines 352-356 (ii-modules.Rmd) 
Error: processing vignette 'ii-modules.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘fastshp’

checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    R     2.5Mb
    doc   2.1Mb
```

## sparklyr (0.5.3)
Maintainer: Javier Luraschi <javier@rstudio.com>  
Bug reports: https://github.com/rstudio/sparklyr/issues

0 errors | 1 warning  | 0 notes

```
checking S3 generic/method consistency ... WARNING
copy_to:
  function(dest, df, name, overwrite, ...)
copy_to.spark_connection:
  function(dest, df, name, memory, repartition, overwrite, ...)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.
```

## sparseHessianFD (0.3.2)
Maintainer: Michael Braun <braunm@smu.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'sparseHessianFD.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.27 \usepackage
                {algorithmic}^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## spbabel (0.4.5)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spbabel/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [34s/35s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
             where = .classEnv(ClassDef)))
         if (identical(ok, FALSE)) 
             stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
                 dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
         TRUE
     })(structure("SpatialPointsDataFrame", package = "sp"), "data", "matrix") at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/SpatialLinesDataFrame-methods.R:93
  10: stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
         dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
  
  testthat results ================================================================
  OK: 57 SKIPPED: 2 FAILED: 1
  1. Error: utils tests work (@test-utils.R#16) 
  
  Error: testthat unit tests failed
  Execution halted
```

## spdplyr (0.1.2)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spdplyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘spdplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr-Spatial
> ### Title: Dplyr verbs for Spatial
> ### Aliases: arrange_.Spatial distinct_.Spatial dplyr-Spatial
> ###   filter_.Spatial group_by_.Spatial inner_join.Spatial
> ###   left_join.Spatial mutate_.Spatial rename_.Spatial select_.Spatial
... 12 lines ...
Attaching package: ‘raster’

The following object is masked from ‘package:dplyr’:

    select

> wrld_simpl %>% mutate(NAME = "allthesame", REGION = row_number())
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "c('SpatialPolygonsDataFrame', 'SpatialPolygons', 'Spatial', 'SpatialVector')"
Calls: %>% ... _fseq -> freduce -> withVisible -> <Anonymous> -> mutate
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: expect_silent(mutate(poly1, NAME = "all same", AREA = row_number())) at testthat/test-basic-dplyr.R:98
  2: evaluate_promise(object) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-output.R:249
  3: with_sink(temp, withCallingHandlers(withVisible(code), warning = handle_warning, 
         message = handle_message)) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:42
  4: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:130
  5: withVisible(code)
  6: mutate(poly1, NAME = "all same", AREA = row_number())
  
  testthat results ================================================================
  OK: 25 SKIPPED: 3 FAILED: 2
  1. Error: distinct works (@test-basic-dplyr.R#92) 
  2. Error: mutate works (@test-basic-dplyr.R#98) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: sp
Checking rgeos availability: TRUE
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 42-58 (spdplyr.Rmd) 
Error: processing vignette 'spdplyr.Rmd' failed with diagnostics:
no applicable method for 'mutate' applied to an object of class "c('SpatialPolygonsDataFrame', 'SpatialPolygons', 'Spatial')"
Execution halted

```

## srvyr (0.2.0)
Maintainer: Greg Freedman Ellis <greg.freedman@gmail.com>  
Bug reports: https://github.com/gergness/srvyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘srvyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: as_survey
> ### Title: Create a tbl_svy from a data.frame
> ### Aliases: as_survey as_survey.data.frame as_survey.survey.design2
> ###   as_survey.svyrepdesign as_survey.tbl_sql as_survey.twophase2
> ###   as_survey_
... 24 lines ...
The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> data(api)
> 
> # stratified sample
> dstrata <- apistrat %>%
+   as_survey(strata = stype, weights = pw)
Error: Cannot convert objects of type `environment` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 3 SKIPPED: 30 FAILED: 14
  1. Error: as_survey correctly uses survey design's arguments (@expect-equality.R#52) 
  2. Error: as_survey correctly uses replicate's arguments (@expect-equality.R#52) 
  3. Error: as_survey correctly uses twophase's arguments (@expect-equality.R#52) 
  4. Error: as_survey_design works with both tbl_dfs and data.frames (@expect-equality.R#52) 
  5. Error: as_survey_design ids argument = 1 doesn't use the first column as an id #7 (@expect-equality.R#52) 
  6. Error: (unknown) (@test_as_survey_rep.r#18) 
  7. Error: (unknown) (@test_as_survey_twophase.r#14) 
  8. Error: (unknown) (@test_basic_example.r#8) 
  9. Error: (unknown) (@test_cascade.r#7) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    dotchart


Attaching package: 'dplyr'

The following objects are masked from 'package:stats':
... 8 lines ...

Attaching package: 'srvyr'

The following object is masked from 'package:stats':

    filter

Quitting from lines 63-69 (srvyr-vs-survey.Rmd) 
Error: processing vignette 'srvyr-vs-survey.Rmd' failed with diagnostics:
Cannot convert objects of type `environment` to `tidy_quote`
Execution halted
```

## ss3sim (0.9.3)
Maintainer: Sean Anderson <sean@seananderson.ca>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ss3sim-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: change_em_binning
> ### Title: Change population and observed length composition bins in an SS
> ###   estimation model
> ### Aliases: change_em_binning
> 
... 69 lines ...
 6: rename_(., Lbin_lo = ~lbin_new_low, Lbin_hi = ~lbin_new_high)
 7: function_list[[i]](value)
 8: freduce(value, `_function_list`)
 9: `_fseq`(`_lhs`)
10: eval(expr, envir, enclos)
11: eval(quote(`_fseq`(`_lhs`)), env, env)
12: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
13: summarise_each_(new_cal, funs_(~sum), vars = dat_cols) %>% rename_(Lbin_lo = ~lbin_new_low,     Lbin_hi = ~lbin_new_high) %>% as.data.frame
14: change_em_binning(olddat, dat_file_out = NULL, bin_vector = c(4,     6, 8), lbin_method = 1, write_file = FALSE)
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)

checking whether package ‘ss3sim’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ss3sim.Rcheck/00install.out’ for details.
```

## stormwindmodel (0.1.0)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>  
Bug reports: https://github.com/geanders/stormwindmodel/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

... 8 lines ...
The following object is masked from 'package:dplyr':

    combine

Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=georgia&zoom=5&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=georgia&sensor=false
Quitting from lines 220-233 (Details.Rmd) 
Error: processing vignette 'Details.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted
```

## stplanr (0.1.7-3)
Maintainer: Robin Lovelace <rob00x@gmail.com>  
Bug reports: https://github.com/ropensci/stplanr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘stplanr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: onewayid
> ### Title: Aggregate ods so they become non-directional
> ### Aliases: onewayid onewayid.SpatialLines onewayid.data.frame
> 
> ### ** Examples
> 
> data(flow)
> flow_oneway = onewayid(flow, attrib = 3)
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Warning: 'dplyr::summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error: Cannot convert objects of type `double` to `tidy_quote`
Execution halted
```

## tadaatoolbox (0.10.0)
Maintainer: Lukas Burk <lukas@quantenbrot.de>  
Bug reports: https://github.com/tadaadata/tadaatoolbox/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tadaatoolbox-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: tadaa_t.test
> ### Title: Tadaa, t-Test!
> ### Aliases: tadaa_t.test
> 
> ### ** Examples
> 
> df <- data.frame(x = runif(100), y = sample(c("A", "B"), 100, TRUE))
> tadaa_t.test(df, x, y)
Warning: attributes are not identical across measure variables; they will be dropped
Error: Cannot convert objects of type `character` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: attributes are not identical across measure variables; they will be dropped
Quitting from lines 37-38 (overview.Rmd) 
Error: processing vignette 'overview.Rmd' failed with diagnostics:
Cannot convert objects of type `character` to `tidy_quote`
Execution halted

```

## texmexseq (0.3)
Maintainer: Scott Olesen <swo@mit.edu>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘texmexseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: quad.table
> ### Title: Create and plot "quads" of samples
> ### Aliases: quad.table quad.plot
> ### Keywords: hplot manip design
> 
... 12 lines ...
Warning in value[[3L]](cond) : fit 2 failed
> 
> # pull out a quad, imagining that samples 1 and 2 were the control samples
> # and 3 and 4 were the treatment
> q <- quad.table(z.table, 'sample1', 'sample2', 'sample3', 'sample4')
> 
> # plot it
> p <- quad.plot(q)
Error in get("d.control") : object 'd.control' not found
Calls: quad.plot ... map -> lapply -> FUN -> overscope_eval -> .Call -> get
Execution halted
```

## textmining (0.0.1)
Maintainer: Jan Idziak <JanIdziak@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking whether package ‘textmining’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/textmining.Rcheck/00install.out’ for details.
```

## textreuse (0.1.4)
Maintainer: Lincoln Mullen <lincoln@lincolnmullen.com>  
Bug reports: https://github.com/ropensci/textreuse/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘textreuse-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: lsh_candidates
> ### Title: Candidate pairs from LSH comparisons
> ### Aliases: lsh_candidates
> 
> ### ** Examples
> 
> dir <- system.file("extdata/legal", package = "textreuse")
> minhash <- minhash_generator(200, seed = 234)
> corpus <- TextReuseCorpus(dir = dir,
+                           tokenizer = tokenize_ngrams, n = 5,
+                           minhash_func = minhash)
> buckets <- lsh(corpus, bands = 50)
> lsh_candidates(buckets)
Error: Cannot convert objects of type `double` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  14: coerce_type(lazy, "tidy_quote", quote = lazy, symbol = , language = quosure(lazy, 
         env), string = {
         if (warn) 
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  15: abort_coercion(.x, .to) at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/types.R:422
  16: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpBT2s6A/devtoolse8e15260319/hadley-rlang-7c42c80/R/types.R:435
  
  testthat results ================================================================
  OK: 129 SKIPPED: 2 FAILED: 1
  1. Error: (unknown) (@test-lsh.R#10) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 75-79 (textreuse-minhash.Rmd) 
Error: processing vignette 'textreuse-minhash.Rmd' failed with diagnostics:
Cannot convert objects of type `double` to `tidy_quote`
Execution halted

```

## tidyjson (0.2.1)
Maintainer: Jeremy Stanley <jeremy.stanley@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘tidyjson-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: companies
> ### Title: Startup company information for 1,000 companies
> ### Aliases: companies
> 
> ### ** Examples
... 34 lines ...
+   enter_object("relationships") %>%
+   gather_array("relationship.index") %>%
+   spread_values(
+     is.past = jlogical("is_past"),
+     name = jstring("person", "permalink"),
+     title = jstring("title")
+   )
Error in eval(expr, envir, enclos) : 
  argument "json.column" is missing, with no default
Calls: %>% ... tryCatchList -> tryCatchOne -> doTryCatch -> eval -> eval
Execution halted

checking R code for possible problems ... NOTE
read_json: no visible global function definition for ‘tail’
Undefined global functions or variables:
  tail
Consider adding
  importFrom("utils", "tail")
to your NAMESPACE file.
```

## tidyquant (0.4.0)
Maintainer: Matt Dancho <mdancho@business-science.io>  
Bug reports: https://github.com/mdancho84/tidyquant/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tidyquant-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: scale_manual
> ### Title: tidyquant colors and fills for ggplot2.
> ### Aliases: scale_manual scale_color_tq scale_fill_tq
> 
> ### ** Examples
> 
> # Load libraries
> library(tidyquant)
> 
> # Get stock prices
> stocks <- c("AAPL", "FB", "NFLX") %>%
+     tq_get(from = "2013-01-01",
+            to   = "2017-01-01")
Error in mutate_impl(.data, dots) : object '.x' not found
Calls: %>% ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [9s/28s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         from = "2015-01-01", to = "2016-01-01"))) at /home/muelleki/git/R/dplyr/R/manip.r:218
  10: mutate_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:83
  
  testthat results ================================================================
  OK: 97 SKIPPED: 2 FAILED: 7
  1. Error: (unknown) (@test_tq_get_compound_getters.R#7) 
  2. Error: (unknown) (@test_tq_get_financials.R#7) 
  3. Error: (unknown) (@test_tq_get_stock_prices.R#10) 
  4. Error: (unknown) (@test_tq_mutate.R#27) 
  5. Error: (unknown) (@test_tq_performance.R#8) 
  6. Error: (unknown) (@test_tq_portfolio.R#8) 
  7. Error: (unknown) (@test_tq_transmute.R#12) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  number of rows of result is not a multiple of vector length (arg 2)
Warning: Computation failed in `stat_b_bands_ribbon()`:
All inputs to rbind.fill must be data.frames
Warning in cbind(new, unique[rep(1, nrow(new)), missing, drop = FALSE]) :
  number of rows of result is not a multiple of vector length (arg 2)
Warning: Computation failed in `stat_b_bands_ribbon()`:
All inputs to rbind.fill must be data.frames
... 8 lines ...
All inputs to rbind.fill must be data.frames
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Getting data...

Getting data...

Quitting from lines 115-117 (TQ01-core-functions-in-tidyquant.Rmd) 
Error: processing vignette 'TQ01-core-functions-in-tidyquant.Rmd' failed with diagnostics:
object '.x' not found
Execution halted
```

## tidyr (0.6.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/tidyverse/tidyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  x[2]: "2"
  y[2]: "b"
  
  x[3]: "2"
  y[3]: "b"
  
  
  testthat results ================================================================
  OK: 191 SKIPPED: 0 FAILED: 2
  1. Failure: .id creates vector of names for vector unnest (@test-unnest.R#59) 
  2. Failure: .id creates vector of names for data frame unnest (@test-unnest.R#69) 
  
  Error: testthat unit tests failed
  Execution halted
```

## tidytext (0.1.2)
Maintainer: Julia Silge <julia.silge@gmail.com>  
Bug reports: http://github.com/juliasilge/tidytext/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [10s/11s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  7: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  8: function_list[[k]](value)
  9: count(., document, wt = gamma)
  10: tally(x, wt = !(!wt), sort = sort) at /home/muelleki/git/R/dplyr/R/count-tally.R:113
  11: summarise(x, `:=`(!(!n_name), !(!n))) at /home/muelleki/git/R/dplyr/R/count-tally.R:81
  12: summarise.tbl_df(x, `:=`(!(!n_name), !(!n))) at /home/muelleki/git/R/dplyr/R/manip.r:143
  13: summarise_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:94
  
  testthat results ================================================================
  OK: 132 SKIPPED: 0 FAILED: 2
  1. Error: can tidy beta matrix (@test-lda-tidiers.R#26) 
  2. Error: can tidy gamma matrix (@test-lda-tidiers.R#48) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union
... 8 lines ...

Attaching package: 'NLP'

The following object is masked from 'package:ggplot2':

    annotate

Quitting from lines 49-57 (tidying_casting.Rmd) 
Error: processing vignette 'tidying_casting.Rmd' failed with diagnostics:
invalid 'type' (closure) of argument
Execution halted
```

## treeplyr (0.1.2)
Maintainer: Josef Uyeda <josef.uyeda@gmail.com>  
Bug reports: https://github.com/uyedaj/treeplyr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘treeplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: filter_.treedata
> ### Title: Function for filtering rows from an object of class 'treedata'
> ### Aliases: filter.grouped_treedata filter.treedata
> ###   filter_.grouped_treedata filter_.treedata
> 
> ### ** Examples
> 
> data(anolis)
> td <- make.treedata(anolis$phy, anolis$dat, name_column=1)
> tdfilter <- filter(td, island=="Cuba", SVL > 3.5)
Error in mutate_impl(.data, dots) : 
  incompatible size (0), expecting 100 (the number of rows) or one
Calls: filter ... filter_.treedata -> mutate -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted
```

## turfR (0.8-7)
Maintainer: Jack Horne <jack@jackhorne.net>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘turfR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: turf
> ### Title: TURF Analysis for R
> ### Aliases: turf
> 
> ### ** Examples
> 
>   ##Example 1
>   ##Evaluate all combinations of 3, 4, 5 and 6 items drawn from 10 items.
>   data(turf_ex_data)
>   ex1 <- turf(turf_ex_data, 10, 3:6)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: turf -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking R code for possible problems ... NOTE
turf: no visible global function definition for ‘read.table’
turf: no visible global function definition for ‘flush.console’
turf.combos: no visible global function definition for ‘combn’
Undefined global functions or variables:
  combn flush.console read.table
Consider adding
  importFrom("utils", "combn", "flush.console", "read.table")
to your NAMESPACE file.
```

## unpivotr (0.1.0)
Maintainer: Duncan Garmonsway <nacnudus@gmail.com>  
Bug reports: https://github.com/nacnudus/unpivotr/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  testthat results ================================================================
  OK: 87 SKIPPED: 0 FAILED: 9
  1. Error: Compass directions N, NNW, W, and WNW work (@test-join_header.R#128) 
  2. Error: Compass directions NNE and WSW work (@test-join_header.R#155) 
  3. Error: Compass directions S, SSE , E and ESE work (@test-join_header.R#178) 
  4. Error: Compass directions SSW and ENE work (@test-join_header.R#205) 
  5. Error: Compass directions ABOVE and LEFT work (@test-join_header.R#228) 
  6. Error: Compass directions BELOW and RIGHT work (@test-join_header.R#251) 
  7. Error: Compass directions ABOVE and LEFT work with boundaries (@test-join_header.R#286) 
  8. Error: Compass directions BELOW and RIGHT work with boundaries (@test-join_header.R#332) 
  9. Error: join_header() works (@test-join_header.R#361) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 56-61 (compass-directions.Rmd) 
Error: processing vignette 'compass-directions.Rmd' failed with diagnostics:
cannot coerce type 'closure' to vector of type 'integer'
Execution halted

```

## useful (1.2.1)
Maintainer: Jared P. Lander <packages@jaredlander.com>

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Objects equal but not identical
  
  
  2. Failure: cart2pol maps cartesian to polar correctly and returns degrees (@test-coordinates.R#150) 
  cart2pol(dAll$x, dAll$y, degrees = TRUE) not identical to bind_rows(...).
  Objects equal but not identical
  
  
  testthat results ================================================================
  OK: 337 SKIPPED: 0 FAILED: 2
  1. Failure: cart2pol maps cartesian to polar correctly (@test-coordinates.R#131) 
  2. Failure: cart2pol maps cartesian to polar correctly and returns degrees (@test-coordinates.R#150) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## valr (0.1.1)
Maintainer: Jay Hesselberth <jay.hesselberth@gmail.com>  
Bug reports: https://github.com/jayhesselberth/valr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘valr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/valr.Rcheck/00install.out’ for details.
```

## vcfR (1.4.0)
Maintainer: Brian J. Knaus <briank.lists@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘vcfR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: Convert to tidy data frames
> ### Title: Convert vcfR objects to tidy data frames
> ### Aliases: 'Convert to tidy data frames' extract_gt_tidy
> ###   extract_info_tidy vcfR2tidy vcf_field_names
> 
> ### ** Examples
> 
> # load the data
> data("vcfR_test")
> vcf <- vcfR_test
> 
> 
> # extract all the INFO and FORMAT fields into a list of tidy
> # data frames: fix, gt, and meta. Here we don't coerce columns
> # to integer or numeric types...
> Z <- vcfR2tidy(vcf)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: vcfR2tidy ... eval -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    libs   5.7Mb
```

## VWPre (0.9.5)
Maintainer: Vincent Porretta <vincentporretta@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Calculating proportions...
There are 320 data points with less than 20 samples per bin.
These can be examined and/or removed using the column 'NSamples'.
These occur at time bin(s): 1180
Subsequent Empirical Logit calculations may be influenced by the number of samples (depending on the number of observations requested).
Sampling rate(s) present in the data are: 50 Hz.

... 8 lines ...
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Quitting from lines 81-87 (SR_Plotting.Rmd) 
Error: processing vignette 'SR_Plotting.Rmd' failed with diagnostics:
object 'talker' not found
Execution halted
```

## WHO (0.2)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: https://www.github.com/expersso/WHO/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > test_check("WHO")
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  not compatible with STRSXP
  1: get_codes(TRUE) at testthat/tests.R:18
  2: dplyr::bind_rows(df_list)
  3: bind_rows_(x, .id) at /home/muelleki/git/R/dplyr/R/bind.r:86
  
  testthat results ================================================================
  OK: 1485 SKIPPED: 0 FAILED: 1
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  
  Error: testthat unit tests failed
  Execution halted
```

## wordbankr (0.2.0)
Maintainer: Mika Braginsky <mika.br@gmail.com>  
Bug reports: http://github.com/langcog/wordbankr/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 27-31 (wordbankr.Rmd) 
Error: processing vignette 'wordbankr.Rmd' failed with diagnostics:
argument is missing, with no default
Execution halted

```

## wrswoR (1.0-1)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/krlmlr/wrswoR/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 622-635 (wrswoR.Rmd) 
Error: processing vignette 'wrswoR.Rmd' failed with diagnostics:

TeX was unable to calculate metrics for the following string
or character:

	36

Common reasons for failure include:
  * The string contains a character which is special to LaTeX unless
    escaped properly, such as % or $.
  * The string makes use of LaTeX commands provided by a package and
    the tikzDevice was not told to load the package.

The contents of the LaTeX log of the aborted run have been printed above,
it may contain additional details as to why the metric calculation failed.
Execution halted

```

## Zelig (5.0-17)
Maintainer: Christopher Gandrud <zelig.zee@gmail.com>  
Bug reports: https://github.com/IQSS/Zelig/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [69s/70s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
            mean         sd       50%      2.5%     97.5%
  [1,] 0.6990619 0.09467077 0.7081549 0.5028567 0.8603348
  pv
          0    1
  [1,] 0.31 0.69
  fd
             mean        sd        50%      2.5%       97.5%
  [1,] -0.2377065 0.1148547 -0.2380952 -0.466211 -0.01659926
  testthat results ================================================================
  OK: 195 SKIPPED: 1 FAILED: 2
  1. Error: REQUIRE TEST Bayes Diagnostics for factors (@test-bayesdiagnostics.R#19) 
  2. Error: REQUIRE TEST for to_zelig_mi (@test-utils.R#28) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 12.1Mb
  sub-directories of 1Mb or more:
    R  11.0Mb
```

