# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.3 (2017-03-06) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |(EN)                         |
|collate  |en_US.UTF-8                  |
|tz       |Zulu                         |
|date     |2017-04-11                   |

## Packages

|package        |*  |version    |date       |source                           |
|:--------------|:--|:----------|:----------|:--------------------------------|
|assertthat     |   |0.1        |2013-12-06 |cran (@0.1)                      |
|BH             |   |1.62.0-1   |2016-11-19 |cran (@1.62.0-)                  |
|bindrcpp       |   |0.1        |2016-12-11 |cran (@0.1)                      |
|bit64          |   |0.9-5      |2015-07-05 |cran (@0.9-5)                    |
|covr           |   |2.2.2      |2017-01-05 |cran (@2.2.2)                    |
|DBI            |   |0.6-11     |2017-04-06 |Github (rstats-db/DBI@20f1f02)   |
|dbplyr         |   |0.0.0.9000 |2017-04-11 |Github (hadley/dbplyr@ce94c15)   |
|dplyr          |   |0.5.0.9001 |2017-04-11 |Github (hadley/dplyr@b72b5dc)    |
|dtplyr         |   |0.0.1      |2016-06-27 |cran (@0.0.1)                    |
|ggplot2        |   |2.2.1      |2016-12-30 |cran (@2.2.1)                    |
|glue           |   |0.0.0.9000 |2017-04-11 |Github (tidyverse/glue@6e77c3b)  |
|hms            |   |0.3        |2016-11-22 |cran (@0.3)                      |
|knitr          |   |1.15.1     |2016-11-22 |cran (@1.15.1)                   |
|Lahman         |   |5.0-0      |2016-08-27 |cran (@5.0-0)                    |
|magrittr       |   |1.5        |2014-11-22 |CRAN (R 3.3.1)                   |
|microbenchmark |   |1.4-2.1    |2015-11-25 |cran (@1.4-2.1)                  |
|nycflights13   |   |0.2.2      |2017-01-27 |cran (@0.2.2)                    |
|pkgconfig      |   |2.0.1      |2017-03-21 |cran (@2.0.1)                    |
|plogr          |   |0.1-1      |2016-09-24 |cran (@0.1-1)                    |
|R6             |   |2.2.0      |2016-10-05 |cran (@2.2.0)                    |
|Rcpp           |   |0.12.10    |2017-04-11 |Github (RcppCore/Rcpp@43e53b0)   |
|rlang          |   |0.0.0.9014 |2017-04-11 |Github (hadley/rlang@ac7498c)    |
|rmarkdown      |   |1.4        |2017-03-24 |cran (@1.4)                      |
|RMySQL         |   |0.10.11    |2017-03-29 |cran (@0.10.11)                  |
|RPostgreSQL    |   |0.4-1      |2016-05-08 |cran (@0.4-1)                    |
|RSQLite        |   |1.1-2      |2017-01-08 |cran (@1.1-2)                    |
|testthat       |   |1.0.2.9000 |2017-02-27 |Github (hadley/testthat@b72a228) |
|tibble         |   |1.3.0      |2017-03-31 |local                            |

# Check results

142 packages with problems

|package              |version | errors| warnings| notes|
|:--------------------|:-------|------:|--------:|-----:|
|ameco                |0.2.6   |      1|        0|     1|
|assertr              |2.0.2   |      1|        0|     0|
|backtestGraphics     |0.1.6   |      1|        0|     0|
|bayesplot            |1.1.0   |      0|        1|     0|
|bibliometrix         |1.5     |      0|        1|     0|
|bigrquery            |0.3.0   |      1|        0|     0|
|bioOED               |0.1.1   |      1|        0|     0|
|blscrapeR            |2.1.2   |      1|        0|     0|
|bossMaps             |0.1.0   |      0|        1|     0|
|broom                |0.4.2   |      2|        1|     0|
|carpenter            |0.2.0   |      2|        1|     0|
|chunked              |0.3     |      1|        0|     1|
|clustrd              |1.1.0   |      0|        1|     0|
|codingMatrices       |0.2.2   |      0|        1|     0|
|cometExactTest       |0.1.3   |      0|        1|     0|
|crawl                |2.0.1   |      0|        1|     1|
|datacheckr           |0.1.2   |      1|        0|     0|
|dat                  |0.1.0   |      1|        0|     0|
|ddpcr                |1.6     |      0|        1|     0|
|decoder              |1.1.12  |      0|        1|     0|
|DeLorean             |1.2.4   |      0|        1|     1|
|describer            |0.2.0   |      1|        0|     0|
|DiagrammeR           |0.9.0   |      1|        0|     1|
|dtplyr               |0.0.1   |      1|        0|     0|
|easyformatr          |0.1.2   |      2|        1|     0|
|eechidna             |0.1     |      0|        1|     0|
|efreadr              |0.1.1   |      0|        1|     0|
|emil                 |2.2.6   |      1|        0|     0|
|etl                  |0.3.5   |      2|        1|     0|
|explor               |0.3.1   |      1|        0|     0|
|eyelinker            |0.1     |      0|        1|     0|
|eyetrackingR         |0.1.6   |      2|        0|     0|
|ezsummary            |0.2.1   |      1|        1|     0|
|fitcoach             |1.0     |      1|        0|     0|
|flextable            |0.1.0   |      0|        1|     0|
|forestmodel          |0.4.0   |      1|        0|     0|
|freqweights          |1.0.2   |      2|        0|     1|
|FRK                  |0.1.1   |      1|        2|     2|
|FSelectorRcpp        |0.1.2   |      1|        0|     2|
|funModeling          |1.6.2   |      0|        1|     0|
|futureheatwaves      |1.0.3   |      0|        1|     0|
|geoknife             |1.5.4   |      0|        1|     0|
|geoSpectral          |0.17    |      1|        0|     0|
|ggCompNet            |0.1.0   |      0|        1|     1|
|ggedit               |0.2.1   |      0|        1|     0|
|ggforce              |0.1.1   |      1|        0|     0|
|ggfortify            |0.4.1   |      2|        0|     1|
|ggraph               |1.0.0   |      1|        0|     1|
|ggvis                |0.4.3   |      2|        0|     0|
|graphicalVAR         |0.2     |      1|        0|     0|
|grattan              |1.4.1.2 |      1|        1|     1|
|groupdata2           |0.1.0   |      0|        1|     0|
|gutenbergr           |0.1.2   |      1|        1|     0|
|harrietr             |0.2.2   |      1|        0|     0|
|heemod               |0.9.0   |      2|        1|     0|
|highcharter          |0.5.0   |      1|        1|     1|
|HTSSIP               |1.0.3   |      1|        0|     0|
|huxtable             |0.1.1   |      1|        1|     0|
|HydeNet              |0.10.5  |      2|        1|     0|
|IATscores            |0.1-2   |      1|        0|     0|
|implyr               |0.1.0   |      1|        0|     0|
|incadata             |0.5.3   |      1|        0|     0|
|inferr               |0.1.0   |      0|        1|     0|
|infuser              |0.2.6   |      0|        2|     0|
|ipft                 |0.5     |      0|        1|     0|
|janitor              |0.2.1   |      1|        0|     0|
|Lahman               |5.0-0   |      0|        1|     1|
|lplyr                |0.1.6   |      1|        1|     0|
|mason                |0.2.5   |      0|        1|     0|
|medicare             |0.2.1   |      0|        1|     0|
|merTools             |0.3.0   |      1|        0|     0|
|metacoder            |0.1.2   |      1|        0|     0|
|metricTester         |1.3.2   |      0|        1|     0|
|Momocs               |1.1.0   |      1|        1|     0|
|MonetDBLite          |0.3.1   |      2|        0|     2|
|mrgsolve             |0.8.6   |      0|        1|     1|
|mtconnectR           |1.1.0   |      2|        1|     0|
|mudata               |0.1     |      1|        0|     0|
|myTAI                |0.5.0   |      1|        1|     1|
|NFP                  |0.99.2  |      0|        1|     2|
|officer              |0.1.1   |      2|        1|     0|
|openair              |2.0-0   |      0|        1|     0|
|padr                 |0.2.1   |      1|        0|     0|
|parsemsf             |0.1.0   |      2|        1|     1|
|pivottabler          |0.1.0   |      1|        1|     0|
|pixiedust            |0.7.4   |      1|        0|     0|
|plotly               |4.5.6   |      1|        0|     0|
|poppr                |2.4.0   |      0|        1|     0|
|prophet              |0.1     |      1|        1|     1|
|purrr                |0.2.2   |      1|        0|     0|
|qualvar              |0.1.0   |      0|        1|     0|
|radiant.basics       |0.6.0   |      0|        1|     0|
|radiant.data         |0.6.0   |      1|        0|     0|
|radiant.design       |0.6.0   |      0|        1|     0|
|radiant.model        |0.6.0   |      1|        0|     0|
|radiant.multivariate |0.6.0   |      0|        1|     0|
|rattle               |4.1.0   |      0|        1|     3|
|recexcavAAR          |0.3.0   |      0|        1|     1|
|REDCapR              |0.9.7   |      0|        1|     0|
|rgho                 |1.0.1   |      1|        1|     0|
|rmcfs                |1.2.4   |      0|        1|     2|
|RPresto              |1.2.1   |      1|        1|     1|
|rsparkling           |0.2.0   |      1|        0|     0|
|rtide                |0.0.3   |      1|        0|     0|
|saeSim               |0.8.0   |      1|        0|     0|
|sejmRP               |1.3.4   |      1|        0|     0|
|sfc                  |0.1.0   |      0|        1|     0|
|sf                   |0.4-1   |      2|        0|     1|
|shazam               |0.1.5   |      0|        1|     0|
|sjmisc               |2.4.0   |      1|        0|     0|
|sjPlot               |2.3.1   |      0|        1|     0|
|SpaDES               |1.3.1   |      0|        2|     2|
|sparklyr             |0.5.3   |      1|        0|     0|
|sparseHessianFD      |0.3.2   |      0|        1|     0|
|spbabel              |0.4.5   |      1|        0|     0|
|spdplyr              |0.1.2   |      0|        1|     0|
|sqlscore             |0.1.1   |      2|        0|     1|
|srvyr                |0.2.0   |      2|        1|     0|
|ss3sim               |0.9.3   |      0|        2|     0|
|stormwindmodel       |0.1.0   |      0|        1|     0|
|stplanr              |0.1.7-3 |      0|        1|     0|
|texmexseq            |0.3     |      1|        0|     0|
|textmining           |0.0.1   |      0|        1|     0|
|tibble               |1.3.0   |      1|        0|     0|
|tidyjson             |0.2.1   |      1|        0|     1|
|tidyquant            |0.5.0   |      1|        0|     0|
|tidyr                |0.6.1   |      1|        0|     0|
|tidytext             |0.1.2   |      1|        1|     0|
|treeplyr             |0.1.2   |      1|        0|     0|
|turfR                |0.8-7   |      1|        0|     1|
|unpivotr             |0.1.0   |      1|        1|     0|
|unvotes              |0.1.0   |      1|        1|     0|
|useful               |1.2.1   |      0|        1|     0|
|valr                 |0.1.2   |      1|        0|     0|
|VWPre                |0.9.5   |      0|        1|     0|
|WHO                  |0.2     |      1|        0|     0|
|wordbankr            |0.2.0   |      0|        1|     1|
|wrswoR               |1.0-1   |      0|        1|     0|
|WufooR               |0.6.1   |      2|        1|     0|
|ZeligChoice          |0.9-4   |      0|        1|     0|
|ZeligEI              |0.1-0   |      0|        1|     0|
|Zelig                |5.0-17  |      1|        0|     1|

## ameco (0.2.6)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: http://github.com/expersso/ameco/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(ameco)
  > 
  > test_check("ameco")
  1. Failure: Test that current version is still latest version (@tests.R#16) ----
  `last_update` not equal to as.Date("2017-02-13").
  'is.NA' value mismatch: 0 in current 1 in target
  
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Failure: Test that current version is still latest version (@tests.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 15.7Mb
  sub-directories of 1Mb or more:
    data  15.6Mb
```

## assertr (2.0.2)
Maintainer: Tony Fischetti <tony.fischetti@gmail.com>  
Bug reports: https://github.com/ropensci/assertr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  4. Failure: insist breaks appropriately (@test-assertions.R#419) ---------------
  error\$message does not match "All select\\(\\) inputs must resolve to integer column positions".
  Actual value: ""vs": must resolve to integer column positions, not string"
  
  
  testthat results ================================================================
  OK: 374 SKIPPED: 0 FAILED: 4
  1. Failure: assert breaks appropriately (@test-assertions.R#232) 
  2. Error: assert_rows raises error if verification fails (using se) (@test-assertions.R#310) 
  3. Failure: assert_rows breaks appropriately (@test-assertions.R#333) 
  4. Failure: insist breaks appropriately (@test-assertions.R#419) 
  
  Error: testthat unit tests failed
  Execution halted
```

## backtestGraphics (0.1.6)
Maintainer: Miller Zijie Zhu <zijie.miller.zhu@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > test_check("backtestGraphics")
  Loading required package: backtestGraphics
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) -------------
  Failed the test for calculating summary statistics not equal to `truth.1`.
  Component "pnl": Component "pnl.drawdown": Component "start": Attributes: < Component "levels": 1 string mismatch >
  Component "pnl": Component "pnl.drawdown": Component "start": 1 string mismatch
  
  
  testthat results ================================================================
  OK: 9 SKIPPED: 0 FAILED: 1
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bayesplot (1.1.0)
Maintainer: Jonah Gabry <jsg2201@columbia.edu>  
Bug reports: https://github.com/stan-dev/bayesplot/issues/

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-14 (./children/SETTINGS-knitr.txt) 
Quitting from lines NA-14 (./children/SETTINGS-knitr.txt) 
Error: processing vignette 'MCMC-diagnostics.Rmd' failed with diagnostics:
object 'params' not found
Execution halted

```

## bibliometrix (1.5)
Maintainer: Massimo Aria <aria@unina.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## bigrquery (0.3.0)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/rstats-db/bigrquery/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘bigrquery’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/bigrquery.Rcheck/00install.out’ for details.
```

## bioOED (0.1.1)
Maintainer: Alberto Garre <garre.alberto@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘MEIGOR’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## blscrapeR (2.1.2)
Maintainer: Kris Eberwein <eberwein@knights.ucf.edu>  
Bug reports: https://github.com/keberwein/blscrapeR/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         on.exit(options(expressions = expressions_opt), add = TRUE)
         e$expectation_calls <- frame_calls(11, 2)
         test_error <<- e
         register_expectation(e)
         e$handled <- TRUE
         test_error <<- e
     }, "attempt to select less than one element in integerOneIndex", quote(datalist[[i]] <- mth_vals)) at testthat/test_bls_csv.R:76
  2: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 2 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_bls_csv.R#76) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bossMaps (0.1.0)
Maintainer: Adam M. Wilson <adamw@buffalo.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

... 8 lines ...

    extract

The following object is masked from 'package:dplyr':

    select

Quitting from lines 82-83 (bossMaps.Rmd) 
Error: processing vignette 'bossMaps.Rmd' failed with diagnostics:
replacement has 2 rows, data has 1
Execution halted
```

## broom (0.4.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/tidyverse/broom/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘broom-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cch_tidiers
> ### Title: tidiers for case-cohort data
> ### Aliases: cch_tidiers glance.cch tidy.cch
> 
> ### ** Examples
... 43 lines ...

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Error in stats::model.frame(formula = `_F`(Surv(edrel, rel), stage + histol +  : 
  could not find function "_F"
Calls: %>% ... .Call -> tidy -> cch -> eval -> eval -> <Anonymous>
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’ [19s/19s]
Running the tests in ‘tests/test-all.R’ failed.
Complete output:
  > library(testthat)
  > test_check("broom")
  Loading required package: broom
  1. Error: (unknown) (@test-dplyr.R#4) ------------------------------------------
  could not find function "lahman_df"
  1: tbl(lahman_df(), "Batting") at testthat/test-dplyr.R:4
  
  testthat results ================================================================
  OK: 887 SKIPPED: 4 FAILED: 1
  1. Error: (unknown) (@test-dplyr.R#4) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Terminated

```

## carpenter (0.2.0)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/carpenter/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘carpenter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: carpenter
> ### Title: Build common tables for your research needs!
> ### Aliases: carpenter
> 
> ### ** Examples
> 
> 
> library(magrittr)
> outline_table(iris, 'Species') %>%
+  add_rows(c('Sepal.Length', 'Petal.Length'), stat_meanSD) %>%
+  add_rows('Sepal.Width', stat_medianIQR) %>%
+  renaming('rows', function(x) gsub('Sepal\\.', 'Sepal ', x)) %>%
+  renaming('header', c('Measures', 'Setosa', 'Versicolor', 'Virginica')) %>%
+  build_table(caption = 'A caption for the table')
Warning: 'dplyr::mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: Variable context not set
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  17: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  18: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  19: dplyr::mutate_each_(., dplyr::funs(as.character), dplyr::everything())
  20: is_empty(vars) at /home/muelleki/git/R/dplyr/R/colwise-mutate.R:232
  21: dplyr::everything() at /tmp/RtmpqNmvln/devtools1b6d73002698/hadley-rlang-ac7498c/R/types.R:240
  22: current_vars() at /home/muelleki/git/R/dplyr/R/select-utils.R:129
  23: cur_vars_env$selected %||% abort("Variable context not set") at /home/muelleki/git/R/dplyr/R/select-utils.R:46
  24: abort("Variable context not set") at /tmp/RtmpqNmvln/devtools1b6d73002698/hadley-rlang-ac7498c/R/operators.R:14
  
  testthat results ================================================================
  OK: 20 SKIPPED: 0 FAILED: 1
  1. Error: build_table outputs correct information (@test-build_tables.R#21) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 87-89 (carpenter.Rmd) 
Error: processing vignette 'carpenter.Rmd' failed with diagnostics:
Variable context not set
Execution halted

```

## chunked (0.3)
Maintainer: Edwin de Jonge <edwindjonge@gmail.com>  
Bug reports: https://github.com/edwindj/chunked/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Error: write_chunkwise to db works (@test-write.R#29) -----------------------
  'sql_render' is not an exported object from 'namespace:dplyr'
  1: write_chunkwise(iris2, tmp, row.names = FALSE) at testthat/test-write.R:29
  2: write_chunkwise.tbl_sql(iris2, tmp, row.names = FALSE)
  3: dplyr::sql_render
  4: getExportedValue(pkg, name)
  5: stop(gettextf("'%s' is not an exported object from 'namespace:%s'", name, getNamespaceName(ns)), 
         call. = FALSE, domain = NA)
  
  testthat results ================================================================
  OK: 41 SKIPPED: 0 FAILED: 1
  1. Error: write_chunkwise to db works (@test-write.R#29) 
  
  Error: testthat unit tests failed
  Execution halted

checking dependencies in R code ... NOTE
Missing or unexported object: ‘dplyr::sql_render’
```

## clustrd (1.1.0)
Maintainer: Angelos Markos <amarkos@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## codingMatrices (0.2.2)
Maintainer: Bill Venables <Bill.Venables@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'car'

The following object is masked from 'package:dplyr':

    recode

Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'codingMatrices.tex' failed.
BibTeX errors:
The top-level auxiliary file: codingMatrices.aux
I couldn't open style file chicago.bst
---line 19 of file codingMatrices.aux
 : \bibstyle{chicago
 :                  }
I'm skipping whatever remains of this command
I found no style file---while reading file codingMatrices.aux
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## cometExactTest (0.1.3)
Maintainer: Max Leiserson <mdml@cs.brown.edu>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## crawl (2.0.1)
Maintainer: Devin S. Johnson <devin.johnson@noaa.gov>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Attaching package: 'lubridate'

The following object is masked from 'package:base':

    date

Loading required package: zoo
... 8 lines ...

Attaching package: 'xts'

The following objects are masked from 'package:dplyr':

    first, last

Quitting from lines 403-414 (crawl_intro.Rmd) 
Error: processing vignette 'crawl_intro.Rmd' failed with diagnostics:
Column `unique_date`: must be length one (the group size), not 9701
Execution halted

checking installed package size ... NOTE
  installed size is  9.3Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   6.3Mb
```

## datacheckr (0.1.2)
Maintainer: Joe Thorley <joe@poissonconsulting.ca>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  many-to-one join between x and y violates referential integrity
  1: expect_identical(check_join(x, y), x) at testthat/test-check-join.R:54
  2: identical(object, expected) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-equality.R:86
  3: check_join(x, y)
  4: check_referential_integrity(data, parent, join, ignore_nas, data_name, parent_name)
  5: error("many-to-one join between ", data_name, " and ", parent_name, " violates referential integrity")
  6: stop(..., call. = FALSE)
  
  testthat results ================================================================
  OK: 374 SKIPPED: 0 FAILED: 2
  1. Error: check_join checks colnames (@test-check-join.R#34) 
  2. Error: check_join missing values (@test-check-join.R#54) 
  
  Error: testthat unit tests failed
  Execution halted
```

## dat (0.1.0)
Maintainer: Sebastian Warnholz <wahani@gmail.com>  
Bug reports: https://github.com/wahani/dat/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [59s/58s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  10: as.function(flatten)(map(x, f, ...))
  11: dplyr::bind_rows(x, .id = id) %>% memClassHandler$wrapClass()
  12: eval(lhs, parent, parent) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:6
  13: eval(expr, envir, enclos)
  14: dplyr::bind_rows(x, .id = id)
  15: bind_rows_(x, .id) at /home/muelleki/git/R/dplyr/R/bind.r:132
  
  testthat results ================================================================
  OK: 105 SKIPPED: 0 FAILED: 3
  1. Failure: Package Style (@test-lintr.R#5) 
  2. Failure: make formulas (@test-makeFormulas.R#45) 
  3. Error: split-apply-combine (@test-map.R#65) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ddpcr (1.6)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/daattali/ddpcr/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 205-213 (extend.Rmd) 
Error: processing vignette 'extend.Rmd' failed with diagnostics:
no function to return from, jumping to top level
Execution halted

```

## decoder (1.1.12)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.com/cancercentrum/decoder/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Joining, by = "key"
Joining, by = "key"
PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable.
Quitting from lines 102-122 (sjukhus.Rmd) 
Error: processing vignette 'sjukhus.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## DeLorean (1.2.4)
Maintainer: John Reid <john.reid@mrc-bsu.cam.ac.uk>

0 errors | 1 warning  | 1 note 

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.cells:
  function(dl, .filter, number, cells)

filter:
  function(x, filter, method, sides, circular, init)
filter.genes:
  function(dl, .filter, number, genes)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.cells filter.genes
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.cells':
  ‘filter.cells’

S3 methods shown with full name in documentation object 'filter.genes':
  ‘filter.genes’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## describer (0.2.0)
Maintainer: Paul Hendricks <paul.hendricks.2013@owu.edu>  
Bug reports: https://github.com/paulhendricks/describer/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [152s/152s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-tbl-cube.R:119:1: style: lines should not be more than 80 characters.
    expect_identical(as.table(nasa, measure = "ozone"), as.table(select(nasa, ozone)))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-ts.R:8:1: style: lines should not be more than 80 characters.
      "`.data`: must be a data source, not ts object, do you want `stats::filter()`?",
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
  
  testthat results ================================================================
  OK: 7 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test-styling.R#4) 
  
  Error: testthat unit tests failed
  Execution halted
```

## DiagrammeR (0.9.0)
Maintainer: Richard Iannone <riannone@me.com>  
Bug reports: https://github.com/rich-iannone/DiagrammeR/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [302s/299s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(DiagrammeR)
  > 
  > test_check("DiagrammeR")
  1. Error: importing a .graphml file is possible (@test-import_graph.R#5) -------
  subscript out of bounds
  1: import_graph(system.file("extdata", "power_grid.graphml", package = "DiagrammeR")) at testthat/test-import_graph.R:5
  2: create_graph(nodes_df = all_nodes, edges_df = all_edges)
  
  testthat results ================================================================
  OK: 1468 SKIPPED: 0 FAILED: 1
  1. Error: importing a .graphml file is possible (@test-import_graph.R#5) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 10.6Mb
  sub-directories of 1Mb or more:
    extdata       2.4Mb
    htmlwidgets   4.7Mb
    img           2.8Mb
```

## dtplyr (0.0.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/dtplyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 112 SKIPPED: 0 FAILED: 20
  1. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#23) 
  2. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#24) 
  3. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#25) 
  4. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#26) 
  5. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#28) 
  6. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#29) 
  7. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#30) 
  8. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#31) 
  9. Failure: grouped_by uses grouping vars & preserves groups (@test-distinct.R#24) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## easyformatr (0.1.2)
Maintainer: Brandon Taylor <brandon.taylor221@gmail.com>  
Bug reports: https://github.com/bramtayl/easyformatr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘easyformatr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: easy_format
> ### Title: Easily build format strings
> ### Aliases: easy_format
> 
> ### ** Examples
> 
> easy_format(year, month, day, integer, octal, double)
Error in function_list[[k]](value) : 

|base  |
|:-----|
|day   |
|month |
|year  |

 has no corresponding code
Calls: easy_format ... eval -> _fseq -> freduce -> withVisible -> <Anonymous>
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  35: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:6
  36: eval(quote(`_fseq`(`_lhs`)), env, env)
  37: eval(expr, envir, enclos)
  38: `_fseq`(`_lhs`)
  39: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  40: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  41: function_list[[k]](value)
  42: stop(.)
  
  testthat results ================================================================
  OK: 15 SKIPPED: 0 FAILED: 1
  1. Error: easy_format (@test.R#48) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 8 lines ...
Error: processing vignette 'easyformatr.Rmd' failed with diagnostics:


|base  |
|:-----|
|day   |
|month |
|year  |

 has no corresponding code
Execution halted
```

## eechidna (0.1)
Maintainer: Ben Marwick <benmarwick@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'purrr'

The following objects are masked from 'package:dplyr':
... 8 lines ...

Attaching package: 'scales'

The following object is masked from 'package:purrr':

    discard

Quitting from lines 178-201 (exploring-election-data.Rmd) 
Error: processing vignette 'exploring-election-data.Rmd' failed with diagnostics:
not compatible with STRSXP
Execution halted
```

## efreadr (0.1.1)
Maintainer: Marco Bascietto <marco.bascietto@crea.gov.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## emil (2.2.6)
Maintainer: Christofer Backlin <emil@christofer.backlin.se>  
Bug reports: https://github.com/Molmed/emil/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘emil-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: learning_curve
> ### Title: Learning curve analysis
> ### Aliases: learning_curve
> 
> ### ** Examples
... 6 lines ...
11 Apr 21:18    Test set test_fraction 1 of 3 (0.7)
11 Apr 21:18      Evaluating modeling performance...
11 Apr 21:18    Test set test_fraction 2 of 3 (0.5)
11 Apr 21:18      Evaluating modeling performance...
11 Apr 21:18    Test set test_fraction 3 of 3 (0.3)
11 Apr 21:18      Evaluating modeling performance...
> plot(lc)
Error in select.list(., test_fraction = TRUE, fold = TRUE, method = TRUE,  : 
  unused arguments (test_fraction = TRUE, fold = TRUE, method = TRUE, performance = "error")
Calls: plot ... withVisible -> <Anonymous> -> select -> select.list
Execution halted
```

## etl (0.3.5)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/etl/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘etl-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: etl_init
> ### Title: Initialize a database using a defined schema
> ### Aliases: etl_init etl_init.default find_schema
> 
> ### ** Examples
> 
> cars <- etl("mtcars")
Not a valid src. Creating a src_sqlite for you at:
/tmp/RtmpszNNXn/filed29c3a3c299a.sqlite3
> cars %>%
+   etl_init()
Could not find schema initialization script
> cars %>%
+   etl_init(script = sql("CREATE TABLE IF NOT EXISTS mtcars_alt (id INTEGER);"))
Error in etl_init.default(., script = sql("CREATE TABLE IF NOT EXISTS mtcars_alt (id INTEGER);")) : 
  could not find function "sql"
Calls: %>% ... withVisible -> <Anonymous> -> etl_init -> etl_init.default
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  downloaded 11 KB
  
  trying URL 'http://www.nytimes.com'
  Content type 'text/html; charset=utf-8' length 211396 bytes (206 KB)
  ==================================================
  downloaded 206 KB
  
  testthat results ================================================================
  OK: 15 SKIPPED: 0 FAILED: 3
  1. Error: sqlite works (@test-etl.R#9) 
  2. Failure: dplyr works (@test-etl.R#25) 
  3. Error: MonetDBLite works (@test-etl.R#54) 
  
  Error: testthat unit tests failed
  Execution halted

checking Rd cross-references ... WARNING
Missing link or links in documentation object 'etl.Rd':
  ‘[dplyr]{src_sql}’

See section 'Cross-references' in the 'Writing R Extensions' manual.

```

## explor (0.3.1)
Maintainer: Julien Barnier <julien.barnier@ens-lyon.fr>  
Bug reports: https://github.com/juba/explor/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2/2 mismatches (average diff: NaN)
  [1] 0.106 - NA == NA
  [2] 0.011 - NA == NA
  
  
  testthat results ================================================================
  OK: 95 SKIPPED: 0 FAILED: 5
  1. Failure: Variables results are equal (@test_prepare_results_PCA.R#15) 
  2. Failure: Variables results are equal (@test_prepare_results_PCA.R#17) 
  3. Failure: Variables results are equal (@test_prepare_results_PCA.R#19) 
  4. Failure: Quantitative supplementary variables results are equal (@test_prepare_results_PCA.R#27) 
  5. Failure: Quantitative supplementary variables results are equal (@test_prepare_results_PCA.R#30) 
  
  Error: testthat unit tests failed
  Execution halted
```

## eyelinker (0.1)
Maintainer: Simon Barthelme <simon.barthelme@gipsa-lab.fr>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: eyelinker
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 226-229 (basics.Rmd) 
Error: processing vignette 'basics.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted

```

## eyetrackingR (0.1.6)
Maintainer: Jacob Dink <jacobwdink@gmail.com>  
Bug reports: https://github.com/jwdink/eyetrackingR/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘eyetrackingR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: analyze_time_clusters
> ### Title: Bootstrap analysis of time-clusters.
> ### Aliases: analyze_time_clusters analyze_time_clusters.time_cluster_data
> 
> ### ** Examples
... 16 lines ...
> response_time <- make_time_sequence_data(response_window, time_bin_size = 500, aois = "Animate", 
+                                          predictor_columns = "Sex")
> 
> time_cluster_data <- make_time_cluster_data(data = response_time, predictor_column = "SexM", 
+                          aoi = "Animate", test = "lmer", 
+                          threshold = 1.5, 
+                          formula = LogitAdjusted ~ Sex + (1|Trial) + (1|ParticipantName))
Error in UseMethod("analyze_time_bins") : 
  no applicable method for 'analyze_time_bins' applied to an object of class "data.frame"
Calls: make_time_cluster_data ... make_time_cluster_data.time_sequence_data -> do.call -> <Anonymous>
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [110s/110s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Predictor:	 Target 
  Formula:	 Prop ~ Target 
  Runs of Significant Time Bins: 
  Positive Run 1  ===== 
  	Time:		 15800 - 21000 1. Error: (unknown) (@test_cluster_analysis.R#143) -----------------------------
  no applicable method for 'make_time_cluster_data' applied to an object of class "data.frame"
  1: make_time_cluster_data(response_time_by_ppt, predictor_column = "Sex", test = "boot_splines", 
         within_subj = FALSE, smoother = "smooth.spline", alpha = 0.05) at testthat/test_cluster_analysis.R:143
  
  testthat results ================================================================
  OK: 46 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_cluster_analysis.R#143) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ezsummary (0.2.1)
Maintainer: Hao Zhu <haozhu233@gmail.com>  
Bug reports: https://github.com/haozhu233/ezsummary/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ezsummary-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ezmarkup
> ### Title: Easy way to "markup" a table before it is sent to be displayed
> ### Aliases: ezmarkup
> 
> ### ** Examples
... 10 lines ...
The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> dt <- mtcars %>% group_by(cyl) %>% select(gear, carb) %>% ezsummary_categorical(n=TRUE)
Adding missing grouping variables: `cyl`
> 
> ezmarkup(dt, "...[.(.)]")
Error in cbind_all(x) : not compatible with STRSXP
Calls: ezmarkup ... ezmarkup_ -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 89-90 (intro_to_ezsummary_0.2.0.Rmd) 
Error: processing vignette 'intro_to_ezsummary_0.2.0.Rmd' failed with diagnostics:
non-numeric argument to mathematical function
Execution halted

```

## fitcoach (1.0)
Maintainer: Niraj Juneja <njuneja@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) --------------------
  res[1] is not strictly less than 2820. Difference: 29.2
  
  
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) --------------------
  `res` is not strictly less than 2518. Difference: 2.3
  
  
  testthat results ================================================================
  OK: 17 SKIPPED: 0 FAILED: 2
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) 
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) 
  
  Error: testthat unit tests failed
  Execution halted
```

## flextable (0.1.0)
Maintainer: David Gohel <david.gohel@ardata.fr>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: officer

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable.
Quitting from lines 39-41 (format.Rmd) 
Error: processing vignette 'format.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## forestmodel (0.4.0)
Maintainer: Nick Kennedy <r@nick-kennedy.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘forestmodel-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: forest_model
> ### Title: Produce a forest plot based on a regression model
> ### Aliases: forest_model
> 
> ### ** Examples
... 16 lines ...
+   transmute(time,
+             status,
+             Age = age,
+             Sex = factor(sex, labels = c("Male", "Female")),
+             ECOG = factor(lung$ph.ecog),
+             `Meal Cal` = meal.cal)
> 
> print(forest_model(coxph(Surv(time, status) ~ ., pretty_lung)))
Error in cbind_all(x) : Argument 2: size must be 2, not 1
Calls: print ... create_term_data -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted
```

## freqweights (1.0.2)
Maintainer: Emilio Torres-Manzanera <torres@uniovi.es>

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘freqweights-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: biglmfreq
> ### Title: Estimates the coefficients of a linear model
> ### Aliases: biglmfreq coef.biglmfreq predict.biglmfreq print.biglmfreq
> ###   update.biglmfreq
> 
> ### ** Examples
> 
> mt <- biglmfreq(Sepal.Length ~ Sepal.Width, iris)
Error in evaldp(., group_by, vars) : 
  could not find function "partial_eval"
Calls: biglmfreq ... _fseq -> freduce -> withVisible -> <Anonymous> -> evaldp
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  5: eval(expr, envir, enclos)
  6: `_fseq`(`_lhs`)
  7: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  8: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  9: function_list[[k]](value)
  10: evaldp(., group_by, vars)
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 3
  1. Error: xxxfreq obtain identical results than linear models (@test-linearmodels.R#10) 
  2. Error: Mean (@test-meanfreq.R#9) 
  3. Error: Table of frequencies. Updating (@test-tablefreq.R#10) 
  
  Error: testthat unit tests failed
  Execution halted

checking R code for possible problems ... NOTE
.corfreq: no visible global function definition for ‘complete.cases’
.covfreq: no visible global function definition for ‘complete.cases’
.hclustvfreq: no visible global function definition for
  ‘complete.cases’
.quantilefreq : <anonymous>: no visible global function definition for
  ‘approx’
coef.biglmfreq: no visible global function definition for ‘coef’
evaldp: no visible global function definition for ‘partial_eval’
predict.biglmfreq: no visible global function definition for ‘predict’
predict.lmfreq: no visible global function definition for ‘predict’
summary.lmfreq: no visible global function definition for ‘coef’
summary.lmfreq: no visible global function definition for ‘pt’
summary.lmfreq: no visible global function definition for ‘AIC’
update.biglmfreq: no visible global function definition for ‘update’
Undefined global functions or variables:
  AIC approx coef complete.cases partial_eval predict pt update
Consider adding
  importFrom("stats", "AIC", "approx", "coef", "complete.cases",
             "predict", "pt", "update")
to your NAMESPACE file.
```

## FRK (0.1.1)
Maintainer: Andrew Zammit-Mangion <andrewzm@gmail.com>  
Bug reports: http://github.com/andrewzm/FRK/issues

1 error  | 2 warnings | 2 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [79s/79s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
    |                                                                            
    |                                                                      |   0%
    |                                                                            
    |===================================                                   |  50%
    |                                                                            
    |======================================================================| 100%
  [1] "Maximum EM iterations reached"
  testthat results ================================================================
  OK: 140 SKIPPED: 1 FAILED: 3
  1. Error: sphere_BAUs (@test_BAUs.R#71) 
  2. Error: SRE sphere works (@test_SRE.R#114) 
  3. Error: SRE space-time sphere works (@test_SRE.R#167) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 525-530 (FRK_intro.Rnw) 
Error: processing vignette 'FRK_intro.Rnw' failed with diagnostics:
invalid list argument: all variables should have the same length
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’

Package which this enhances but not available for checking: ‘dggrids’

checking installed package size ... NOTE
  installed size is  7.7Mb
  sub-directories of 1Mb or more:
    data   4.8Mb
    doc    1.6Mb
```

## FSelectorRcpp (0.1.2)
Maintainer: Zygmunt Zawadzki <zygmunt@zstat.pl>  
Bug reports: https://github.com/mi2-warsaw/FSelectorRcpp/issues

1 error  | 0 warnings | 2 notes

```
checking examples ... ERROR
Running examples in ‘FSelectorRcpp-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: feature_search
> ### Title: General Feature Searching Engine
> ### Aliases: feature_search
> 
> ### ** Examples
> 
> 
> # Enable parallelization in examples
>  library(doSNOW) # doSNOW has an option for progress bar
Loading required package: foreach
Loading required package: iterators
Loading required package: snow
>  cl <- makeCluster(2)
Loading required namespace: Rmpi
Error in Rmpi::mpi.comm.spawn(slave = mpitask, slavearg = args, nslaves = count,  : 
  MPI_Comm_spawn is not supported.
Calls: makeCluster -> makeMPIcluster -> <Anonymous>
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘RTCGA.rnaseq’

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   7.3Mb
```

## funModeling (1.6.2)
Maintainer: Pablo Casas <pcasas.biz@gmail.com>  
Bug reports: https://github.com/pablo14/funModeling/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
... 6 lines ...
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## futureheatwaves (1.0.3)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 604-605 (futureheatwaves.Rmd) 
Error: processing vignette 'futureheatwaves.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## geoknife (1.5.4)
Maintainer: Jordan Read <jread@usgs.gov>  
Bug reports: https://github.com/USGS-R/geoknife/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 360-364 (geoknife.Rmd) 
Error: processing vignette 'geoknife.Rmd' failed with diagnostics:
need finite 'xlim' values
Execution halted

```

## geoSpectral (0.17)
Maintainer: Servet Ahmet Cizmeli <ahmet@pranageo.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘geoSpectral-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: spc.Read_NOMAD_v2
> ### Title: Read the NOMAD v2 bio-optical database
> ### Aliases: spc.Read_NOMAD_v2
> 
> ### ** Examples
> 
> ap = spc.Read_NOMAD_v2()
[1] "Reading the NOMAD file /home/muelleki/git/R/dplyr/revdep/checks/geoSpectral.Rcheck/geoSpectral/test_data/nomad_seabass_v2.a_2008200.txt.gz off disk."
Error: Variable context not set
Execution halted
```

## ggCompNet (0.1.0)
Maintainer: Sam Tyner <sctyner@iastate.edu>  
Bug reports: https://github.com/sctyner/ggCompNet/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: Removed 8 rows containing missing values (geom_segment).
Quitting from lines 617-629 (examples-from-paper.Rmd) 
Error: processing vignette 'examples-from-paper.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted


checking installed package size ... NOTE
  installed size is  6.4Mb
  sub-directories of 1Mb or more:
    doc   6.1Mb
```

## ggedit (0.2.1)
Maintainer: Jonathan Sidi <yonis@metrumrg.com>  
Bug reports: https://github.com/metrumresearchgroup/ggedit/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
** found \donttest examples: check also with --run-donttest
```

## ggforce (0.1.1)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggforce-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: geom_arc_bar
> ### Title: Arcs and wedges as polygons
> ### Aliases: geom_arc_bar stat_arc_bar stat_pie
> 
> ### ** Examples
... 27 lines ...
> ggplot() + geom_arc_bar(aes(x0=0, y0=0, r0=r0, r=1, amount=amount,
+                             fill=state, explode=focus),
+                         data=pie, stat='pie') +
+   facet_wrap(~type, ncol=1) +
+   coord_fixed() +
+   theme_no_axes() +
+   scale_fill_brewer('', type='qual')
Warning: Ignoring unknown aesthetics: explode
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted
```

## ggfortify (0.4.1)
Maintainer: Masaaki Horikoshi <sinhrks@gmail.com>  
Bug reports: https://github.com/sinhrks/ggfortify/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggfortify-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: gglagplot
> ### Title: Plot time series against lagged versions of themselves
> ### Aliases: gglagplot
> 
> ### ** Examples
> 
> gglagplot(AirPassengers)
Error: `x`: must be a vector, not ts object, do you want `stats::lag()`?
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’ [62s/62s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  8: eval_bare(dot$expr, dot$env) at /tmp/RtmpqNmvln/devtools1b6d73002698/hadley-rlang-ac7498c/R/dots.R:72
  9: lapply(seq(1:lags), .lag)
  10: FUN(X[[i]], ...)
  11: as.vector(lag(ts, k))
  12: lag(ts, k)
  13: bad_args("x", "must be a vector, not ts object, do you want `stats::lag()`?") at /home/muelleki/git/R/dplyr/R/lead-lag.R:65
  14: glubort(fmt_args(args), ..., .envir = .envir) at /home/muelleki/git/R/dplyr/R/error.R:20
  15: abort(text) at /home/muelleki/git/R/dplyr/R/error.R:46
  
  testthat results ================================================================
  OK: 1442 SKIPPED: 9 FAILED: 1
  1. Error: gglagplot (@test-tslib.R#103) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  5.7Mb
  sub-directories of 1Mb or more:
    doc   5.0Mb
```

## ggraph (1.0.0)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggraph-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: facet_edges
> ### Title: Create small multiples based on edge attributes
> ### Aliases: facet_edges
> 
> ### ** Examples
... 13 lines ...

> gr <- graph_from_data_frame(highschool)
> 
> ggraph(gr) +
+     geom_edge_link() +
+     geom_node_point() +
+     facet_edges(~year)
Using `nicely` as default layout
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted

checking installed package size ... NOTE
  installed size is  6.1Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    libs   2.0Mb
```

## ggvis (0.4.3)
Maintainer: Winston Chang <winston@rstudio.com>

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggvis-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: explain
> ### Title: Explain details of an object
> ### Aliases: explain
> 
> ### ** Examples
> 
> p <- mtcars %>% ggvis(x = ~cyl) %>% layer_bars()
Error in eval(expr, envir, enclos) : object 'xmin_' not found
Calls: %>% ... prop_value -> prop_value.prop_variable -> eval -> eval
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
     }) at /tmp/RtmpbLc7Bm/devtoolsd9aa64444a/shiny/R/react.R:21
  25: env$runWith(self, func) at /tmp/RtmpbLc7Bm/devtoolsd9aa64444a/shiny/R/react.R:25
  26: contextFunc() at /tmp/RtmpbLc7Bm/devtoolsd9aa64444a/shiny/R/react.R:113
  27: ..stacktraceon..(expr) at /tmp/RtmpbLc7Bm/devtoolsd9aa64444a/shiny/R/reactives.R:1721
  28: prop_value(prop, data()) at /tmp/RtmpbLc7Bm/devtoolsd9aa64444a/shiny/R/conditions.R:331
  29: prop_value.prop_variable(prop, data())
  30: eval(value(x$value), envir = data, enclos = x$env)
  31: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 443 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test-specs.r#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## graphicalVAR (0.2)
Maintainer: Sacha Epskamp <mail@sachaepskamp.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘graphicalVAR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: graphicalVAR
> ### Title: Estimate the graphical VAR model.
> ### Aliases: graphicalVAR
> 
> ### ** Examples
> 
> # Simulate model:
> Mod <- randomGVARmodel(4,probKappaEdge = 0.8,probBetaEdge = 0.8)
> 
> # Simulate data:
> Data <- graphicalVARsim(100,Mod$beta,Mod$kappa)
> 
> # Estimate model:
> Res <- graphicalVAR(Data, gamma = 0, nLambda = 10)
Warning: 'dplyr::summarize_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error in cbind_all(x) : Argument 1: must have names
Calls: graphicalVAR ... tsData -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted
```

## grattan (1.4.1.2)
Maintainer: Hugh Parsonage <hugh.parsonage@gmail.com>  
Bug reports: https://github.com/HughParsonage/grattan/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [54s/57s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  13: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  14: filter(., fy_year == "2016-17")
  15: filter.data.frame(., fy_year == "2016-17") at /home/muelleki/git/R/dplyr/R/manip.r:52
  16: as.data.frame(filter(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:62
  17: filter(tbl_df(.data), ...)
  18: filter.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:52
  19: filter_impl(.data, quo) at /home/muelleki/git/R/dplyr/R/tbl-df.r:65
  
  testthat results ================================================================
  OK: 205 SKIPPED: 35 FAILED: 1
  1. Error: New SAPTO matches old SAPTO for SAPTO (@test_sapto.R#83) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘taxstats’
```

## groupdata2 (0.1.0)
Maintainer: Ludvig Renbo Olsen <r-pkgs@ludvigolsen.dk>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Loading required package: lme4

Attaching package: 'lmerTest'

The following object is masked from 'package:lme4':

... 8 lines ...

Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Quitting from lines 293-301 (introduction_to_groupdata2.Rmd) 
Error: processing vignette 'introduction_to_groupdata2.Rmd' failed with diagnostics:
the table must have a header (column names)
Execution halted
```

## gutenbergr (0.1.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/ropenscilabs/gutenbergr/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [6s/10s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  6. Failure: read_zip_url can download and read a zip file (@test-utils.R#7) ----
  any(z == "Congress shall make no law respecting an establishment of religion,") isn't true.
  
  
  testthat results ================================================================
  OK: 41 SKIPPED: 0 FAILED: 6
  1. Error: gutenberg_get_mirror works (@test-download.R#13) 
  2. Error: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#19) 
  3. Failure: Can download books from a data frame with gutenberg_id column (@test-download.R#43) 
  4. Failure: We can download a file that only has a -8 version (@test-download.R#50) 
  5. Error: Trying to download a non-existent book raises a warning (@test-download.R#55) 
  6. Failure: read_zip_url can download and read a zip file (@test-utils.R#7) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':
... 8 lines ...
Warning: Unknown or uninitialised column: 'text'.
Warning in is.na(text) :
  is.na() applied to non-(list or vector) of type 'NULL'
Warning in .f(.x[[i]], ...) :
  Could not download a book at http://www.gutenberg.lib.md.us/7/6/768/768.zip
Warning in .f(.x[[i]], ...) :
  Could not download a book at http://www.gutenberg.lib.md.us/1/2/6/1260/1260.zip
Quitting from lines 78-81 (intro.Rmd) 
Error: processing vignette 'intro.Rmd' failed with diagnostics:
Column `gutenberg_id`: must be length 0 (the number of rows) or one, not 2
Execution halted
```

## harrietr (0.2.2)
Maintainer: Anders Gonçalves da Silva <andersgs@gmail.com>  
Bug reports: https://github.com/andersgs/harrietr/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘ggtree’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## heemod (0.9.0)
Maintainer: Antoine Filipovic-Pierucci <pierucci@gmail.com>  
Bug reports: https://github.com/pierucci/heemod/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘heemod-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: run_dsa
> ### Title: Run Sensitivity Analysis
> ### Aliases: run_dsa
> 
> ### ** Examples
... 70 lines ...
p1 0.1  0.9 
p2 0.1  0.3 
r  0.05 0.1 
> 
> x <- run_dsa(res2, ds)
Running DSA on strategy 'I'...
Running DSA on strategy 'II'...
> 
> plot(x, value = "cost")
Error: evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [458s/454s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 392 SKIPPED: 0 FAILED: 13
  1. Error: HIV model works (@test_dmhee.R#108) 
  2. Error: HRT model works (@test_dmhee.R#259) 
  3. Error: Identify Frontier Scenarios (@test_efficiency_frontier.R#15) 
  4. Error: Demographic analysis (@test_heterogeneity.R#58) 
  5. Error: Heterogeneity analysis (@test_heterogeneity.R#134) 
  6. Error: Same results using 1 core or 2. (@test_parallel.R#7) 
  7. Error: Probabilistic analysis works (@test_probabilistic.R#101) 
  8. Error: run_model behaves as expected (@test_run_model.R#213) 
  9. Error: Discounting (@test_run_model.R#353) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Scale for 'colour' is already present. Adding another scale for
'colour', which will replace the existing scale.
Scale for 'colour' is already present. Adding another scale for
'colour', which will replace the existing scale.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Error: processing vignette 'c-homogeneous.Rmd' failed with diagnostics:
evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted

```

## highcharter (0.5.0)
Maintainer: Joshua Kunst <jbkunst@gmail.com>  
Bug reports: https://github.com/jbkunst/highcharter/issues

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘highcharter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: hcboxplot
> ### Title: Shortcut to make a boxplot
> ### Aliases: hcboxplot
> 
> ### ** Examples
> 
> hcboxplot(x = iris$Sepal.Length, var = iris$Species, color = "red")
Error in mutate_impl(.data, dots) : 
  Column `data`: must be length one (the group size), not 5
Calls: hcboxplot ... transmute.default -> mutate -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Highcharts (www.highcharts.com) is a Highsoft software product which is
not free for commercial and Governmental use
Quitting from lines 45-46 (charting-data-frames.Rmd) 
Error: processing vignette 'charting-data-frames.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is 16.5Mb
  sub-directories of 1Mb or more:
    doc          13.7Mb
    htmlwidgets   1.9Mb
```

## HTSSIP (1.0.3)
Maintainer: Nicholas Youngblut <nyoungb2@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘phyloseq’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## huxtable (0.1.1)
Maintainer: David Hugh-Jones <davidhughjones@gmail.com>  
Bug reports: https://github.com/hughjonesd/huxtable/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         message = handle_message)) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:42
  4: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:130
  5: withVisible(code)
  6: rmarkdown::render("rowheight-multicol-test.Rmd", quiet = TRUE)
  7: convert(output_file, run_citeproc) at /tmp/Rtmp3nAXji/devtools8a9b3238f6f1/rmarkdown/R/render.R:655
  8: pandoc_convert(utf8_input, pandoc_to, output_format$pandoc$from, output, citeproc, 
         output_format$pandoc$args, !quiet) at /tmp/Rtmp3nAXji/devtools8a9b3238f6f1/rmarkdown/R/render.R:585
  9: stop("pandoc document conversion failed with error ", result, call. = FALSE) at /tmp/Rtmp3nAXji/devtools8a9b3238f6f1/rmarkdown/R/pandoc.R:100
  
  testthat results ================================================================
  OK: 9 SKIPPED: 10 FAILED: 1
  1. Error: Row heights do not screw up latex multicol (@test-output.R#44) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 51-102 (design-principles.Rmd) 
Error: processing vignette 'design-principles.Rmd' failed with diagnostics:
argument is of length zero
Execution halted

```

## HydeNet (0.10.5)
Maintainer: Benjamin Nutter <benjamin.nutter@gmail.com>  
Bug reports: https://github.com/nutterb/HydeNet/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘HydeNet-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: HydePosterior
> ### Title: Posterior Distributions of a Decision Network
> ### Aliases: HydePosterior
> 
> ### ** Examples
> 
> data(PE, package="HydeNet")
> Net <- HydeNetwork(~ wells + 
+                      pe | wells + 
+                      d.dimer | pregnant*pe + 
+                      angio | pe + 
+                      treat | d.dimer*angio + 
+                      death | pe*treat,
+                      data = PE) 
>   
>                  
> compiledNet <- compileJagsModel(Net, n.chains=5)
Error in cbind_all(x) : Argument 2: must have names
Calls: compileJagsModel ... cpt_workhorse -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [20s/20s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 55 SKIPPED: 0 FAILED: 12
  1. Error: HydeNetwork.list returns expected attributes (@test-HydeNetwork.R#41) 
  2. Error: (unknown) (@test-HydePosterior.R#11) 
  3. Error: (unknown) (@test-bindPosterior.R#12) 
  4. Error: compileJagsModel returns an object of class 'compiledHydeNetwork' (@test-compileJagsModel.R#14) 
  5. Error: cpt.list (@test-cpt.R#12) 
  6. Error: cpt with weights (@test-cpt.R#20) 
  7. Error: (unknown) (@test-print.HydePosterior.R#11) 
  8. Failure: fit the model for dnorm (@test-setNode.R#40) 
  9. Error: (unknown) (@test-setNodeModels.R#14) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: nnet
Quitting from lines 227-233 (DecisionNetworks.Rmd) 
Error: processing vignette 'DecisionNetworks.Rmd' failed with diagnostics:
Argument 2: must have names
Execution halted

```

## IATscores (0.1-2)
Maintainer: Giulio Costantini <costantinigiulio@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘nem’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## implyr (0.1.0)
Maintainer: Ian Cook <ian@cloudera.com>  
Bug reports: https://github.com/ianmcook/implyr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘implyr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/implyr.Rcheck/00install.out’ for details.
```

## incadata (0.5.3)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.org/cancercentrum/incadata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(incadata)
  > 
  > test_check("incadata")
  1. Failure: filter (@test-dplyr_methods.R#13) ----------------------------------
  dplyr::filter(testdata, persnr == 198505043334) inherits from `tbl_df/tbl/data.frame` not `incadata`.
  
  
  [1] "ex_data48223bcd4f01.csv2"
  testthat results ================================================================
  OK: 40 SKIPPED: 0 FAILED: 1
  1. Failure: filter (@test-dplyr_methods.R#13) 
  
  Error: testthat unit tests failed
  Execution halted
```

## inferr (0.1.0)
Maintainer: Aravind Hebbali <hebbali.aravind@gmail.com>  
Bug reports: https://github.com/rsquaredacademy/inferr/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
... 14 lines ...
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## infuser (0.2.6)
Maintainer: Bart Smeets <bartsmeets86@gmail.com>  
Bug reports: https://github.com/bart6114/infuser/issues

0 errors | 2 warnings | 0 notes

```
checking Rd cross-references ... WARNING
Missing link or links in documentation object 'infuse.Rd':
  ‘[dplyr]{build_sql}’

See section 'Cross-references' in the 'Writing R Extensions' manual.


checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 157-158 (getting_started.Rmd) 
Error: processing vignette 'getting_started.Rmd' failed with diagnostics:
'build_sql' is not an exported object from 'namespace:dplyr'
Execution halted

```

## ipft (0.5)
Maintainer: Emilio Sansano <esansano@uji.es>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## janitor (0.2.1)
Maintainer: Sam Firke <samuel.firke@gmail.com>  
Bug reports: https://github.com/sfirke/janitor/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  [3] 0.333 - 0.286 ==  0.0476
  [4] 0.000 - 0.143 == -0.1429
  
  
  testthat results ================================================================
  OK: 191 SKIPPED: 1 FAILED: 6
  1. Failure: missing factor levels are displayed with NA values (@test-tabyl.R#58) 
  2. Failure: missing factor levels are displayed with NA values (@test-tabyl.R#59) 
  3. Failure: sort parameter works (@test-tabyl.R#77) 
  4. Failure: sort parameter works (@test-tabyl.R#78) 
  5. Failure: NA results are treated appropriately (@test-top-levels.R#27) 
  6. Failure: NA results are treated appropriately (@test-top-levels.R#28) 
  
  Error: testthat unit tests failed
  Execution halted
```

## Lahman (5.0-0)
Maintainer: Chris Dalzell <cdalzell@gmail.com>

0 errors | 1 warning  | 1 note 

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
       user system elapsed
Teams 5.336  0.088    5.37

checking installed package size ... NOTE
  installed size is  7.6Mb
  sub-directories of 1Mb or more:
    data   7.4Mb
```

## lplyr (0.1.6)
Maintainer: Paul Poncet <paulponcet@yahoo.fr>  
Bug reports: https://github.com/paulponcet/lplyr/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘lplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mutate_.list
> ### Title: Dplyr verbs for lists and pairlists
> ### Aliases: mutate_.list mutate_.pairlist rename_.list rename_.pairlist
> ###   select_.list select_.pairlist transmute_.list transmute_.pairlist
> 
... 35 lines ...
[1] "alpha"

$x3[[2]]
[1] "beta"  "gamma"


> select(xs, -x3)
Error in select.list(xs, -x3) : 
  select.list() cannot be used non-interactively
Calls: select -> select.list
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 7 lines ...

Attaching package: 'lplyr'

The following object is masked from 'package:dplyr':

    pull

Quitting from lines 22-30 (lplyr-vignette.Rmd) 
Error: processing vignette 'lplyr-vignette.Rmd' failed with diagnostics:
select.list() cannot be used non-interactively
Execution halted
```

## mason (0.2.5)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/mason/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'dplyr::mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## medicare (0.2.1)
Maintainer: Robert Gambrel <robert.gambrel@gmail.com>  
Bug reports: http://www.github.com/robertgambrel/medicare/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag
... 8 lines ...
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Quitting from lines 197-222 (cost-report-analysis.Rmd) 
Error: processing vignette 'cost-report-analysis.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted
```

## merTools (0.3.0)
Maintainer: Jared E. Knowles <jknowles@gmail.com>  
Bug reports: https://www.github.com/jknowles/merTools

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘merTools-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotREsim
> ### Title: Plot the results of a simulation of the random effects
> ### Aliases: plotREsim
> 
> ### ** Examples
> 
>  require(ggplot2); require(lme4);
Loading required package: ggplot2
>  fm1 <- lmer(Reaction ~ Days + (Days | Subject), sleepstudy)
>  (p1 <- plotREsim(REsim(fm1)))
>  #Plot just the random effects for the Days slope
>  (p2 <- plotREsim(REsim(fm1)[REsim(fm1)$term=="Days",]))
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L)) : 
  replacement has 2 rows, data has 1
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## metacoder (0.1.2)
Maintainer: Zachary Foster <zacharyfoster1989@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [12s/12s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  result$taxon_data$taxon_info_2 not equal to c(NA, NA, NA, "9639", "9688").
  target is NULL, current is character
  
  
  testthat results ================================================================
  OK: 155 SKIPPED: 5 FAILED: 5
  1. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#111) 
  2. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#112) 
  3. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#135) 
  4. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#136) 
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  
  Error: testthat unit tests failed
  Execution halted
```

## metricTester (1.3.2)
Maintainer: Eliot Miller <eliot.isaac@gmail.com>  
Bug reports: https://github.com/eliotmiller/metricTester/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## Momocs (1.1.0)
Maintainer: Vincent Bonhomme <bonhomme.vincent@gmail.com>  
Bug reports: https://github.com/vbonhomme/Momocs/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘Momocs-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: transmute
> ### Title: Transmutes (ala dplyr) on Momocs objects
> ### Aliases: transmute
> 
> ### ** Examples
> 
> olea
An Opn object with: 
--------------------
 - $coo: 210 open outlines (99 +/- 4 coordinates)
 - $fac: 4 classifiers:
     'var' (factor 4): Aglan, Cypre, MouBo1, PicMa.
     'domes' (factor 2): cult, wild.
     'view' (factor 2): VD, VL.
     'ind' (factor 30): O1, O10, O11, O12, O13, O14, O15, O16, O17, O18, O19 ... + 19 more.
> transmute(olea, id=factor(1:length(olea)))
Error: evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following object is masked from 'package:base':

    table

method not provided. efourier is used
method not provided. efourier is used
... 8 lines ...
'nb.pts' missing and set to 91
'degree' missing and set to 5
'nb.h' not provided and set to 10 (99% harmonic power)
PC axes 1 to 7 were retained
PC axes 1 to 7 were retained
'nb.h' not provided and set to 10 (99% harmonic power)
PC axes 1 to 7 were retained
no 'fac' provided, returns meanshape
Error: processing vignette 'Momocs_speed_dating.Rmd' failed with diagnostics:
evaluation nested too deeply: infinite recursion / options(expressions=)?
Execution halted
```

## MonetDBLite (0.3.1)
Maintainer: Hannes Muehleisen <hannes@cwi.nl>  
Bug reports: https://github.com/hannesmuehleisen/MonetDBLite/issues

2 errors | 0 warnings | 2 notes

```
checking examples ... ERROR
Running examples in ‘MonetDBLite-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: src_monetdb
> ### Title: dplyr integration from MonetDBLite
> ### Aliases: src_monetdb src_monetdblite tbl.src_monetdb
> ###   src_desc.src_monetdb src_translate_env.src_monetdb
> ###   sample_frac.tbl_monetdb sample_n.tbl_monetdb
... 18 lines ...

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> # Connection basics ---------------------------------------------------------
> # To connect to a database first create a src:
> dbdir <- file.path(tempdir(), "dplyrdir")
> my_db <- MonetDBLite::src_monetdb(embedded=dbdir)
Error: 'src_sql' is not an exported object from 'namespace:dplyr'
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [18s/61s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 132 SKIPPED: 3 FAILED: 13
  1. Error: we can connect (@test_03_dplyr.R#15) 
  2. Error: dplyr copy_to() (@test_03_dplyr.R#27) 
  3. Error: dplyr tbl( sql() ) (@test_03_dplyr.R#35) 
  4. Error: dplyr select() (@test_03_dplyr.R#43) 
  5. Error: dplyr filter() (@test_03_dplyr.R#51) 
  6. Error: dplyr arrange() (@test_03_dplyr.R#59) 
  7. Error: dplyr mutate() (@test_03_dplyr.R#67) 
  8. Error: dplyr summarise() (@test_03_dplyr.R#75) 
  9. Error: dplyr multiple objects (@test_03_dplyr.R#84) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is  7.0Mb
  sub-directories of 1Mb or more:
    libs   6.7Mb

checking dependencies in R code ... NOTE
Missing or unexported objects:
  ‘dplyr::base_agg’ ‘dplyr::base_scalar’ ‘dplyr::base_win’
  ‘dplyr::build_sql’ ‘dplyr::ident’ ‘dplyr::is.ident’ ‘dplyr::sql’
  ‘dplyr::sql_infix’ ‘dplyr::sql_prefix’ ‘dplyr::sql_translator’
  ‘dplyr::sql_variant’ ‘dplyr::src_sql’ ‘dplyr::tbl_sql’
```

## mrgsolve (0.8.6)
Maintainer: Kyle T Baron  <kyleb@metrumrg.com>  
Bug reports: https://github.com/metrumresearchgroup/mrgsolve/issues

0 errors | 1 warning  | 1 note 

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking installed package size ... NOTE
  installed size is  6.2Mb
  sub-directories of 1Mb or more:
    libs   5.0Mb
```

## mtconnectR (1.1.0)
Maintainer: Subramanyam Ravishankar <subramanyam@systeminsights.com>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘mtconnectR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: map_gcode_mtc
> ### Title: Create a mapping between simulated and actual data
> ### Aliases: map_gcode_mtc
> 
> ### ** Examples
> 
> data("example_gcode_parsed") # Parsed gcode
> data("example_mtc_device_3") # MTCDevice object of actual log data
> simulated_gcode_data = na.omit(simulate_data_from_gcode(example_gcode_parsed, 
+ start_time = 0, data_res = 0.1, data_type = "HH"))
Error in if (str_detect(current_values$state_motion, "ARC_")) distance_moved = arc_position_change(current_values,  : 
  argument is of length zero
Calls: na.omit ... simulate_data_from_gcode -> update_timestamps -> pfr_time
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [34s/34s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
             1404 
  1. Error: (unknown) (@test-simulate_gcode.R#6) ---------------------------------
  argument is of length zero
  1: na.omit(simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, 
         data_type = "HH")) at testthat/test-simulate_gcode.R:6
  2: simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, data_type = "HH")
  3: update_timestamps(current_values, previous_values)
  4: pfr_time(current_values, previous_values, current_values$pfr, current_values$state_motion)
  
  testthat results ================================================================
  OK: 38 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test-simulate_gcode.R#6) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 66-74 (simulate_map_gcode.Rmd) 
Error: processing vignette 'simulate_map_gcode.Rmd' failed with diagnostics:
argument is of length zero
Execution halted

```

## mudata (0.1)
Maintainer: Dewey Dunnington <dewey@fishandwhistle.net>  
Bug reports: https://github.com/paleolimbot/mudata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
             return(TRUE)
         }
         else {
             return(FALSE)
         }
     }, kentvillegreenwood$data$flags, reexpanded$data$flags, USE.NAMES = FALSE)
  
  testthat results ================================================================
  OK: 187 SKIPPED: 0 FAILED: 3
  1. Error: expand.tags behaviour in constructor is consistent (@test_mudata.R#75) 
  2. Error: expand/condense functions work properly for data frames (@test_tags.R#10) 
  3. Error: expand/condense tags work properly for mudata objects (@test_tags.R#30) 
  
  Error: testthat unit tests failed
  Execution halted
```

## myTAI (0.5.0)
Maintainer: Hajk-Georg Drost <hgd23@cam.ac.uk>  
Bug reports: https://github.com/HajkD/myTAI/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [77s/77s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: PlotContribution() works properly with PhyloExpressionSet input... (@test-PlotContribution.R#11) 
  PlotContribution(PhyloExpressionSetExample, legendName = "PS") produced warnings.
  
  
  2. Failure: PlotContribution() works properly with DivergenceExpressionSet input... (@test-PlotContribution.R#16) 
  PlotContribution(DivergenceExpressionSetExample, legendName = "DS") produced warnings.
  
  
  testthat results ================================================================
  OK: 197 SKIPPED: 1 FAILED: 2
  1. Failure: PlotContribution() works properly with PhyloExpressionSet input... (@test-PlotContribution.R#11) 
  2. Failure: PlotContribution() works properly with DivergenceExpressionSet input... (@test-PlotContribution.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    data   2.0Mb
    doc    2.3Mb
```

## NFP (0.99.2)
Maintainer: Yang Cao <yiluheihei@gmail.com>  
Bug reports: https://github.com/yiluheihei/NFP/issues

0 errors | 1 warning  | 2 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    clusterApply, clusterApplyLB, clusterCall, clusterEvalQ,
    clusterExport, clusterMap, parApply, parCapply, parLapply,
    parLapplyLB, parRapply, parSapply, parSapplyLB

The following objects are masked from 'package:stats':

... 8 lines ...
    match, mget, order, paste, pmax, pmax.int, pmin, pmin.int, rank,
    rbind, rownames, sapply, setdiff, sort, table, tapply, union,
    unique, unsplit, which, which.max, which.min

Loading required package: graphite
Warning in library(package, lib.loc = lib.loc, character.only = TRUE, logical.return = TRUE,  :
  there is no package called 'graphite'
Quitting from lines 198-206 (NFP.Rnw) 
Error: processing vignette 'NFP.Rnw' failed with diagnostics:
could not find function "pathways"
Execution halted

checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘graphite’ ‘NFPdata’

checking installed package size ... NOTE
  installed size is  8.4Mb
  sub-directories of 1Mb or more:
    data   7.5Mb
```

## officer (0.1.1)
Maintainer: David Gohel <david.gohel@ardata.fr>  
Bug reports: https://github.com/davidgohel/officer/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘officer-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: slip_in_img
> ### Title: append an image
> ### Aliases: slip_in_img
> 
> ### ** Examples
> 
> library(magrittr)
> img.file <- file.path( Sys.getenv("R_HOME"), "doc", "html", "logo.jpg" )
> x <- read_docx() %>%
+   body_add_par("R logo: ", style = "Normal") %>%
+   slip_in_img(src = img.file, style = "strong", width = .3, height = .3)
Error: file.exists(src) is not TRUE
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [42s/42s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  9: ph_with_img(., type = "body", src = img.file, height = 1.06, width = 1.39)
  10: external_img(src, width = width, height = height)
  11: stopifnot(file.exists(src))
  12: stop(sprintf(ngettext(length(r), "%s is not TRUE", "%s are not all TRUE"), ch), call. = FALSE, 
         domain = NA)
  
  testthat results ================================================================
  OK: 317 SKIPPED: 0 FAILED: 4
  1. Error: image add  (@test-docx-add.R#68) 
  2. Error: pml fp_border (@test-fp_cell.R#75) 
  3. Error: css fp_border (@test-fp_cell.R#165) 
  4. Error: add img into placeholder (@test-pptx-add.R#67) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 181-191 (powerpoint.Rmd) 
Error: processing vignette 'powerpoint.Rmd' failed with diagnostics:
file.exists(src) is not TRUE
Execution halted

```

## openair (2.0-0)
Maintainer: David Carslaw <david.carslaw@york.ac.uk>  
Bug reports: https://github.com/davidcarslaw/openair/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
           user system elapsed
polarPlot 5.988  0.004   5.921
```

## padr (0.2.1)
Maintainer: Edwin Thoen <edwinthoen@gmail.com>  
Bug reports: https://github.com/EdwinTh/padr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [150s/150s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-tbl-cube.R:119:1: style: lines should not be more than 80 characters.
    expect_identical(as.table(nasa, measure = "ozone"), as.table(select(nasa, ozone)))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-ts.R:8:1: style: lines should not be more than 80 characters.
      "`.data`: must be a data source, not ts object, do you want `stats::filter()`?",
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  
  
  testthat results ================================================================
  OK: 192 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test_zzz_lintr.R#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## parsemsf (0.1.0)
Maintainer: Benjamin Jack <benjamin.r.jack@gmail.com>  
Bug reports: https://github.com/benjaminjack/parsemsf/issues

2 errors | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘parsemsf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: make_area_table
> ### Title: Make a table of peptide areas
> ### Aliases: make_area_table
> 
> ### ** Examples
> 
> make_area_table(parsemsf_example("test_db.msf"))
Error in inherits(x, "sql") : could not find function "sql"
Calls: make_area_table ... tbl -> tbl.src_dbi -> tbl_sql -> is.sql -> inherits
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  6: tbl(my_db, sql("SELECT FieldID, PeptideID, CAST(FieldValue as REAL) AS FieldValue FROM CustomDataPeptides"))
  7: tbl.src_dbi(my_db, sql("SELECT FieldID, PeptideID, CAST(FieldValue as REAL) AS FieldValue FROM CustomDataPeptides")) at /home/muelleki/git/R/dplyr/R/tbl.r:9
  8: tbl_sql("dbi", src = src, from = from) at /tmp/RtmpqNmvln/devtools1b6db9ea4b4/hadley-dbplyr-ce94c15/R/src_dbi.R:115
  9: is.sql(from) at /tmp/RtmpqNmvln/devtools1b6db9ea4b4/hadley-dbplyr-ce94c15/R/tbl-sql.r:16
  10: inherits(x, "sql") at /tmp/RtmpqNmvln/devtools1b6db9ea4b4/hadley-dbplyr-ce94c15/R/sql-escape.r:78
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 4
  1. Error: make_area_table creates a data frame with the correct column names (@test_make_area_table.R#16) 
  2. Error: make_pep_table creates a data frame with the correct column names (@test_make_pep_table.R#13) 
  3. Error: map_peptides creates a data frame with the correct column names (@test_map_peptides.R#16) 
  4. Error: (unknown) (@test_quantitate.R#9) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 20-25 (introduction.Rmd) 
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
could not find function "sql"
Execution halted


checking R code for possible problems ... NOTE
make_pep_table: no visible global function definition for ‘sql’
Undefined global functions or variables:
  sql
```

## pivottabler (0.1.0)
Maintainer: Christopher Bailiss <cbailiss@gmail.com>  
Bug reports: https://github.com/cbailiss/pivottabler/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [25s/25s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  x[1]: "9b3e30ce7c3378371c41b1232cd96441"
  y[1]: "e82ee929b0e423df8b48adc926a264d8"
  
  
  testthat results ================================================================
  OK: 67 SKIPPED: 0 FAILED: 6
  1. Failure: basic layout tests:  two measures (@testGeneral.R#132) 
  2. Failure: basic layout tests:  rows plus two measures (@testGeneral.R#187) 
  3. Failure: basic layout tests:  columns plus two totals (@testGeneral.R#242) 
  4. Failure: basic layout tests:  rows, columns and two calculations (@testGeneral.R#300) 
  5. Failure: basic layout tests:  columns plus two totals on rows (@testGeneral.R#341) 
  6. Failure: basic layout tests:  rows, columns and two calculations on rows (@testGeneral.R#384) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in normalizePath(f2) :
  path[1]="./webshotf1f14d867732.png": No such file or directory
Warning in file(con, "rb") :
  cannot open file './webshotf1f14d867732.png': No such file or directory
Quitting from lines 53-60 (calculations.Rmd) 
Error: processing vignette 'calculations.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## pixiedust (0.7.4)
Maintainer: Benjamin Nutter <nutter@battelle.org>  
Bug reports: https://github.com/nutterb/pixiedust/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [24s/24s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 144 SKIPPED: 1 FAILED: 12
  1. Failure: dust with additional descriptors (@test-dust.R#58) 
  2. Failure: dust with additional descriptors and term_plain numeric_label (@test-dust.R#64) 
  3. Failure: dust with glance_foot (@test-dust.R#71) 
  4. Failure: dust with glance_foot and col_pairs a divisor of total_cols (@test-dust.R#77) 
  5. Failure: dust with descriptors (@test-dust.R#94) 
  6. Failure: glance_foot by column (@test-glance_foot.R#14) 
  7. Failure: glance_foot by row (@test-glance_foot.R#19) 
  8. Failure: medley_model (@test-medley.R#17) 
  9. Failure: output for a model with no interactions (@test-tidy_label_level.R#19) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## plotly (4.5.6)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: https://github.com/ropensci/plotly/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [81s/80s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  [1] "Running test: plotly-group-within-trace"
  [1] "Running test: plotly-alpha-blending"
  [1] "Running test: plotly-alpha-no-color"
  [1] "Running test: plotly-factor-axis"
  [1] "Running test: plotly-character-axis"
  [1] "Running test: plotly-histogram"
  [1] "Running test: plotly-histogram-vert"
  [1] "Running test: plotly-inherit-FALSE"
  [1] "Running test: plotly-time-series-summary"
  testthat results ================================================================
  OK: 814 SKIPPED: 17 FAILED: 1
  1. Error: datetimes are converted to e.g. 2013-01-02 05:00:00 (@test-ggplot-date.R#11) 
  
  Error: testthat unit tests failed
  Execution halted
```

## poppr (2.4.0)
Maintainer: Zhian N. Kamvar <zkamvar@gmail.com>  
Bug reports: https://github.com/grunwaldlab/poppr/issues

0 errors | 1 warning  | 0 notes

```
checking whether package ‘poppr’ can be installed ... WARNING
Found the following significant warnings:
  Warning: namespace ‘DBI’ is not available and has been replaced
See ‘/home/muelleki/git/R/dplyr/revdep/checks/poppr.Rcheck/00install.out’ for details.
```

## prophet (0.1)
Maintainer: Sean Taylor <sjt@fb.com>

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [12s/12s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Optimization terminated normally: 
    Convergence detected: absolute parameter change was below tolerance
  3. Failure: fit_with_holidays (@test_prophet.R#188) ----------------------------
  predict(m) threw an error.
  object 'lower' not found
  
  
  testthat results ================================================================
  OK: 32 SKIPPED: 0 FAILED: 3
  1. Failure: fit_predict (@test_prophet.R#18) 
  2. Failure: fit_predict_no_changepoints (@test_prophet.R#30) 
  3. Failure: fit_with_holidays (@test_prophet.R#188) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: Rcpp

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 43-45 (quick_start.Rmd) 
Error: processing vignette 'quick_start.Rmd' failed with diagnostics:
object 'lower' not found
Execution halted


checking installed package size ... NOTE
  installed size is 18.8Mb
  sub-directories of 1Mb or more:
    libs  18.6Mb
```

## purrr (0.2.2)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/purrr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘purrr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/purrr.Rcheck/00install.out’ for details.
```

## qualvar (0.1.0)
Maintainer: Joel Gombin <joel.gombin@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 92-106 (wilcox1973.Rmd) 
Error: processing vignette 'wilcox1973.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## radiant.basics (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.basics/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 26 lines ...
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.data (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.data/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.data-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dtab.pivotr
> ### Title: Make a pivot tabel in DT
> ### Aliases: dtab.pivotr
> 
> ### ** Examples
> 
> pivotr("diamonds", cvars = "cut") %>% dtab
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: cut: must resolve to integer column positions, not closure
Execution halted
```

## radiant.design (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.design/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.model (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.model/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.model-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.model.predict
> ### Title: Plot method for model.predict functions
> ### Aliases: plot.model.predict
> 
> ### ** Examples
... 25 lines ...
See help("Deprecated")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error in rep(yes, length.out = length(ans)) : 
  attempt to replicate an object of type 'closure'
Calls: %>% ... <Anonymous> -> plot -> plot.model.predict -> ifelse
Execution halted
```

## radiant.multivariate (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.multivariate/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 30 lines ...
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## rattle (4.1.0)
Maintainer: Graham Williams <Graham.Williams@togaware.com>

0 errors | 1 warning  | 3 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'rattle.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm2e.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.14 \usepackage
                [^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘pkgDepTools’

checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    data   2.5Mb
    etc    1.9Mb
    po     1.2Mb

checking dependencies in R code ... NOTE

(R:5622): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
```

## recexcavAAR (0.3.0)
Maintainer: Clemens Schmid <clemens@nevrome.de>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: kriging

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 70-71 (recexcavAAR-vignette-1.Rmd) 
Error: processing vignette 'recexcavAAR-vignette-1.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is  5.9Mb
  sub-directories of 1Mb or more:
    doc    2.5Mb
    libs   3.3Mb
```

## REDCapR (0.9.7)
Maintainer: Will Beasley <wibeasley@hotmail.com>  
Bug reports: https://github.com/OuhscBbmc/REDCapR/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 42-45 (BasicREDCapROperations.Rmd) 
Error: processing vignette 'BasicREDCapROperations.Rmd' failed with diagnostics:
object 'ds' not found
Execution halted

```

## rgho (1.0.1)
Maintainer: Antoine Filipovic-Pierucci <pierucci@gmail.com>  
Bug reports: https://github.com/pierucci/rgho/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘rgho-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: get_gho_data
> ### Title: Returns GHO Data
> ### Aliases: get_gho_data
> 
> ### ** Examples
... 36 lines ...
> result <- get_gho_data(
+   dimension = "GHO",
+   code = "MDG_0000000001",
+   filter = list(
+     REGION = "EUR",
+     YEAR = "2015"
+   )
+ )
Error in get_gho_data(dimension = "GHO", code = "MDG_0000000001", filter = list(REGION = "EUR",  : 
  No data returned by WHO GHO server.
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 101-111 (a-intro.Rmd) 
Error: processing vignette 'a-intro.Rmd' failed with diagnostics:
No data returned by WHO GHO server.
Execution halted

```

## rmcfs (1.2.4)
Maintainer: Michal Draminski <michal.draminski@ipipan.waw.pl>

0 errors | 1 warning  | 2 notes

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.data:
  function(data, mcfs_result, size)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.data
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    java   4.5Mb

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.data':
  ‘filter.data’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## RPresto (1.2.1)
Maintainer: Onur Ismail Filiz <onur@fb.com>  
Bug reports: https://github.com/prestodb/RPresto/issues

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [5s/10s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: setup_mock_dplyr_connection() at testthat/test-db_query_fields.R:42
  2: src_presto(RPresto::Presto(), schema = "test", catalog = "catalog", host = "http://localhost", 
         port = 8000, user = Sys.getenv("USER"), session.timezone = test.timezone(), parameters = list())
  3: dplyr::src_sql
  4: getExportedValue(pkg, name)
  5: stop(gettextf("'%s' is not an exported object from 'namespace:%s'", name, getNamespaceName(ns)), 
         call. = FALSE, domain = NA)
  
  testthat results ================================================================
  OK: 175 SKIPPED: 33 FAILED: 2
  1. Error: db_explain works with mock (@test-db_explain.R#25) 
  2. Error: db_query_fields works with mock (@test-db_query_fields.R#42) 
  
  Error: testthat unit tests failed
  Execution halted

checking Rd cross-references ... WARNING
Missing link or links in documentation object 'dplyr_function_implementations.Rd':
  ‘[dplyr]{src_desc}’

Missing link or links in documentation object 'src_presto.Rd':
  ‘[dplyr]{sql}’

See section 'Cross-references' in the 'Writing R Extensions' manual.


checking dependencies in R code ... NOTE
Missing or unexported objects:
  ‘dplyr::base_agg’ ‘dplyr::base_scalar’ ‘dplyr::base_win’
  ‘dplyr::build_sql’ ‘dplyr::ident’ ‘dplyr::sql’ ‘dplyr::sql_prefix’
  ‘dplyr::sql_translator’ ‘dplyr::sql_variant’ ‘dplyr::src_sql’
  ‘dplyr::tbl_sql’
```

## rsparkling (0.2.0)
Maintainer: Navdeep Gill <navdeep@h2o.ai>  
Bug reports: http://jira.h2o.ai

1 error  | 0 warnings | 0 notes

```
checking whether package ‘rsparkling’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/rsparkling.Rcheck/00install.out’ for details.
```

## rtide (0.0.3)
Maintainer: Joe Thorley <joe@poissonconsulting.ca>  
Bug reports: https://github.com/poissonconsulting/rtide/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [35s/35s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  attributes are different
  1: expect_equal(tide_height_data(data), tide_height_data(data2)) at testthat/test-tide-height-data.R:49
  2: compare(object, expected, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-equality.R:52
  3: compare.default(object, expected, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/compare.R:12
  4: all.equal(x, y, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/compare.R:47
  5: all.equal.tbl_df(x, y, ...)
  6: equal_data_frame(target, current, ignore_col_order = ignore_col_order, ignore_row_order = ignore_row_order, 
         convert = convert) at /home/muelleki/git/R/dplyr/R/all-equal.r:35
  
  testthat results ================================================================
  OK: 30 SKIPPED: 0 FAILED: 1
  1. Error: tide_height_data tz (@test-tide-height-data.R#49) 
  
  Error: testthat unit tests failed
  Execution halted
```

## saeSim (0.8.0)
Maintainer: Sebastian Warnholz <Sebastian.Warnholz@fu-berlin.de>  
Bug reports: https://github.com/wahani/saeSim/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’ [14s/13s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  
  [[1]]
  [1] 1
  
  
  [[1]]
  [1] 1
  
  
  testthat results ================================================================
  OK: 184 SKIPPED: 0 FAILED: 1
  1. Error: base_add_id (@test-sim_base.R#6) 
  
  Error: testthat unit tests failed
  Execution halted
```

## sejmRP (1.3.4)
Maintainer: Piotr Smuda <piotrsmuda@gmail.com>  
Bug reports: http://github.com/mi2-warsaw/sejmRP/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘sejmRP’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/sejmRP.Rcheck/00install.out’ for details.
```

## sfc (0.1.0)
Maintainer: Hu Sheng <shenghu@nju.edu.cn>  
Bug reports: https://github.com/ctfysh/sfc/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## sf (0.4-1)
Maintainer: Edzer Pebesma <edzer.pebesma@uni-muenster.de>  
Bug reports: https://github.com/edzer/sfr/issues/

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr
> ### Title: Dplyr verb methods for sf objects
> ### Aliases: dplyr filter_.sf dplyr arrange_.sf dplyr distinct_.sf dplyr
> ###   group_by_.sf dplyr mutate_.sf dplyr transmute_.sf dplyr select_.sf
> ###   dplyr rename_.sf dplyr slice_.sf dplyr summarise_.sf dplyr gather_.sf
... 22 lines ...
Simple feature collection with 100 features and 14 fields
geometry type:  MULTIPOLYGON
dimension:      XY
bbox:           xmin: -84.32385 ymin: 33.88199 xmax: -75.45698 ymax: 36.58965
epsg (SRID):    4267
proj4string:    +proj=longlat +datum=NAD27 +no_defs
> nc %>% filter(AREA > .1) %>% plot()
Error in data.matrix(x) : 
  (list) object cannot be coerced to type 'double'
Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
Execution halted

checking tests ... ERROR
  Running ‘cast.R’
  Comparing ‘cast.Rout’ to ‘cast.Rout.save’ ... OK
  Running ‘crs.R’
  Comparing ‘crs.Rout’ to ‘crs.Rout.save’ ... OK
  Running ‘dist.R’
  Comparing ‘dist.Rout’ to ‘dist.Rout.save’ ... OK
  Running ‘dplyr.R’
Running the tests in ‘tests/dplyr.R’ failed.
Last 13 lines of output:
  
  The following objects are masked from 'package:stats':
  
      filter, lag
  
  The following objects are masked from 'package:base':
  
      intersect, setdiff, setequal, union
  
  > nc = st_read(system.file("shape/nc.shp", package="sf"), quiet = TRUE)
  > nc %>% filter(AREA > .1) %>% plot()
  Error in data.matrix(x) : 
    (list) object cannot be coerced to type 'double'
  Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
  Execution halted

checking installed package size ... NOTE
  installed size is 10.2Mb
  sub-directories of 1Mb or more:
    doc    4.0Mb
    libs   4.2Mb
```

## shazam (0.1.5)
Maintainer: Jason Vander Heiden <jason.vanderheiden@yale.edu>  
Bug reports: https://bitbucket.org/kleinstein/shazam/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 43 lines ...
plotTune               8.716  0.020   8.612
plotBaselineSummary    7.640  0.192   7.764
observedMutations      7.428  0.112   7.583
testBaseline           7.064  0.184   7.101
summarizeBaseline      6.988  0.204   7.120
getBaselineStats       6.720  0.208   6.973
extendMutabilityMatrix 6.672  0.104   6.706
distToNearest          6.044  0.032   6.017
minNumSeqMutationsTune 5.960  0.040   5.930
plotBaselineDensity    5.576  0.220   5.654
** found \donttest examples: check also with --run-donttest
```

## sjmisc (2.4.0)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/strengejacke/sjmisc/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjmisc-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: merge_df
> ### Title: Merge labelled data frames
> ### Aliases: merge_df
> 
> ### ** Examples
... 71 lines ...
> ##D library(sjPlot)
> ##D view_df(mydf)
> ## End(Not run)
> 
> x3 <- efc %>% select(5:9) %>% slice(21:30)
> x4 <- efc %>% select(11:14) %>% slice(31:40)
> 
> mydf <- merge_df(x1, x2, x3, x4, id = "subsets")
Error in cbind_all(x) : Argument 2: must have names
Calls: merge_df -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted
```

## sjPlot (2.3.1)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/devel/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
    select

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':
... 8 lines ...
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Argument `recodes` is deprecated. Please use `rec` instead.
Computing p-values via Kenward-Roger approximation. Use `p.kr = FALSE` if computation takes too long.
Quitting from lines 66-67 (sjtlmer.Rmd) 
Error: processing vignette 'sjtlmer.Rmd' failed with diagnostics:
is.data.frame(df) is not TRUE
Execution halted
```

## SpaDES (1.3.1)
Maintainer: Alex M Chubaty <alexander.chubaty@canada.ca>  
Bug reports: https://github.com/PredictiveEcology/SpaDES/issues

0 errors | 2 warnings | 2 notes

```
checking whether package ‘SpaDES’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/SpaDES.Rcheck/00install.out’ for details.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    RFoptions

The following object is masked from 'package:raster':

    atan2

... 8 lines ...

Attaching package: 'grid'

The following object is masked from 'package:SpaDES':

    gpar

Quitting from lines 352-356 (ii-modules.Rmd) 
Error: processing vignette 'ii-modules.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘fastshp’

checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    R     2.5Mb
    doc   2.1Mb
```

## sparklyr (0.5.3)
Maintainer: Javier Luraschi <javier@rstudio.com>  
Bug reports: https://github.com/rstudio/sparklyr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘sparklyr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/sparklyr.Rcheck/00install.out’ for details.
```

## sparseHessianFD (0.3.2)
Maintainer: Michael Braun <braunm@smu.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'sparseHessianFD.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.27 \usepackage
                {algorithmic}^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## spbabel (0.4.5)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spbabel/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [43s/43s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
             where = .classEnv(ClassDef)))
         if (identical(ok, FALSE)) 
             stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
                 dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
         TRUE
     })(structure("SpatialPointsDataFrame", package = "sp"), "data", "matrix") at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/SpatialLinesDataFrame-methods.R:93
  10: stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
         dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
  
  testthat results ================================================================
  OK: 57 SKIPPED: 2 FAILED: 1
  1. Error: utils tests work (@test-utils.R#16) 
  
  Error: testthat unit tests failed
  Execution halted
```

## spdplyr (0.1.2)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spdplyr/issues

0 errors | 1 warning  | 0 notes

```
checking whether package ‘spdplyr’ can be installed ... WARNING
Found the following significant warnings:
  Warning: namespace ‘DBI’ is not available and has been replaced
See ‘/home/muelleki/git/R/dplyr/revdep/checks/spdplyr.Rcheck/00install.out’ for details.
```

## sqlscore (0.1.1)
Maintainer: William Brannon <wwbrannon@email.wm.edu>  
Bug reports: https://github.com/wwbrannon/sqlscore/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sqlscore-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: create_statement
> ### Title: Generate a CREATE TABLE statement from a model
> ### Aliases: create_statement
> 
> ### ** Examples
> 
> # Basic create statements
> mod <- glm(Sepal.Length ~ Sepal.Width + Petal.Length + Petal.Width + Species,
+            data=datasets::iris)
> create_statement(mod, src_table="tbl_name", dest_table="target_tbl")
Error: 'ident' is not an exported object from 'namespace:dplyr'
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [17s/17s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: fqtn(src_tbl, schema = src_schema, catalog = src_catalog) at testthat/test-select_statement.R:9
  2: dplyr::ident
  3: getExportedValue(pkg, name)
  4: stop(gettextf("'%s' is not an exported object from 'namespace:%s'", name, getNamespaceName(ns)), 
         call. = FALSE, domain = NA)
  
  testthat results ================================================================
  OK: 79 SKIPPED: 0 FAILED: 4
  1. Error: (unknown) (@test-create_statement.R#9) 
  2. Error: Basic table names are escaped (@test-fqtn.R#4) 
  3. Error: More complicated names are escaped (@test-fqtn.R#10) 
  4. Error: (unknown) (@test-select_statement.R#9) 
  
  Error: testthat unit tests failed
  Execution halted

checking dependencies in R code ... NOTE
Missing or unexported objects:
  ‘dplyr::build_sql’ ‘dplyr::ident’ ‘dplyr::translate_sql’
```

## srvyr (0.2.0)
Maintainer: Greg Freedman Ellis <greg.freedman@gmail.com>  
Bug reports: https://github.com/gergness/srvyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘srvyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: as_survey_design
> ### Title: Create a tbl_svy survey object using sampling design
> ### Aliases: as_survey_design as_survey_design.data.frame
> ###   as_survey_design.survey.design2 as_survey_design.tbl_sql
> ###   as_survey_design_
... 21 lines ...
> 
> # one-stage cluster sample
> dclus1 <- apiclus1 %>%
+   as_survey_design(dnum, weights = pw, fpc = fpc)
> 
> # two-stage cluster sample: weights computed from population sizes.
> dclus2 <- apiclus2 %>%
+   as_survey_design(c(dnum, snum), fpc = c(fpc1, fpc2))
Error in overscope_eval_next(overscope, f) : object 'dnum' not found
Calls: %>% ... map -> lapply -> FUN -> overscope_eval_next -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [11s/11s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 64 SKIPPED: 30 FAILED: 15
  1. Error: as_survey correctly uses survey design's arguments (@expect-equality.R#52) 
  2. Failure: as_survey_twophase gets same mean / total / median / ratio in srvyr 
  3. Failure: as_survey_twophase gets same mean / total / median / ratio in (grouped) 
  4. Failure: srvyr and survey get same mean (grouped) 
  5. Failure: srvyr and survey get same total (grouped) 
  6. Failure: survey_total is correct when doing props with multiple groups 
  7. Failure: survey_mean and survey_total work with cis 
  8. Failure: survey_total is correct with special chars in peel 
  9. Failure: ungrouped proportion works correctly 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    dotchart


Attaching package: 'dplyr'

The following objects are masked from 'package:stats':
... 8 lines ...

Attaching package: 'srvyr'

The following object is masked from 'package:stats':

    filter

Quitting from lines 75-83 (srvyr-vs-survey.Rmd) 
Error: processing vignette 'srvyr-vs-survey.Rmd' failed with diagnostics:
object 'stype' not found
Execution halted
```

## ss3sim (0.9.3)
Maintainer: Sean Anderson <sean@seananderson.ca>

0 errors | 2 warnings | 0 notes

```
checking whether package ‘ss3sim’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ss3sim.Rcheck/00install.out’ for details.

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## stormwindmodel (0.1.0)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>  
Bug reports: https://github.com/geanders/stormwindmodel/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

... 8 lines ...
The following object is masked from 'package:dplyr':

    combine

Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=georgia&zoom=5&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=georgia&sensor=false
Quitting from lines 220-233 (Details.Rmd) 
Error: processing vignette 'Details.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted
```

## stplanr (0.1.7-3)
Maintainer: Robin Lovelace <rob00x@gmail.com>  
Bug reports: https://github.com/ropensci/stplanr/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'dplyr::summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'dplyr::summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'dplyr::summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'dplyr::summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'dplyr::summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## texmexseq (0.3)
Maintainer: Scott Olesen <swo@mit.edu>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘texmexseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: quad.table
> ### Title: Create and plot "quads" of samples
> ### Aliases: quad.table quad.plot
> ### Keywords: hplot manip design
> 
... 12 lines ...
Warning in value[[3L]](cond) : fit 2 failed
> 
> # pull out a quad, imagining that samples 1 and 2 were the control samples
> # and 3 and 4 were the treatment
> q <- quad.table(z.table, 'sample1', 'sample2', 'sample3', 'sample4')
> 
> # plot it
> p <- quad.plot(q)
Error in get("d.control") : object 'd.control' not found
Calls: quad.plot ... lapply -> FUN -> overscope_eval_next -> .Call -> get
Execution halted
```

## textmining (0.0.1)
Maintainer: Jan Idziak <JanIdziak@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking whether package ‘textmining’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/textmining.Rcheck/00install.out’ for details.
```

## tibble (1.3.0)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/tidyverse/tibble/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 368 SKIPPED: 4 FAILED: 10
  1.  Error: add_column() keeps class of object (@test-add.R#121) 
  2.  Error: add_column() keeps class of object when adding in the middle (@test-add.R#129) 
  3.  Error: add_column() keeps class of object when adding in the beginning (@test-add.R#137) 
  4.  Error: missing row names stay missing when adding column (@test-add.R#220) 
  5.  Failure: data frames not equal if missing col (@test-equality.R#32) 
  6.  Failure: data frames not equal if missing col (@test-equality.R#33) 
  7.  Failure: data frames not equal if missing col (@test-equality.R#34) 
  8.  Failure: data frames not equal if missing col (@test-equality.R#35) 
  9.  Failure: factors equal only if levels equal (@test-equality.R#44) 
  10. Failure: equality test fails when convert is FALSE and types don't match (#1484) (@test-equality.R#82) 
  
  Error: testthat unit tests failed
  Execution halted
```

## tidyjson (0.2.1)
Maintainer: Jeremy Stanley <jeremy.stanley@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘tidyjson-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: companies
> ### Title: Startup company information for 1,000 companies
> ### Aliases: companies
> 
> ### ** Examples
... 34 lines ...
+   enter_object("relationships") %>%
+   gather_array("relationship.index") %>%
+   spread_values(
+     is.past = jlogical("is_past"),
+     name = jstring("person", "permalink"),
+     title = jstring("title")
+   )
Error in eval(expr, envir, enclos) : 
  argument "json.column" is missing, with no default
Calls: %>% ... tryCatchList -> tryCatchOne -> doTryCatch -> eval -> eval
Execution halted

checking R code for possible problems ... NOTE
read_json: no visible global function definition for ‘tail’
Undefined global functions or variables:
  tail
Consider adding
  importFrom("utils", "tail")
to your NAMESPACE file.
```

## tidyquant (0.5.0)
Maintainer: Matt Dancho <mdancho@business-science.io>  
Bug reports: https://github.com/business-science/tidyquant/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [23s/49s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  testthat results ================================================================
  OK: 183 SKIPPED: 2 FAILED: 8
  1. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_exchange_rates.R#20) 
  2. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_exchange_rates.R#21) 
  3. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_exchange_rates.R#25) 
  4. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_exchange_rates.R#27) 
  5. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_metal_prices.R#20) 
  6. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_metal_prices.R#21) 
  7. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_metal_prices.R#25) 
  8. Failure: Test returns tibble with correct rows and columns. (@test_tq_get_metal_prices.R#27) 
  
  Error: testthat unit tests failed
  Execution halted
```

## tidyr (0.6.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/tidyverse/tidyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  x[2]: "2"
  y[2]: "b"
  
  x[3]: "2"
  y[3]: "b"
  
  
  testthat results ================================================================
  OK: 190 SKIPPED: 0 FAILED: 3
  1. Failure: elements must all be of same type (@test-unnest.R#24) 
  2. Failure: .id creates vector of names for vector unnest (@test-unnest.R#59) 
  3. Failure: .id creates vector of names for data frame unnest (@test-unnest.R#69) 
  
  Error: testthat unit tests failed
  Execution halted
```

## tidytext (0.1.2)
Maintainer: Julia Silge <julia.silge@gmail.com>  
Bug reports: http://github.com/juliasilge/tidytext/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [13s/13s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2: new("dictionary", x, format = format, file = file, concatenator = concatenator) at /tmp/RtmpXUxKF8/devtools3f681a456fd7/quanteda/R/dictionaries.R:189
  3: initialize(value, ...)
  4: initialize(value, ...)
  5: validObject(.Object)
  6: anyStrings(validityMethod(object))
  7: validityMethod(object)
  8: validate_dictionary(object) at /tmp/RtmpXUxKF8/devtools3f681a456fd7/quanteda/R/dictionaries.R:20
  9: stop("dictionary elements must be named") at /tmp/RtmpXUxKF8/devtools3f681a456fd7/quanteda/R/dictionaries.R:28
  
  testthat results ================================================================
  OK: 130 SKIPPED: 0 FAILED: 1
  1. Error: can tidy a quanteda dictionary (@test-dictionary-tidiers.R#7) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Using 15 of 16 cores for parallel computing

Attaching package: 'quanteda'

The following objects are masked from 'package:tm':

    as.DocumentTermMatrix, stopwords
... 8 lines ...
    View

The following object is masked from 'package:base':

    sample

Joining, by = "word"
Quitting from lines 210-221 (tidying_casting.Rmd) 
Error: processing vignette 'tidying_casting.Rmd' failed with diagnostics:
is_symbol(wt) is not TRUE
Execution halted
```

## treeplyr (0.1.2)
Maintainer: Josef Uyeda <josef.uyeda@gmail.com>  
Bug reports: https://github.com/uyedaj/treeplyr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘treeplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: filter_.treedata
> ### Title: Function for filtering rows from an object of class 'treedata'
> ### Aliases: filter.grouped_treedata filter.treedata
> ###   filter_.grouped_treedata filter_.treedata
> 
> ### ** Examples
> 
> data(anolis)
> td <- make.treedata(anolis$phy, anolis$dat, name_column=1)
> tdfilter <- filter(td, island=="Cuba", SVL > 3.5)
Error in mutate_impl(.data, dots) : 
  Column `tip.label`: must be a vector, not NULL
Calls: filter ... filter_.treedata -> mutate -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted
```

## turfR (0.8-7)
Maintainer: Jack Horne <jack@jackhorne.net>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘turfR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: turf
> ### Title: TURF Analysis for R
> ### Aliases: turf
> 
> ### ** Examples
> 
>   ##Example 1
>   ##Evaluate all combinations of 3, 4, 5 and 6 items drawn from 10 items.
>   data(turf_ex_data)
>   ex1 <- turf(turf_ex_data, 10, 3:6)
Error in cbind_all(x) : Argument 2: must have names
Calls: turf -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking R code for possible problems ... NOTE
turf: no visible global function definition for ‘read.table’
turf: no visible global function definition for ‘flush.console’
turf.combos: no visible global function definition for ‘combn’
Undefined global functions or variables:
  combn flush.console read.table
Consider adding
  importFrom("utils", "combn", "flush.console", "read.table")
to your NAMESPACE file.
```

## unpivotr (0.1.0)
Maintainer: Duncan Garmonsway <nacnudus@gmail.com>  
Bug reports: https://github.com/nacnudus/unpivotr/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  testthat results ================================================================
  OK: 87 SKIPPED: 0 FAILED: 9
  1. Error: Compass directions N, NNW, W, and WNW work (@test-join_header.R#128) 
  2. Error: Compass directions NNE and WSW work (@test-join_header.R#155) 
  3. Error: Compass directions S, SSE , E and ESE work (@test-join_header.R#178) 
  4. Error: Compass directions SSW and ENE work (@test-join_header.R#205) 
  5. Error: Compass directions ABOVE and LEFT work (@test-join_header.R#228) 
  6. Error: Compass directions BELOW and RIGHT work (@test-join_header.R#251) 
  7. Error: Compass directions ABOVE and LEFT work with boundaries (@test-join_header.R#286) 
  8. Error: Compass directions BELOW and RIGHT work with boundaries (@test-join_header.R#332) 
  9. Error: join_header() works (@test-join_header.R#361) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 56-61 (compass-directions.Rmd) 
Error: processing vignette 'compass-directions.Rmd' failed with diagnostics:
cannot coerce type 'closure' to vector of type 'integer'
Execution halted

```

## unvotes (0.1.0)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/dgrtwo/unvotes/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘unvotes-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: un_roll_calls
> ### Title: Data on each roll call of the United Nations General Assembly
> ### Aliases: un_roll_calls
> ### Keywords: datasets
> 
... 13 lines ...

    intersect, setdiff, setequal, union

> 
> # combine with per-country-vote information
> un_votes %>%
+   inner_join(un_roll_calls, by = "rcid")
Error in inner_join_impl(x, y, by$x, by$y, suffix$x, suffix$y, accept_na_match) : 
  attributes of different sizes
Calls: %>% ... inner_join -> inner_join.tbl_df -> inner_join_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

If you use data from the unvotes package, please cite the following:

Erik Voeten "Data and Analyses of Voting in the UN General Assembly" Routledge Handbook of International Organization, edited by Bob Reinalda (published May 27, 2013)
Warning: code chunks must not depend on the uncached chunk "joined"
Quitting from lines 52-58 (unvotes.Rmd) 
Error: processing vignette 'unvotes.Rmd' failed with diagnostics:
attributes of different sizes
Execution halted

```

## useful (1.2.1)
Maintainer: Jared P. Lander <packages@jaredlander.com>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## valr (0.1.2)
Maintainer: Jay Hesselberth <jay.hesselberth@gmail.com>  
Bug reports: https://github.com/rnabioco/valr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘valr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/valr.Rcheck/00install.out’ for details.
```

## VWPre (0.9.5)
Maintainer: Vincent Porretta <vincentporretta@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Calculating proportions...
There are 320 data points with less than 20 samples per bin.
These can be examined and/or removed using the column 'NSamples'.
These occur at time bin(s): 1180
Subsequent Empirical Logit calculations may be influenced by the number of samples (depending on the number of observations requested).
Sampling rate(s) present in the data are: 50 Hz.

... 8 lines ...
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Quitting from lines 81-87 (SR_Plotting.Rmd) 
Error: processing vignette 'SR_Plotting.Rmd' failed with diagnostics:
object 'talker' not found
Execution halted
```

## WHO (0.2)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: https://www.github.com/expersso/WHO/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [12s/13s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > test_check("WHO")
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  argumemt is not a character vector
  1: get_codes(TRUE) at testthat/tests.R:18
  2: dplyr::bind_rows(df_list)
  3: bind_rows_(x, .id) at /home/muelleki/git/R/dplyr/R/bind.r:132
  
  testthat results ================================================================
  OK: 1485 SKIPPED: 0 FAILED: 1
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  
  Error: testthat unit tests failed
  Execution halted
```

## wordbankr (0.2.0)
Maintainer: Mika Braginsky <mika.br@gmail.com>  
Bug reports: http://github.com/langcog/wordbankr/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Auto-disconnecting MySQLConnection
Quitting from lines 27-31 (wordbankr.Rmd) 
Error: processing vignette 'wordbankr.Rmd' failed with diagnostics:
'sql' is not an exported object from 'namespace:dplyr'
Execution halted


checking dependencies in R code ... NOTE
Missing or unexported object: ‘dplyr::sql’
```

## wrswoR (1.0-1)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/krlmlr/wrswoR/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 622-635 (wrswoR.Rmd) 
Error: processing vignette 'wrswoR.Rmd' failed with diagnostics:

TeX was unable to calculate metrics for the following string
or character:

	36

Common reasons for failure include:
  * The string contains a character which is special to LaTeX unless
    escaped properly, such as % or $.
  * The string makes use of LaTeX commands provided by a package and
    the tikzDevice was not told to load the package.

The contents of the LaTeX log of the aborted run have been printed above,
it may contain additional details as to why the metric calculation failed.
Execution halted

```

## WufooR (0.6.1)
Maintainer: John Malc <cincenko@outlook.com>  
Bug reports: http://github.com/dmpe/wufoor/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘WufooR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: form_entries
> ### Title: Return responses of your form
> ### Aliases: form_entries
> 
> ### ** Examples
> 
> form_entries(formIdentifier = "z5kqx7h1gtvg4g")
Error in `colnames<-`(`*tmp*`, value = c("Entry Id", "First Name", "First",  : 
  'names' attribute [21] must be the same length as the vector [19]
Calls: form_entries -> colnames<-
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > test_check("WufooR")
  1. Error: Form returns entries, with the requested URL (@test-forms.R#14) ------
  'names' attribute [21] must be the same length as the vector [19]
  1: form_entries(formIdentifier = "z5kqx7h1gtvg4g", systemFields = "false", showRequestURL = FALSE) at testthat/test-forms.R:14
  2: `colnames<-`(`*tmp*`, value = c("Entry Id", "First Name", "First", "Last Name", "Email", 
     "Untitled", "Address", "Street Address", "Address Line 2", "City", "State / Province / Region", 
     "Postal / Zip Code", "Country", "Amount", "Time", "Website", "Phone Number", "Date Created", 
     "Created By", "DateUpdated", "Updated By"))
  
  testthat results ================================================================
  OK: 6 SKIPPED: 0 FAILED: 1
  1. Error: Form returns entries, with the requested URL (@test-forms.R#14) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 52-59 (Tutorial.Rmd) 
Error: processing vignette 'Tutorial.Rmd' failed with diagnostics:
'names' attribute [21] must be the same length as the vector [19]
Execution halted

```

## ZeligChoice (0.9-4)
Maintainer: Christopher Gandrud <zelig.zee@gmail.com>  
Bug reports: https://github.com/IQSS/Zelig/issues

0 errors | 1 warning  | 0 notes

```
checking whether package ‘ZeligChoice’ can be installed ... WARNING
Found the following significant warnings:
  Warning: namespace ‘DBI’ is not available and has been replaced
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ZeligChoice.Rcheck/00install.out’ for details.
```

## ZeligEI (0.1-0)
Maintainer: James Honaker <zelig.zee@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking whether package ‘ZeligEI’ can be installed ... WARNING
Found the following significant warnings:
  Warning: namespace ‘DBI’ is not available and has been replaced
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ZeligEI.Rcheck/00install.out’ for details.
```

## Zelig (5.0-17)
Maintainer: Christopher Gandrud <zelig.zee@gmail.com>  
Bug reports: https://github.com/IQSS/Zelig/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [79s/78s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ev
            mean         sd       50%      2.5%     97.5%
  [1,] 0.6990619 0.09467077 0.7081549 0.5028567 0.8603348
  pv
          0    1
  [1,] 0.31 0.69
  fd
             mean        sd        50%      2.5%       97.5%
  [1,] -0.2377065 0.1148547 -0.2380952 -0.466211 -0.01659926
  testthat results ================================================================
  OK: 196 SKIPPED: 1 FAILED: 1
  1. Error: REQUIRE TEST Bayes Diagnostics for factors (@test-bayesdiagnostics.R#19) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 12.1Mb
  sub-directories of 1Mb or more:
    R  11.0Mb
```

