# Setup

## Platform

|setting  |value                        |
|:--------|:----------------------------|
|version  |R version 3.3.3 (2017-03-06) |
|system   |x86_64, linux-gnu            |
|ui       |X11                          |
|language |(EN)                         |
|collate  |en_US.UTF-8                  |
|tz       |Zulu                         |
|date     |2017-03-14                   |

## Packages

|package        |*  |version    |date       |source                           |
|:--------------|:--|:----------|:----------|:--------------------------------|
|assertthat     |   |0.1        |2013-12-06 |cran (@0.1)                      |
|BH             |   |1.62.0-1   |2016-11-19 |cran (@1.62.0-)                  |
|bindrcpp       |   |0.1        |2016-12-11 |cran (@0.1)                      |
|bit64          |   |0.9-5      |2015-07-05 |cran (@0.9-5)                    |
|covr           |   |2.2.2      |2017-01-05 |cran (@2.2.2)                    |
|DBI            |   |0.6        |2017-03-13 |Github (rstats-db/DBI@76d8d65)   |
|dplyr          |   |0.5.0      |2016-06-24 |CRAN (R 3.3.2)                   |
|dtplyr         |   |0.0.1      |2016-06-27 |cran (@0.0.1)                    |
|ggplot2        |   |2.2.1      |2016-12-30 |cran (@2.2.1)                    |
|glue           |   |0.0.0.9000 |2017-03-14 |Github (tidyverse/glue@099b283)  |
|hms            |   |0.3        |2016-11-22 |cran (@0.3)                      |
|knitr          |   |1.15.1     |2016-11-22 |cran (@1.15.1)                   |
|Lahman         |   |5.0-0      |2016-08-27 |cran (@5.0-0)                    |
|magrittr       |   |1.5        |2014-11-22 |CRAN (R 3.3.1)                   |
|microbenchmark |   |1.4-2.1    |2015-11-25 |cran (@1.4-2.1)                  |
|nycflights13   |   |0.2.2      |2017-01-27 |cran (@0.2.2)                    |
|plogr          |   |0.1-1      |2016-09-24 |cran (@0.1-1)                    |
|R6             |   |2.2.0      |2016-10-05 |cran (@2.2.0)                    |
|Rcpp           |   |0.12.9.4   |2017-03-08 |Github (RcppCore/Rcpp@0566d7c)   |
|rlang          |   |0.0.0.9000 |2017-03-14 |Github (hadley/rlang@2376dd6)    |
|rmarkdown      |   |1.3        |2016-12-21 |cran (@1.3)                      |
|RMySQL         |   |0.10.9     |2016-05-08 |cran (@0.10.9)                   |
|RPostgreSQL    |   |0.4-1      |2016-05-08 |cran (@0.4-1)                    |
|RSQLite        |   |1.1-2      |2017-01-08 |cran (@1.1-2)                    |
|testthat       |   |1.0.2.9000 |2017-02-27 |Github (hadley/testthat@b72a228) |
|tibble         |   |1.2        |2016-08-26 |cran (@1.2)                      |

# Check results

517 packages

|package                |version   | errors| warnings| notes|
|:----------------------|:---------|------:|--------:|-----:|
|abjutils               |0.0.1     |      0|        0|     0|
|ACDm                   |1.0.4     |      0|        0|     0|
|adegenet               |2.0.1     |      0|        0|     0|
|admixturegraph         |1.0.2     |      0|        0|     0|
|ADPclust               |0.7       |      0|        0|     0|
|aemo                   |0.2.0     |      0|        0|     0|
|afex                   |0.16-1    |      0|        0|     0|
|AIG                    |0.1.2     |      0|        0|     0|
|alakazam               |0.2.5     |      1|        1|     0|
|alluvial               |0.1-2     |      0|        0|     0|
|alphabetr              |0.2.2     |      0|        0|     0|
|ameco                  |0.2.6     |      1|        0|     1|
|anametrix              |2.0.0     |      1|        0|     0|
|ANLP                   |1.3       |      0|        0|     0|
|apa                    |0.2.0     |      0|        0|     0|
|archivist              |2.1.2     |      0|        0|     2|
|ARTool                 |0.10.4    |      0|        0|     0|
|assertive.types        |0.0-3     |      0|        0|     0|
|assertr                |1.0.2     |      1|        0|     0|
|automagic              |0.3       |      0|        0|     0|
|AutoModel              |0.4.9     |      0|        0|     0|
|backtestGraphics       |0.1.6     |      1|        0|     0|
|bannerCommenter        |0.1.0     |      0|        0|     0|
|bayesGDS               |0.6.2     |      0|        0|     0|
|bayesplot              |1.1.0     |      0|        1|     0|
|bib2df                 |0.1.1     |      0|        0|     0|
|bibliometrix           |1.4       |      0|        1|     0|
|bigrquery              |0.3.0     |      0|        0|     1|
|binford                |0.1.0     |      0|        0|     0|
|binomen                |0.1.0     |      0|        0|     0|
|bioinactivation        |1.1.5     |      2|        1|     0|
|biomartr               |0.3.0     |      0|        0|     0|
|bioOED                 |0.1.1     |      1|        0|     0|
|bkmr                   |0.1.0     |      0|        0|     0|
|blkbox                 |1.0       |      0|        0|     2|
|blscrapeR              |2.1.2     |      1|        0|     0|
|bmlm                   |1.2.10    |      0|        0|     1|
|bodenmiller            |0.1       |      0|        0|     1|
|bootnet                |0.4       |      0|        0|     0|
|bossMaps               |0.1.0     |      2|        1|     0|
|boxr                   |0.3.4     |      0|        0|     0|
|braQCA                 |0.9.9.6   |      0|        0|     0|
|broom                  |0.4.2     |      1|        1|     0|
|bsam                   |1.1.1     |      0|        0|     0|
|CARBayesST             |2.4       |      0|        0|     0|
|carpenter              |0.2.0     |      2|        1|     0|
|causaldrf              |0.3       |      0|        0|     0|
|cbsodataR              |0.2.1     |      0|        0|     0|
|cdcfluview             |0.5.1     |      0|        0|     0|
|censusr                |0.0.2     |      0|        0|     0|
|choroplethr            |3.5.3     |      0|        0|     0|
|chromer                |0.1       |      0|        0|     2|
|chunked                |0.3       |      2|        0|     0|
|cleanNLP               |0.24      |      0|        0|     0|
|climbeR                |0.0.1     |      0|        0|     0|
|clustRcompaR           |0.1.0     |      0|        0|     0|
|clustrd                |1.1.0     |      1|        0|     0|
|codingMatrices         |0.2.2     |      0|        1|     0|
|codyn                  |1.1.0     |      0|        0|     0|
|cofeatureR             |1.0.1     |      0|        0|     0|
|CollapsABEL            |0.10.11   |      0|        0|     0|
|cometExactTest         |0.1.3     |      0|        1|     0|
|compareDF              |1.1.0     |      2|        0|     0|
|condformat             |0.5.0     |      2|        1|     0|
|coreSim                |0.2.3     |      0|        0|     0|
|corrr                  |0.2.1     |      1|        1|     0|
|countytimezones        |1.0.0     |      1|        1|     0|
|countyweather          |0.1.0     |      0|        0|     0|
|cpr                    |0.2.3     |      1|        1|     0|
|crawl                  |2.0.1     |      0|        1|     1|
|cricketr               |0.0.14    |      0|        0|     0|
|d3r                    |0.6.2     |      0|        0|     1|
|darksky                |1.0.0     |      0|        0|     0|
|datacheckr             |0.1.2     |      0|        0|     0|
|DataCombine            |0.2.21    |      0|        0|     0|
|datadr                 |0.8.6     |      0|        0|     1|
|dataonderivatives      |0.2.1     |      0|        0|     0|
|dataRetrieval          |2.6.3     |      0|        0|     0|
|datastepr              |0.0.2     |      0|        0|     0|
|dat                    |0.1.0     |      1|        0|     0|
|dbfaker                |0.1.0     |      0|        0|     0|
|ddpcr                  |1.6       |      1|        0|     0|
|decoder                |1.1.12    |      0|        1|     0|
|DeLorean               |1.2.4     |      0|        1|     1|
|denovolyzeR            |0.2.0     |      0|        0|     0|
|DepthProc              |1.0.7     |      0|        0|     1|
|describer              |0.2.0     |      1|        0|     0|
|descriptr              |0.1.0     |      0|        0|     0|
|dggridR                |0.1.11    |      1|        0|     0|
|DiagrammeR             |0.9.0     |      2|        0|     1|
|diffrprojects          |0.1.14    |      0|        0|     0|
|diffrprojectswidget    |0.1.5     |      0|        0|     0|
|DisimForMixed          |0.2       |      0|        0|     0|
|DiversityOccupancy     |1.0.6     |      0|        0|     0|
|dmutate                |0.1.1     |      0|        0|     0|
|DMwR2                  |0.0.2     |      0|        0|     0|
|doctr                  |0.2.0     |      1|        1|     0|
|docxtools              |0.1.1     |      0|        0|     0|
|docxtractr             |0.2.0     |      0|        0|     0|
|dotwhisker             |0.2.3     |      1|        1|     0|
|downsize               |0.2.1     |      0|        0|     0|
|dtplyr                 |0.0.1     |      1|        0|     0|
|eAnalytics             |0.1.3     |      0|        0|     0|
|easyformatr            |0.1.2     |      0|        0|     0|
|ecb                    |0.2       |      0|        0|     0|
|ecoengine              |1.10.0    |      0|        0|     0|
|edeaR                  |0.4.7     |      0|        0|     0|
|eechidna               |0.1       |      0|        1|     0|
|eemR                   |0.1.4     |      0|        0|     0|
|EFDR                   |0.1.1     |      0|        0|     1|
|efreadr                |0.1.1     |      0|        1|     0|
|electionsBR            |0.2.0     |      0|        0|     0|
|emil                   |2.2.6     |      2|        0|     0|
|emuR                   |0.2.2     |      0|        0|     0|
|engsoccerdata          |0.1.5     |      0|        0|     0|
|epidata                |0.1.0     |      0|        0|     0|
|episheet               |0.2.0     |      0|        0|     0|
|estatapi               |0.3.0     |      0|        0|     0|
|etl                    |0.3.5     |      2|        0|     0|
|europepmc              |0.1.3     |      0|        0|     0|
|europop                |0.3.1     |      0|        0|     0|
|eurostat               |2.2.43    |      0|        0|     0|
|evaluator              |0.1.0     |      0|        0|     0|
|explor                 |0.3.1     |      0|        0|     0|
|extdplyr               |0.1.4     |      1|        0|     0|
|eyelinker              |0.1       |      0|        1|     0|
|eyetrackingR           |0.1.6     |      2|        0|     0|
|ezec                   |1.0.1     |      0|        0|     0|
|ezsummary              |0.2.1     |      2|        1|     0|
|fbar                   |0.1.23    |      1|        0|     1|
|feather                |0.3.1     |      0|        0|     0|
|FedData                |2.4.5     |      0|        0|     0|
|feedeR                 |0.0.7     |      0|        0|     0|
|fiftystater            |1.0.1     |      0|        0|     0|
|filesstrings           |0.3.1     |      0|        0|     0|
|finreportr             |1.0.1     |      0|        0|     0|
|fitcoach               |1.0       |      1|        0|     0|
|fivethirtyeight        |0.1.0     |      0|        0|     0|
|flora                  |0.2.8     |      0|        0|     0|
|foghorn                |0.4.2     |      0|        0|     0|
|forestmodel            |0.4.0     |      1|        0|     0|
|fractional             |0.1.3     |      0|        0|     0|
|freqweights            |1.0.2     |      0|        0|     1|
|FRK                    |0.1.1     |      1|        2|     2|
|FSA                    |0.8.12    |      0|        0|     1|
|FSelectorRcpp          |0.1.2     |      0|        0|     2|
|fueleconomy            |0.1       |      0|        0|     0|
|funModeling            |1.6.1     |      1|        0|     0|
|funrar                 |1.0.2     |      0|        0|     0|
|futureheatwaves        |1.0.3     |      1|        1|     0|
|fuzzyjoin              |0.1.2     |      0|        0|     0|
|GADMTools              |2.0-1     |      1|        0|     0|
|gaiah                  |0.0.2     |      0|        0|     0|
|gapminder              |0.2.0     |      0|        0|     0|
|GenCAT                 |1.0.3     |      1|        1|     0|
|gender                 |0.5.1     |      0|        0|     1|
|geoknife               |1.5.2     |      0|        1|     0|
|geomnet                |0.2.0     |      0|        0|     0|
|geoparser              |0.1.0     |      0|        0|     0|
|GerminaR               |1.1       |      0|        0|     0|
|getCRUCLdata           |0.1.4     |      0|        0|     0|
|GetHFData              |1.2.4     |      0|        0|     0|
|ggalt                  |0.4.0     |      0|        0|     0|
|ggCompNet              |0.1.0     |      0|        1|     1|
|ggenealogy             |0.3.0     |      0|        0|     0|
|ggforce                |0.1.1     |      1|        0|     0|
|ggfortify              |0.4.1     |      2|        1|     1|
|ggguitar               |0.1.1     |      0|        0|     0|
|gglogo                 |0.1.3     |      0|        0|     0|
|ggmap                  |2.6.1     |      0|        0|     0|
|ggmcmc                 |1.1       |      0|        0|     0|
|ggmosaic               |0.1.2     |      0|        0|     0|
|ggpmisc                |0.2.14    |      0|        0|     0|
|ggRandomForests        |2.0.1     |      0|        0|     0|
|ggraph                 |1.0.0     |      1|        0|     1|
|ggraptR                |0.1       |      0|        0|     0|
|ggspectra              |0.2.0     |      0|        0|     0|
|ggswissmaps            |0.1.1     |      0|        0|     0|
|ggvis                  |0.4.3     |      2|        0|     0|
|gistr                  |0.3.6     |      0|        0|     0|
|gitgadget              |0.2.1     |      0|        0|     0|
|gitlabr                |0.7       |      0|        0|     0|
|glycanr                |0.3.0     |      0|        0|     0|
|gogamer                |0.4.3     |      0|        0|     0|
|goldi                  |1.0.0     |      1|        0|     0|
|googleAnalyticsR       |0.3.0     |      0|        0|     0|
|googlesheets           |0.2.1     |      0|        0|     0|
|graphicalVAR           |0.1.4     |      0|        0|     0|
|graphTweets            |0.3.2     |      0|        0|     0|
|grattan                |1.4.1.2   |      2|        1|     1|
|Greg                   |1.2       |      0|        0|     0|
|groupdata2             |0.1.0     |      0|        1|     0|
|growthcurver           |0.2.1     |      0|        0|     0|
|GSODR                  |1.0.1     |      0|        0|     1|
|gunsales               |0.1.2     |      0|        0|     0|
|gutenbergr             |0.1.2     |      1|        0|     0|
|gwdegree               |0.1.1     |      0|        0|     0|
|harrietr               |0.2.2     |      1|        0|     0|
|hdr                    |0.1       |      0|        0|     0|
|heemod                 |0.8.0     |      2|        1|     0|
|highcharter            |0.5.0     |      1|        1|     1|
|highlightHTML          |0.1.1     |      0|        0|     0|
|hrbrthemes             |0.1.0     |      0|        0|     0|
|HTSSIP                 |1.0.3     |      1|        0|     0|
|hurricaneexposure      |0.0.1     |      0|        0|     1|
|huxtable               |0.1.0     |      1|        1|     0|
|HydeNet                |0.10.5    |      2|        1|     0|
|hydrostats             |0.2.5     |      0|        0|     0|
|hypothesisr            |0.1.1     |      0|        0|     0|
|IAT                    |0.3       |      1|        0|     0|
|IATscores              |0.1-2     |      1|        0|     0|
|idbr                   |0.2       |      0|        0|     0|
|IDmining               |1.0.0     |      0|        0|     0|
|imager                 |0.40.1    |      0|        0|     1|
|imfr                   |0.1.3     |      0|        0|     0|
|IMP                    |1.1       |      0|        0|     0|
|imputeTestbench        |3.0.0     |      0|        0|     0|
|incadata               |0.5.3     |      1|        0|     0|
|IncucyteDRC            |0.5.4     |      0|        0|     0|
|inferr                 |0.1.0     |      0|        1|     0|
|InformativeCensoring   |0.3.4     |      0|        0|     0|
|infuser                |0.2.5     |      0|        0|     0|
|inlmisc                |0.2.4     |      1|        0|     0|
|internetarchive        |0.1.6     |      0|        0|     0|
|interplot              |0.1.5     |      0|        0|     0|
|ipft                   |0.2.8     |      0|        1|     0|
|IRISMustangMetrics     |2.0.4     |      0|        0|     0|
|JacobiEigen            |0.2-2     |      0|        0|     0|
|janeaustenr            |0.1.4     |      0|        0|     0|
|janitor                |0.2.1     |      0|        0|     0|
|jpmesh                 |0.3.0     |      1|        0|     0|
|kntnr                  |0.4.0     |      0|        0|     0|
|kokudosuuchi           |0.2.0     |      1|        0|     0|
|KraljicMatrix          |0.1.1     |      0|        0|     0|
|labelled               |1.0.0     |      0|        0|     2|
|Lahman                 |5.0-0     |      0|        1|     1|
|laketemps              |0.5.1     |      0|        0|     0|
|LBSPR                  |0.1.0     |      0|        0|     0|
|LendingClub            |1.0.2     |      0|        0|     0|
|lexRankr               |0.4.0     |      0|        0|     0|
|livechatR              |0.1.0     |      0|        0|     0|
|lmeresampler           |0.1.0     |      0|        0|     0|
|LocFDRPois             |1.0.0     |      0|        0|     1|
|longurl                |0.3.0     |      0|        0|     0|
|lookupTable            |0.1       |      0|        0|     0|
|loopr                  |1.0.1     |      0|        0|     1|
|lplyr                  |0.1.6     |      1|        1|     0|
|lucid                  |1.4       |      0|        0|     0|
|lvnet                  |0.3       |      0|        0|     0|
|macleish               |0.3.0     |      0|        0|     0|
|makeFlow               |1.0.2     |      0|        0|     0|
|manifestoR             |1.2.3     |      0|        0|     0|
|mason                  |0.2.5     |      1|        0|     0|
|MazamaSpatialUtils     |0.4.8     |      1|        0|     0|
|mdsr                   |0.1.3     |      1|        0|     1|
|medicare               |0.2.0     |      0|        1|     0|
|merTools               |0.3.0     |      1|        0|     0|
|metacoder              |0.1.2     |      1|        1|     0|
|MetaComp               |1.0.1     |      0|        0|     0|
|metricsgraphics        |0.9.0     |      0|        0|     0|
|metricTester           |1.3.2     |      0|        1|     0|
|mglR                   |0.1.0     |      0|        0|     0|
|MIAmaxent              |0.4.0     |      0|        0|     0|
|mixOmics               |6.1.2     |      0|        0|     0|
|mixpack                |0.3.6     |      0|        0|     0|
|mlVAR                  |0.3.2     |      0|        0|     0|
|modelr                 |0.1.0     |      0|        0|     0|
|Momocs                 |1.1.0     |      1|        1|     0|
|MonetDBLite            |0.3.1     |      0|        0|     1|
|monkeylearn            |0.1.1     |      0|        0|     0|
|morse                  |2.2.0     |      0|        0|     0|
|mosaicData             |0.14.0    |      0|        0|     0|
|mosaic                 |0.14.4    |      0|        0|     2|
|mousetrap              |3.0.0     |      0|        0|     0|
|mplot                  |0.7.9     |      0|        0|     0|
|mpoly                  |1.0.3     |      0|        0|     0|
|mrgsolve               |0.8.2     |      2|        0|     1|
|mscstexta4r            |0.1.2     |      0|        0|     0|
|mtconnectR             |1.0.1     |      2|        1|     0|
|mudata                 |0.1       |      1|        0|     0|
|muir                   |0.1.0     |      0|        0|     0|
|myTAI                  |0.4.0     |      0|        1|     1|
|nasadata               |0.9.0     |      0|        0|     0|
|ncappc                 |0.2.1.1   |      0|        0|     0|
|ncar                   |0.3.1     |      0|        0|     0|
|NestedCategBayesImpute |1.0.0     |      0|        0|     0|
|networkreporting       |0.1.1     |      0|        0|     0|
|NetworkRiskMeasures    |0.1.1     |      0|        0|     0|
|neurobase              |1.13.2    |      0|        0|     0|
|neuropsychology        |0.3.0     |      0|        0|     0|
|NFP                    |0.99.2    |      1|        0|     0|
|nlstimedist            |1.0.4     |      2|        1|     0|
|notifyme               |0.3.0     |      0|        0|     0|
|NPC                    |1.1.0     |      0|        0|     0|
|nullabor               |0.3.1     |      0|        0|     1|
|nycflights13           |0.2.2     |      0|        0|     1|
|nzelect                |0.3.0     |      0|        0|     0|
|observer               |0.1.2     |      0|        0|     1|
|officer                |0.1.1     |      2|        1|     0|
|openadds               |0.2.0     |      0|        0|     0|
|openair                |2.0-0     |      0|        1|     0|
|opencage               |0.1.0     |      0|        0|     0|
|packagetrackr          |0.1.1     |      0|        0|     0|
|PAC                    |1.0.7     |      0|        0|     0|
|padr                   |0.2.1     |      1|        0|     0|
|parsemsf               |0.1.0     |      2|        1|     0|
|patternplot            |0.1       |      0|        0|     1|
|pdp                    |0.5.2     |      0|        0|     1|
|peptider               |0.2.2     |      0|        0|     0|
|performanceEstimation  |1.1.0     |      0|        0|     0|
|petrinetR              |0.1.0     |      0|        0|     0|
|photobiologyInOut      |0.4.12    |      1|        1|     0|
|photobiology           |0.9.13    |      0|        0|     0|
|phylopath              |0.2.2     |      0|        0|     0|
|pinnacle.API           |1.92      |      0|        0|     0|
|pitchRx                |1.8.2     |      0|        0|     1|
|pixiedust              |0.7.4     |      2|        1|     0|
|PKNCA                  |0.8.1     |      0|        0|     0|
|Plasmidprofiler        |0.1.6     |      0|        0|     0|
|plater                 |1.0.0     |      1|        0|     0|
|platetools             |0.0.2     |      0|        0|     0|
|pleiades               |0.1.0     |      0|        0|     0|
|plotly                 |4.5.6     |      2|        0|     0|
|plotrr                 |0.2.0     |      0|        0|     0|
|pmc                    |1.0.2     |      0|        0|     0|
|PogromcyDanych         |1.5       |      0|        0|     1|
|poio                   |0.0-3     |      0|        0|     0|
|pollen                 |0.50.0    |      0|        0|     0|
|PopED                  |0.3.2     |      0|        0|     0|
|poplite                |0.99.17.3 |      2|        1|     0|
|poppr                  |2.3.0     |      0|        0|     0|
|prcr                   |0.1.1     |      0|        0|     0|
|prepdat                |1.0.8     |      0|        0|     0|
|pRF                    |1.2       |      0|        0|     0|
|prophet                |0.1       |      1|        1|     1|
|ptstem                 |0.0.3     |      0|        0|     1|
|purrr                  |0.2.2     |      1|        0|     0|
|PWFSLSmoke             |0.99.9    |      1|        0|     0|
|qdap                   |2.2.5     |      0|        0|     0|
|quadmesh               |0.1.0     |      0|        0|     0|
|qualvar                |0.1.0     |      0|        1|     0|
|queuecomputer          |0.6.1     |      0|        0|     0|
|quickpsy               |0.1.4     |      0|        0|     0|
|qwraps2                |0.2.4     |      1|        0|     0|
|r2glmm                 |0.1.1     |      0|        0|     0|
|R6Frame                |0.1.0     |      1|        0|     0|
|radiant.basics         |0.6.0     |      0|        1|     0|
|radiant.data           |0.6.0     |      2|        0|     0|
|radiant.design         |0.6.0     |      0|        1|     0|
|radiant.model          |0.6.0     |      2|        0|     0|
|radiant.multivariate   |0.6.0     |      2|        0|     0|
|randomizr              |0.4.1     |      0|        0|     0|
|raptr                  |0.0.3     |      1|        0|     0|
|rattle                 |4.1.0     |      0|        1|     3|
|raw                    |0.1.4     |      0|        0|     0|
|rbgm                   |0.0.4     |      0|        1|     0|
|rbison                 |0.5.0     |      0|        0|     0|
|rccmisc                |0.3.7     |      0|        0|     0|
|rchess                 |0.1       |      0|        0|     0|
|rcicr                  |0.3.4.1   |      0|        0|     0|
|rclinicaltrials        |1.4.7     |      0|        0|     0|
|RCMIP5                 |1.2.0     |      0|        0|     0|
|rcrossref              |0.6.0     |      0|        0|     0|
|RDML                   |0.9-5     |      0|        0|     0|
|rdrop2                 |0.7.0     |      0|        0|     0|
|readODS                |1.6.4     |      0|        0|     0|
|rebird                 |0.3.0     |      0|        0|     0|
|recexcavAAR            |0.3.0     |      0|        1|     1|
|REDCapR                |0.9.7     |      0|        0|     0|
|refund.shiny           |0.3.0     |      0|        0|     0|
|replyr                 |0.2.3     |      2|        1|     0|
|rerddap                |0.3.4     |      0|        0|     0|
|resumer                |0.0.3     |      0|        0|     0|
|reval                  |2.0.0     |      0|        0|     0|
|rex                    |1.1.1     |      0|        0|     0|
|rfishbase              |2.1.1     |      1|        1|     0|
|rgho                   |1.0.1     |      0|        0|     0|
|rivr                   |1.2       |      0|        0|     0|
|RmarineHeatWaves       |0.15.1    |      0|        0|     0|
|rmcfs                  |1.2.4     |      0|        1|     2|
|rmdHelpers             |1.2       |      0|        0|     0|
|RNeXML                 |2.0.7     |      2|        1|     1|
|RNHANES                |1.1.0     |      0|        0|     0|
|rnoaa                  |0.6.6     |      0|        0|     0|
|robotstxt              |0.3.2     |      0|        0|     0|
|ropenaq                |0.1.3     |      0|        0|     0|
|ROpenFIGI              |0.2.8     |      0|        0|     0|
|rpcdsearch             |1.0       |      0|        0|     0|
|rpdo                   |0.2.1     |      0|        0|     0|
|rpivotTable            |0.1.5.20  |      0|        0|     0|
|rplos                  |0.6.4     |      0|        0|     0|
|rPref                  |1.2       |      0|        0|     1|
|RPresto                |1.2.1     |      0|        0|     0|
|rprev                  |0.2.3     |      0|        1|     0|
|rrr                    |1.0.0     |      0|        0|     0|
|rscorecard             |0.3.5     |      0|        0|     0|
|rslp                   |0.1.0     |      0|        0|     0|
|rsparkling             |0.1.1     |      0|        0|     0|
|RSSL                   |0.6.1     |      0|        0|     1|
|rtable                 |0.1.5     |      0|        0|     0|
|rtdists                |0.6-6     |      0|        0|     0|
|rtide                  |0.0.3     |      1|        0|     0|
|rtimes                 |0.3.0     |      0|        0|     0|
|rtrends                |0.1.0     |      0|        0|     0|
|RtutoR                 |0.3       |      0|        0|     0|
|rvertnet               |0.5.0     |      0|        0|     0|
|rwunderground          |0.1.0     |      1|        0|     0|
|saeSim                 |0.8.0     |      2|        1|     0|
|scholar                |0.1.4     |      0|        0|     0|
|SEERaBomb              |2016.2    |      0|        0|     0|
|sejmRP                 |1.3       |      0|        0|     0|
|sfc                    |0.1.0     |      0|        1|     0|
|sf                     |0.3-4     |      2|        0|     1|
|shazam                 |0.1.4     |      1|        1|     0|
|shinyAce               |0.2.1     |      0|        0|     1|
|ShinyTester            |0.1.0     |      0|        0|     0|
|SimDesign              |1.5       |      0|        0|     1|
|simmer.plot            |0.1.8     |      2|        1|     0|
|simmer                 |3.6.0     |      0|        0|     1|
|simPH                  |1.3.9     |      0|        0|     0|
|simputation            |0.2.1     |      0|        0|     0|
|sjmisc                 |2.3.1     |      1|        1|     0|
|sjPlot                 |2.3.1     |      1|        1|     0|
|sjstats                |0.9.0     |      1|        0|     1|
|slackr                 |1.4.2     |      0|        0|     0|
|solrium                |0.4.0     |      0|        0|     0|
|sophisthse             |0.7.0     |      0|        0|     0|
|sorvi                  |0.7.26    |      0|        0|     1|
|sourceR                |1.0.0     |      0|        0|     0|
|sp500SlidingWindow     |0.1.0     |      0|        0|     0|
|SpaCCr                 |0.1.0     |      0|        0|     0|
|SpaDES                 |1.3.1     |      1|        2|     2|
|sparklyr               |0.5.3     |      0|        1|     0|
|sparsediscrim          |0.2.3     |      0|        0|     0|
|sparseHessianFD        |0.3.2     |      0|        1|     0|
|SpatialEpiApp          |0.1       |      0|        0|     1|
|spbabel                |0.4.5     |      1|        0|     0|
|spdplyr                |0.1.2     |      2|        1|     0|
|spellcheckr            |0.1.2     |      0|        0|     0|
|sqlscore               |0.1.1     |      0|        0|     0|
|srvyr                  |0.2.0     |      2|        1|     0|
|ss3sim                 |0.9.3     |      1|        1|     0|
|statar                 |0.6.2     |      2|        0|     0|
|stationaRy             |0.4.1     |      0|        0|     1|
|statip                 |0.1.4     |      0|        0|     1|
|statisticalModeling    |0.3.0     |      0|        0|     0|
|sticky                 |0.4.0     |      0|        0|     0|
|stormwindmodel         |0.1.0     |      0|        1|     0|
|stplanr                |0.1.7-3   |      1|        0|     1|
|superheat              |0.1.0     |      0|        0|     0|
|surveybootstrap        |0.0.1     |      0|        0|     0|
|SWMPr                  |2.2.0     |      0|        0|     0|
|taber                  |0.1.0     |      0|        0|     0|
|tadaatoolbox           |0.10.0    |      1|        1|     0|
|Tcomp                  |1.0.0     |      0|        0|     0|
|tcR                    |2.2.1.11  |      0|        0|     3|
|TeachBayes             |0.1.0     |      0|        0|     0|
|tempcyclesdata         |1.0.1     |      0|        0|     1|
|testassay              |0.1.0     |      0|        0|     0|
|texmexseq              |0.3       |      1|        0|     0|
|textmining             |0.0.1     |      0|        1|     0|
|textreuse              |0.1.4     |      2|        1|     0|
|textstem               |0.0.1     |      0|        0|     0|
|TH.data                |1.0-8     |      0|        0|     1|
|tictactoe              |0.2.0     |      0|        0|     0|
|tidyjson               |0.2.1     |      1|        0|     1|
|tidyquant              |0.4.0     |      2|        1|     0|
|tidyr                  |0.6.1     |      1|        0|     0|
|tidyRSS                |1.0.1     |      0|        0|     0|
|tidytext               |0.1.2     |      1|        1|     0|
|tidyverse              |1.1.1     |      0|        0|     0|
|tigger                 |0.2.8     |      1|        1|     0|
|tigris                 |0.3.3     |      1|        0|     0|
|timelineS              |0.1.1     |      0|        0|     0|
|titanic                |0.1.0     |      0|        0|     0|
|tmap                   |1.8-1     |      1|        0|     0|
|Tmisc                  |0.1.12    |      0|        0|     0|
|traits                 |0.2.0     |      0|        0|     0|
|treeplyr               |0.1.2     |      1|        0|     0|
|trelliscope            |0.9.7     |      0|        0|     0|
|trelloR                |0.1.0     |      0|        0|     0|
|turfR                  |0.8-7     |      1|        0|     1|
|uaparserjs             |0.1.0     |      0|        0|     0|
|ubeR                   |0.1.4     |      0|        0|     0|
|unpivotr               |0.1.0     |      2|        1|     0|
|unvotes                |0.1.0     |      0|        0|     0|
|useful                 |1.2.1     |      2|        0|     0|
|vaersNDvax             |1.0.4     |      0|        0|     1|
|vaersvax               |1.0.4     |      0|        0|     1|
|valr                   |0.1.1     |      1|        0|     0|
|vcfR                   |1.4.0     |      1|        0|     1|
|vdmR                   |0.2.3     |      0|        0|     0|
|VIM                    |4.6.0     |      0|        0|     1|
|vkR                    |0.1       |      0|        0|     0|
|vqtl                   |1.0       |      0|        0|     0|
|vtreat                 |0.5.30    |      0|        0|     0|
|VWPre                  |0.9.5     |      0|        1|     0|
|wakefield              |0.3.0     |      0|        0|     0|
|wallace                |0.6.3     |      0|        0|     0|
|wand                   |0.2.0     |      0|        0|     0|
|wec                    |0.4       |      0|        0|     0|
|wfindr                 |0.1.0     |      0|        0|     0|
|WHO                    |0.2       |      1|        0|     0|
|WikidataQueryServiceR  |0.1.0     |      0|        0|     0|
|wikipediatrend         |1.1.10    |      0|        0|     1|
|wordbankr              |0.2.0     |      0|        1|     0|
|worldmet               |0.7.5     |      0|        0|     0|
|wrangle                |0.3       |      0|        0|     0|
|wrswoR.benchmark       |0.1-1     |      0|        0|     0|
|wrswoR                 |1.0-1     |      0|        1|     0|
|WRTDStidal             |1.0.1     |      0|        0|     0|
|WufooR                 |0.6.1     |      0|        0|     0|
|yorkr                  |0.0.7     |      0|        0|     0|
|ZeligChoice            |0.9-4     |      0|        0|     0|
|ZeligEI                |0.1-0     |      0|        0|     0|
|Zelig                  |5.0-17    |      1|        0|     1|
|ztype                  |0.1.0     |      0|        0|     0|

## abjutils (0.0.1)
Maintainer: Fernando Correa <fcorrea@abj.org.br>

0 errors | 0 warnings | 0 notes

## ACDm (1.0.4)
Maintainer: Markus Belfrage <markus.belfrage@gmail.com>

0 errors | 0 warnings | 0 notes

## adegenet (2.0.1)
Maintainer: Thibaut Jombart <thibautjombart@gmail.com>

0 errors | 0 warnings | 0 notes

## admixturegraph (1.0.2)
Maintainer: Thomas Mailund <mailund@birc.au.dk>  
Bug reports: https://github.com/mailund/admixture_graph/issues

0 errors | 0 warnings | 0 notes

## ADPclust (0.7)
Maintainer: Yifan (Ethan) Xu <ethan.yifanxu@gmail.com>  
Bug reports: https://github.com/ethanyxu/ADPclust/issues

0 errors | 0 warnings | 0 notes

## aemo (0.2.0)
Maintainer: Imanuel Costigan <i.costigan@me.com>

0 errors | 0 warnings | 0 notes

## afex (0.16-1)
Maintainer: Henrik Singmann <singmann+afex@gmail.com>

0 errors | 0 warnings | 0 notes

## AIG (0.1.2)
Maintainer: Bao Sheng Loe (Aiden) <bsl28@cam.ac.uk>

0 errors | 0 warnings | 0 notes

## alakazam (0.2.5)
Maintainer: Jason Vander Heiden <jason.vanderheiden@yale.edu>  
Bug reports: https://bitbucket.org/kleinstein/alakazam/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘alakazam-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotSubtrees
> ### Title: Plots subtree statistics for multiple trees
> ### Aliases: plotSubtrees
> 
> ### ** Examples
> 
> # Define example tree set
> graphs <- ExampleTrees[1-10]
> 
> # Plot violins of outdegree by sample
> plotSubtrees(graphs, "SAMPLE", "out", main_title="Node outdegree", 
+              style="v")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted
** found \donttest examples: check also with --run-donttest

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'igraph'

... 8 lines ...

The following object is masked from 'package:base':

    union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 171-176 (Topology-Vignette.Rmd) 
Error: processing vignette 'Topology-Vignette.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted
```

## alluvial (0.1-2)
Maintainer: Michal Bojanowski <michal2992@gmail.com>  
Bug reports: https://github.com/mbojan/alluvial/issues

0 errors | 0 warnings | 0 notes

## alphabetr (0.2.2)
Maintainer: Edward Lee <email.edwardlee@gmail.com>  
Bug reports: http://github.com/edwardslee/alphabetr

0 errors | 0 warnings | 0 notes

## ameco (0.2.6)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: http://github.com/expersso/ameco/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(ameco)
  > 
  > test_check("ameco")
  1. Failure: Test that current version is still latest version (@tests.R#16) ----
  `last_update` not equal to as.Date("2017-02-13").
  'is.NA' value mismatch: 0 in current 1 in target
  
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Failure: Test that current version is still latest version (@tests.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 15.7Mb
  sub-directories of 1Mb or more:
    data  15.6Mb
```

## anametrix (2.0.0)
Maintainer: Caroline Cochet-Escartin <caroline.escartin@ensighten.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘anametrix-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: authenticate
> ### Title: authenticate
> ### Aliases: authenticate
> 
> ### ** Examples
> 
> username <- "demo.user@demoanametrix.com"
> password <- "password"
> auth     <- authenticate("https://demo.anametrix.com/api", username, password)
Failed to authenticate with Anametrix API 
Error: Connection timed out after 100000 milliseconds 
Error in xmlTreeParse(resp) : object 'resp' not found
Calls: authenticate -> xmlTreeParse
Execution halted
```

## ANLP (1.3)
Maintainer: Achal Shah <achalshah20@gmail.com>

0 errors | 0 warnings | 0 notes

## apa (0.2.0)
Maintainer: Daniel Gromer <dgromer@mailbox.org>  
Bug reports: https://github.com/dgromer/apa/issues

0 errors | 0 warnings | 0 notes

## archivist (2.1.2)
Maintainer: Przemyslaw Biecek <przemyslaw.biecek@gmail.com>  
Bug reports: https://github.com/pbiecek/archivist/issues

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘archivist.github’

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘archivist.github’
```

## ARTool (0.10.4)
Maintainer: Matthew Kay <mjskay@umich.edu>  
Bug reports: https://github.com/mjskay/ARTool/issues/new

0 errors | 0 warnings | 0 notes

## assertive.types (0.0-3)
Maintainer: Richard Cotton <richierocks@gmail.com>  
Bug reports: https://bitbucket.org/richierocks/assertive.types/issues

0 errors | 0 warnings | 0 notes

## assertr (1.0.2)
Maintainer: Tony Fischetti <tony.fischetti@gmail.com>  
Bug reports: https://github.com/ropenscilabs/assertr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  testthat results ================================================================
  OK: 284 SKIPPED: 0 FAILED: 8
  1. Failure: insist raises error if verification fails (@test-assertions.R#302) 
  2. Failure: insist raises error if verification fails (@test-assertions.R#304) 
  3. Failure: insist raises error if verification fails (using se) (@test-assertions.R#313) 
  4. Failure: insist raises error if verification fails (using se) (@test-assertions.R#315) 
  5. Failure: insist raises *custom error* if verification fails (@test-assertions.R#324) 
  6. Failure: insist raises *custom error* if verification fails (@test-assertions.R#326) 
  7. Failure: insist raises *custom error* if verification fails (using se) (@test-assertions.R#339) 
  8. Failure: insist raises *custom error* if verification fails (using se) (@test-assertions.R#341) 
  
  Error: testthat unit tests failed
  Execution halted
```

## automagic (0.3)
Maintainer: Cole Brokamp <cole.brokamp@gmail.com>  
Bug reports: https://github.com/cole-brokamp/automagic/issues

0 errors | 0 warnings | 0 notes

## AutoModel (0.4.9)
Maintainer: Alex Lishinski <alexlishinski@gmail.com>

0 errors | 0 warnings | 0 notes

## backtestGraphics (0.1.6)
Maintainer: Miller Zijie Zhu <zijie.miller.zhu@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > test_check("backtestGraphics")
  Loading required package: backtestGraphics
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) -------------
  Failed the test for calculating summary statistics not equal to `truth.1`.
  Component "pnl": Component "pnl.drawdown": Component "start": Attributes: < Component "levels": 1 string mismatch >
  Component "pnl": Component "pnl.drawdown": Component "start": 1 string mismatch
  
  
  testthat results ================================================================
  OK: 9 SKIPPED: 0 FAILED: 1
  1. Failure: stat_calculation function (@test_stat_calculation.R#8) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bannerCommenter (0.1.0)
Maintainer: Bill Venables <Bill.Venables@gmail.com>

0 errors | 0 warnings | 0 notes

## bayesGDS (0.6.2)
Maintainer: Michael Braun <braunm@smu.edu>

0 errors | 0 warnings | 0 notes

## bayesplot (1.1.0)
Maintainer: Jonah Gabry <jsg2201@columbia.edu>  
Bug reports: https://github.com/stan-dev/bayesplot/issues/

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 2-14 (./children/SETTINGS-knitr.txt) 
Quitting from lines NA-14 (./children/SETTINGS-knitr.txt) 
Error: processing vignette 'MCMC-diagnostics.Rmd' failed with diagnostics:
object 'params' not found
Execution halted

```

## bib2df (0.1.1)
Maintainer: Philipp Ottolinger <philipp@ottolinger.de>

0 errors | 0 warnings | 0 notes

## bibliometrix (1.4)
Maintainer: Massimo Aria <aria@unina.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## bigrquery (0.3.0)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/rstats-db/bigrquery/issues

0 errors | 0 warnings | 1 note 

```
checking dependencies in R code ... NOTE
Missing objects imported by ':::' calls:
  ‘dplyr:::over’ ‘dplyr:::partition_group’ ‘dplyr:::partition_order’
```

## binford (0.1.0)
Maintainer: Ben Marwick <benmarwick@gmail.com>

0 errors | 0 warnings | 0 notes

## binomen (0.1.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/binomen/issues

0 errors | 0 warnings | 0 notes

## bioinactivation (1.1.5)
Maintainer: Alberto Garre <garre.alberto@gmail.com>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘bioinactivation-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: fit_dynamic_inactivation
> ### Title: Fitting of Dynamic Inactivation Models
> ### Aliases: fit_dynamic_inactivation
> 
> ### ** Examples
... 21 lines ...
> 
> starting_points <- c(n = 1, k_b = 0.25, N0 = 1e+05)
> upper_bounds <- c(n = 2, k_b = 1, N0 = Inf)
> lower_bounds <- c(n = 0, k_b = 0, N0 = 1e4)
> 
> dynamic_fit <- fit_dynamic_inactivation(dynamic_inactivation, simulation_model,
+                                         dummy_temp, starting_points,
+                                         upper_bounds, lower_bounds,
+                                         known_params)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Error: Test failed: 'Prediction Bigelow'
  * Cannot convert objects of type `closure` to `tidy_quote`
  1: predict_inactivation(simulation_model, times, parms, temp_profile)
  2: mutate_(out, logN = interp(~log10(N), N = as.name("N")), S = interp(~N/parms[["N0"]], 
         N = as.name("N")))
  3: mutate_.data.frame(out, logN = interp(~log10(N), N = as.name("N")), S = interp(~N/parms[["N0"]], 
         N = as.name("N"))) at /home/muelleki/git/R/dplyr/R/manip.r:211
  4: as.data.frame(mutate_(tbl_df(.data), ..., .dots = dots)) at /home/muelleki/git/R/dplyr/R/dataframe.R:95
  5: mutate_(tbl_df(.data), ..., .dots = dots)
  6: mutate_.tbl_df(tbl_df(.data), ..., .dots = dots) at /home/muelleki/git/R/dplyr/R/manip.r:211
  7: compat_lazy_dots(.dots, caller_env(), ..., .named = TRUE) at /home/muelleki/git/R/dplyr/R/tbl-df.r:87
  8: compat_lazy(dots[[i]], env, warn) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:47
  9: coerce_type(lazy, "tidy_quote", quote = lazy, symbol = , language = quosure(lazy, 
         env), 
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 360-361 (inactivation.Rmd) 
Error: processing vignette 'inactivation.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## biomartr (0.3.0)
Maintainer: Hajk-Georg Drost <hgd23@cam.ac.uk>  
Bug reports: https://github.com/HajkD/biomartr/issues

0 errors | 0 warnings | 0 notes

## bioOED (0.1.1)
Maintainer: Alberto Garre <garre.alberto@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘MEIGOR’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## bkmr (0.1.0)
Maintainer: Jennifer F. Bobb <jenniferfederbobb@gmail.com>  
Bug reports: https://github.com/jenfb/bkmr/issues

0 errors | 0 warnings | 0 notes

## blkbox (1.0)
Maintainer: Boris Guennewig <b.guennewig@garvan.org.au>

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘bigrf’

checking dependencies in R code ... NOTE
Missing or unexported object: ‘xgboost::predict’
```

## blscrapeR (2.1.2)
Maintainer: Kris Eberwein <eberwein@knights.ucf.edu>  
Bug reports: https://github.com/keberwein/blscrapeR/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         on.exit(options(expressions = expressions_opt), add = TRUE)
         e$expectation_calls <- frame_calls(11, 2)
         test_error <<- e
         register_expectation(e)
         e$handled <- TRUE
         test_error <<- e
     }, "attempt to select less than one element in integerOneIndex", quote(datalist[[i]] <- mth_vals)) at testthat/test_bls_csv.R:76
  2: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 2 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_bls_csv.R#76) 
  
  Error: testthat unit tests failed
  Execution halted
```

## bmlm (1.2.10)
Maintainer: Matti Vuorre <mv2521@columbia.edu>  
Bug reports: http://github.com/mvuorre/bmlm/issues/

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 28.5Mb
  sub-directories of 1Mb or more:
    libs  28.2Mb
```

## bodenmiller (0.1)
Maintainer: Yann Abraham <yann.abraham@gmail.com>  
Bug reports: https://github.com/yannabraham/bodenmiller/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  8.9Mb
  sub-directories of 1Mb or more:
    data   8.7Mb
```

## bootnet (0.4)
Maintainer: Sacha Epskamp <mail@sachaepskamp.com>

0 errors | 0 warnings | 0 notes

## bossMaps (0.1.0)
Maintainer: Adam M. Wilson <adamw@buffalo.edu>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘bossMaps-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: checkRates
> ### Title: Evaluate whether potential curve parameters are feasible given
> ###   the range and domain geometry.
> ### Aliases: checkRates
> 
> ### ** Examples
> 
> data(Beamys_hindei_range)
> # Generate domain
>  domain=raster::raster(xmn=-180, xmx=180, ymn=-90, ymx=90,
+  crs="+proj=longlat +ellps=WGS84 +towgs84=0,0,0,0,0,0,0 +no_defs",
+  resolution=.1, vals=NULL)
>  # registerDoParallel(2) #optionally use to speed up calculation
>  # Calculate distance to range
>  rdist=rangeDist(range=Beamys_hindei_range,domain=domain,
+                  domainkm=1000,mask=FALSE,fact=2,verbose=FALSE)
Error in spTransform(x, CRS(CRSobj), ...) : 
  package rgdal is required for spTransform methods
Calls: rangeDist ... spTransform -> spTransform -> spTransform -> spTransform
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: rangeDist(Tinamus_solitarius_range, domain = domain, domainkm = 1000, mask = F, fact = 2, 
         verbose = F) at testthat/test_Tinamou.R:16
  2: spTransform(urange, bproj)
  3: spTransform(urange, bproj) at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/Spatial-methods.R:93
  4: spTransform(x, CRS(CRSobj), ...) at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/Spatial-methods.R:93
  5: spTransform(x, CRS(CRSobj), ...) at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/Spatial-methods.R:93
  6: stop("package rgdal is required for spTransform methods") at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/Spatial-methods.R:97
  
  testthat results ================================================================
  OK: 2 SKIPPED: 0 FAILED: 2
  1. Error: (unknown) (@test_Beamys_hindei.R#16) 
  2. Error: (unknown) (@test_Tinamou.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

... 8 lines ...

    extract

The following object is masked from 'package:dplyr':

    select

Quitting from lines 53-72 (bossMaps.Rmd) 
Error: processing vignette 'bossMaps.Rmd' failed with diagnostics:
package rgdal is required for spTransform methods
Execution halted
```

## boxr (0.3.4)
Maintainer: Brendan Rocks <foss@brendanrocks.com>  
Bug reports: https://github.com/brendan-r/boxr/issues

0 errors | 0 warnings | 0 notes

## braQCA (0.9.9.6)
Maintainer: C. Ben Gibson <cbgibson@uci.edu>

0 errors | 0 warnings | 0 notes

## broom (0.4.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/tidyverse/broom/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘broom-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cch_tidiers
> ### Title: tidiers for case-cohort data
> ### Aliases: cch_tidiers glance.cch tidy.cch
> 
> ### ** Examples
... 42 lines ...
The following objects are masked from ‘package:stats’:

    filter, lag

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

Error in ~seqno : object 'seqno' not found
Calls: %>% ... overscope_eval -> .Call -> tidy -> cch -> ~ -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Terminated

```

## bsam (1.1.1)
Maintainer: Ian Jonsen <ian.jonsen@mq.edu.au>  
Bug reports: https://github.com/ianjonsen/bsam/issues

0 errors | 0 warnings | 0 notes

## CARBayesST (2.4)
Maintainer: Duncan Lee <Duncan.Lee@glasgow.ac.uk>

0 errors | 0 warnings | 0 notes

## carpenter (0.2.0)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/carpenter/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘carpenter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: carpenter
> ### Title: Build common tables for your research needs!
> ### Aliases: carpenter
> 
> ### ** Examples
> 
> 
> library(magrittr)
> outline_table(iris, 'Species') %>%
+  add_rows(c('Sepal.Length', 'Petal.Length'), stat_meanSD) %>%
+  add_rows('Sepal.Width', stat_medianIQR) %>%
+  renaming('rows', function(x) gsub('Sepal\\.', 'Sepal ', x)) %>%
+  renaming('header', c('Measures', 'Setosa', 'Versicolor', 'Virginica')) %>%
+  build_table(caption = 'A caption for the table')
Warning: 'dplyr::mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: Variable context not set
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  13: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  14: dplyr::mutate_each_(., dplyr::funs(as.character), dplyr::everything())
  15: is_empty(vars) at /home/muelleki/git/R/dplyr/R/colwise.R:277
  16: dplyr::everything() at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:240
  17: current_vars() at /home/muelleki/git/R/dplyr/R/select-utils.R:129
  18: cur_vars_env$selected %||% stop("Variable context not set", call. = FALSE) at /home/muelleki/git/R/dplyr/R/select-utils.R:46
  19: stop("Variable context not set", call. = FALSE) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/operators.R:14
  
  testthat results ================================================================
  OK: 20 SKIPPED: 0 FAILED: 1
  1. Error: build_table outputs correct information (@test-build_tables.R#21) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 87-89 (carpenter.Rmd) 
Error: processing vignette 'carpenter.Rmd' failed with diagnostics:
Variable context not set
Execution halted

```

## causaldrf (0.3)
Maintainer: Douglas Galagate <galagated@gmail.com>

0 errors | 0 warnings | 0 notes

## cbsodataR (0.2.1)
Maintainer: Edwin de Jonge <edwindjonge@gmail.com>  
Bug reports: https://github.com/edwindj/cbsodataR/issues

0 errors | 0 warnings | 0 notes

## cdcfluview (0.5.1)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/cdcfluview/issues

0 errors | 0 warnings | 0 notes

## censusr (0.0.2)
Maintainer: Greg Macfarlane <gregmacfarlane@gmail.com>

0 errors | 0 warnings | 0 notes

## choroplethr (3.5.3)
Maintainer: Ari Lamstein <ari@lamsteinconsulting.com>

0 errors | 0 warnings | 0 notes

## chromer (0.1)
Maintainer: Matthew Pennell <mwpennell@gmail.com>  
Bug reports: http://www.github.com/ropensci/chromer/issues/new

0 errors | 0 warnings | 2 notes

```
checking DESCRIPTION meta-information ... NOTE
Malformed Description field: should contain one or more complete sentences.

checking R code for possible problems ... NOTE
parse_counts: no visible global function definition for ‘na.omit’
Undefined global functions or variables:
  na.omit
Consider adding
  importFrom("stats", "na.omit")
to your NAMESPACE file.
```

## chunked (0.3)
Maintainer: Edwin de Jonge <edwindjonge@gmail.com>  
Bug reports: https://github.com/edwindj/chunked/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘chunked-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: read_csv_chunkwise
> ### Title: Read chunkwise data from text files
> ### Aliases: read_csv2_chunkwise read_csv_chunkwise read_laf_chunkwise
> ###   read_table_chunkwise
> 
... 7 lines ...
> #
> women_chunked <-
+   read_chunkwise(in_file) %>%  #open chunkwise connection
+   mutate(ratio = weight/height) %>%
+   filter(ratio > 2) %>%
+   select(height, ratio) %>%
+   inner_join(data.frame(height=63:66)) # you can join with data.frames!
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "c('chunkwise', 'tbl')"
Calls: %>% ... eval -> eval -> _fseq -> freduce -> <Anonymous> -> mutate
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  testthat results ================================================================
  OK: 16 SKIPPED: 0 FAILED: 9
  1. Error: insert_chunkwise_into: can insert modified table into a db (@test-insert-into.R#15) 
  2. Error: print groups: should print groups chunked info (@test-print.R#14) 
  3. Error: mutate(): can add a new column (@test-verbs.R#9) 
  4. Error: mutate(): can add a new column (@test-verbs.R#14) 
  5. Error: select(): select a column (@test-verbs.R#21) 
  6. Error: filter(): can filter rows (@test-verbs.R#28) 
  7. Error: rename(): can rename columns (@test-verbs.R#35) 
  8. Error: (unknown) (@test-verbs.R#40) 
  9. Error: write_chunkwise to db works (@test-write.R#29) 
  
  Error: testthat unit tests failed
  Execution halted
```

## cleanNLP (0.24)
Maintainer: Taylor B. Arnold <taylor.arnold@acm.org>

0 errors | 0 warnings | 0 notes

## climbeR (0.0.1)
Maintainer: John Karlen <jkarlen@massmutual.com>

0 errors | 0 warnings | 0 notes

## clustRcompaR (0.1.0)
Maintainer: Alex Lishinski <alexlishinski@gmail.com>

0 errors | 0 warnings | 0 notes

## clustrd (1.1.0)
Maintainer: Angelos Markos <amarkos@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘clustrd-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cluspca
> ### Title: Joint dimension reduction and clustering of continuous data.
> ### Aliases: cluspca
> 
> ### ** Examples
> 
> data(macro)
> outRKM = cluspca(macro, 3, 2, method = "RKM", rotation = "varimax")
> plot(outRKM, cludesc = TRUE)
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error in bind_rows_(x, .id) : cannot convert object to a data frame
Calls: plot ... clu_means -> rbind -> rbind -> bind_rows -> bind_rows_ -> .Call
Execution halted
```

## codingMatrices (0.2.2)
Maintainer: Bill Venables <Bill.Venables@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'car'

The following object is masked from 'package:dplyr':

    recode

Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'codingMatrices.tex' failed.
BibTeX errors:
The top-level auxiliary file: codingMatrices.aux
I couldn't open style file chicago.bst
---line 19 of file codingMatrices.aux
 : \bibstyle{chicago
 :                  }
I'm skipping whatever remains of this command
I found no style file---while reading file codingMatrices.aux
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## codyn (1.1.0)
Maintainer: Matthew B. Jones <jones@nceas.ucsb.edu>  
Bug reports: https://github.com/laurenmh/codyn/issues

0 errors | 0 warnings | 0 notes

## cofeatureR (1.0.1)
Maintainer: Fong Chun Chan <fongchunchan@gmail.com>  
Bug reports: https://github.com/tinyheero/cofeatureR/issues

0 errors | 0 warnings | 0 notes

## CollapsABEL (0.10.11)
Maintainer: Kaiyin Zhong <kindlychung@gmail.com>  
Bug reports: https://bitbucket.org/kindlychung/collapsabel2/issues

0 errors | 0 warnings | 0 notes

## cometExactTest (0.1.3)
Maintainer: Max Leiserson <mdml@cs.brown.edu>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## compareDF (1.1.0)
Maintainer: Alex Joseph <alexsanjoseph@gmail.com>

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘compareDF-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: compare_df
> ### Title: Compare Two dataframes
> ### Aliases: compare_df
> 
> ### ** Examples
> 
> old_df = data.frame(var1 = c("A", "B", "C"),
+                     val1 = c(1, 2, 3))
> new_df = data.frame(var1 = c("A", "B", "C"),
+                     val1 = c(1, 2, 4))
> ctable = compare_df(new_df, old_df, c("var1"))
Creating comparison table...
Error in arrange_impl(.data, dots) : incorrect size (1), expecting : 2
Calls: compare_df ... as.data.frame -> arrange -> arrange.tbl_df -> arrange_impl -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  8: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  9: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  10: arrange_(., group_col)
  11: arrange_.data.frame(., group_col) at /home/muelleki/git/R/dplyr/R/manip.r:263
  12: as.data.frame(arrange(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:104
  13: arrange(tbl_df(.data), ...)
  14: arrange.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:258
  15: arrange_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:46
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test-fnsComparison.R#14) 
  
  Error: testthat unit tests failed
  Execution halted
```

## condformat (0.5.0)
Maintainer: Sergio Oller Moreno <sergioller@gmail.com>  
Bug reports: http://github.com/zeehio/condformat/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘condformat-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: condformat
> ### Title: Converts a data frame or matrix to a condformat_tbl class.
> ### Aliases: condformat
> 
> ### ** Examples
... 50 lines ...
<td style='border-bottom: 2px solid grey; text-align: center;'>5.0</td>
<td style='border-bottom: 2px solid grey; text-align: center;'>3.6</td>
<td style='border-bottom: 2px solid grey; text-align: center;'>1.4</td>
<td style='border-bottom: 2px solid grey; text-align: center;'>0.2</td>
<td style='border-bottom: 2px solid grey; text-align: center;'>setosa</td>
</tr>
</tbody>
</table>
> 
> condformat(iris[1:5,]) + rule_fill_gradient(Sepal.Length)
Error: All select() inputs must resolve to integer column positions. The following do not:

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 20 SKIPPED: 0 FAILED: 10
  1.  Error: rule_fill_discrete works (@test_rule_fill_discrete.R#10) 
  2.  Error: rule_fill_discrete lock cells (@test_rule_fill_discrete.R#35) 
  3.  Error: rule_fill_discrete_ works (@test_rule_fill_discrete.R#64) 
  4.  Error: rule_fill_discrete_ works with formula (@test_rule_fill_discrete.R#86) 
  5.  Error: rule_fill_gradient works in the limits (@test_rule_fill_gradient.R#8) 
  6.  Error: rule_fill_gradient_ works in the limits (@test_rule_fill_gradient.R#24) 
  7.  Error: rule_fill_gradient2 works (@test_rule_fill_gradient.R#41) 
  8.  Error: show_column works (@test_show.R#8) 
  9.  Error: show_column_ works (@test_show.R#20) 
  10. Error: show_column works with custom names (@test_show.R#32) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 31-39 (introduction.Rmd) 
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
All select() inputs must resolve to integer column positions. The following do not:
```

## coreSim (0.2.3)
Maintainer: Christopher Gandrud <christopher.gandrud@gmail.com>  
Bug reports: https://github.com/christophergandrud/coreSim/issues

0 errors | 0 warnings | 0 notes

## corrr (0.2.1)
Maintainer: Simon Jackson <drsimonjackson@gmail.com>  
Bug reports: https://github.com/drsimonj/corrr/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2/3 mismatches
  x[2]: "dplyr::contains(\"Length\")1"
  y[2]: "Sepal.Length"
  
  x[3]: "dplyr::contains(\"Length\")2"
  y[3]: "Petal.Length"
  
  
  testthat results ================================================================
  OK: 52 SKIPPED: 0 FAILED: 2
  1. Failure: Column select works as dplyr::select (@test-focus.R#12) 
  2. Failure: Column select works as dplyr::select (@test-focus.R#16) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 8 lines ...
Attaching package: 'purrr'

The following objects are masked from 'package:dplyr':

    contains, order_by

Warning: Unknown column 'PANEL'
Quitting from lines 111-116 (using-corrr.Rmd) 
Error: processing vignette 'using-corrr.Rmd' failed with diagnostics:
a dimension is zero
Execution halted
```

## countytimezones (1.0.0)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>  
Bug reports: https://github.com/geanders/countytimezones/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘countytimezones-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: add_local_time
> ### Title: Add local time to dataset
> ### Aliases: add_local_time
> 
> ### ** Examples
> 
> ex_df <- data.frame(datetime = c("1999-01-01 08:00", "1999-01-01 09:00",
+                                  "1999-01-01 10:00"),
+                     fips = c("36061", "17031", "06037"))
> add_local_time(df = ex_df, fips = ex_df$fips,
+                datetime_colname = "datetime")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 61-62 (countytimezones.Rmd) 
Error: processing vignette 'countytimezones.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## countyweather (0.1.0)
Maintainer: Rachel Severson <rachel.severson@colostate.edu>

0 errors | 0 warnings | 0 notes

## cpr (0.2.3)
Maintainer: Peter DeWitt <dewittpe@gmail.com>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘cpr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cp
> ### Title: Control Polygons
> ### Aliases: cp cp.cpr_bs cp.formula print.cpr_cp summary.cpr_cp
> ###   plot.cpr_cp
> 
... 12 lines ...
> theta1 <- c(1, 0, 3.5, 4.2, 3.7, -0.5, -0.7, 2, 1.5)
> theta2 <- c(1, 3.4, -2, 1.7)
> 
> # build the two control polygons
> cp1 <- cp(bmat1, theta1)
> cp2 <- cp(bmat2, theta2)
> 
> # black and white plot
> plot(cp1)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 93-95 (cpr-pkg.Rmd) 
Error: processing vignette 'cpr-pkg.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## crawl (2.0.1)
Maintainer: Devin S. Johnson <devin.johnson@noaa.gov>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
crawl 2.0.1 (2016-07-22) 
 Type 'vignette('crawl_intro')' to see examples of package use, and
 'demo(package='crawl')' will provide a list of demos.
 The raw code for the demos can be found by typing:
 'system.file('demo', package='crawl')'

Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/muelleki/R/x86_64-pc-linux-gnu-library/3.3/rgdal/libs/rgdal.so':
  libgdal.so.1: cannot open shared object file: No such file or directory
Quitting from lines 80-83 (crawl_intro.Rmd) 
Error: processing vignette 'crawl_intro.Rmd' failed with diagnostics:
package or namespace load failed for 'rgdal'
Execution halted


checking installed package size ... NOTE
  installed size is  9.3Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   6.3Mb
```

## cricketr (0.0.14)
Maintainer: Tinniam V Ganesh <tvganesh.85@gmail.com>  
Bug reports: https://github.com/tvganesh/cricketr/issues

0 errors | 0 warnings | 0 notes

## d3r (0.6.2)
Maintainer: Kent Russell <kent.russell@timelyportfolio.com>  
Bug reports: https://github.com/timelyportfolio/d3r/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘treemap’
```

## darksky (1.0.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/darksky/issues

0 errors | 0 warnings | 0 notes

## datacheckr (0.1.2)
Maintainer: Joe Thorley <joe@poissonconsulting.ca>

0 errors | 0 warnings | 0 notes

## DataCombine (0.2.21)
Maintainer: Christopher Gandrud <christopher.gandrud@gmail.com>  
Bug reports: https://github.com/christophergandrud/DataCombine/issues

0 errors | 0 warnings | 0 notes

## datadr (0.8.6)
Maintainer: Ryan Hafen <rhafen@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘Rhipe’
```

## dataonderivatives (0.2.1)
Maintainer: Imanuel Costigan <i.costigan@me.com>  
Bug reports: https://github.com/imanuelcostigan/dataonderivatives/issues

0 errors | 0 warnings | 0 notes

## dataRetrieval (2.6.3)
Maintainer: Laura DeCicco <ldecicco@usgs.gov>  
Bug reports: https://github.com/USGS-R/dataRetrieval/issues

0 errors | 0 warnings | 0 notes

## datastepr (0.0.2)
Maintainer: Brandon Taylor <brandon.taylor221@gmail.com>  
Bug reports: https://github.com/bramtayl/datastepr/issues

0 errors | 0 warnings | 0 notes

## dat (0.1.0)
Maintainer: Sebastian Warnholz <wahani@gmail.com>  
Bug reports: https://github.com/wahani/dat/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [47s/48s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  31: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  32: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 82 SKIPPED: 0 FAILED: 5
  1. Failure: Basic syntax of a DataFrame (@test-DataFrame.R#66) 
  2. Error: Parameterized formulas (@test-DataFrame.R#169) 
  3. Failure: Package Style (@test-lintr.R#5) 
  4. Error: make formulas (@test-makeFormulas.R#40) 
  5. Error: map (@test-map.R#13) 
  
  Error: testthat unit tests failed
  Execution halted
```

## dbfaker (0.1.0)
Maintainer: Hari Mohanraj <hari@shipt.com>

0 errors | 0 warnings | 0 notes

## ddpcr (1.6)
Maintainer: Dean Attali <daattali@gmail.com>  
Bug reports: https://github.com/daattali/ddpcr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘ddpcr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ddpcr.Rcheck/00install.out’ for details.
```

## decoder (1.1.12)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.com/cancercentrum/decoder/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Joining, by = "key"
Joining, by = "key"
PhantomJS not found. You can install it with webshot::install_phantomjs(). If it is installed, please make sure the phantomjs executable can be found via the PATH variable.
Quitting from lines 102-122 (sjukhus.Rmd) 
Error: processing vignette 'sjukhus.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## DeLorean (1.2.4)
Maintainer: John Reid <john.reid@mrc-bsu.cam.ac.uk>

0 errors | 1 warning  | 1 note 

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.cells:
  function(dl, .filter, number, cells)

filter:
  function(x, filter, method, sides, circular, init)
filter.genes:
  function(dl, .filter, number, genes)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.cells filter.genes
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.cells':
  ‘filter.cells’

S3 methods shown with full name in documentation object 'filter.genes':
  ‘filter.genes’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## denovolyzeR (0.2.0)
Maintainer: James Ware <j.ware@imperial.ac.uk>  
Bug reports: http://github.com/jamesware/denovolyzeR/issues

0 errors | 0 warnings | 0 notes

## DepthProc (1.0.7)
Maintainer: Zygmunt Zawadzki <zawadzkizygmunt@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.3Mb
  sub-directories of 1Mb or more:
    libs   4.4Mb
```

## describer (0.2.0)
Maintainer: Paul Hendricks <paul.hendricks.2013@owu.edu>  
Bug reports: https://github.com/paulhendricks/describer/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [124s/128s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-window.R:74:8: style: Variable and function names should be all lowercase.
      ), sqlText),
         ^~~~~~~
  tests/testthat/test-window.R:76:12: style: Variable and function names should be all lowercase.
      info = sqlText
             ^~~~~~~
  
  
  testthat results ================================================================
  OK: 7 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test-styling.R#4) 
  
  Error: testthat unit tests failed
  Execution halted
```

## descriptr (0.1.0)
Maintainer: Aravind Hebbali <hebbali.aravind@gmail.com>  
Bug reports: https://github.com/rsquaredacademy/descriptr/issues

0 errors | 0 warnings | 0 notes

## dggridR (0.1.11)
Maintainer: Richard Barnes <rbarnes@umn.edu>  
Bug reports: https://github.com/r-barnes/dggridR/

1 error  | 0 warnings | 0 notes

```
checking whether package ‘dggridR’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/dggridR.Rcheck/00install.out’ for details.
```

## DiagrammeR (0.9.0)
Maintainer: Richard Iannone <riannone@me.com>  
Bug reports: https://github.com/rich-iannone/DiagrammeR/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘DiagrammeR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: drop_edge_attrs
> ### Title: Drop an edge attribute column
> ### Aliases: drop_edge_attrs
> 
> ### ** Examples
> 
> # Create a random graph
> graph <-
+   create_random_graph(
+     5, 6, set_seed = 23) %>%
+     set_edge_attrs("value", 3) %>%
+     mutate_edge_attrs(
+       "value", "2 * ~", "penwidth")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [169s/174s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  20: as.data.frame(arrange(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:104
  21: arrange(tbl_df(.data), ...)
  22: arrange.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:258
  23: arrange_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:46
  
  testthat results ================================================================
  OK: 1453 SKIPPED: 0 FAILED: 5
  1. Error: adding nodes from a table to a graph is possible (@test-add_nodes_edges_from_table.R#256) 
  2. Error: Mutating node attributes is possible (@test-copy_drop_mutate_recode_attrs.R#302) 
  3. Error: Mutating edge attributes is possible (@test-copy_drop_mutate_recode_attrs.R#363) 
  4. Error: importing a .graphml file is possible (@test-import_graph.R#5) 
  5. Error: setting positions with a text string is possible (@test-set_node_positions.R#171) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 10.6Mb
  sub-directories of 1Mb or more:
    extdata       2.4Mb
    htmlwidgets   4.7Mb
    img           2.8Mb
```

## diffrprojects (0.1.14)
Maintainer: Peter Meissner <retep.meissner@gmail.com>  
Bug reports: https://github.com/petermeissner/diffrprojects/issues

0 errors | 0 warnings | 0 notes

## diffrprojectswidget (0.1.5)
Maintainer: Peter Meissner <retep.meissner@gmail.com>  
Bug reports: https://github.com/petermeissner/diffrprojectswidget/issues

0 errors | 0 warnings | 0 notes

## DisimForMixed (0.2)
Maintainer: Hasanthi A. Pathberiya <hasanthi@sjp.ac.lk>

0 errors | 0 warnings | 0 notes

## DiversityOccupancy (1.0.6)
Maintainer: Derek Corcoran <derek.corcoran.barrios@gmail.com>

0 errors | 0 warnings | 0 notes

## dmutate (0.1.1)
Maintainer: Kyle T Baron <kyleb@metrumrg.com>  
Bug reports: https://github.com/kylebmetrum/dmutate/issues

0 errors | 0 warnings | 0 notes

## DMwR2 (0.0.2)
Maintainer: Luis Torgo <ltorgo@dcc.fc.up.pt>  
Bug reports: https://github.com/ltorgo/DMwR2/issues

0 errors | 0 warnings | 0 notes

## doctr (0.2.0)
Maintainer: Caio Lente <me@ctlente.com>

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  13: eval(expr, envir, enclos)
  14: `_fseq`(`_lhs`)
  15: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  16: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  17: dplyr::mutate(., data = purrr::map(exams[[1]], ~X[[.x]]))
  18: mutate.tbl_df(., data = purrr::map(exams[[1]], ~X[[.x]])) at /home/muelleki/git/R/dplyr/R/manip.r:206
  19: mutate_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:83
  
  testthat results ================================================================
  OK: 26 SKIPPED: 0 FAILED: 2
  1. Error: (unknown) (@test_diagnose.R#7) 
  2. Error: (unknown) (@test_issues.R#13) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading tidyverse: ggplot2
Loading tidyverse: tibble
Loading tidyverse: tidyr
Loading tidyverse: readr
Loading tidyverse: purrr
Loading tidyverse: dplyr
Conflicts with tidy packages ----------------------------------------------
filter(): dplyr, stats
lag():    dplyr, stats
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 43-45 (doctr_diagnose.Rmd) 
Error: processing vignette 'doctr_diagnose.Rmd' failed with diagnostics:
object '.x' not found
Execution halted

```

## docxtools (0.1.1)
Maintainer: Richard Layton <graphdoctor@gmail.com>

0 errors | 0 warnings | 0 notes

## docxtractr (0.2.0)
Maintainer: Bob Rudis <bob@rudis.net>

0 errors | 0 warnings | 0 notes

## dotwhisker (0.2.3)
Maintainer: Yue Hu <yue-hu-1@uiowa.edu>  
Bug reports: https://github.com/fsolt/dotwhisker/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘dotwhisker-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dwplot
> ### Title: Dot-and-Whisker Plots of Regression Results
> ### Aliases: dwplot
> 
> ### ** Examples
... 51 lines ...
 4: rename(., model = am)
 5: function_list[[k]](value)
 6: withVisible(function_list[[k]](value))
 7: freduce(value, `_function_list`)
 8: `_fseq`(`_lhs`)
 9: eval(expr, envir, enclos)
10: eval(quote(`_fseq`(`_lhs`)), env, env)
11: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env))
12: mtcars %>% group_by(am) %>% do(tidy(lm(mpg ~ wt + cyl + disp,     data = .))) %>% rename(model = am)
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
14: withVisible(eval(expr, envir, enclos))
15: withCallingHandlers(withVisible(eval(expr, envir, enclos)), warning = wHandler,     error = eHandler, message = mHandler)
16: handle(ev <- withCallingHandlers(withVisible(eval(expr, envir,     enclos)), warning = wHandler, error = eHandler, message = mHandler))
17: timing_fn(handle(ev <- withCallingHandlers(withVisible(eval(expr,     envir, enclos)), warning = wHandler, error = eHandler, message = mHandler)))
18: evaluate_call(expr, parsed$src[[i]], envir = envir, enclos = enclos,     debug = debug, last = i == length(out), use_try = stop_on_error !=         2L, keep_warning = keep_warning, keep_message = keep_message,     output_handler = output_handler, include_timing = include_timing)
19: evaluate(code, envir = env, new_device = FALSE, keep_warning = !isFALSE(options$warning),     keep_message = !isFALSE(options$message), stop_on_error = if (options$error &&         options$include) 0L else 2L, output_handler = knit_handlers(options$render,         options))
20: in_dir(input_dir(), evaluate(code, envir = env, new_device = FALSE,     keep_warning = !isFALSE(options$warning), keep_message = !isFALSE(options$message),     stop_on_error = if (options$error && options$include) 0L else 2L,     output_handler = knit_handlers(options$render, options)))
... 8 lines ...
28: knit2html(..., force_v1 = TRUE)
29: (if (grepl("\\.[Rr]md$", file)) knit2html_v1 else if (grepl("\\.[Rr]rst$",     file)) knit2pdf else knit)(file, encoding = encoding, quiet = quiet,     envir = globalenv())
30: vweave(...)
31: engine$weave(file, quiet = quiet, encoding = enc)
32: doTryCatch(return(expr), name, parentenv, handler)
33: tryCatchOne(expr, names, parentenv, handlers[[1L]])
34: tryCatchList(expr, classes, parentenv, handlers)
35: tryCatch({    engine$weave(file, quiet = quiet, encoding = enc)    setwd(startdir)    find_vignette_product(name, by = "weave", engine = engine)}, error = function(e) {    stop(gettextf("processing vignette '%s' failed with diagnostics:\n%s",         file, conditionMessage(e)), domain = NA, call. = FALSE)})
36: buildVignettes(dir = "/home/muelleki/git/R/dplyr/revdep/checks/dotwhisker.Rcheck/vign_test/dotwhisker")
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)
```

## downsize (0.2.1)
Maintainer: William Landau <will.landau@gmail.com>  
Bug reports: https://github.com/wlandau/downsize/issues

0 errors | 0 warnings | 0 notes

## dtplyr (0.0.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/dtplyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 113 SKIPPED: 0 FAILED: 19
  1. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#23) 
  2. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#24) 
  3. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#25) 
  4. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#26) 
  5. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#28) 
  6. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#29) 
  7. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#30) 
  8. Failure: arrange results agree with data same regardless of backend (@test-arrange.R#31) 
  9. Failure: grouped_by uses grouping vars & preserves groups (@test-distinct.R#24) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## eAnalytics (0.1.3)
Maintainer: Paul Govan <pgovan1@aggienetwork.com>  
Bug reports: https://github.com/paulgovan/eanalytics/issues

0 errors | 0 warnings | 0 notes

## easyformatr (0.1.2)
Maintainer: Brandon Taylor <brandon.taylor221@gmail.com>  
Bug reports: https://github.com/bramtayl/easyformatr/issues

0 errors | 0 warnings | 0 notes

## ecb (0.2)
Maintainer: Eric Persson <expersso5@gmail.com>

0 errors | 0 warnings | 0 notes

## ecoengine (1.10.0)
Maintainer: Karthik Ram <karthik.ram@gmail.com>  
Bug reports: https://github.com/ropensci/ecoengine/issues

0 errors | 0 warnings | 0 notes

## edeaR (0.4.7)
Maintainer: Gert Janssenswillen <gert.janssenswillen@uhasselt.be>

0 errors | 0 warnings | 0 notes

## eechidna (0.1)
Maintainer: Ben Marwick <benmarwick@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'purrr'

The following objects are masked from 'package:dplyr':
... 8 lines ...

Attaching package: 'scales'

The following object is masked from 'package:purrr':

    discard

Quitting from lines 154-171 (exploring-election-data.Rmd) 
Error: processing vignette 'exploring-election-data.Rmd' failed with diagnostics:
Value of SET_STRING_ELT() must be a 'CHARSXP' not a 'logical'
Execution halted
```

## eemR (0.1.4)
Maintainer: Philippe Massicotte <pm@bios.au.dk>  
Bug reports: https://github.com/PMassicotte/eemR/issues

0 errors | 0 warnings | 0 notes

## EFDR (0.1.1)
Maintainer: Andrew Zammit-Mangion <andrewzm@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.gdf : find_loss: no visible global function definition for ‘rnorm’
.p.values : <anonymous>: no visible global function definition for
  ‘pnorm’
.relist.dwt: no visible global function definition for ‘relist’
.relist.dwt: no visible global function definition for ‘as’
.std.wav.coeff : <anonymous>: no visible global function definition for
  ‘mad’
regrid: no visible global function definition for ‘predict’
regrid: no visible global function definition for ‘var’
regrid: no visible global function definition for ‘medpolish’
Undefined global functions or variables:
  as mad medpolish pnorm predict relist rnorm var
Consider adding
  importFrom("methods", "as")
  importFrom("stats", "mad", "medpolish", "pnorm", "predict", "rnorm",
             "var")
  importFrom("utils", "relist")
to your NAMESPACE file (and ensure that your DESCRIPTION Imports field
contains 'methods').
```

## efreadr (0.1.1)
Maintainer: Marco Bascietto <marco.bascietto@crea.gov.it>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## electionsBR (0.2.0)
Maintainer: Denisson Silva <denissonsilva@ufmg.br>  
Bug reports: https://github.com/silvadenisson/electionsBR/issues

0 errors | 0 warnings | 0 notes

## emil (2.2.6)
Maintainer: Christofer Backlin <emil@christofer.backlin.se>  
Bug reports: https://github.com/Molmed/emil/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘emil-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: get_importance
> ### Title: Feature (variable) importance of a fitted model
> ### Aliases: get_importance
> 
> ### ** Examples
... 31 lines ...
14 Mär 00:38    Repeat 2, fold 2
14 Mär 00:38      Extracting fitting and testing datasets.
14 Mär 00:38      Fitting pamr
14 Mär 00:38    Repeat 2, fold 3
14 Mär 00:38      Extracting fitting and testing datasets.
14 Mär 00:38      Fitting pamr
> get_importance(result)
Error in UseMethod("select") : 
  no applicable method for 'select' applied to an object of class "modeling_result"
Calls: get_importance -> get_importance.modeling_result -> select
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [7s/10s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  3: eval(quote(`_fseq`(`_lhs`)), env, env)
  4: eval(expr, envir, enclos)
  5: `_fseq`(`_lhs`)
  6: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  7: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  8: function_list[[k]](value)
  9: select(., fold = TRUE, method = TRUE, error = "error")
  
  testthat results ================================================================
  OK: 172 SKIPPED: 1 FAILED: 2
  1. Error: Parameter tuning (@test-extractors.r#14) 
  2. Error: dplyr integration (@test-helpers.r#52) 
  
  Error: testthat unit tests failed
  Execution halted
```

## emuR (0.2.2)
Maintainer: Raphael Winkelmann <raphael@phonetik.uni-muenchen.de>  
Bug reports: https://github.com/IPS-LMU/emuR/issues

0 errors | 0 warnings | 0 notes

## engsoccerdata (0.1.5)
Maintainer: James Curley <jc3181@columbia.edu>

0 errors | 0 warnings | 0 notes

## epidata (0.1.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/epidata/issues

0 errors | 0 warnings | 0 notes

## episheet (0.2.0)
Maintainer: James Black <james@epijim.uk>  
Bug reports: https://github.com/epijim/episheet/issues

0 errors | 0 warnings | 0 notes

## estatapi (0.3.0)
Maintainer: Hiroaki Yutani <yutani.ini@gmail.com>  
Bug reports: https://github.com/yutannihilation/estatapi/issues

0 errors | 0 warnings | 0 notes

## etl (0.3.5)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/etl/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘etl-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: valid_year_month
> ### Title: Ensure that years and months are within a certain time span
> ### Aliases: valid_year_month
> 
> ### ** Examples
> 
> 
> valid_year_month(years = 1999:2001, months = c(1:3, 7))
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  downloaded 10 KB
  
  trying URL 'http://www.nytimes.com'
  Content type 'text/html; charset=utf-8' length 218062 bytes (212 KB)
  ==================================================
  downloaded 212 KB
  
  testthat results ================================================================
  OK: 18 SKIPPED: 0 FAILED: 3
  1. Error: sqlite works (@test-etl.R#9) 
  2. Failure: dplyr works (@test-etl.R#25) 
  3. Error: valid_year_month works (@test-etl.R#69) 
  
  Error: testthat unit tests failed
  Execution halted
```

## europepmc (0.1.3)
Maintainer: Najko Jahn <najko.jahn@gmail.com>  
Bug reports: http://github.com/ropensci/europepmc/issues

0 errors | 0 warnings | 0 notes

## europop (0.3.1)
Maintainer: Matthew Lincoln <matthew.d.lincoln@gmail.com>  
Bug reports: https://github.com/mdlincoln/europop/issues

0 errors | 0 warnings | 0 notes

## eurostat (2.2.43)
Maintainer: Leo Lahti <louhos@googlegroups.com>  
Bug reports: https://github.com/ropengov/eurostat/issues

0 errors | 0 warnings | 0 notes

## evaluator (0.1.0)
Maintainer: David Severski <davidski@deadheaven.com>  
Bug reports: https://github.com/davidski/evaluator/issues

0 errors | 0 warnings | 0 notes

## explor (0.3.1)
Maintainer: Julien Barnier <julien.barnier@ens-lyon.fr>  
Bug reports: https://github.com/juba/explor/issues

0 errors | 0 warnings | 0 notes

## extdplyr (0.1.4)
Maintainer: Yuchen Wang <ycwang0712@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘extdplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: grp_routine
> ### Title: Mutate a character/factor based on conditions.
> ### Aliases: grp_routine grp_routine_
> 
> ### ** Examples
... 7 lines ...
3  c  3
4  d  4
5  e  5
> 
> # By default, it creates new groups
> grp_routine(df, "group",
+                first = v1 %in% c("a", "b"),
+                second = v2 == 3,
+                third = v2 >= 4)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted
```

## eyelinker (0.1)
Maintainer: Simon Barthelme <simon.barthelme@gipsa-lab.fr>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: eyelinker
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 226-229 (basics.Rmd) 
Error: processing vignette 'basics.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted

```

## eyetrackingR (0.1.6)
Maintainer: Jacob Dink <jacobwdink@gmail.com>  
Bug reports: https://github.com/jwdink/eyetrackingR/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘eyetrackingR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: analyze_time_clusters
> ### Title: Bootstrap analysis of time-clusters.
> ### Aliases: analyze_time_clusters analyze_time_clusters.time_cluster_data
> 
> ### ** Examples
... 16 lines ...
> response_time <- make_time_sequence_data(response_window, time_bin_size = 500, aois = "Animate", 
+                                          predictor_columns = "Sex")
> 
> time_cluster_data <- make_time_cluster_data(data = response_time, predictor_column = "SexM", 
+                          aoi = "Animate", test = "lmer", 
+                          threshold = 1.5, 
+                          formula = LogitAdjusted ~ Sex + (1|Trial) + (1|ParticipantName))
Error in UseMethod("analyze_time_bins") : 
  no applicable method for 'analyze_time_bins' applied to an object of class "data.frame"
Calls: make_time_cluster_data ... make_time_cluster_data.time_sequence_data -> do.call -> <Anonymous>
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [82s/85s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Predictor:	 Target 
  Formula:	 Prop ~ Target 
  Runs of Significant Time Bins: 
  Positive Run 1  ===== 
  	Time:		 15800 - 21000 1. Error: (unknown) (@test_cluster_analysis.R#143) -----------------------------
  no applicable method for 'make_time_cluster_data' applied to an object of class "data.frame"
  1: make_time_cluster_data(response_time_by_ppt, predictor_column = "Sex", test = "boot_splines", 
         within_subj = FALSE, smoother = "smooth.spline", alpha = 0.05) at testthat/test_cluster_analysis.R:143
  
  testthat results ================================================================
  OK: 46 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_cluster_analysis.R#143) 
  
  Error: testthat unit tests failed
  Execution halted
```

## ezec (1.0.1)
Maintainer: Zhian N. Kamvar <zkamvar@gmail.com>  
Bug reports: https://github.com/grunwaldlab/ezec/issues

0 errors | 0 warnings | 0 notes

## ezsummary (0.2.1)
Maintainer: Hao Zhu <haozhu233@gmail.com>  
Bug reports: https://github.com/haozhu233/ezsummary/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ezsummary-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ezmarkup
> ### Title: Easy way to "markup" a table before it is sent to be displayed
> ### Aliases: ezmarkup
> 
> ### ** Examples
... 10 lines ...
The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> dt <- mtcars %>% group_by(cyl) %>% select(gear, carb) %>% ezsummary_categorical(n=TRUE)
Adding missing grouping variables: `cyl`
> 
> ezmarkup(dt, "...[.(.)]")
Error in cbind_all(x) : cannot convert object to a data frame
Calls: ezmarkup ... ezmarkup_ -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  26: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  27: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 23 SKIPPED: 0 FAILED: 4
  1. Error: ezsummary_quantitative can evaluate grouping info correctly with 1 variable (@test-ezsummary_quantitative.R#19) 
  2. Error: ezsummary_quantitative can work with 2 variables (@test-ezsummary_quantitative.R#26) 
  3. Error: ezsummary_quantitative can work with 2 variables with grouping info (@test-ezsummary_quantitative.R#33) 
  4. Error: ezsummary_quantitative can handle NAs when running with quantile=TRUE (@test-ezsummary_quantitative.R#40) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 89-90 (intro_to_ezsummary_0.2.0.Rmd) 
Error: processing vignette 'intro_to_ezsummary_0.2.0.Rmd' failed with diagnostics:
incorrect size (1), expecting : 11
Execution halted

```

## fbar (0.1.23)
Maintainer: Max Conway <conway.max1@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [11s/11s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  16: coerce_type(lazy, "tidy_quote", quote = lazy, symbol = , language = quosure(lazy, 
         env), string = {
         if (warn) 
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  17: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  18: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 40 SKIPPED: 10 FAILED: 1
  1. Error: find_fluxes_df works in grouped context (@test-full-size.R#11) 
  
  Error: testthat unit tests failed
  Execution halted

checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘Rglpk’ ‘ROI.plugin.glpk’
```

## feather (0.3.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/wesm/feather/issues

0 errors | 0 warnings | 0 notes

## FedData (2.4.5)
Maintainer: R. Kyle Bocinsky <bocinsky@gmail.com>  
Bug reports: https://github.com/bocinsky/FedData/issues

0 errors | 0 warnings | 0 notes

## feedeR (0.0.7)
Maintainer: Andrew Collier <andrew@exegetic.biz>  
Bug reports: https://github.com/DataWookie/feedeR/issues

0 errors | 0 warnings | 0 notes

## fiftystater (1.0.1)
Maintainer: William Murphy <william.murphy.rd@gmail.com>  
Bug reports: https://github.com/wmurphyrd/fiftystater/issues

0 errors | 0 warnings | 0 notes

## filesstrings (0.3.1)
Maintainer: Rory Nolan <rorynoolan@gmail.com>  
Bug reports: https://www.github.com/rorynolan/filesstrings/issues

0 errors | 0 warnings | 0 notes

## finreportr (1.0.1)
Maintainer: Seward Lee <sewardlee337@gmail.com>

0 errors | 0 warnings | 0 notes

## fitcoach (1.0)
Maintainer: Niraj Juneja <njuneja@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) --------------------
  res[1] is not strictly less than 2820. Difference: 29.2
  
  
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) --------------------
  `res` is not strictly less than 2518. Difference: 2.3
  
  
  testthat results ================================================================
  OK: 17 SKIPPED: 0 FAILED: 2
  1. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#28) 
  2. Failure: FitAnalyzer test cases (@test-fitanalyzer.R#49) 
  
  Error: testthat unit tests failed
  Execution halted
```

## fivethirtyeight (0.1.0)
Maintainer: Albert Y. Kim <albert.ys.kim@gmail.com>  
Bug reports: https://github.com/rudeboybert/fivethirtyeight/issues

0 errors | 0 warnings | 0 notes

## flora (0.2.8)
Maintainer: Gustavo Carvalho <gustavo.bio@gmail.com>  
Bug reports: http://www.github.com/gustavobio/flora/issues

0 errors | 0 warnings | 0 notes

## foghorn (0.4.2)
Maintainer: Francois Michonneau <francois.michonneau@gmail.com>  
Bug reports: https://github.com/fmichonneau/foghorn/issues

0 errors | 0 warnings | 0 notes

## forestmodel (0.4.0)
Maintainer: Nick Kennedy <r@nick-kennedy.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘forestmodel-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: forest_model
> ### Title: Produce a forest plot based on a regression model
> ### Aliases: forest_model
> 
> ### ** Examples
... 16 lines ...
+   transmute(time,
+             status,
+             Age = age,
+             Sex = factor(sex, labels = c("Male", "Female")),
+             ECOG = factor(lung$ph.ecog),
+             `Meal Cal` = meal.cal)
> 
> print(forest_model(coxph(Surv(time, status) ~ ., pretty_lung)))
Error in cbind_all(x) : incompatible number of rows (2, expecting 1)
Calls: print ... create_term_data -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted
```

## fractional (0.1.3)
Maintainer: Bill Venables <bill.venables@gmail.com>

0 errors | 0 warnings | 0 notes

## freqweights (1.0.2)
Maintainer: Emilio Torres-Manzanera <torres@uniovi.es>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
.corfreq: no visible global function definition for ‘complete.cases’
.covfreq: no visible global function definition for ‘complete.cases’
.hclustvfreq: no visible global function definition for
  ‘complete.cases’
.quantilefreq : <anonymous>: no visible global function definition for
  ‘approx’
coef.biglmfreq: no visible global function definition for ‘coef’
predict.biglmfreq: no visible global function definition for ‘predict’
predict.lmfreq: no visible global function definition for ‘predict’
summary.lmfreq: no visible global function definition for ‘coef’
summary.lmfreq: no visible global function definition for ‘pt’
summary.lmfreq: no visible global function definition for ‘AIC’
update.biglmfreq: no visible global function definition for ‘update’
Undefined global functions or variables:
  AIC approx coef complete.cases predict pt update
Consider adding
  importFrom("stats", "AIC", "approx", "coef", "complete.cases",
             "predict", "pt", "update")
to your NAMESPACE file.
```

## FRK (0.1.1)
Maintainer: Andrew Zammit-Mangion <andrewzm@gmail.com>  
Bug reports: http://github.com/andrewzm/FRK/issues

1 error  | 2 warnings | 2 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [58s/59s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
    |                                                                            
    |                                                                      |   0%
    |                                                                            
    |===================================                                   |  50%
    |                                                                            
    |======================================================================| 100%
  [1] "Maximum EM iterations reached"
  testthat results ================================================================
  OK: 140 SKIPPED: 1 FAILED: 3
  1. Error: sphere_BAUs (@test_BAUs.R#71) 
  2. Error: SRE sphere works (@test_SRE.R#114) 
  3. Error: SRE space-time sphere works (@test_SRE.R#167) 
  
  Error: testthat unit tests failed
  Execution halted

checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 525-530 (FRK_intro.Rnw) 
Error: processing vignette 'FRK_intro.Rnw' failed with diagnostics:
invalid list argument: all variables should have the same length
Execution halted


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’

Package which this enhances but not available for checking: ‘dggrids’

checking installed package size ... NOTE
  installed size is  7.7Mb
  sub-directories of 1Mb or more:
    data   4.8Mb
    doc    1.6Mb
```

## FSA (0.8.12)
Maintainer: Derek Ogle <derek@derekogle.com>  
Bug reports: https://github.com/droglenc/FSA/issues

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Packages unavailable to check Rd xrefs: ‘alr3’, ‘RMark’, ‘asbio’, ‘PMCMR’, ‘pgirmess’
```

## FSelectorRcpp (0.1.2)
Maintainer: Zygmunt Zawadzki <zygmunt@zstat.pl>  
Bug reports: https://github.com/mi2-warsaw/FSelectorRcpp/issues

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘RTCGA.rnaseq’

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    2.2Mb
    libs   7.3Mb
```

## fueleconomy (0.1)
Maintainer: 'Hadley Wickham' <h.wickham@gmail.com>

0 errors | 0 warnings | 0 notes

## funModeling (1.6.1)
Maintainer: Pablo Casas <pcasas.biz@gmail.com>  
Bug reports: https://github.com/pablo14/funModeling/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘funModeling-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: auto_grouping
> ### Title: Reduce cardinality in categorical variable by automatic grouping
> ### Aliases: auto_grouping
> 
> ### ** Examples
> 
> # Reducing quantity of countries based on has_flu variable
> auto_grouping(data=data_country, input='country', target="has_flu", n_groups=8)
Error in arrange_impl(.data, dots) : incorrect size (1), expecting : 70
Calls: auto_grouping ... as.data.frame -> arrange -> arrange.tbl_df -> arrange_impl -> .Call
Execution halted
```

## funrar (1.0.2)
Maintainer: Matthias Grenié <matthias.grenie@gmail.com>  
Bug reports: https://github.com/Rekyt/funrar/issues

0 errors | 0 warnings | 0 notes

## futureheatwaves (1.0.3)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘futureheatwaves-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: apply_all_models
> ### Title: Apply a function across heat waves from all projections
> ### Aliases: apply_all_models
> 
> ### ** Examples
> 
> ex_results <- system.file("extdata/example_results",
+                           package = "futureheatwaves")
> apply_all_models(ex_results, FUN = average_mean_temp)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted
** found \donttest examples: check also with --run-donttest

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 326-328 (futureheatwaves.Rmd) 
Error: processing vignette 'futureheatwaves.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## fuzzyjoin (0.1.2)
Maintainer: David Robinson <drobinson@stackoverflow.com>

0 errors | 0 warnings | 0 notes

## GADMTools (2.0-1)
Maintainer: Jean Pierre Decorps <jp.decorps@epiconcept.fr>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘GADMTools’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/GADMTools.Rcheck/00install.out’ for details.
```

## gaiah (0.0.2)
Maintainer: Eric C. Anderson <eric.anderson@noaa.gov>

0 errors | 0 warnings | 0 notes

## gapminder (0.2.0)
Maintainer: Jennifer Bryan <jenny@stat.ubc.ca>  
Bug reports: https://github.com/jennybc/gapminder/issues

0 errors | 0 warnings | 0 notes

## GenCAT (1.0.3)
Maintainer: Eric Reed <reeder@bu.edu>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘GenCAT-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: GenCAT
> ### Title: Running GenCAT
> ### Aliases: GenCAT
> ### Keywords: ~kwd1 ~kwd2
> 
... 42 lines ...
rs624673   13  rs624673       NA 19743996  G  A
rs9511877  13 rs9511877       NA 19744070  A  G
rs638773   13  rs638773       NA 19744848  A  G
rs9511880  13 rs9511880       NA 19745096  G  T
rs482278   13  rs482278       NA 19745251  A  G
rs9507552  13 rs9507552       NA 19745903  G  A
> 
> GenCATtest <- GenCAT(CardioMappedSub, genoData=genoData, snpInfo = snpInfo)
Error in `[.data.frame`(SNPdata, , i) : undefined columns selected
Calls: GenCAT -> [ -> [.data.frame
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':
... 6 lines ...
    intersect, setdiff, setequal, union

Loading required package: doParallel
Loading required package: foreach
Loading required package: iterators
Loading required package: parallel
Loading required package: ggplot2
Quitting from lines 62-69 (GenCAT-vignette.Rmd) 
Error: processing vignette 'GenCAT-vignette.Rmd' failed with diagnostics:
undefined columns selected
Execution halted
```

## gender (0.5.1)
Maintainer: Lincoln Mullen <lincoln@lincolnmullen.com>  
Bug reports: https://github.com/ropensci/gender/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘genderdata’
```

## geoknife (1.5.2)
Maintainer: Jordan Read <jread@usgs.gov>  
Bug reports: https://github.com/USGS-R/geoknife/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 360-364 (geoknife.Rmd) 
Error: processing vignette 'geoknife.Rmd' failed with diagnostics:
need finite 'xlim' values
Execution halted

```

## geomnet (0.2.0)
Maintainer: Samantha Tyner <sctyner@iastate.edu>  
Bug reports: https://github.com/sctyner/geomnet/issues

0 errors | 0 warnings | 0 notes

## geoparser (0.1.0)
Maintainer: Maëlle Salmon <maelle.salmon@yahoo.se>  
Bug reports: http://github.com/ropenscilabs/geoparser/issues

0 errors | 0 warnings | 0 notes

## GerminaR (1.1)
Maintainer: Flavio Lozano Isla <flavjack@gmail.com>  
Bug reports: https://github.com/Flavjack/GerminaR/issues

0 errors | 0 warnings | 0 notes

## getCRUCLdata (0.1.4)
Maintainer: Adam Sparks <adamhsparks@gmail.com>  
Bug reports: https://github.com/adamhsparks/getCRUCLdata/issues

0 errors | 0 warnings | 0 notes

## GetHFData (1.2.4)
Maintainer: Marcelo Perlin <marceloperlin@gmail.com>

0 errors | 0 warnings | 0 notes

## ggalt (0.4.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/ggalt/issues

0 errors | 0 warnings | 0 notes

## ggCompNet (0.1.0)
Maintainer: Sam Tyner <sctyner@iastate.edu>  
Bug reports: https://github.com/sctyner/ggCompNet/issues

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: Removed 8 rows containing missing values (geom_segment).
Quitting from lines 617-629 (examples-from-paper.Rmd) 
Error: processing vignette 'examples-from-paper.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted


checking installed package size ... NOTE
  installed size is  6.4Mb
  sub-directories of 1Mb or more:
    doc   6.1Mb
```

## ggenealogy (0.3.0)
Maintainer: Lindsay Rutter <lrutter@iastate.edu>

0 errors | 0 warnings | 0 notes

## ggforce (0.1.1)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggforce-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: geom_arc_bar
> ### Title: Arcs and wedges as polygons
> ### Aliases: geom_arc_bar stat_arc_bar stat_pie
> 
> ### ** Examples
... 27 lines ...
> ggplot() + geom_arc_bar(aes(x0=0, y0=0, r0=r0, r=1, amount=amount,
+                             fill=state, explode=focus),
+                         data=pie, stat='pie') +
+   facet_wrap(~type, ncol=1) +
+   coord_fixed() +
+   theme_no_axes() +
+   scale_fill_brewer('', type='qual')
Warning: Ignoring unknown aesthetics: explode
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted
```

## ggfortify (0.4.1)
Maintainer: Masaaki Horikoshi <sinhrks@gmail.com>  
Bug reports: https://github.com/sinhrks/ggfortify/issues

2 errors | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggfortify-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: autoplot.acf
> ### Title: Autoplot 'stats::acf'. Note to pass 'plot = FALSE' to original
> ###   function to suppress standard plot output
> ### Aliases: autoplot.acf
> 
> ### ** Examples
> 
> autoplot(stats::acf(AirPassengers, plot = FALSE))
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’ [31s/32s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 995 SKIPPED: 9 FAILED: 13
  1. Error: Check ggmultiplot arithmetics (@test-plotlib.R#4) 
  2. Error: Check ggmultiplot extraction (@test-plotlib.R#24) 
  3. Error: Check ggmultiplot multiple instances (@test-plotlib.R#112) 
  4. Error: Check autoplot works for list of ggplot (@test-plotlib.R#209) 
  5. Error: Check autoplot works for list of ggpmultilot (@test-plotlib.R#217) 
  6. Error: autoplot.lm can accept + operator (@test-stats-lm.R#22) 
  7. Error: autoplot.lm works for USArrests (@test-stats-lm.R#34) 
  8. Error: autoplot.lm works for binomial (@test-stats-lm.R#196) 
  9. Error: autoplot.lm works for polynomial fit (@test-stats-lm.R#353) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Loading required package: ggplot2
Quitting from lines 82-85 (basics.Rmd) 
Error: processing vignette 'basics.Rmd' failed with diagnostics:
incorrect size (1), expecting : 31
Execution halted


checking installed package size ... NOTE
  installed size is  5.7Mb
  sub-directories of 1Mb or more:
    doc   5.0Mb
```

## ggguitar (0.1.1)
Maintainer: Casimir Saternos <cas.saternos@gmail.com>  
Bug reports: https://github.com/ezgraphs/ggguitar/issues

0 errors | 0 warnings | 0 notes

## gglogo (0.1.3)
Maintainer: Eric Hare <erichare@iastate.edu>  
Bug reports: https://github.com/heike/gglogo/issues

0 errors | 0 warnings | 0 notes

## ggmap (2.6.1)
Maintainer: David Kahle <david.kahle@gmail.com>  
Bug reports: https://github.com/dkahle/ggmap/issues

0 errors | 0 warnings | 0 notes

## ggmcmc (1.1)
Maintainer: Xavier Fernández i Marín <xavier.fim@gmail.com>  
Bug reports: https://github.com/xfim/ggmcmc/issues

0 errors | 0 warnings | 0 notes

## ggmosaic (0.1.2)
Maintainer: Haley Jeppson <hjeppson@iastate.edu>  
Bug reports: https://github.com/haleyjeppson/ggmosaic

0 errors | 0 warnings | 0 notes

## ggpmisc (0.2.14)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/ggpmisc/issues

0 errors | 0 warnings | 0 notes

## ggRandomForests (2.0.1)
Maintainer: John Ehrlinger <john.ehrlinger@gmail.com>  
Bug reports: https://github.com/ehrlinger/ggRandomForests/issues

0 errors | 0 warnings | 0 notes

## ggraph (1.0.0)
Maintainer: Thomas Lin Pedersen <thomasp85@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘ggraph-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: facet_edges
> ### Title: Create small multiples based on edge attributes
> ### Aliases: facet_edges
> 
> ### ** Examples
... 13 lines ...

> gr <- graph_from_data_frame(highschool)
> 
> ggraph(gr) +
+     geom_edge_link() +
+     geom_node_point() +
+     facet_edges(~year)
Using `nicely` as default layout
Error in order(panels$PANEL) : argument 1 is not a vector
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> <Anonymous> -> f -> order
Execution halted

checking installed package size ... NOTE
  installed size is  6.1Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    libs   2.0Mb
```

## ggraptR (0.1)
Maintainer: Eugene Dubossarsky <eugene@presciient.com>  
Bug reports: https://github.com/cargomoose/raptR/issues

0 errors | 0 warnings | 0 notes

## ggspectra (0.2.0)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/ggspectra

0 errors | 0 warnings | 0 notes

## ggswissmaps (0.1.1)
Maintainer: Sandro Petrillo Burri <gibo.gaf@gmail.com>

0 errors | 0 warnings | 0 notes

## ggvis (0.4.3)
Maintainer: Winston Chang <winston@rstudio.com>

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘ggvis-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: auto_group
> ### Title: Automatically group data by grouping variables
> ### Aliases: auto_group
> 
> ### ** Examples
> 
> # One line
> mtcars %>% ggvis(~disp, ~mpg, stroke = ~factor(cyl)) %>% layer_paths()
> # One line for each level of cyl
> mtcars %>% ggvis(~disp, ~mpg, stroke = ~factor(cyl)) %>% group_by(cyl) %>%
+   layer_paths()
Error in UseMethod("group_by") : 
  no applicable method for 'group_by' applied to an object of class "ggvis"
Calls: %>% ... eval -> _fseq -> freduce -> <Anonymous> -> group_by
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
     }) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:21
  25: env$runWith(self, func) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:25
  26: contextFunc() at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/react.R:113
  27: ..stacktraceon..(expr) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/reactives.R:1480
  28: prop_value(prop, data()) at /tmp/RtmpYXN6qC/devtoolse936a98de1c/shiny/R/conditions.R:331
  29: prop_value.prop_variable(prop, data())
  30: eval(value(x$value), envir = data, enclos = x$env)
  31: eval(expr, envir, enclos)
  
  testthat results ================================================================
  OK: 443 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test-specs.r#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## gistr (0.3.6)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: http://www.github.com/ropensci/gistr/issues

0 errors | 0 warnings | 0 notes

## gitgadget (0.2.1)
Maintainer: Vincent Nijs <vnijs@ucsd.edu>  
Bug reports: https://github.com/vnijs/gitgadget/issues

0 errors | 0 warnings | 0 notes

## gitlabr (0.7)
Maintainer: Jirka Lewandowski <jirka.lewandowski@wzb.eu>  
Bug reports: http://gitlab.points-of-interest.cc/points-of-interest/gitlabr/issues/

0 errors | 0 warnings | 0 notes

## glycanr (0.3.0)
Maintainer: Ivo Ugrina <ivo@iugrina.com>  
Bug reports: https://github.com/iugrina/glycanr/issues

0 errors | 0 warnings | 0 notes

## gogamer (0.4.3)
Maintainer: Kota Mori <kmori05@gmail.com>  
Bug reports: https://github.com/kota7/gogamer/issues

0 errors | 0 warnings | 0 notes

## goldi (1.0.0)
Maintainer: Christopher B. Cole <chris.c.1221@gmail.com>  
Bug reports: https://github.com/Chris1221/goldi/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: expect_equal(goldi(doc = doc, terms = "empty", output = output, lims = lims, log = log, 
         object = TRUE, reader = "local", term_tdm = TDM.go.df), out) at testthat/test-goldi.R:23
  2: compare(object, expected, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-equality.R:52
  3: goldi(doc = doc, terms = "empty", output = output, lims = lims, log = log, object = TRUE, 
         reader = "local", term_tdm = TDM.go.df) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/compare.R:12
  4: tm_map(doc.corpus, content_transformer(tolower), mc.cores = 1)
  5: tm_map.SimpleCorpus(doc.corpus, content_transformer(tolower), mc.cores = 1) at /tmp/RtmptEsk2b/devtools8d9d1c68fc45/tm/R/transform.R:5
  6: FUN(content(x), ...) at /tmp/RtmptEsk2b/devtools8d9d1c68fc45/tm/R/transform.R:27
  
  testthat results ================================================================
  OK: 0 SKIPPED: 0 FAILED: 1
  1. Error: testing main function (@test-goldi.R#23) 
  
  Error: testthat unit tests failed
  Execution halted
```

## googleAnalyticsR (0.3.0)
Maintainer: Mark Edmondson <m@sunholo.com>  
Bug reports: https://github.com/MarkEdmondson1234/googleAnalyticsR/issues

0 errors | 0 warnings | 0 notes

## googlesheets (0.2.1)
Maintainer: Jennifer Bryan <jenny@stat.ubc.ca>  
Bug reports: https://github.com/jennybc/googlesheets/issues

0 errors | 0 warnings | 0 notes

## graphicalVAR (0.1.4)
Maintainer: Sacha Epskamp <mail@sachaepskamp.com>

0 errors | 0 warnings | 0 notes

## graphTweets (0.3.2)
Maintainer: John Coene <jcoenep@gmail.com>  
Bug reports: https://github.com/JohnCoene/graphTweets/issues

0 errors | 0 warnings | 0 notes

## grattan (1.4.1.2)
Maintainer: Hugh Parsonage <hugh.parsonage@gmail.com>  
Bug reports: https://github.com/HughParsonage/grattan/issues

2 errors | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘grattan-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cpi_inflator
> ### Title: CPI inflator
> ### Aliases: cpi_inflator
> 
> ### ** Examples
> 
> cpi_inflator(100, from_fy = "2005-06", to_fy = "2014-15")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [44s/44s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  14: filter(., fy_year == "2016-17")
  15: filter.data.frame(., fy_year == "2016-17") at /home/muelleki/git/R/dplyr/R/manip.r:40
  16: as.data.frame(filter(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:62
  17: filter(tbl_df(.data), ...)
  18: filter.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:40
  19: filter_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:61
  
  testthat results ================================================================
  OK: 197 SKIPPED: 35 FAILED: 3
  1. Error: generic inflator doesn't fail! (@test_generic_inflator.R#5) 
  2. Error: generic inflator gives higher/lower for upper/lower (@test_generic_inflator.R#9) 
  3. Error: New SAPTO matches old SAPTO for SAPTO (@test_sapto.R#83) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Installing package into '/home/muelleki/git/R/dplyr/revdep/checks/grattan.Rcheck'
(as 'lib' is unspecified)
trying URL 'https://hughparsonage.github.io/drat/src/contrib/taxstats_0.0.4.1314.tar.gz'
Content type 'application/octet-stream' length 52654612 bytes (50.2 MB)
==================================================
downloaded 50.2 MB


checking package dependencies ... NOTE
Package suggested but not available for checking: ‘taxstats’
```

## Greg (1.2)
Maintainer: Max Gordon <max@gforge.se>

0 errors | 0 warnings | 0 notes

## groupdata2 (0.1.0)
Maintainer: Ludvig Renbo Olsen <r-pkgs@ludvigolsen.dk>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Loading required package: lme4

Attaching package: 'lmerTest'

The following object is masked from 'package:lme4':

... 8 lines ...

Attaching package: 'zoo'

The following objects are masked from 'package:base':

    as.Date, as.Date.numeric

Quitting from lines 293-301 (introduction_to_groupdata2.Rmd) 
Error: processing vignette 'introduction_to_groupdata2.Rmd' failed with diagnostics:
the table must have a header (column names)
Execution halted
```

## growthcurver (0.2.1)
Maintainer: Kathleen sprouffske <sprouffske@gmail.com>  
Bug reports: https://github.com/sprouffske/growthcurver/issues

0 errors | 0 warnings | 0 notes

## GSODR (1.0.1)
Maintainer: Adam Sparks <adamhsparks@gmail.com>  
Bug reports: https://github.com/ropensci/GSODR/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘plotKML’
```

## gunsales (0.1.2)
Maintainer: Dirk Eddelbuettel <edd@debian.org>

0 errors | 0 warnings | 0 notes

## gutenbergr (0.1.2)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/ropenscilabs/gutenbergr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  
  testthat results ================================================================
  OK: 50 SKIPPED: 0 FAILED: 8
  1. Error: gutenberg_get_mirror works (@test-download.R#13) 
  2. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#22) 
  3. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#23) 
  4. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#31) 
  5. Failure: Can download Charles Dickens' Christmas Carol and Jane Austen's Persuasion (@test-download.R#32) 
  6. Failure: Can download books from a data frame with gutenberg_id column (@test-download.R#43) 
  7. Failure: We can download a file that only has a -8 version (@test-download.R#50) 
  8. Failure: read_zip_url can download and read a zip file (@test-utils.R#7) 
  
  Error: testthat unit tests failed
  Execution halted
```

## gwdegree (0.1.1)
Maintainer: Michael Levy <malevy@ucdavis.edu>  
Bug reports: https://github.com/michaellevy/gwdegree/issues

0 errors | 0 warnings | 0 notes

## harrietr (0.2.2)
Maintainer: Anders Gonçalves da Silva <andersgs@gmail.com>  
Bug reports: https://github.com/andersgs/harrietr/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘ggtree’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## hdr (0.1)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: https://github.com/expersso/hdr

0 errors | 0 warnings | 0 notes

## heemod (0.8.0)
Maintainer: Antoine Filipovic-Pierucci <pierucci@gmail.com>  
Bug reports: https://github.com/pierucci/heemod/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘heemod-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: eval_strategy
> ### Title: Evaluate Strategy
> ### Aliases: eval_strategy
> ### Keywords: internal
> 
... 20 lines ...
+   mod,
+   param,
+   init = c(10, 5),
+   cycles = 5,
+   method = "end",
+   inflow = c(0, 0),
+   strategy_name = "A",
+   expand_limit = c(A = 5, B = 5)
+ )
Error: Cannot convert objects of type `double` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [9s/11s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 278 SKIPPED: 0 FAILED: 26
  1. Error: HIV model works (@test_dmhee.R#98) 
  2. Error: HRT model works (@test_dmhee.R#249) 
  3. Error: Identify Frontier Scenarios (@test_efficiency_frontier.R#15) 
  4. Error: Expansion works. (@test_expand.R#203) 
  5. Error: Translating objects to code (@test_get_code.R#42) 
  6. Error: Demographic analysis (@test_heterogeneity.R#37) 
  7. Error: Heterogeneity analysis (@test_heterogeneity.R#115) 
  8. Failure: Functions on matrix objects (@test_matrix.R#119) 
  9. Error: Functions on matrix objects (@test_matrix.R#127) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 98-105 (a-introduction.Rmd) 
Error: processing vignette 'a-introduction.Rmd' failed with diagnostics:
Cannot convert objects of type `double` to `tidy_quote`
Execution halted

```

## highcharter (0.5.0)
Maintainer: Joshua Kunst <jbkunst@gmail.com>  
Bug reports: https://github.com/jbkunst/highcharter/issues

1 error  | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘highcharter-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: hc_add_series_map
> ### Title: Add a map series
> ### Aliases: hc_add_series_map
> 
> ### ** Examples
... 22 lines ...
+   hc_title(text = "Violent Crime Rates by US State") %>% 
+   hc_subtitle(text = "Source: USArrests data") %>% 
+   hc_add_series_map(usgeojson, USArrests, name = "Murder arrests (per 100,000)",
+                     value = "Murder", joinBy = c("woename", "state"),
+                     dataLabels = list(enabled = TRUE,
+                                       format = '{point.properties.postalcode}')) %>% 
+   hc_colorAxis(stops = color_stops()) %>% 
+   hc_legend(valueDecimals = 0, valueSuffix = "%") %>%
+   hc_mapNavigation(enabled = TRUE) 
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Highcharts (www.highcharts.com) is a Highsoft software product which is
not free for commercial and Governmental use
Quitting from lines 45-46 (charting-data-frames.Rmd) 
Error: processing vignette 'charting-data-frames.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is 16.5Mb
  sub-directories of 1Mb or more:
    doc          13.7Mb
    htmlwidgets   1.9Mb
```

## highlightHTML (0.1.1)
Maintainer: Brandon LeBeau <lebebr01+highlightHTML@gmail.com>  
Bug reports: https://github.com/lebebr01/highlightHTML/issues

0 errors | 0 warnings | 0 notes

## hrbrthemes (0.1.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/hrbrthemes/issues

0 errors | 0 warnings | 0 notes

## HTSSIP (1.0.3)
Maintainer: Nicholas Youngblut <nyoungb2@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘phyloseq’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## hurricaneexposure (0.0.1)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>  
Bug reports: https://github.com/geanders/hurricaneexposure/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘hurricaneexposuredata’
```

## huxtable (0.1.0)
Maintainer: David Hugh-Jones <davidhughjones@gmail.com>  
Bug reports: https://github.com/hughjonesd/huxtable/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         message = handle_message)) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:42
  4: withCallingHandlers(withVisible(code), warning = handle_warning, message = handle_message) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/evaluate-promise.R:130
  5: withVisible(code)
  6: rmarkdown::render("rowheight-multicol-test.Rmd", quiet = TRUE)
  7: convert(output_file, run_citeproc) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/render.R:656
  8: pandoc_convert(utf8_input, pandoc_to, output_format$pandoc$from, output, citeproc, 
         output_format$pandoc$args, !quiet) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/render.R:589
  9: stop("pandoc document conversion failed with error ", result, call. = FALSE) at /tmp/RtmpvuUUir/devtools34505bf56457/rmarkdown/R/pandoc.R:100
  
  testthat results ================================================================
  OK: 9 SKIPPED: 9 FAILED: 1
  1. Error: Row heights do not screw up latex multicol (@test-output.R#42) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 52-103 (design-principles.Rmd) 
Error: processing vignette 'design-principles.Rmd' failed with diagnostics:
argument is of length zero
Execution halted

```

## HydeNet (0.10.5)
Maintainer: Benjamin Nutter <benjamin.nutter@gmail.com>  
Bug reports: https://github.com/nutterb/HydeNet/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘HydeNet-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: HydeNetwork
> ### Title: Define a Probablistic Graphical Network
> ### Aliases: HydeNetwork HydeNetwork.formula HydeNetwork.list
> 
> ### ** Examples
... 44 lines ...
> g3 <- lm(d.dimer ~ pe + pregnant, data=PE)
> g4 <- xtabs(~ pregnant, data=PE)
> g5 <- glm(angio ~ pe, data=PE, family="binomial")
> g6 <- glm(treat ~ d.dimer + angio, data=PE, family="binomial")
> g7 <- glm(death ~ pe + treat, data=PE, family="binomial")
> 
> bagOfModels <- list(g1,g2,g3,g4,g5,g6,g7)
> 
> bagNet <- HydeNetwork(bagOfModels)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [11s/11s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 45 SKIPPED: 0 FAILED: 18
  1. Error: HydeNetwork.list returns expected attributes (@test-HydeNetwork.R#41) 
  2. Error: (unknown) (@test-HydePosterior.R#11) 
  3. Error: (unknown) (@test-bindPosterior.R#12) 
  4. Error: compileJagsModel returns an object of class 'compiledHydeNetwork' (@test-compileJagsModel.R#14) 
  5. Error: cpt.list (@test-cpt.R#12) 
  6. Error: cpt with weights (@test-cpt.R#20) 
  7. Error: modelToNode: multinom (@test-modelToNode.R#17) 
  8. Error: (unknown) (@test-print.HydePosterior.R#11) 
  9. Error: fit the model for dbern (@test-setNode.R#28) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: nnet
Quitting from lines 227-233 (DecisionNetworks.Rmd) 
Error: processing vignette 'DecisionNetworks.Rmd' failed with diagnostics:
cannot convert object to a data frame
Execution halted

```

## hydrostats (0.2.5)
Maintainer: Nick Bond <n.bond@latrobe.edu.au>

0 errors | 0 warnings | 0 notes

## hypothesisr (0.1.1)
Maintainer: Matthew Lincoln <matthew.d.lincoln@gmail.com>  
Bug reports: https://github.com/mdlincoln/hypothesisr/issues

0 errors | 0 warnings | 0 notes

## IAT (0.3)
Maintainer: Dan Martin <dpmartin42@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘IAT-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: cleanIAT
> ### Title: Clean IAT data using the updated D-Scoring algorithm
> ### Aliases: cleanIAT
> 
> ### ** Examples
> 
> # Get Ps who recieve Math-Male sorting task in first blocks
> 
> cong_first <- IATData[IATData$isCongruentFirst == 1, ]
> 
> dscore_first <- cleanIAT(my_data = cong_first,
+                          block_name = "BLOCK_NAME_S",
+                          trial_blocks = c("BLOCK2", "BLOCK3", "BLOCK5", "BLOCK6"),
+                          session_id = "SESSION_ID",
+                          trial_latency = "TRIAL_LATENCY",
+                          trial_error = "TRIAL_ERROR",
+                          v_error = 1, v_extreme = 2, v_std = 1)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted
```

## IATscores (0.1-2)
Maintainer: Giulio Costantini <costantinigiulio@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘nem’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## idbr (0.2)
Maintainer: Kyle Walker <kyle.walker@tcu.edu>

0 errors | 0 warnings | 0 notes

## IDmining (1.0.0)
Maintainer: Jean Golay <Jean.Golay@unil.ch>

0 errors | 0 warnings | 0 notes

## imager (0.40.1)
Maintainer: Simon Barthelme <simon.barthelme@gipsa-lab.fr>  
Bug reports: https://github.com/dahtah/imager/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 42.1Mb
  sub-directories of 1Mb or more:
    data      1.4Mb
    doc       4.2Mb
    extdata   1.0Mb
    include   2.8Mb
    libs     32.2Mb
```

## imfr (0.1.3)
Maintainer: Christopher Gandrud <christopher.gandrud@gmail.com>  
Bug reports: https://github.com/christophergandrud/imfr/issues

0 errors | 0 warnings | 0 notes

## IMP (1.1)
Maintainer: Anup Nair <nairanup50695@gmail.com>

0 errors | 0 warnings | 0 notes

## imputeTestbench (3.0.0)
Maintainer: Marcus W. Beck <mbafs2012@gmail.com>  
Bug reports: https://github.com/neerajdhanraj/imputeTestbench/issues

0 errors | 0 warnings | 0 notes

## incadata (0.5.3)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://www.bitbucket.org/cancercentrum/incadata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > library(testthat)
  > library(incadata)
  > 
  > test_check("incadata")
  1. Failure: filter (@test-dplyr_methods.R#13) ----------------------------------
  dplyr::filter(testdata, persnr == 198505043334) inherits from `tbl_df/tbl/data.frame` not `incadata`.
  
  
  [1] "ex_databbfd2a8db8bb.csv2"
  testthat results ================================================================
  OK: 40 SKIPPED: 0 FAILED: 1
  1. Failure: filter (@test-dplyr_methods.R#13) 
  
  Error: testthat unit tests failed
  Execution halted
```

## IncucyteDRC (0.5.4)
Maintainer: Phil Chapman <phil.chapman@cruk.manchester.ac.uk>  
Bug reports: https://github.com/chapmandu2/IncucyteDRC/issues

0 errors | 0 warnings | 0 notes

## inferr (0.1.0)
Maintainer: Aravind Hebbali <hebbali.aravind@gmail.com>  
Bug reports: https://github.com/rsquaredacademy/inferr/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
... 14 lines ...
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## InformativeCensoring (0.3.4)
Maintainer: Jonathan Bartlett <jonathan.bartlett1@astrazeneca.com>

0 errors | 0 warnings | 0 notes

## infuser (0.2.5)
Maintainer: Bart Smeets <bartsmeets86@gmail.com>  
Bug reports: https://github.com/bart6114/infuser/issues

0 errors | 0 warnings | 0 notes

## inlmisc (0.2.4)
Maintainer: Jason C. Fisher <jfisher@usgs.gov>  
Bug reports: https://github.com/USGS-R/inlmisc/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘inlmisc’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/inlmisc.Rcheck/00install.out’ for details.
```

## internetarchive (0.1.6)
Maintainer: Lincoln Mullen <lincoln@lincolnmullen.com>  
Bug reports: https://github.com/ropensci/internetarchive/issues

0 errors | 0 warnings | 0 notes

## interplot (0.1.5)
Maintainer: Yue Hu <yue-hu-1@uiowa.edu>

0 errors | 0 warnings | 0 notes

## ipft (0.2.8)
Maintainer: Emilio Sansano <esansano@uji.es>

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
             user system elapsed
ipfProb     9.936  2.800  13.064
ipfEstimate 7.156  1.668   9.141
```

## IRISMustangMetrics (2.0.4)
Maintainer: Gillian Sharer <gillian@iris.washington.edu>

0 errors | 0 warnings | 0 notes

## JacobiEigen (0.2-2)
Maintainer: Bill Venables <Bill.Venables@gmail.com>

0 errors | 0 warnings | 0 notes

## janeaustenr (0.1.4)
Maintainer: Julia Silge <julia.silge@gmail.com>  
Bug reports: https://github.com/juliasilge/janeaustenr/issues

0 errors | 0 warnings | 0 notes

## janitor (0.2.1)
Maintainer: Sam Firke <samuel.firke@gmail.com>  
Bug reports: https://github.com/sfirke/janitor/issues

0 errors | 0 warnings | 0 notes

## jpmesh (0.3.0)
Maintainer: Shinya Uryu <suika1127@gmail.com>  
Bug reports: https://github.com/uribo/jpmesh/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘jpmesh’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/jpmesh.Rcheck/00install.out’ for details.
```

## kntnr (0.4.0)
Maintainer: Hiroaki Yutani <yutani.ini@gmail.com>

0 errors | 0 warnings | 0 notes

## kokudosuuchi (0.2.0)
Maintainer: Hiroaki Yutani <yutani.ini@gmail.com>  
Bug reports: https://github.com/yutannihilation/kokudosuuchi/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [2s/424s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2: rgdal::ogrListLayers
  3: getExportedValue(pkg, name)
  4: asNamespace(ns)
  5: getNamespace(ns)
  6: tryCatch(loadNamespace(name), error = function(e) stop(e))
  7: tryCatchList(expr, classes, parentenv, handlers)
  8: tryCatchOne(expr, names, parentenv, handlers[[1L]])
  9: value[[3L]](cond)
  
  testthat results ================================================================
  OK: 1 SKIPPED: 0 FAILED: 1
  1. Error: getKSJData works (@test-getData.R#4) 
  
  Error: testthat unit tests failed
  Execution halted
```

## KraljicMatrix (0.1.1)
Maintainer: Bradley Boehmke <bradleyboehmke@gmail.com>  
Bug reports: https://github.com/bradleyboehmke/KraljicMatrix/issues

0 errors | 0 warnings | 0 notes

## labelled (1.0.0)
Maintainer: Joseph Larmarange <joseph@larmarange.net>  
Bug reports: https://github.com/larmarange/labelled/issues

0 errors | 0 warnings | 2 notes

```
checking package dependencies ... NOTE
Package which this enhances but not available for checking: ‘memisc’

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘memisc’
```

## Lahman (5.0-0)
Maintainer: Chris Dalzell <cdalzell@gmail.com>

0 errors | 1 warning  | 1 note 

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
Examples with CPU or elapsed time > 5s
        user system elapsed
Teams 11.112  2.624  14.195

checking installed package size ... NOTE
  installed size is  7.6Mb
  sub-directories of 1Mb or more:
    data   7.4Mb
```

## laketemps (0.5.1)
Maintainer: Jordan S Read <jread@usgs.gov>  
Bug reports: https://github.com/USGS-R/laketemps/issues

0 errors | 0 warnings | 0 notes

## LBSPR (0.1.0)
Maintainer: Adrian Hordyk <ar.hordyk@gmail.com>  
Bug reports: https://github.com/AdrianHordyk/LBSPR/issues

0 errors | 0 warnings | 0 notes

## LendingClub (1.0.2)
Maintainer: Ryan Kuhn <kuhnrl30@gmail.com>  
Bug reports: https://github.com/kuhnrl30/LendingClub/issues

0 errors | 0 warnings | 0 notes

## lexRankr (0.4.0)
Maintainer: Adam Spannbauer <spannbaueradam@gmail.com>

0 errors | 0 warnings | 0 notes

## livechatR (0.1.0)
Maintainer: Lawrence Wu <lwu@payoff.com>

0 errors | 0 warnings | 0 notes

## lmeresampler (0.1.0)
Maintainer: Adam Loy <loyad01@gmail.com>  
Bug reports: https://github.com/aloy/lmeresampler/issues

0 errors | 0 warnings | 0 notes

## LocFDRPois (1.0.0)
Maintainer: Kris Sankaran <kriss1@stanford.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
AnalyticalOptim: no visible global function definition for ‘optim’
LLConstructor : LL: no visible global function definition for ‘dpois’
MixtureDensity: no visible global function definition for ‘glm’
MixtureDensity : f_hat: no visible global function definition for
  ‘predict’
NullDensity : f0: no visible global function definition for ‘dpois’
Undefined global functions or variables:
  dpois glm optim predict
Consider adding
  importFrom("stats", "dpois", "glm", "optim", "predict")
to your NAMESPACE file.
```

## longurl (0.3.0)
Maintainer: Bob Rudis <bob@rud.is>

0 errors | 0 warnings | 0 notes

## lookupTable (0.1)
Maintainer: Enzo Jia <enzo.jia@gmail.com>

0 errors | 0 warnings | 0 notes

## loopr (1.0.1)
Maintainer: Brandon Taylor <Brandon.Taylor221@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
amendColumns: no visible global function definition for ‘setNames’
fillColumns: no visible global function definition for ‘setNames’
Undefined global functions or variables:
  setNames
Consider adding
  importFrom("stats", "setNames")
to your NAMESPACE file.
```

## lplyr (0.1.6)
Maintainer: Paul Poncet <paulponcet@yahoo.fr>  
Bug reports: https://github.com/paulponcet/lplyr/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘lplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: mutate_.list
> ### Title: Dplyr verbs for lists and pairlists
> ### Aliases: mutate_.list mutate_.pairlist rename_.list rename_.pairlist
> ###   select_.list select_.pairlist transmute_.list transmute_.pairlist
> 
> ### ** Examples
> 
> xs <- list(x1 = 1:3, 
+            x2 = 2:5, 
+            x3 = list("alpha", c("beta", "gamma")))
> 
> # Non-standard evaluation
> mutate(xs, x4 = 4)
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "list"
Calls: mutate
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 7 lines ...

Attaching package: 'lplyr'

The following object is masked from 'package:dplyr':

    pull

Quitting from lines 22-30 (lplyr-vignette.Rmd) 
Error: processing vignette 'lplyr-vignette.Rmd' failed with diagnostics:
no applicable method for 'mutate' applied to an object of class "list"
Execution halted
```

## lucid (1.4)
Maintainer: Kevin Wright <kw.stat@gmail.com>  
Bug reports: https://github.com/kwstat/lucid/issues

0 errors | 0 warnings | 0 notes

## lvnet (0.3)
Maintainer: Sacha Epskamp <mail@sachaepskamp.com>

0 errors | 0 warnings | 0 notes

## macleish (0.3.0)
Maintainer: Ben Baumer <ben.baumer@gmail.com>

0 errors | 0 warnings | 0 notes

## makeFlow (1.0.2)
Maintainer: Alex J. Krebs <Krebs.AlexJ@gmail.com>

0 errors | 0 warnings | 0 notes

## manifestoR (1.2.3)
Maintainer: Jirka Lewandowski <jirka.lewandowski@wzb.eu>  
Bug reports: https://github.com/ManifestoProject/manifestoR/issues

0 errors | 0 warnings | 0 notes

## mason (0.2.5)
Maintainer: Luke Johnston <lwjohnst@gmail.com>  
Bug reports: https://github.com/lwjohnst86/mason/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘mason-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: polish
> ### Title: Do some final polishing of the scrubbed mason analysis data.
> ### Aliases: polish polish_adjust_pvalue polish_filter polish_renaming
> ###   polish_transform_estimates
> 
... 6 lines ...
+  add_settings() %>%
+  add_variables('yvar', c('Fertility', 'Education')) %>%
+  add_variables('xvar', c('Agriculture', 'Catholic')) %>%
+  add_variables('covariates', 'Examination') %>%
+  construct() %>%
+  scrub()
> polish_renaming(ds, function(x) gsub('Education', 'Schooling', x))
Warning: 'dplyr::mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: All select() inputs must resolve to integer column positions. The following do not:
```

## MazamaSpatialUtils (0.4.8)
Maintainer: Jonathan Callahan <jonathan.s.callahan@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘MazamaSpatialUtils’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/MazamaSpatialUtils.Rcheck/00install.out’ for details.
```

## mdsr (0.1.3)
Maintainer: Ben Baumer <ben.baumer@gmail.com>  
Bug reports: https://github.com/beanumber/mdsr/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘mdsr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: make_babynames_dist
> ### Title: Wrangle babynames data
> ### Aliases: make_babynames_dist
> 
> ### ** Examples
> 
> 
> BabynameDist <- make_babynames_dist()
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking installed package size ... NOTE
  installed size is  5.4Mb
  sub-directories of 1Mb or more:
    data   5.3Mb
```

## medicare (0.2.0)
Maintainer: Robert Gambrel <robert.gambrel@gmail.com>  
Bug reports: http://www.github.com/robertgambrel/medicare/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag
... 8 lines ...
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Joining, by = "rpt_rec_num"
Quitting from lines 197-222 (cost-report-analysis.Rmd) 
Error: processing vignette 'cost-report-analysis.Rmd' failed with diagnostics:
argument 1 is not a vector
Execution halted
```

## merTools (0.3.0)
Maintainer: Jared E. Knowles <jknowles@gmail.com>  
Bug reports: https://www.github.com/jknowles/merTools

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘merTools-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plotREsim
> ### Title: Plot the results of a simulation of the random effects
> ### Aliases: plotREsim
> 
> ### ** Examples
> 
>  require(ggplot2); require(lme4);
Loading required package: ggplot2
>  fm1 <- lmer(Reaction ~ Days + (Days | Subject), sleepstudy)
>  (p1 <- plotREsim(REsim(fm1)))
>  #Plot just the random effects for the Days slope
>  (p2 <- plotREsim(REsim(fm1)[REsim(fm1)$term=="Days",]))
Error in `$<-.data.frame`(`*tmp*`, "PANEL", value = c(1L, 1L)) : 
  replacement has 2 rows, data has 1
Calls: <Anonymous> ... <Anonymous> -> f -> <Anonymous> -> f -> $<- -> $<-.data.frame
Execution halted
```

## metacoder (0.1.2)
Maintainer: Zachary Foster <zacharyfoster1989@gmail.com>

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  result$taxon_data$taxon_info_2 not equal to c(NA, NA, NA, "9639", "9688").
  target is NULL, current is character
  
  
  testthat results ================================================================
  OK: 155 SKIPPED: 5 FAILED: 5
  1. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#111) 
  2. Failure: Taxon info columns from key are added (@test--extract_taxonomy.R#112) 
  3. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#135) 
  4. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#136) 
  5. Failure: Taxon info columns from both key and class are added (@test--extract_taxonomy.R#137) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
1000 inputs used to classify 1000 observations by 955 taxa.
Quitting from lines 72-73 (introduction.Rmd) 
Error: processing vignette 'introduction.Rmd' failed with diagnostics:
cannot convert object to a data frame
Execution halted

```

## MetaComp (1.0.1)
Maintainer: Pavel Senin <psenin@lanl.gov>  
Bug reports: https://github.com/seninp-bioinfo/MetaComp/issues

0 errors | 0 warnings | 0 notes

## metricsgraphics (0.9.0)
Maintainer: Bob Rudis <bob@rudis.net>  
Bug reports: https://github.com/hrbrmstr/metricsgraphics/issues

0 errors | 0 warnings | 0 notes

## metricTester (1.3.2)
Maintainer: Eliot Miller <eliot.isaac@gmail.com>  
Bug reports: https://github.com/eliotmiller/metricTester/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## mglR (0.1.0)
Maintainer: Katherine Hartmann <klh64@cornell.edu>

0 errors | 0 warnings | 0 notes

## MIAmaxent (0.4.0)
Maintainer: Julien Vollering <julien.vollering@hvl.no>  
Bug reports: https://github.com/julienvollering/MIAmaxent/issues

0 errors | 0 warnings | 0 notes

## mixOmics (6.1.2)
Maintainer: Kim-Anh Le Cao <k.lecao@uq.edu.au>  
Bug reports: https://bitbucket.org/klecao/package-mixomics/issues

0 errors | 0 warnings | 0 notes

## mixpack (0.3.6)
Maintainer: Marc Comas-Cufí <mcomas@imae.udg.edu>

0 errors | 0 warnings | 0 notes

## mlVAR (0.3.2)
Maintainer: Sacha Epskamp <mail@sachaepskamp.com>

0 errors | 0 warnings | 0 notes

## modelr (0.1.0)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/modelr/issues

0 errors | 0 warnings | 0 notes

## Momocs (1.1.0)
Maintainer: Vincent Bonhomme <bonhomme.vincent@gmail.com>  
Bug reports: https://github.com/vbonhomme/Momocs/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘Momocs-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: PCcontrib
> ### Title: Shape variation along PC axes
> ### Aliases: PCcontrib PCcontrib.PCA
> 
> ### ** Examples
> 
> data(bot)
> bot.p <- PCA(efourier(bot, 12))
> PCcontrib(bot.p)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
This is Momocs 1.1.0

Attaching package: 'Momocs'

The following object is masked from 'package:stats':

    filter

The following object is masked from 'package:base':

    table

method not provided. efourier is used
method not provided. efourier is used
method not provided. efourier is used
Quitting from lines 227-229 (Momocs_speed_dating.Rmd) 
Error: processing vignette 'Momocs_speed_dating.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## MonetDBLite (0.3.1)
Maintainer: Hannes Muehleisen <hannes@cwi.nl>  
Bug reports: https://github.com/hannesmuehleisen/MonetDBLite/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  7.0Mb
  sub-directories of 1Mb or more:
    libs   6.7Mb
```

## monkeylearn (0.1.1)
Maintainer: Maëlle Salmon <maelle.salmon@yahoo.se>  
Bug reports: http://github.com/ropenscilabs/monkeylearn/issues

0 errors | 0 warnings | 0 notes

## morse (2.2.0)
Maintainer: Philippe Veber <philippe.veber@univ-lyon1.fr>

0 errors | 0 warnings | 0 notes

## mosaicData (0.14.0)
Maintainer: Randall Pruim <rpruim@calvin.edu>

0 errors | 0 warnings | 0 notes

## mosaic (0.14.4)
Maintainer: Randall Pruim <rpruim@calvin.edu>  
Bug reports: https://github.com/ProjectMOSAIC/mosaic/issues

0 errors | 0 warnings | 2 notes

```
checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    R     2.2Mb
    doc   6.9Mb

checking R code for possible problems ... NOTE
mMap: no visible binding for global variable ‘facet’
mScatter: no visible binding for global variable ‘facet’
mUniplot: no visible binding for global variable ‘facet’
Undefined global functions or variables:
  facet
```

## mousetrap (3.0.0)
Maintainer: Pascal J. Kieslich <kieslich@psychologie.uni-mannheim.de>  
Bug reports: https://github.com/pascalkieslich/mousetrap/issues

0 errors | 0 warnings | 0 notes

## mplot (0.7.9)
Maintainer: Garth Tarr <garth.tarr@gmail.com>

0 errors | 0 warnings | 0 notes

## mpoly (1.0.3)
Maintainer: David Kahle <david.kahle@gmail.com>  
Bug reports: https://github.com/dkahle/mpoly/issues

0 errors | 0 warnings | 0 notes

## mrgsolve (0.8.2)
Maintainer: Kyle T Baron  <kyleb@metrumrg.com>  
Bug reports: https://github.com/metrumresearchgroup/mrgsolve/issues

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘mrgsolve-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: data_set
> ### Title: Select and modify a data set for simulation.
> ### Aliases: data_set data_set,mrgmod,data.frame-method
> ###   data_set,mrgmod,ANY-method data_set,mrgmod,missing-method
> 
> ### ** Examples
> 
> 
> mod <- mrgsolve:::house()
> 
> data <- expand.ev(ID=1:3, amt=c(10,20))
> 
> mod %>% data_set(data, ID > 1) %>% mrgsim
Error: Cannot convert objects of type `environment` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [18s/22s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  5: as.vector(object)
  6: lim(out10, CENT > 0)
  7: x %>% dplyr::filter(...) %>% as.data.frame at testthat/test-z-alag-f.R:24
  8: withVisible(eval(quote(`_fseq`(`_lhs`)), env, env)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:6
  9: eval(quote(`_fseq`(`_lhs`)), env, env)
  10: eval(expr, envir, enclos)
  11: `_fseq`(`_lhs`)
  12: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  13: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  14: dplyr::filter(., ...)  ... and 1 more
  
  
  DONE ===========================================================================
  Error: Test failures
  Execution halted

checking installed package size ... NOTE
  installed size is  6.3Mb
  sub-directories of 1Mb or more:
    libs   5.0Mb
```

## mscstexta4r (0.1.2)
Maintainer: Phil Ferriere <pferriere@hotmail.com>  
Bug reports: http://www.github.com/philferriere/mscstexta4r/issues

0 errors | 0 warnings | 0 notes

## mtconnectR (1.0.1)
Maintainer: Subramanyam Ravishankar <subramanyam@systeminsights.com>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘mtconnectR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: create_mtc_device_from_dmtcd
> ### Title: Create MTCDevice class from Delimited MTC Data and log file
> ### Aliases: create_mtc_device_from_dmtcd
> 
> ### ** Examples
... 105 lines ...
  |                                                                            
  |==================================================================    |  94%
  |                                                                            
  |===================================================================   |  96%
  |                                                                            
  |===================================================================== |  98%
  |                                                                            
  |======================================================================| 100%
Error in arrange_impl(.data, dots) : incorrect size (1), expecting : 251
Calls: create_mtc_device_from_dmtcd ... as.data.frame -> arrange -> arrange.tbl_df -> arrange_impl -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [16s/17s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  argument is of length zero
  1: na.omit(simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, 
         data_type = "HH")) at testthat/test-simulate_gcode.R:6
  2: simulate_data_from_gcode(example_gcode_parsed, start_time = 0, data_res = 0.1, data_type = "HH")
  3: update_timestamps(current_values, previous_values)
  4: pfr_time(current_values, previous_values, current_values$pfr, current_values$state_motion)
  
  testthat results ================================================================
  OK: 24 SKIPPED: 0 FAILED: 3
  1. Error: (unknown) (@test-dataExtraction_mtc.R#12) 
  2. Failure: (unknown) (@test-map_gcode_mtc.R#16) 
  3. Error: (unknown) (@test-simulate_gcode.R#6) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 21-30 (create_mtc_device.Rmd) 
Error: processing vignette 'create_mtc_device.Rmd' failed with diagnostics:
incorrect size (1), expecting : 251
Execution halted

```

## mudata (0.1)
Maintainer: Dewey Dunnington <dewey@fishandwhistle.net>  
Bug reports: https://github.com/paleolimbot/mudata/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘test-all.R’ [15s/16s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
             return(TRUE)
         }
         else {
             return(FALSE)
         }
     }, kentvillegreenwood$data$flags, reexpanded$data$flags, USE.NAMES = FALSE)
  
  testthat results ================================================================
  OK: 187 SKIPPED: 0 FAILED: 3
  1. Error: expand.tags behaviour in constructor is consistent (@test_mudata.R#75) 
  2. Error: expand/condense functions work properly for data frames (@test_tags.R#10) 
  3. Error: expand/condense tags work properly for mudata objects (@test_tags.R#30) 
  
  Error: testthat unit tests failed
  Execution halted
```

## muir (0.1.0)
Maintainer: Justin Alford <justin.alford@gmail.com>  
Bug reports: https://github.com/alforj/muir/issues

0 errors | 0 warnings | 0 notes

## myTAI (0.4.0)
Maintainer: Hajk-Georg Drost <hgd23@cam.ac.uk>  
Bug reports: https://github.com/HajkD/myTAI/issues

0 errors | 1 warning  | 1 note 

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.

checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    data   2.0Mb
    doc    2.4Mb
```

## nasadata (0.9.0)
Maintainer: Eduardo Flores <eduardo@enelmargen.org>

0 errors | 0 warnings | 0 notes

## ncappc (0.2.1.1)
Maintainer: Chayan Acharya <chayan.acharya@farmbio.uu.se>

0 errors | 0 warnings | 0 notes

## ncar (0.3.1)
Maintainer: Sungpil Han <shan@acp.kr>  
Bug reports: https://github.com/asancpt/ncar/issues

0 errors | 0 warnings | 0 notes

## NestedCategBayesImpute (1.0.0)
Maintainer: Quanli Wang <quanli@stat.duke.edu>

0 errors | 0 warnings | 0 notes

## networkreporting (0.1.1)
Maintainer: Dennis M. Feehan <feehan@berkeley.edu>

0 errors | 0 warnings | 0 notes

## NetworkRiskMeasures (0.1.1)
Maintainer: Carlos Cinelli <carloscinelli@hotmail.com>

0 errors | 0 warnings | 0 notes

## neurobase (1.13.2)
Maintainer: John Muschelli <muschellij2@gmail.com>  
Bug reports: https://github.com/muschellij2/nifti/issues

0 errors | 0 warnings | 0 notes

## neuropsychology (0.3.0)
Maintainer: Dominique Makowski <dom.makowski@gmail.com>  
Bug reports: https://github.com/neuropsychology/neuropsychology.R/issues

0 errors | 0 warnings | 0 notes

## NFP (0.99.2)
Maintainer: Yang Cao <yiluheihei@gmail.com>  
Bug reports: https://github.com/yiluheihei/NFP/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘KEGGgraph’

Packages suggested but not available for checking: ‘graphite’ ‘NFPdata’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## nlstimedist (1.0.4)
Maintainer: Nathan Eastwood <nathan.eastwood@icloud.com>  
Bug reports: https://github.com/nathaneastwood/nlstimedist/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘nlstimedist-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: tdCdfPlot
> ### Title: Plot the timedist PDF or CDF
> ### Aliases: tdCdfPlot tdPdfPlot
> 
> ### ** Examples
> 
> tdTilia <- tdData(tilia, x = "Day", y = "Trees")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  17: coerce_type(lazy, "tidy_quote", quote = lazy, symbol = , language = quosure(lazy, 
         env), string = {
         if (warn) 
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  18: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  19: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 6 SKIPPED: 0 FAILED: 1
  1. Error: tdData output is as expected (@test-tdData.R#5) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 28-30 (nlstimedist.Rmd) 
Error: processing vignette 'nlstimedist.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## notifyme (0.3.0)
Maintainer: James Black <james@epijim.uk>  
Bug reports: https://github.com/epijim/notifyme/issues

0 errors | 0 warnings | 0 notes

## NPC (1.1.0)
Maintainer: Devin Caughey <devin.caughey@gmail.com>

0 errors | 0 warnings | 0 notes

## nullabor (0.3.1)
Maintainer: Di Cook <dicook@iastate.edu>

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
box_dist: no visible global function definition for ‘quantile’
calc_mean_dist: no visible global function definition for ‘filter’
n: no visible global function definition for ‘resid’
null_dist : <anonymous>: no visible global function definition for
  ‘coef’
null_lm : <anonymous>: no visible global function definition for
  ‘predict’
reg_dist: no visible global function definition for ‘coef’
reg_dist: no visible global function definition for ‘lm’
... 10 lines ...
sep_dist: no visible global function definition for ‘cutree’
sep_dist: no visible global function definition for ‘hclust’
uni_dist: no visible global function definition for ‘sd’
Undefined global functions or variables:
  coef cutree dist filter hclust lm predict quantile rbinom resid rnorm
  sd update
Consider adding
  importFrom("stats", "coef", "cutree", "dist", "filter", "hclust", "lm",
             "predict", "quantile", "rbinom", "resid", "rnorm", "sd",
             "update")
to your NAMESPACE file.
```

## nycflights13 (0.2.2)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/nycflights13/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  7.0Mb
  sub-directories of 1Mb or more:
    data   6.9Mb
```

## nzelect (0.3.0)
Maintainer: Peter Ellis <peter.ellis2013nz@gmail.com>

0 errors | 0 warnings | 0 notes

## observer (0.1.2)
Maintainer: Paul Poncet <paulponcet@yahoo.fr>  
Bug reports: https://github.com/paulponcet/observer/issues

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘validate’
```

## officer (0.1.1)
Maintainer: David Gohel <david.gohel@ardata.fr>  
Bug reports: https://github.com/davidgohel/officer/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘officer-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: slip_in_img
> ### Title: append an image
> ### Aliases: slip_in_img
> 
> ### ** Examples
> 
> library(magrittr)
> img.file <- file.path( Sys.getenv("R_HOME"), "doc", "html", "logo.jpg" )
> x <- read_docx() %>%
+   body_add_par("R logo: ", style = "Normal") %>%
+   slip_in_img(src = img.file, style = "strong", width = .3, height = .3)
Error: file.exists(src) is not TRUE
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [23s/25s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  9: ph_with_img(., type = "body", src = img.file, height = 1.06, width = 1.39)
  10: external_img(src, width = width, height = height)
  11: stopifnot(file.exists(src))
  12: stop(sprintf(ngettext(length(r), "%s is not TRUE", "%s are not all TRUE"), ch), call. = FALSE, 
         domain = NA)
  
  testthat results ================================================================
  OK: 317 SKIPPED: 0 FAILED: 4
  1. Error: image add  (@test-docx-add.R#68) 
  2. Error: pml fp_border (@test-fp_cell.R#75) 
  3. Error: css fp_border (@test-fp_cell.R#165) 
  4. Error: add img into placeholder (@test-pptx-add.R#67) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 181-191 (powerpoint.Rmd) 
Error: processing vignette 'powerpoint.Rmd' failed with diagnostics:
file.exists(src) is not TRUE
Execution halted

```

## openadds (0.2.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/sckott/openadds/issues

0 errors | 0 warnings | 0 notes

## openair (2.0-0)
Maintainer: David Carslaw <david.carslaw@york.ac.uk>  
Bug reports: https://github.com/davidcarslaw/openair/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## opencage (0.1.0)
Maintainer: Maëlle Salmon <maelle.salmon@yahoo.se>  
Bug reports: http://github.com/ropenscilabs/opencage/issues

0 errors | 0 warnings | 0 notes

## packagetrackr (0.1.1)
Maintainer: Jirka Lewandowski <jirka.lewandowski@wzb.eu>  
Bug reports: http://gitlab.points-of-interest.cc/points-of-interest/packagetrackr/issues

0 errors | 0 warnings | 0 notes

## PAC (1.0.7)
Maintainer: Ye Henry Li <ywli@stanford.edu>

0 errors | 0 warnings | 0 notes

## padr (0.2.1)
Maintainer: Edwin Thoen <edwinthoen@gmail.com>  
Bug reports: https://github.com/EdwinTh/padr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [132s/136s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  tests/testthat/test-window.R:74:8: style: Variable and function names should be all lowercase.
      ), sqlText),
         ^~~~~~~
  tests/testthat/test-window.R:76:12: style: Variable and function names should be all lowercase.
      info = sqlText
             ^~~~~~~
  
  
  testthat results ================================================================
  OK: 192 SKIPPED: 0 FAILED: 1
  1. Failure: Package Style (@test_zzz_lintr.R#5) 
  
  Error: testthat unit tests failed
  Execution halted
```

## parsemsf (0.1.0)
Maintainer: Benjamin Jack <benjamin.r.jack@gmail.com>  
Bug reports: https://github.com/benjaminjack/parsemsf/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘parsemsf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: make_area_table
> ### Title: Make a table of peptide areas
> ### Aliases: make_area_table
> 
> ### ** Examples
... 6 lines ...

Traceback:
 1: .Call("dplyr_select_impl", PACKAGE = "dplyr", df, vars)
 2: select_impl(.data, vars)
 3: select.grouped_df(.data, !(!(!dots)))
 4: select_.grouped_df(pep_table, .dots = setNames(old_names, new_names))
 5: select_(pep_table, .dots = setNames(old_names, new_names))
 6: make_pep_table(msf_file, min_conf, prot_regex, collapse)
 7: make_area_table(parsemsf_example("test_db.msf"))
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  34: test_code(NULL, exprs, env)
  35: source_file(path, new.env(parent = env), chdir = TRUE, encoding = encoding,     wrap = wrap)
  36: force(code)
  37: with_reporter(reporter = reporter, start_end_reporter = start_end_reporter,     {        lister$start_file(basename(path))        source_file(path, new.env(parent = env), chdir = TRUE,             encoding = encoding, wrap = wrap)        end_context()    })
  38: FUN(X[[i]], ...)
  39: lapply(paths, test_file, env = env, reporter = current_reporter,     start_end_reporter = FALSE, load_helpers = FALSE, encoding = encoding,     wrap = TRUE)
  40: force(code)
  41: with_reporter(reporter = current_reporter, results <- lapply(paths,     test_file, env = env, reporter = current_reporter, start_end_reporter = FALSE,     load_helpers = FALSE, encoding = encoding, wrap = TRUE))
  42: test_files(paths, reporter = reporter, env = env, encoding = encoding)
  43: test_dir(test_path, reporter = reporter, env = env, filter = filter,     ...)
  44: with_top_env(env, {    test_dir(test_path, reporter = reporter, env = env, filter = filter,         ...)})
  45: run_tests(package, test_path, filter, reporter, ...)
  46: test_check("parsemsf")
  An irrecoverable exception occurred. R is aborting now ...
  Segmentation fault (core dumped)

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
10: withVisible(eval(expr, envir, enclos))
11: withCallingHandlers(withVisible(eval(expr, envir, enclos)), warning = wHandler,     error = eHandler, message = mHandler)
12: handle(ev <- withCallingHandlers(withVisible(eval(expr, envir,     enclos)), warning = wHandler, error = eHandler, message = mHandler))
13: timing_fn(handle(ev <- withCallingHandlers(withVisible(eval(expr,     envir, enclos)), warning = wHandler, error = eHandler, message = mHandler)))
14: evaluate_call(expr, parsed$src[[i]], envir = envir, enclos = enclos,     debug = debug, last = i == length(out), use_try = stop_on_error !=         2L, keep_warning = keep_warning, keep_message = keep_message,     output_handler = output_handler, include_timing = include_timing)
15: evaluate(code, envir = env, new_device = FALSE, keep_warning = !isFALSE(options$warning),     keep_message = !isFALSE(options$message), stop_on_error = if (options$error &&         options$include) 0L else 2L, output_handler = knit_handlers(options$render,         options))
16: in_dir(input_dir(), evaluate(code, envir = env, new_device = FALSE,     keep_warning = !isFALSE(options$warning), keep_message = !isFALSE(options$message),     stop_on_error = if (options$error && options$include) 0L else 2L,     output_handler = knit_handlers(options$render, options)))
... 8 lines ...
24: knit2html(..., force_v1 = TRUE)
25: (if (grepl("\\.[Rr]md$", file)) knit2html_v1 else if (grepl("\\.[Rr]rst$",     file)) knit2pdf else knit)(file, encoding = encoding, quiet = quiet,     envir = globalenv())
26: vweave(...)
27: engine$weave(file, quiet = quiet, encoding = enc)
28: doTryCatch(return(expr), name, parentenv, handler)
29: tryCatchOne(expr, names, parentenv, handlers[[1L]])
30: tryCatchList(expr, classes, parentenv, handlers)
31: tryCatch({    engine$weave(file, quiet = quiet, encoding = enc)    setwd(startdir)    find_vignette_product(name, by = "weave", engine = engine)}, error = function(e) {    stop(gettextf("processing vignette '%s' failed with diagnostics:\n%s",         file, conditionMessage(e)), domain = NA, call. = FALSE)})
32: buildVignettes(dir = "/home/muelleki/git/R/dplyr/revdep/checks/parsemsf.Rcheck/vign_test/parsemsf")
An irrecoverable exception occurred. R is aborting now ...
Segmentation fault (core dumped)
```

## patternplot (0.1)
Maintainer: Chunqiao Luo <cluo@uams.edu>

0 errors | 0 warnings | 1 note 

```
checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
```

## pdp (0.5.2)
Maintainer: Brandon Greenwell <greenwell.brandon@gmail.com>  
Bug reports: https://github.com/bgreenwell/pdp/issues

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘ICEbox’
```

## peptider (0.2.2)
Maintainer: Eric Hare <erichare@iastate.edu>  
Bug reports: https://github.com/heike/peptider/issues

0 errors | 0 warnings | 0 notes

## performanceEstimation (1.1.0)
Maintainer: Luis Torgo <ltorgo@dcc.fc.up.pt>  
Bug reports: https://github.com/ltorgo/performanceEstimation/issues

0 errors | 0 warnings | 0 notes

## petrinetR (0.1.0)
Maintainer: Gert Janssenswillen <gert.janssenswillen@uhasselt.be>

0 errors | 0 warnings | 0 notes

## photobiologyInOut (0.4.12)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/photobiologyinout/

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  3. Failure: SpectraSuite (@test-oo.R#158) --------------------------------------
  as.numeric(getWhenMeasured(ss.spct), tz = "CET") not equal to as.numeric(ymd_hms("2013-05-06 15:13:40", tz = "CET"), tz = "CET").
  1/1 mismatches
  [1] 1.37e+09 - 1.37e+09 == 3450400
  
  
  Read 4 items
  testthat results ================================================================
  OK: 188 SKIPPED: 0 FAILED: 3
  1. Error: single spectrum (quantum) (@test-licor.R#9) 
  2. Failure: SpectraSuite (@test-oo.R#76) 
  3. Failure: SpectraSuite (@test-oo.R#158) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Read 3 items
Read 16 items
Read 4 items
Read 18 items
Read 4 items
Read 18 items
Read 4 items
Read 18 items
Read 4 items
Read 18 items
Read 3 items
Read 16 items
Read 12 items
Read 4 items
Read 151 records
Read 7 items
Quitting from lines 390-391 (user-guide.Rnw) 
Error: processing vignette 'user-guide.Rnw' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## photobiology (0.9.13)
Maintainer: Pedro J. Aphalo <pedro.aphalo@helsinki.fi>  
Bug reports: https://bitbucket.org/aphalo/photobiology/issues

0 errors | 0 warnings | 0 notes

## phylopath (0.2.2)
Maintainer: Wouter van der Bijl <wouter.van.der.bijl@zoologi.su.se>  
Bug reports: https://github.com/Ax3man/phylopath/issues

0 errors | 0 warnings | 0 notes

## pinnacle.API (1.92)
Maintainer: Marco Blume <marco.blume@pinnaclesports.com>

0 errors | 0 warnings | 0 notes

## pitchRx (1.8.2)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: http://github.com/cpsievert/pitchRx/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘ggsubplot’
```

## pixiedust (0.7.4)
Maintainer: Benjamin Nutter <nutter@battelle.org>  
Bug reports: https://github.com/nutterb/pixiedust/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘pixiedust-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: as.data.frame.dust
> ### Title: Convert 'dust' Object to Data Frame
> ### Aliases: as.data.frame.dust as.data.frame.dust_list
> 
> ### ** Examples
> 
> fit <- lm(mpg ~ qsec + factor(am) + wt * factor(gear), data = mtcars)
> Dust <- dust(fit) %>%
+   sprinkle(cols = 2:4, round = 2) %>%
+   sprinkle(cols = 5, fn = quote(pvalString(value))) %>%
+   sprinkle(cols = 3, font_color = "#DA70D6") %>%
+   sprinkle_print_method("html")
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 18 SKIPPED: 0 FAILED: 125
  1. Error: as.data.frame.dust (@test-as.data.frame.R#6) 
  2. Error: as.data.frame.dust unsprinkled (@test-as.data.frame.R#17) 
  3. Error: (unknown) (@test-colors.R#3) 
  4. Error: Create a dust object (@test-dust.R#7) 
  5. Error: dust object has expected names (@test-dust.R#14) 
  6. Error: dust object body component has correct dimensions (@test-dust.R#26) 
  7. Error: dust runs when passed a data frame with tidy_df = FALSE (@test-dust.R#42) 
  8. Error: dust runs when passed a data frame with tidy_df = TRUE (@test-dust.R#47) 
  9. Error: dust with keep_rownames = TRUE adds rownames to object (@test-dust.R#52) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 84-91 (advancedMagic.Rmd) 
Error: processing vignette 'advancedMagic.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## PKNCA (0.8.1)
Maintainer: Bill Denney <wdenney@humanpredictions.com>  
Bug reports: https://github.com/billdenney/pknca/issues

0 errors | 0 warnings | 0 notes

## Plasmidprofiler (0.1.6)
Maintainer: Adrian Zetner <adrian.zetner@phac-aspc.gc.ca>

0 errors | 0 warnings | 0 notes

## plater (1.0.0)
Maintainer: Sean Hughes <smhughes@uw.edu>  
Bug reports: https://github.com/ropenscilabs/plater/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 751 SKIPPED: 10 FAILED: 20
  1. Failure: read_plates should contain all columns present in *any* of the input data frames (@test-read_plates.R#30) 
  2. Failure: Plate should be first column in result of read_plates (@test-read_plates.R#35) 
  3. Failure: read_plates without plate_names arg should be named by files (@test-read_plates.R#40) 
  4. Failure: read_plates with plate_names arg should be correctly named (@test-read_plates.R#45) 
  5. Failure: read_plates should contain all columns present in *any* of the input data frames (@test-read_plates.R#30) 
  6. Failure: Plate should be first column in result of read_plates (@test-read_plates.R#35) 
  7. Failure: read_plates without plate_names arg should be named by files (@test-read_plates.R#40) 
  8. Failure: read_plates with plate_names arg should be correctly named (@test-read_plates.R#45) 
  9. Failure: read_plates should contain all columns present in *any* of the input data frames (@test-read_plates.R#30) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## platetools (0.0.2)
Maintainer: Scott Warchal <s.warchal@sms.ed.ac.uk>

0 errors | 0 warnings | 0 notes

## pleiades (0.1.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/pleiades/issues

0 errors | 0 warnings | 0 notes

## plotly (4.5.6)
Maintainer: Carson Sievert <cpsievert1@gmail.com>  
Bug reports: https://github.com/ropensci/plotly/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘plotly-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: add_annotations
> ### Title: Add an annotation(s) to a plot
> ### Aliases: add_annotations
> 
> ### ** Examples
> 
> 
> # single annotation
> plot_ly(mtcars, x = ~wt, y = ~mpg) %>%
+   slice(which.max(mpg)) %>%
+   add_annotations(text = "Good mileage")
Error in UseMethod("slice") : 
  no applicable method for 'slice' applied to an object of class "c('plotly', 'htmlwidget')"
Calls: %>% ... eval -> eval -> _fseq -> freduce -> <Anonymous> -> slice
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [50s/52s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  4: eval(expr, envir, enclos)
  5: `_fseq`(`_lhs`)
  6: freduce(value, `_function_list`) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/pipe.R:28
  7: function_list[[i]](value) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  8: group_by(., city)
  
  testthat results ================================================================
  OK: 806 SKIPPED: 17 FAILED: 4
  1. Error: datetimes are converted to e.g. 2013-01-02 05:00:00 (@test-ggplot-date.R#11) 
  2. Error: can filter data returned by ggplotly (@test-ggplot-ggplotly.R#13) 
  3. Error: add_fun can apply two different chunks of data to a plot (@test-plotly-data.R#23) 
  4. Error: Complex example works (@test-plotly.R#177) 
  
  Error: testthat unit tests failed
  Execution halted
```

## plotrr (0.2.0)
Maintainer: Charles Crabtree <ccrabtr@umich.edu>

0 errors | 0 warnings | 0 notes

## pmc (1.0.2)
Maintainer: Carl Boettiger <cboettig@gmail.com>  
Bug reports: https://github.com/cboettig/pmc/issues

0 errors | 0 warnings | 0 notes

## PogromcyDanych (1.5)
Maintainer: Przemyslaw Biecek <przemyslaw.biecek@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.2Mb
  sub-directories of 1Mb or more:
    data   6.0Mb
```

## poio (0.0-3)
Maintainer: Richard Cotton <richierocks@gmail.com>  
Bug reports: https://github.com/RL10N/poio/issues

0 errors | 0 warnings | 0 notes

## pollen (0.50.0)
Maintainer: Jakub Nowosad <nowosad.jakub@gmail.com>  
Bug reports: https://github.com/Nowosad/pollen/issues

0 errors | 0 warnings | 0 notes

## PopED (0.3.2)
Maintainer: Andrew C. Hooker <andrew.hooker@farmbio.uu.se>  
Bug reports: https://github.com/andrewhooker/PopED/issues

0 errors | 0 warnings | 0 notes

## poplite (0.99.17.3)
Maintainer: Daniel Bottomly <bottomly@ohsu.edu>

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘poplite-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: External methods
> ### Title: Specific methods for generics defined in external packages.
> ### Aliases: filter select
> ### Keywords: utilities
> 
... 20 lines ...
+       filter(baseball.db, active == "Y")
+       
+       select(filter(baseball.db, active == "Y" & W > 50 & teamID == "CAL"), active, W, teamID)
+   }
Loading required package: Lahman
Starting team_franch
Starting teams
Error in UseMethod("filter") : 
  no applicable method for 'filter' applied to an object of class "Database"
Calls: filter
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  [1] "normal"
  [1] "merge"
  [1] "clinical"
  [1] "dna"
  [1] "samples"
  1. Error: Querying with Database objects (@test-poplite.R#711) -----------------
  no applicable method for 'filter' applied to an object of class "Database"
  1: filter(sample.tracking.db, samples.sample_id == 1) at testthat/test-poplite.R:711
  
  testthat results ================================================================
  OK: 149 SKIPPED: 0 FAILED: 1
  1. Error: Querying with Database objects (@test-poplite.R#711) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Loading required package: DBI

Attaching package: ‘poplite’

The following object is masked from ‘package:dplyr’:

... 8 lines ...
  ERROR: The names of the supplied data.frame need to be modified for the database see correct.df.names
Starting clinical
Warning: RSQLite::dbGetPreparedQuery() is deprecated, please switch to DBI::dbGetQuery(params = bind.data).
Starting samples
Starting dna

Error: processing vignette 'poplite.Rnw' failed with diagnostics:
 chunk 10 
Error in UseMethod("filter") : 
  no applicable method for 'filter' applied to an object of class "Database"
Execution halted
```

## poppr (2.3.0)
Maintainer: Zhian N. Kamvar <zkamvar@gmail.com>  
Bug reports: https://github.com/grunwaldlab/poppr/issues

0 errors | 0 warnings | 0 notes

## prcr (0.1.1)
Maintainer: Joshua Rosenberg <jmichaelrosenberg@gmail.com>

0 errors | 0 warnings | 0 notes

## prepdat (1.0.8)
Maintainer: Ayala S. Allon <ayalaallon@gmail.com>  
Bug reports: http://github.com/ayalaallon/prepdat/issues

0 errors | 0 warnings | 0 notes

## pRF (1.2)
Maintainer: Ankur Chakravarthy <ankur.chakravarthy.10@ucl.ac.uk>

0 errors | 0 warnings | 0 notes

## prophet (0.1)
Maintainer: Sean Taylor <sjt@fb.com>

1 error  | 1 warning  | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  Optimization terminated normally: 
    Convergence detected: absolute parameter change was below tolerance
  3. Failure: fit_with_holidays (@test_prophet.R#188) ----------------------------
  predict(m) threw an error.
  object 'lower' not found
  
  
  testthat results ================================================================
  OK: 32 SKIPPED: 0 FAILED: 3
  1. Failure: fit_predict (@test_prophet.R#18) 
  2. Failure: fit_predict_no_changepoints (@test_prophet.R#30) 
  3. Failure: fit_with_holidays (@test_prophet.R#188) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: Rcpp

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 43-45 (quick_start.Rmd) 
Error: processing vignette 'quick_start.Rmd' failed with diagnostics:
object 'lower' not found
Execution halted


checking installed package size ... NOTE
  installed size is 18.8Mb
  sub-directories of 1Mb or more:
    libs  18.5Mb
```

## ptstem (0.0.3)
Maintainer: Daniel Falbel <dfalbel@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    dict   5.1Mb
```

## purrr (0.2.2)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/hadley/purrr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘purrr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/purrr.Rcheck/00install.out’ for details.
```

## PWFSLSmoke (0.99.9)
Maintainer: Jonathan Callahan <jonathan.s.callahan@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘MazamaSpatialUtils’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## qdap (2.2.5)
Maintainer: Tyler Rinker <tyler.rinker@gmail.com>  
Bug reports: http://github.com/trinker/qdap/issues

0 errors | 0 warnings | 0 notes

## quadmesh (0.1.0)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>

0 errors | 0 warnings | 0 notes

## qualvar (0.1.0)
Maintainer: Joel Gombin <joel.gombin@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 92-106 (wilcox1973.Rmd) 
Error: processing vignette 'wilcox1973.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

```

## queuecomputer (0.6.1)
Maintainer: Anthony Ebert <anthonyebert+CRAN@gmail.com>

0 errors | 0 warnings | 0 notes

## quickpsy (0.1.4)
Maintainer: Linares Daniel <danilinares@gmail.com>

0 errors | 0 warnings | 0 notes

## qwraps2 (0.2.4)
Maintainer: Peter DeWitt <dewittpe@gmail.com>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘qwraps2-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: qkmplot
> ### Title: Kaplan-Meier Plot
> ### Aliases: qkmplot qkmplot_bulid_data_frame
> 
> ### ** Examples
> 
> require(survival)
Loading required package: survival
> leukemia.surv <- survival::survfit(survival::Surv(time, status) ~ x, data = survival::aml) 
> survival:::plot.survfit(leukemia.surv, conf.int = TRUE, lty = 2:3, col = 1:2)
> 
> qkmplot(leukemia.surv, conf_int = TRUE) 
Error: Cannot convert objects of type `logical` to `tidy_quote`
Execution halted
```

## r2glmm (0.1.1)
Maintainer: Byron Jaeger <byron.jaeger@gmail.com>  
Bug reports: https://github.com/bcjaeger/r2glmm/issues

0 errors | 0 warnings | 0 notes

## R6Frame (0.1.0)
Maintainer: Kristian D. Olsen <kristian@doingit.no>

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 105 SKIPPED: 1 FAILED: 26
  1. Error: bind_rows work with R6 data.frame (@test-dplyr_merges.R#10) 
  2. Error: bind_rows work with R6 data.table (@test-dplyr_merges.R#22) 
  3. Error: bind_rows work with R6 tbl_df (@test-dplyr_merges.R#34) 
  4. Error: bind_cols work with R6 data.frame (@test-dplyr_merges.R#47) 
  5. Error: bind_cols work with R6 data.table (@test-dplyr_merges.R#57) 
  6. Error: bind_cols work with R6 tbl_df (@test-dplyr_merges.R#67) 
  7. Error: left_join work with R6 data.frame (@test-dplyr_merges.R#78) 
  8. Error: left_join work with R6 data.table (@test-dplyr_merges.R#89) 
  9. Error: left_join work with R6 tbl_df (@test-dplyr_merges.R#101) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted
```

## radiant.basics (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.basics/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
... 26 lines ...
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'summarise_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.data (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.data/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.data-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dtab.explore
> ### Title: Make a tabel of summary statistics in DT
> ### Aliases: dtab.explore
> 
> ### ** Examples
> 
> tab <- explore("diamonds", "price:x") %>% dtab
Error in overscope_eval(overscope, f) : object 'price' not found
Calls: %>% ... map_if -> map -> lapply -> FUN -> overscope_eval -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  The following object is masked from 'package:testthat':
  
      describe
  
  1. Failure: getdata (@test_funs.R#31) ------------------------------------------------------------------------------------------------------------------------------------------------------------------
  `_lhs` not equal to `%>%`(...).
  Names: 3 string mismatches
  
  
  testthat results ========================================================================================================================================================================================
  OK: 17 SKIPPED: 0 FAILED: 1
  1. Failure: getdata (@test_funs.R#31) 
  
  Error: testthat unit tests failed
  Execution halted
```

## radiant.design (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.design/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## radiant.model (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.model/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.model-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: ann
> ### Title: Artificial Neural Networks
> ### Aliases: ann
> 
> ### ** Examples
... 7 lines ...
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
> result <- ann("diamonds", "price", c("carat","clarity"), type = "regression")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error: All select() inputs must resolve to integer column positions. The following do not:

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2: scaledf(dat, wts = wts)
  3: summarise_each_(dat, funs(mean(., na.rm = TRUE)), vars = cn)
  4: apply_vars(funs, vars, tbl) at /home/muelleki/git/R/dplyr/R/colwise.R:251
  5: select_vars(tbl_vars(tbl), !(!(!vars)), exclude = group_vars(tbl)) at /home/muelleki/git/R/dplyr/R/colwise.R:191
  6: abort(glue("All select() inputs must resolve to integer column positions. \\\n       The following do not:\n       {labels}", labels = paste("* ", labels, collapse = "\n"))) at /home/muelleki/git/R/dplyr/R/select-vars.R:76
  
  testthat results ========================================================================================================================================================================================
  OK: 19 SKIPPED: 0 FAILED: 4
  1. Error: regress - predict (@test_stats.R#22) 
  2. Error: logistic - predict (@test_stats.R#38) 
  3. Error: ann - predict for classification (@test_stats.R#51) 
  4. Error: ann - predict for regression (@test_stats.R#63) 
  
  Error: testthat unit tests failed
  Execution halted
```

## radiant.multivariate (0.6.0)
Maintainer: Vincent Nijs <radiant@rady.ucsd.edu>  
Bug reports: https://github.com/radiant-rstats/radiant.multivariate/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘radiant.multivariate-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: conjoint
> ### Title: Conjoint analysis
> ### Aliases: conjoint
> 
> ### ** Examples
> 
> result <- conjoint("mp3", rvar = "Rating", evar = "Memory:Shape")
Error in eval(expr, envir, enclos) : object 'Memory' not found
Calls: conjoint ... eval -> <Anonymous> -> model.frame.default -> eval -> eval
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  2: lm(formula, data = dat)
  3: eval(mf, parent.frame())
  4: eval(expr, envir, enclos)
  5: stats::model.frame(formula = formula, data = dat, drop.unused.levels = TRUE)
  6: model.frame.default(formula = formula, data = dat, drop.unused.levels = TRUE)
  7: eval(predvars, data, env)
  8: eval(expr, envir, enclos)
  
  testthat results ========================================================================================================================================================================================
  OK: 5 SKIPPED: 0 FAILED: 2
  1. Error: Kmeans cluster analysis (@test_stats.R#77) 
  2. Error: Conjoint on mp3 data (@test_stats.R#91) 
  
  Error: testthat unit tests failed
  Execution halted
```

## randomizr (0.4.1)
Maintainer: Alexander Coppock <acoppock@gmail.com>

0 errors | 0 warnings | 0 notes

## raptr (0.0.3)
Maintainer: Jeffrey O Hanson <jeffrey.hanson@uqconnect.edu.au>  
Bug reports: https://github.com/jeffreyhanson/raptr/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Package required but not available: ‘gdalUtils’

Packages suggested but not available for checking: ‘gurobi’ ‘rgurobi’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## rattle (4.1.0)
Maintainer: Graham Williams <Graham.Williams@togaware.com>

0 errors | 1 warning  | 3 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'rattle.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm2e.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.14 \usepackage
                [^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted


checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘rggobi’ ‘pkgDepTools’

checking installed package size ... NOTE
  installed size is  6.9Mb
  sub-directories of 1Mb or more:
    data   2.5Mb
    etc    1.9Mb
    po     1.2Mb

checking dependencies in R code ... NOTE

(R:36636): Gtk-WARNING **: gtk_disable_setlocale() must be called before gtk_init()
```

## raw (0.1.4)
Maintainer: Brian A. Fannin <FanninQED@yahoo.com>

0 errors | 0 warnings | 0 notes

## rbgm (0.0.4)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/AustralianAntarcticDivision/rbgm/issues/

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: sp

Attaching package: 'dplyr'

The following objects are masked from 'package:raster':
... 8 lines ...
The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/home/muelleki/R/x86_64-pc-linux-gnu-library/3.3/rgdal/libs/rgdal.so':
  libgdal.so.1: cannot open shared object file: No such file or directory
Quitting from lines 16-24 (BGM_Spatial.Rmd) 
Error: processing vignette 'BGM_Spatial.Rmd' failed with diagnostics:
package or namespace load failed for 'rgdal'
Execution halted
```

## rbison (0.5.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/rbison/issues

0 errors | 0 warnings | 0 notes

## rccmisc (0.3.7)
Maintainer: Erik Bulow <erik.bulow@rccvast.se>  
Bug reports: https://bitbucket.com/cancercentrum/rccmisc/issues

0 errors | 0 warnings | 0 notes

## rchess (0.1)
Maintainer: Joshua Kunst <jbkunst@gmail.com>  
Bug reports: https://github.com/jbkunst/rchess/issues

0 errors | 0 warnings | 0 notes

## rcicr (0.3.4.1)
Maintainer: Ron Dotsch <r.dotsch@uu.nl>

0 errors | 0 warnings | 0 notes

## rclinicaltrials (1.4.7)
Maintainer: Michael C Sachs <sachsmc@gmail.com>

0 errors | 0 warnings | 0 notes

## RCMIP5 (1.2.0)
Maintainer: Kathe Todd-Brown <ktoddbrown@gmail.com>

0 errors | 0 warnings | 0 notes

## rcrossref (0.6.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/rcrossref/issues

0 errors | 0 warnings | 0 notes

## RDML (0.9-5)
Maintainer: Konstantin A. Blagodatskikh <k.blag@yandex.ru>

0 errors | 0 warnings | 0 notes

## rdrop2 (0.7.0)
Maintainer: Karthik Ram <karthik.ram@gmail.com>  
Bug reports: https://github.com/karthik/rdrop2/issues

0 errors | 0 warnings | 0 notes

## readODS (1.6.4)
Maintainer: Chung-hong Chan <chainsawtiney@gmail.com>

0 errors | 0 warnings | 0 notes

## rebird (0.3.0)
Maintainer: Sebastian Pardo <sebpardo@gmail.com>  
Bug reports: http://github.com/ropensci/rebird/issues

0 errors | 0 warnings | 0 notes

## recexcavAAR (0.3.0)
Maintainer: Clemens Schmid <clemens@nevrome.de>

0 errors | 1 warning  | 1 note 

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: kriging

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 70-71 (recexcavAAR-vignette-1.Rmd) 
Error: processing vignette 'recexcavAAR-vignette-1.Rmd' failed with diagnostics:
cannot open the connection
Execution halted


checking installed package size ... NOTE
  installed size is  5.9Mb
  sub-directories of 1Mb or more:
    doc    2.5Mb
    libs   3.3Mb
```

## REDCapR (0.9.7)
Maintainer: Will Beasley <wibeasley@hotmail.com>  
Bug reports: https://github.com/OuhscBbmc/REDCapR/issues

0 errors | 0 warnings | 0 notes

## refund.shiny (0.3.0)
Maintainer: Julia Wrobel <jw3134@cumc.columbia.edu>

0 errors | 0 warnings | 0 notes

## replyr (0.2.3)
Maintainer: John Mount <jmount@win-vector.com>  
Bug reports: https://github.com/WinVector/replyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘replyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: gapply
> ### Title: grouped ordered apply
> ### Aliases: gapply
> 
> ### ** Examples
... 59 lines ...
  <dbl> <dbl>  <dbl> <dbl>
1     2   0.5      8     1
2     2   0.4      4     2
3     2   0.3      2     3
4     1   0.2     20     1
5     1   0.1     10     2
[1] "split"
[1] "cumulative sum example"
Error in arrange_impl(.data, dots) : incorrect size (1), expecting : 2
Calls: print ... as.data.frame -> arrange -> arrange.tbl_df -> arrange_impl -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  14: arrange_.data.frame(df, columnName) at /home/muelleki/git/R/dplyr/R/manip.r:263
  15: as.data.frame(arrange(tbl_df(.data), ...)) at /home/muelleki/git/R/dplyr/R/dataframe.R:104
  16: arrange(tbl_df(.data), ...)
  17: arrange.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:258
  18: arrange_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:46
  
  nrows: 2
  Observations: 2
  Variables: 1
  $ x <dbl> 1, 2testthat results ================================================================
  OK: 1 SKIPPED: 20 FAILED: 1
  1. Error: test_gapply.R (@test_gapply.R#23) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 103-108 (ParametricExample.Rmd) 
Error: processing vignette 'ParametricExample.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## rerddap (0.3.4)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: http://www.github.com/ropensci/rerddap/issues

0 errors | 0 warnings | 0 notes

## resumer (0.0.3)
Maintainer: Jared Lander <packages@jaredlander.com>  
Bug reports: https://github.com/jaredlander/resumer/issues

0 errors | 0 warnings | 0 notes

## reval (2.0.0)
Maintainer: Michael C Koohafkan <michael.koohafkan@gmail.com>  
Bug reports: https://github.com/mkoohafkan/reval/issues

0 errors | 0 warnings | 0 notes

## rex (1.1.1)
Maintainer: Jim Hester <james.f.hester@gmail.com>  
Bug reports: https://github.com/kevinushey/rex/issues

0 errors | 0 warnings | 0 notes

## rfishbase (2.1.1)
Maintainer: Carl Boettiger <cboettig@ropensci.org>  
Bug reports: https://github.com/ropensci/rfishbase/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         tidy_table(data, server = server)
     }) at /home/muelleki/git/R/dplyr/R/bind.r:120
  5: FUN(X[[i]], ...)
  6: tidy_table(data, server = server)
  7: cbind(x, SpecCode = code)
  8: cbind(deparse.level, ...)
  9: bind_cols(...) at /home/muelleki/git/R/dplyr/R/bind.r:105
  10: cbind_all(x) at /home/muelleki/git/R/dplyr/R/bind.r:99
  
  testthat results ================================================================
  OK: 35 SKIPPED: 31 FAILED: 1
  1. Error: Custom queries give desired result (@test_endpoint.R#10) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning: FishBase says that 'Salmo trutta' can also be misapplied to other species
                    but is returning only the best match.  
                    See synonyms('Salmo trutta') for details
Quitting from lines 88-89 (tutorial.Rmd) 
Error: processing vignette 'tutorial.Rmd' failed with diagnostics:
cannot convert object to a data frame
Execution halted

```

## rgho (1.0.1)
Maintainer: Antoine Filipovic-Pierucci <pierucci@gmail.com>  
Bug reports: https://github.com/pierucci/rgho/issues

0 errors | 0 warnings | 0 notes

## rivr (1.2)
Maintainer: Michael C Koohafkan <michael.koohafkan@gmail.com>  
Bug reports: https://github.com/mkoohafkan/rivr/issues

0 errors | 0 warnings | 0 notes

## RmarineHeatWaves (0.15.1)
Maintainer: Albertus J. Smit <albertus.smit@gmail.com>

0 errors | 0 warnings | 0 notes

## rmcfs (1.2.4)
Maintainer: Michal Draminski <michal.draminski@ipipan.waw.pl>

0 errors | 1 warning  | 2 notes

```
checking S3 generic/method consistency ... WARNING
filter:
  function(x, filter, method, sides, circular, init)
filter.data:
  function(data, mcfs_result, size)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.

Found the following apparent S3 methods exported but not registered:
  filter.data
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking installed package size ... NOTE
  installed size is  5.2Mb
  sub-directories of 1Mb or more:
    java   4.5Mb

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'filter.data':
  ‘filter.data’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## rmdHelpers (1.2)
Maintainer: Mark Peterson <mark.phillip.peterson@gmail.com>

0 errors | 0 warnings | 0 notes

## RNeXML (2.0.7)
Maintainer: Carl Boettiger <cboettig@gmail.com>  
Bug reports: https://github.com/ropensci/RNeXML/issues

2 errors | 1 warning  | 1 note 

```
checking examples ... ERROR
Running examples in ‘RNeXML-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: get_taxa
> ### Title: get_taxa
> ### Aliases: get_otu get_taxa
> 
> ### ** Examples
> 
> comp_analysis <- system.file("examples", "comp_analysis.xml", package="RNeXML")
> nex <- nexml_read(comp_analysis)
> get_taxa(nex)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’ [38s/71s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 164 SKIPPED: 1 FAILED: 19
  1. Error: we can extract character matrix with get_characters (@test_characters.R#44) 
  2. Error: add_otu can append only unmatched taxa to an existing otus block (@test_characters.R#63) 
  3. Error: we can add characters to a nexml file using a data.frame (@test_characters.R#88) 
  4. Error: We can extract tree and trait data to run fitContinuous and fitDiscrete (@test_comp_analysis.R#9) 
  5. Error: we can read in a file with existing meta and append without overwriting (@test_concatenate.R#99) 
  6. Error: Getting characters (@test_get_characters.R#7) 
  7. Error: get_characters throws appropriate warnings (@test_get_characters.R#15) 
  8. Error: (unknown) (@test_get_level.R#5) 
  9. Error: We can add additional metadata (@test_meta.R#14) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: ape
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 73-74 (metadata.Rmd) 
Error: processing vignette 'metadata.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted


checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘rrdf’ ‘Sxslt’
```

## RNHANES (1.1.0)
Maintainer: Herb Susmann <susmann@silentspring.org>  
Bug reports: https://github.com/silentspringinstitute/RNHANES/issues

0 errors | 0 warnings | 0 notes

## rnoaa (0.6.6)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/rnoaa/issues

0 errors | 0 warnings | 0 notes

## robotstxt (0.3.2)
Maintainer: Peter Meissner <retep.meissner@gmail.com>  
Bug reports: https://github.com/ropenscilabs/robotstxt/issues

0 errors | 0 warnings | 0 notes

## ropenaq (0.1.3)
Maintainer: Maëlle Salmon <maelle.salmon@yahoo.se>  
Bug reports: http://github.com/ropenscilabs/ropenaq/issues

0 errors | 0 warnings | 0 notes

## ROpenFIGI (0.2.8)
Maintainer: Ruokun Huang <hruokun.2008@gmail.com>

0 errors | 0 warnings | 0 notes

## rpcdsearch (1.0)
Maintainer: David Springate <daspringate@gmail.com>

0 errors | 0 warnings | 0 notes

## rpdo (0.2.1)
Maintainer: Joe Thorley <joe@poissonconsulting.ca>

0 errors | 0 warnings | 0 notes

## rpivotTable (0.1.5.20)
Maintainer: Enzo Martoglio  <enzo@smartinsightsfromdata.com>

0 errors | 0 warnings | 0 notes

## rplos (0.6.4)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/rplos/issues

0 errors | 0 warnings | 0 notes

## rPref (1.2)
Maintainer: Patrick Roocks <mail@p-roocks.de>

0 errors | 0 warnings | 1 note 

```
checking for GNU extensions in Makefiles ... NOTE
GNU make is a SystemRequirements.
```

## RPresto (1.2.1)
Maintainer: Onur Ismail Filiz <onur@fb.com>  
Bug reports: https://github.com/prestodb/RPresto/issues

0 errors | 0 warnings | 0 notes

## rprev (0.2.3)
Maintainer: Stuart Lacy <stuart.lacy@york.ac.uk>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: survival
Quitting from lines 269-270 (user_guide.Rmd) 
Error: processing vignette 'user_guide.Rmd' failed with diagnostics:
All select() inputs must resolve to integer column positions. The following do not:
```

## rrr (1.0.0)
Maintainer: Chris Addy <chris.william.addy@gmail.com>

0 errors | 0 warnings | 0 notes

## rscorecard (0.3.5)
Maintainer: Benjamin Skinner <b.skinner@vanderbilt.edu>  
Bug reports: http://github.com/btskinner/rscorecard/issues

0 errors | 0 warnings | 0 notes

## rslp (0.1.0)
Maintainer: Daniel Falbel <dfalbel@gmail.com>

0 errors | 0 warnings | 0 notes

## rsparkling (0.1.1)
Maintainer: Navdeep Gill <navdeep@h2o.ai>  
Bug reports: http://jira.h2o.ai

0 errors | 0 warnings | 0 notes

## RSSL (0.6.1)
Maintainer: Jesse Krijthe <jkrijthe@gmail.com>  
Bug reports: http://www.github.com/jkrijthe/RSSL

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.7Mb
  sub-directories of 1Mb or more:
    R      2.2Mb
    libs   2.5Mb
```

## rtable (0.1.5)
Maintainer: David Gohel <david.gohel@lysis-consultants.fr>  
Bug reports: https://github.com/davidgohel/rtable/issues

0 errors | 0 warnings | 0 notes

## rtdists (0.6-6)
Maintainer: Henrik Singmann <singmann+rtdists@gmail.com>  
Bug reports: https://github.com/rtdists/rtdists/issues

0 errors | 0 warnings | 0 notes

## rtide (0.0.3)
Maintainer: Joe Thorley <joe@poissonconsulting.ca>  
Bug reports: https://github.com/poissonconsulting/rtide/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  40: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  41: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 23 SKIPPED: 0 FAILED: 5
  1. Error: tide_height_data works (@test-tide-height-data.R#10) 
  2. Error: tide_height_data predictions (@test-tide-height-data.R#17) 
  3. Failure: tide_height_data checks (@test-tide-height-data.R#36) 
  4. Error: tide_height_data tz (@test-tide-height-data.R#49) 
  5. Error: tide_height works (@test-tide-height.R#6) 
  
  Error: testthat unit tests failed
  Execution halted
```

## rtimes (0.3.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropengov/rtimes/issues

0 errors | 0 warnings | 0 notes

## rtrends (0.1.0)
Maintainer: Avi Blinder <aviblinder@gmail.com>

0 errors | 0 warnings | 0 notes

## RtutoR (0.3)
Maintainer: Anup Nair <nairanup50695@gmail.com>

0 errors | 0 warnings | 0 notes

## rvertnet (0.5.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: https://github.com/ropensci/rvertnet/issues

0 errors | 0 warnings | 0 notes

## rwunderground (0.1.0)
Maintainer: Alex Shum <alex@ALShum.com>  
Bug reports: https://github.com/alshum/rwunderground/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘rwunderground-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: set_location
> ### Title: Specifies location of request
> ### Aliases: set_location
> 
> ### ** Examples
> 
> set_location(zip_code = "90210")
[1] "90210"
> set_location(territory = "Hawaii", city = "Honolulu")
Error in `[.data.frame`(countrycode::countrycode_data, , c("country.name",  : 
  undefined columns selected
Calls: set_location ... is_valid_territory -> list_countries -> [ -> [.data.frame
Execution halted
```

## saeSim (0.8.0)
Maintainer: Sebastian Warnholz <Sebastian.Warnholz@fu-berlin.de>  
Bug reports: https://github.com/wahani/saeSim/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘saeSim-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: comp_var
> ### Title: Compute variables in data
> ### Aliases: comp_var
> 
> ### ** Examples
> 
> sim_base_lm() %>% sim_comp_pop(comp_var(yExp = exp(y)))
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘test-all.R’
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 120 SKIPPED: 0 FAILED: 10
  1.  Error: Attributes are preserved (@test-preserve_attributes.R#9) 
  2.  Error: Save incremental files (@test-sim.R#45) 
  3.  Error: Basic functionality (@test-sim_agg.R#11) 
  4.  Error: base_add_id (@test-sim_base.R#6) 
  5.  Error: sim_comp and comp_var (@test-sim_comp.R#12) 
  6.  Error: gen_generic (@test-sim_generate.R#31) 
  7.  Error: sim_gen_generic (@test-sim_generate.R#52) 
  8.  Error: Construction of response works correctly (@test-sim_resp.R#6) 
  9.  Error: as.data.frame (@test-sim_setup.R#28) 
  10. Error: autoplot (@test-sim_setup.R#35) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 121-126 (Introduction.Rmd) 
Error: processing vignette 'Introduction.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## scholar (0.1.4)
Maintainer: James Keirstead <james.keirstead@gmail.com>  
Bug reports: https://github.com/jkeirstead/scholar/issues

0 errors | 0 warnings | 0 notes

## SEERaBomb (2016.2)
Maintainer: Tomas Radivoyevitch <radivot@ccf.org>

0 errors | 0 warnings | 0 notes

## sejmRP (1.3)
Maintainer: Piotr Smuda <piotrsmuda@gmail.com>  
Bug reports: http://github.com/mi2-warsaw/sejmRP/issues

0 errors | 0 warnings | 0 notes

## sfc (0.1.0)
Maintainer: Hu Sheng <shenghu@nju.edu.cn>  
Bug reports: https://github.com/ctfysh/sfc/issues

0 errors | 1 warning  | 0 notes

```
checking examples ... WARNING
Found the following significant warnings:

  Warning: 'mutate_each_' is deprecated.
  Warning: 'mutate_each_' is deprecated.
Deprecated functions may be defunct as soon as of the next release of
R.
See ?Deprecated.
```

## sf (0.3-4)
Maintainer: Edzer Pebesma <edzer.pebesma@uni-muenster.de>  
Bug reports: https://github.com/edzer/sfr/issues/

2 errors | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sf-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr
> ### Title: Dplyr verb methods for sf objects
> ### Aliases: dplyr filter_.sf dplyr arrange_.sf dplyr distinct_.sf dplyr
> ###   group_by_.sf dplyr mutate_.sf dplyr transmute_.sf dplyr select_.sf
> ###   dplyr rename_.sf dplyr slice_.sf dplyr summarise_.sf dplyr gather_.sf
... 22 lines ...
Simple feature collection with 100 features and 14 fields
geometry type:  MULTIPOLYGON
dimension:      XY
bbox:           xmin: -84.32385 ymin: 33.88199 xmax: -75.45698 ymax: 36.58965
epsg (SRID):    4267
proj4string:    +proj=longlat +datum=NAD27 +no_defs
> nc %>% filter(AREA > .1) %>% plot()
Error in data.matrix(x) : 
  (list) object cannot be coerced to type 'double'
Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
Execution halted

checking tests ... ERROR
  Running ‘cast.R’
  Comparing ‘cast.Rout’ to ‘cast.Rout.save’ ... OK
  Running ‘crs.R’
  Comparing ‘crs.Rout’ to ‘crs.Rout.save’ ... OK
  Running ‘dist.R’
  Comparing ‘dist.Rout’ to ‘dist.Rout.save’ ... OK
  Running ‘dplyr.R’
Running the tests in ‘tests/dplyr.R’ failed.
Last 13 lines of output:
  
  The following objects are masked from 'package:stats':
  
      filter, lag
  
  The following objects are masked from 'package:base':
  
      intersect, setdiff, setequal, union
  
  > nc = st_read(system.file("shape/nc.shp", package="sf"), quiet = TRUE)
  > nc %>% filter(AREA > .1) %>% plot()
  Error in data.matrix(x) : 
    (list) object cannot be coerced to type 'double'
  Calls: %>% ... <Anonymous> -> plot -> plot.data.frame -> pairs -> data.matrix
  Execution halted

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    3.9Mb
    libs   3.8Mb
```

## shazam (0.1.4)
Maintainer: Jason Vander Heiden <jason.vanderheiden@yale.edu>  
Bug reports: https://bitbucket.org/kleinstein/shazam/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘shazam-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: calcBaseline
> ### Title: Calculate the BASELINe PDFs
> ### Aliases: calcBaseline
> 
> ### ** Examples
... 7 lines ...
> baseline <- calcBaseline(db, 
+                          sequenceColumn="SEQUENCE_IMGT",
+                          germlineColumn="GERMLINE_IMGT_D_MASK", 
+                          testStatistic="focused",
+                          regionDefinition=IMGT_V_NO_CDR3,
+                          targetingModel = HS5FModel,
+                          nproc=1)
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: All select() inputs must resolve to integer column positions. The following do not:

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: ggplot2
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Quitting from lines 72-74 (Baseline-Vignette.Rmd) 
Error: processing vignette 'Baseline-Vignette.Rmd' failed with diagnostics:
All select() inputs must resolve to integer column positions. The following do not:
```

## shinyAce (0.2.1)
Maintainer: Jeff Allen <cran@trestletechnology.net>  
Bug reports: https://github.com/trestletech/shinyAce/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  9.9Mb
  sub-directories of 1Mb or more:
    www   9.7Mb
```

## ShinyTester (0.1.0)
Maintainer: Amit Kohli <amit@amitkohli.com>

0 errors | 0 warnings | 0 notes

## SimDesign (1.5)
Maintainer: Phil Chalmers <rphilip.chalmers@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘doMPI’
```

## simmer.plot (0.1.8)
Maintainer: Iñaki Ucar <i.ucar86@gmail.com>  
Bug reports: https://github.com/r-simmer/simmer.plot/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘simmer.plot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: plot.simmer
> ### Title: Plot method for simmer objects
> ### Aliases: plot.simmer
> 
> ### ** Examples
... 21 lines ...
> 
> env %>% run(until=80)
simmer environment: SuperDuperSim | now: 80 | next: 82.9893723637843
{ Resource: nurse | monitored: 1 | server status: 1(1) | queue status: 2(Inf) }
{ Resource: doctor | monitored: 1 | server status: 1(2) | queue status: 0(Inf) }
{ Resource: administration | monitored: 1 | server status: 0(1) | queue status: 0(Inf) }
{ Generator: patient | monitored: 1 | n_generated: 8 }
> 
> plot(env, what="resources", metric="usage", "doctor", items = "server", steps = TRUE)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  6: arrange.tbl_df(tbl_df(.data), ...) at /home/muelleki/git/R/dplyr/R/manip.r:258
  7: arrange_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:46
  
  testthat results ================================================================
  OK: 40 SKIPPED: 0 FAILED: 7
  1. Error: single replication plots (@test-plot-arrivals.R#22) 
  2. Error: multiple replication plots (@test-plot-arrivals.R#37) 
  3. Error: single replication plots (@test-plot-resources.R#22) 
  4. Error: multiple replication plots (@test-plot-resources.R#40) 
  5. Error: a simple trajectory is correctly converted to graph (@test-plot-trajectory.R#31) 
  6. Error: a rollback with variable amount is correctly converted to graph (@test-plot-trajectory.R#37) 
  7. Error: a complex trajectory is correctly converted to graph (@test-plot-trajectory.R#68) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: ggplot2
Quitting from lines 72-75 (plot.simmer.Rmd) 
Error: processing vignette 'plot.simmer.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## simmer (3.6.0)
Maintainer: Iñaki Ucar <i.ucar86@gmail.com>  
Bug reports: https://github.com/r-simmer/simmer/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is 11.8Mb
  sub-directories of 1Mb or more:
    libs  10.6Mb
```

## simPH (1.3.9)
Maintainer: Christopher Gandrud <christopher.gandrud@gmail.com>  
Bug reports: https://github.com/christophergandrud/simPH/issues

0 errors | 0 warnings | 0 notes

## simputation (0.2.1)
Maintainer: Mark van der Loo <mark.vanderloo@gmail.com>  
Bug reports: https://github.com/markvanderloo/simputation/issues

0 errors | 0 warnings | 0 notes

## sjmisc (2.3.1)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/sjmisc/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjmisc-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: descr
> ### Title: Basic descriptive statistics
> ### Aliases: descr
> 
> ### ** Examples
> 
> data(efc)
> descr(efc, e17age, c160age)
Error in mutate_impl(.data, dots) : object '.x' not found
Calls: descr ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 117-128 (exploringdatasets.Rmd) 
Error: processing vignette 'exploringdatasets.Rmd' failed with diagnostics:
object 'quol_5' not found
Execution halted

```

## sjPlot (2.3.1)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/devel/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘sjPlot-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: sjp.kfold_cv
> ### Title: Plot model fit from k-fold cross-validation
> ### Aliases: sjp.kfold_cv
> 
> ### ** Examples
> 
> library(sjmisc)
> data(efc)
> 
> sjp.kfold_cv(efc, neg_c_7 ~ e42dep + c172code + c12hour)
Error in mutate_impl(.data, dots) : object 'neg_c_7' not found
Calls: sjp.kfold_cv ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    select

The following objects are masked from 'package:stats':

    filter, lag

... 8 lines ...
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Fitted models have different coefficients. Grouping may not work properly. Set `group.pred = FALSE` if you encouter cluttered labelling.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Computing p-values via Kenward-Roger approximation. Use `p.kr = FALSE` if computation takes too long.
Quitting from lines 66-67 (sjtlmer.Rmd) 
Error: processing vignette 'sjtlmer.Rmd' failed with diagnostics:
is.data.frame(df) is not TRUE
Execution halted
```

## sjstats (0.9.0)
Maintainer: Daniel Lüdecke <d.luedecke@uke.de>  
Bug reports: https://github.com/sjPlot/sjstats/issues

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘sjstats-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: check_assumptions
> ### Title: Check model assumptions
> ### Aliases: check_assumptions outliers heteroskedastic autocorrelation
> ###   normality multicollin
> 
... 61 lines ...
+   bootstrap(50) %>%
+   mutate(models = lapply(.$strap, function(x) {
+     lm(neg_c_7 ~ e42dep + c12hour + c161sex, data = x)
+   }))
> 
> # for list-variables, argument 'model.column' is the
> # quoted name of the list-variable with fitted models
> tmp %>% heteroskedastic("models")
Error in eval(expr, envir, enclos) : object 'x' not found
Calls: %>% ... .Call -> .f -> <Anonymous> -> ncvTest.lm -> eval -> eval
Execution halted

checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘piecewiseSEM’
```

## slackr (1.4.2)
Maintainer: Bob Rudis <bob@rudis.net>  
Bug reports: https://github.com/hrbrmstr/slackr/issues

0 errors | 0 warnings | 0 notes

## solrium (0.4.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: http://www.github.com/ropensci/solrium/issues

0 errors | 0 warnings | 0 notes

## sophisthse (0.7.0)
Maintainer: Boris Demeshev <boris.demeshev@gmail.com>

0 errors | 0 warnings | 0 notes

## sorvi (0.7.26)
Maintainer: Leo Lahti <louhos@googlegroups.com>  
Bug reports: https://github.com/ropengov/sorvi/issues

0 errors | 0 warnings | 1 note 

```
checking R code for possible problems ... NOTE
convert_municipality_names: no visible global function definition for
  ‘read.csv’
regression_plot: no visible binding for global variable ‘loess’
regression_plot: no visible global function definition for
  ‘colorRampPalette’
regression_plot: no visible global function definition for
  ‘loess.control’
regression_plot: no visible global function definition for ‘predict’
regression_plot : <anonymous>: no visible global function definition
  for ‘quantile’
regression_plot : <anonymous>: no visible global function definition
  for ‘pnorm’
regression_plot: no visible global function definition for
  ‘flush.console’
regression_plot: no visible global function definition for ‘density’
Undefined global functions or variables:
  colorRampPalette density flush.console loess loess.control pnorm
  predict quantile read.csv
Consider adding
  importFrom("grDevices", "colorRampPalette")
  importFrom("stats", "density", "loess", "loess.control", "pnorm",
             "predict", "quantile")
  importFrom("utils", "flush.console", "read.csv")
to your NAMESPACE file.
```

## sourceR (1.0.0)
Maintainer: Poppy Miller <p.miller@lancaster.ac.uk>

0 errors | 0 warnings | 0 notes

## sp500SlidingWindow (0.1.0)
Maintainer: George Fisher <george@georgefisher.com>

0 errors | 0 warnings | 0 notes

## SpaCCr (0.1.0)
Maintainer: John Nagorski<jn13@rice.edu>

0 errors | 0 warnings | 0 notes

## SpaDES (1.3.1)
Maintainer: Alex M Chubaty <alexander.chubaty@canada.ca>  
Bug reports: https://github.com/PredictiveEcology/SpaDES/issues

1 error  | 2 warnings | 2 notes

```
checking tests ... ERROR
  Running ‘test-all.R’ [69s/154s]
Running the tests in ‘tests/test-all.R’ failed.
Last 13 lines of output:
  [35m## RMarkdown[0m
  
  RMarkdown syntax allows R code, outputs, and figures to be rendered in the documm[19;1Hentation.
  
  For help writing in RMarkdown, see http://rmarkdown.rstudio.com/.
  
  [35m# Usage[0m[24;63H1,1[11CTop[1;1H[34h[?25h[24;1H[?1l>[?1049lVim: Error reading input, exiting...
  
  Vim: Finished.
  [24;1Htestthat results ================================================================
  OK: 882 SKIPPED: 23 FAILED: 1
  1. Failure: spades calls with different signatures don't work (@test-simulation.R#195) 
  
  Error: testthat unit tests failed
  Execution halted

checking whether package ‘SpaDES’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/SpaDES.Rcheck/00install.out’ for details.

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

    RFoptions

The following object is masked from 'package:raster':

    atan2

... 8 lines ...

Attaching package: 'grid'

The following object is masked from 'package:SpaDES':

    gpar

Quitting from lines 352-356 (ii-modules.Rmd) 
Error: processing vignette 'ii-modules.Rmd' failed with diagnostics:
cannot open the connection
Execution halted

checking package dependencies ... NOTE
Package suggested but not available for checking: ‘fastshp’

checking installed package size ... NOTE
  installed size is  5.6Mb
  sub-directories of 1Mb or more:
    R     2.5Mb
    doc   2.1Mb
```

## sparklyr (0.5.3)
Maintainer: Javier Luraschi <javier@rstudio.com>  
Bug reports: https://github.com/rstudio/sparklyr/issues

0 errors | 1 warning  | 0 notes

```
checking S3 generic/method consistency ... WARNING
copy_to:
  function(dest, df, name, overwrite, ...)
copy_to.spark_connection:
  function(dest, df, name, memory, repartition, overwrite, ...)

See section ‘Generic functions and methods’ in the ‘Writing R
Extensions’ manual.
```

## sparsediscrim (0.2.3)
Maintainer: John A. Ramey <johnramey@gmail.com>

0 errors | 0 warnings | 0 notes

## sparseHessianFD (0.3.2)
Maintainer: Michael Braun <braunm@smu.edu>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Error in texi2dvi(file = file, pdf = TRUE, clean = clean, quiet = quiet,  : 
  Running 'texi2dvi' on 'sparseHessianFD.tex' failed.
LaTeX errors:
! LaTeX Error: File `algorithm.sty' not found.

Type X to quit or <RETURN> to proceed,
or enter new name. (Default extension: sty)

! Emergency stop.
<read *> 
         
l.27 \usepackage
                {algorithmic}^^M
!  ==> Fatal error occurred, no output PDF file produced!
Calls: buildVignettes -> texi2pdf -> texi2dvi
Execution halted

```

## SpatialEpiApp (0.1)
Maintainer: Paula Moraga <p.e.moraga-serrano@lancaster.ac.uk>

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Package suggested but not available for checking: ‘INLA’
```

## spbabel (0.4.5)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spbabel/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’ [32s/33s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
             where = .classEnv(ClassDef)))
         if (identical(ok, FALSE)) 
             stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
                 dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
         TRUE
     })(structure("SpatialPointsDataFrame", package = "sp"), "data", "matrix") at /tmp/RtmpvuUUir/devtools345049746f77/sp/R/SpatialLinesDataFrame-methods.R:93
  10: stop(gettextf("assignment of an object of class %s is not valid for @%s in an object of class %s; is(value, \"%s\") is not TRUE", 
         dQuote(valueClass), sQuote(name), dQuote(cl), slotClass), domain = NA)
  
  testthat results ================================================================
  OK: 57 SKIPPED: 2 FAILED: 1
  1. Error: utils tests work (@test-utils.R#16) 
  
  Error: testthat unit tests failed
  Execution halted
```

## spdplyr (0.1.2)
Maintainer: Michael D. Sumner <mdsumner@gmail.com>  
Bug reports: https://github.com/mdsumner/spdplyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘spdplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: dplyr-Spatial
> ### Title: Dplyr verbs for Spatial
> ### Aliases: arrange_.Spatial distinct_.Spatial dplyr-Spatial
> ###   filter_.Spatial group_by_.Spatial inner_join.Spatial
> ###   left_join.Spatial mutate_.Spatial rename_.Spatial select_.Spatial
... 12 lines ...
Attaching package: ‘raster’

The following object is masked from ‘package:dplyr’:

    select

> wrld_simpl %>% mutate(NAME = "allthesame", REGION = row_number())
Error in UseMethod("mutate") : 
  no applicable method for 'mutate' applied to an object of class "c('SpatialPolygonsDataFrame', 'SpatialPolygons', 'Spatial', 'SpatialVector')"
Calls: %>% ... _fseq -> freduce -> withVisible -> <Anonymous> -> mutate
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  5. Error: (unknown) (@test-basic-dplyr.R#127) ----------------------------------
  no applicable method for 'filter' applied to an object of class "c('SpatialPolygonsDataFrame', 'SpatialPolygons', 'Spatial')"
  1: filter(wrld_simpl, NAME %in% c("Portugal", "France", "Spain")) at testthat/test-basic-dplyr.R:127
  
  testthat results ================================================================
  OK: 2 SKIPPED: 3 FAILED: 5
  1. Error: filter works for all geometric types (@test-basic-dplyr.R#20) 
  2. Error: arrange works (@test-basic-dplyr.R#46) 
  3. Error: distinct works (@test-basic-dplyr.R#92) 
  4. Error: mutate works (@test-basic-dplyr.R#98) 
  5. Error: (unknown) (@test-basic-dplyr.R#127) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: sp
Checking rgeos availability: TRUE
Loading required package: dplyr

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 31-37 (spdplyr.Rmd) 
Error: processing vignette 'spdplyr.Rmd' failed with diagnostics:
no applicable method for 'filter' applied to an object of class "c('SpatialPointsDataFrame', 'SpatialPoints', 'Spatial')"
Execution halted

```

## spellcheckr (0.1.2)
Maintainer: Selva Prabhakaran <selva86@gmail.com>

0 errors | 0 warnings | 0 notes

## sqlscore (0.1.1)
Maintainer: William Brannon <wwbrannon@email.wm.edu>  
Bug reports: https://github.com/wwbrannon/sqlscore/issues

0 errors | 0 warnings | 0 notes

## srvyr (0.2.0)
Maintainer: Greg Freedman Ellis <greg.freedman@gmail.com>  
Bug reports: https://github.com/gergness/srvyr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘srvyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: as_survey
> ### Title: Create a tbl_svy from a data.frame
> ### Aliases: as_survey as_survey.data.frame as_survey.survey.design2
> ###   as_survey.svyrepdesign as_survey.tbl_sql as_survey.twophase2
> ###   as_survey_
... 23 lines ...

The following objects are masked from ‘package:base’:

    intersect, setdiff, setequal, union

> data(api)
> 
> # stratified sample
> dstrata <- apistrat %>%
+   as_survey(strata = stype, weights = pw)
Error: All select() inputs must resolve to integer column positions. The following do not:

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  testthat results ================================================================
  OK: 3 SKIPPED: 30 FAILED: 14
  1. Error: as_survey correctly uses survey design's arguments (@expect-equality.R#52) 
  2. Error: as_survey correctly uses replicate's arguments (@expect-equality.R#52) 
  3. Error: as_survey correctly uses twophase's arguments (@expect-equality.R#52) 
  4. Error: as_survey_design works with both tbl_dfs and data.frames (@expect-equality.R#52) 
  5. Error: as_survey_design ids argument = 1 doesn't use the first column as an id #7 (@expect-equality.R#52) 
  6. Error: (unknown) (@test_as_survey_rep.r#18) 
  7. Error: (unknown) (@test_as_survey_twophase.r#14) 
  8. Error: (unknown) (@test_basic_example.r#8) 
  9. Error: (unknown) (@test_cascade.r#7) 
  1. ...
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
    dotchart


Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

... 6 lines ...


Attaching package: 'srvyr'

The following object is masked from 'package:stats':

    filter

Quitting from lines 63-69 (srvyr-vs-survey.Rmd) 
Error: processing vignette 'srvyr-vs-survey.Rmd' failed with diagnostics:
All select() inputs must resolve to integer column positions. The following do not:
```

## ss3sim (0.9.3)
Maintainer: Sean Anderson <sean@seananderson.ca>

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘ss3sim-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: change_em_binning
> ### Title: Change population and observed length composition bins in an SS
> ###   estimation model
> ### Aliases: change_em_binning
> 
... 49 lines ...
14 2002    1   1      0    0      1       4       5     3  1  1  1
15 2002    1   1      0    0      1       5       6     3  1  1  1
16 2002    1   1      0    0      1       6       7     3  1  1  1
17 2002    1   1      0    0      1       7       8     3  1  1  1
18 2002    1   1      0    0      1       8      -1     3  1  1  1
> newdat <- change_em_binning(olddat, dat_file_out = NULL, bin_vector = c(4, 6, 8),
+   lbin_method = 1, write_file = FALSE)
Warning: 'summarise_each_' is deprecated.
Use 'summarise_all' instead.
See help("Deprecated")
Error: All select() inputs must resolve to integer column positions. The following do not:

checking whether package ‘ss3sim’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/ss3sim.Rcheck/00install.out’ for details.
```

## statar (0.6.2)
Maintainer: Matthieu Gomez <mattg@princeton.edu>  
Bug reports: https://github.com/matthieugomez/statar/issues

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘statar-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: fill_gap
> ### Title: Add rows corresponding to gaps in some variable
> ### Aliases: fill_gap fill_gap_
> 
> ### ** Examples
... 22 lines ...
The following object is masked from ‘package:base’:

    date

> df <- data_frame(
+     id    = c(1, 1, 1, 2),
+     datem  = as.monthly(mdy(c("04/03/1992", "01/04/1992", "03/15/1992", "05/11/1992"))),
+     value = c(4.1, 4.5, 3.3, 3.2)
+ )
> df %>% group_by(id) %>% fill_gap(datem)
Error: All select() inputs must resolve to integer column positions. The following do not:

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  1: expect_equal(fill_gap(groupeddf, datem)$value, c(4.5, NA, 3.3, 4.1, 3.2)) at testthat/test_fill.R:11
  2: compare(object, expected, ...) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/expect-equality.R:52
  3: fill_gap(groupeddf, datem) at /tmp/Rtmp8QnkSI/devtoolse928789ce822/hadley-testthat-b72a228/R/compare.R:12
  4: fill_gap_(x, .dots = lazy_dots(...), full = full, roll = roll, rollends = rollends)
  5: select_vars_(names(x), dots, exclude = byvars)
  6: select_vars(vars, !(!(!args)), include = include, exclude = exclude) at /home/muelleki/git/R/dplyr/R/select-vars.R:107
  7: abort(glue("All select() inputs must resolve to integer column positions. \\\n       The following do not:\n       {labels}", 
         labels = paste("* ", labels, collapse = "\n"))) at /home/muelleki/git/R/dplyr/R/select-vars.R:76
  
  testthat results ================================================================
  OK: 39 SKIPPED: 0 FAILED: 1
  1. Error: (unknown) (@test_fill.R#11) 
  
  Error: testthat unit tests failed
  Execution halted
```

## stationaRy (0.4.1)
Maintainer: Richard Iannone <riannone@me.com>  
Bug reports: https://github.com/rich-iannone/stationaRy/issues

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  8.1Mb
```

## statip (0.1.4)
Maintainer: Paul Poncet <paulponcet@yahoo.fr>  
Bug reports: https://github.com/paulponcet/statip/issues

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘distrEx’
```

## statisticalModeling (0.3.0)
Maintainer: Daniel Kaplan <kaplan@macalester.edu>

0 errors | 0 warnings | 0 notes

## sticky (0.4.0)
Maintainer: Christopher Brown <chris.brown@decisionpatterns.com>  
Bug reports: https://github.com/decisionpatterns/sticky/issues

0 errors | 0 warnings | 0 notes

## stormwindmodel (0.1.0)
Maintainer: Brooke Anderson <brooke.anderson@colostate.edu>  
Bug reports: https://github.com/geanders/stormwindmodel/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

... 8 lines ...
The following object is masked from 'package:dplyr':

    combine

Map from URL : http://maps.googleapis.com/maps/api/staticmap?center=georgia&zoom=5&size=640x640&scale=2&maptype=terrain&language=en-EN&sensor=false
Information from URL : http://maps.googleapis.com/maps/api/geocode/json?address=georgia&sensor=false
Quitting from lines 220-233 (Details.Rmd) 
Error: processing vignette 'Details.Rmd' failed with diagnostics:
GeomRasterAnn was built with an incompatible version of ggproto.
Please reinstall the package that provides this extension.
Execution halted
```

## stplanr (0.1.7-3)
Maintainer: Robin Lovelace <rob00x@gmail.com>  
Bug reports: https://github.com/ropensci/stplanr/issues

1 error  | 0 warnings | 1 note 

```
checking whether package ‘stplanr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/stplanr.Rcheck/00install.out’ for details.

checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘tmap’ ‘tmaptools’ ‘OpenStreetMap’
```

## superheat (0.1.0)
Maintainer: Rebecca Barter <rebeccabarter@berkeley.edu>

0 errors | 0 warnings | 0 notes

## surveybootstrap (0.0.1)
Maintainer: Dennis M. Feehan <feehan@berkeley.edu>

0 errors | 0 warnings | 0 notes

## SWMPr (2.2.0)
Maintainer: Marcus W. Beck <mbafs2012@gmail.com>  
Bug reports: http://github.com/fawda123/SWMPr/issues

0 errors | 0 warnings | 0 notes

## taber (0.1.0)
Maintainer: Seth Wenchel <seth@wenchel.com>  
Bug reports: http://github.com/restonslacker/taber/issues

0 errors | 0 warnings | 0 notes

## tadaatoolbox (0.10.0)
Maintainer: Lukas Burk <lukas@quantenbrot.de>  
Bug reports: https://github.com/tadaadata/tadaatoolbox/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tadaatoolbox-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: tadaa_nom
> ### Title: Get all the nominal stats
> ### Aliases: tadaa_nom
> 
> ### ** Examples
> 
> tadaa_nom(ngo$abschalt, ngo$geschl)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 37-38 (overview.Rmd) 
Error: processing vignette 'overview.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## Tcomp (1.0.0)
Maintainer: Peter Ellis <peter.ellis2013nz@gmail.com>  
Bug reports: https://github.com/ellisp/Tcomp-r-package/issues

0 errors | 0 warnings | 0 notes

## tcR (2.2.1.11)
Maintainer: Vadim Nazarov <vdm.nazarov@gmail.com>  
Bug reports: https://github.com/imminfo/tcr/issues

0 errors | 0 warnings | 3 notes

```
checking installed package size ... NOTE
  installed size is  6.3Mb
  sub-directories of 1Mb or more:
    data   1.2Mb
    doc    3.9Mb

checking S3 generic/method consistency ... NOTE
Found the following apparent S3 methods exported but not registered:
  slice.fun
See section ‘Registering S3 methods’ in the ‘Writing R Extensions’
manual.

checking Rd \usage sections ... NOTE
S3 methods shown with full name in documentation object 'top.fun':
  ‘slice.fun’

The \usage entries for S3 methods should use the \method markup and not
their full name.
See chapter ‘Writing R documentation files’ in the ‘Writing R
Extensions’ manual.
```

## TeachBayes (0.1.0)
Maintainer: Jim Albert <albert@bgsu.edu>

0 errors | 0 warnings | 0 notes

## tempcyclesdata (1.0.1)
Maintainer: George Wang <biogeek@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  6.0Mb
  sub-directories of 1Mb or more:
    data   5.9Mb
```

## testassay (0.1.0)
Maintainer: Michael C Sachs <sachsmc@gmail.com>

0 errors | 0 warnings | 0 notes

## texmexseq (0.3)
Maintainer: Scott Olesen <swo@mit.edu>

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘texmexseq-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: quad.table
> ### Title: Create and plot "quads" of samples
> ### Aliases: quad.table quad.plot
> ### Keywords: hplot manip design
> 
... 12 lines ...
Warning in value[[3L]](cond) : fit 2 failed
> 
> # pull out a quad, imagining that samples 1 and 2 were the control samples
> # and 3 and 4 were the treatment
> q <- quad.table(z.table, 'sample1', 'sample2', 'sample3', 'sample4')
> 
> # plot it
> p <- quad.plot(q)
Error in get("d.control") : object 'd.control' not found
Calls: quad.plot ... map -> lapply -> FUN -> overscope_eval -> .Call -> get
Execution halted
```

## textmining (0.0.1)
Maintainer: Jan Idziak <JanIdziak@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking whether package ‘textmining’ can be installed ... WARNING
Found the following significant warnings:
  Warning: no DISPLAY variable so Tk is not available
See ‘/home/muelleki/git/R/dplyr/revdep/checks/textmining.Rcheck/00install.out’ for details.
```

## textreuse (0.1.4)
Maintainer: Lincoln Mullen <lincoln@lincolnmullen.com>  
Bug reports: https://github.com/ropensci/textreuse/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘textreuse-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: lsh_candidates
> ### Title: Candidate pairs from LSH comparisons
> ### Aliases: lsh_candidates
> 
> ### ** Examples
> 
> dir <- system.file("extdata/legal", package = "textreuse")
> minhash <- minhash_generator(200, seed = 234)
> corpus <- TextReuseCorpus(dir = dir,
+                           tokenizer = tokenize_ngrams, n = 5,
+                           minhash_func = minhash)
> buckets <- lsh(corpus, bands = 50)
> lsh_candidates(buckets)
Error: Cannot convert objects of type `double` to `tidy_quote`
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  14: coerce_type(lazy, "tidy_quote", quote = lazy, symbol = , language = quosure(lazy, 
         env), string = {
         if (warn) 
             warn_text_se()
         parse_f(lazy, env)
     }, list = coerce_class(lazy, "tidy_quote", lazy = quosure(lazy$expr, lazy$env))) at /home/muelleki/git/R/dplyr/R/compat-lazyeval.R:23
  15: abort_coercion(.x, .to) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:422
  16: abort(paste0("Cannot convert objects of type `", type_of(x), "` to `", to, "`")) at /tmp/RtmpTRHGp3/devtoolsa70b5bb6f77f/hadley-rlang-2376dd6/R/types.R:435
  
  testthat results ================================================================
  OK: 129 SKIPPED: 2 FAILED: 1
  1. Error: (unknown) (@test-lsh.R#10) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 75-79 (textreuse-minhash.Rmd) 
Error: processing vignette 'textreuse-minhash.Rmd' failed with diagnostics:
Cannot convert objects of type `double` to `tidy_quote`
Execution halted

```

## textstem (0.0.1)
Maintainer: Tyler Rinker <tyler.rinker@gmail.com>  
Bug reports: http://github.com/trinker/textstem/issues

0 errors | 0 warnings | 0 notes

## TH.data (1.0-8)
Maintainer: Torsten Hothorn <Torsten.Hothorn@R-project.org>

0 errors | 0 warnings | 1 note 

```
checking installed package size ... NOTE
  installed size is  5.3Mb
  sub-directories of 1Mb or more:
    data   1.2Mb
    rda    3.8Mb
```

## tictactoe (0.2.0)
Maintainer: Kota Mori <kmori05@gmail.com>  
Bug reports: https://github.com/kota7/tictactoe/issues

0 errors | 0 warnings | 0 notes

## tidyjson (0.2.1)
Maintainer: Jeremy Stanley <jeremy.stanley@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘tidyjson-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: companies
> ### Title: Startup company information for 1,000 companies
> ### Aliases: companies
> 
> ### ** Examples
... 34 lines ...
+   enter_object("relationships") %>%
+   gather_array("relationship.index") %>%
+   spread_values(
+     is.past = jlogical("is_past"),
+     name = jstring("person", "permalink"),
+     title = jstring("title")
+   )
Error in eval(expr, envir, enclos) : 
  argument "json.column" is missing, with no default
Calls: %>% ... tryCatchList -> tryCatchOne -> doTryCatch -> eval -> eval
Execution halted

checking R code for possible problems ... NOTE
read_json: no visible global function definition for ‘tail’
Undefined global functions or variables:
  tail
Consider adding
  importFrom("utils", "tail")
to your NAMESPACE file.
```

## tidyquant (0.4.0)
Maintainer: Matt Dancho <mdancho@business-science.io>  
Bug reports: https://github.com/mdancho84/tidyquant/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tidyquant-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: scale_manual
> ### Title: tidyquant colors and fills for ggplot2.
> ### Aliases: scale_manual scale_color_tq scale_fill_tq
> 
> ### ** Examples
> 
> # Load libraries
> library(tidyquant)
> 
> # Get stock prices
> stocks <- c("AAPL", "FB", "NFLX") %>%
+     tq_get(from = "2013-01-01",
+            to   = "2017-01-01")
Error in mutate_impl(.data, dots) : object '.x' not found
Calls: %>% ... <Anonymous> -> mutate.tbl_df -> mutate_impl -> .Call
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’ [9s/28s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
         from = "2015-01-01", to = "2016-01-01"))) at /home/muelleki/git/R/dplyr/R/manip.r:206
  10: mutate_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:83
  
  testthat results ================================================================
  OK: 97 SKIPPED: 2 FAILED: 7
  1. Error: (unknown) (@test_tq_get_compound_getters.R#7) 
  2. Error: (unknown) (@test_tq_get_financials.R#7) 
  3. Error: (unknown) (@test_tq_get_stock_prices.R#10) 
  4. Error: (unknown) (@test_tq_mutate.R#27) 
  5. Error: (unknown) (@test_tq_performance.R#8) 
  6. Error: (unknown) (@test_tq_portfolio.R#8) 
  7. Error: (unknown) (@test_tq_transmute.R#12) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
  number of rows of result is not a multiple of vector length (arg 2)
Warning: Computation failed in `stat_b_bands_ribbon()`:
All inputs to rbind.fill must be data.frames
Warning in cbind(new, unique[rep(1, nrow(new)), missing, drop = FALSE]) :
  number of rows of result is not a multiple of vector length (arg 2)
Warning: Computation failed in `stat_b_bands_ribbon()`:
All inputs to rbind.fill must be data.frames
... 8 lines ...
All inputs to rbind.fill must be data.frames
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Getting data...

Getting data...

Quitting from lines 115-117 (TQ01-core-functions-in-tidyquant.Rmd) 
Error: processing vignette 'TQ01-core-functions-in-tidyquant.Rmd' failed with diagnostics:
object '.x' not found
Execution halted
```

## tidyr (0.6.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/tidyverse/tidyr/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  x[2]: "2"
  y[2]: "b"
  
  x[3]: "2"
  y[3]: "b"
  
  
  testthat results ================================================================
  OK: 191 SKIPPED: 0 FAILED: 2
  1. Failure: .id creates vector of names for vector unnest (@test-unnest.R#59) 
  2. Failure: .id creates vector of names for data frame unnest (@test-unnest.R#69) 
  
  Error: testthat unit tests failed
  Execution halted
```

## tidyRSS (1.0.1)
Maintainer: Robert Myles McDonnell <robertmylesmcdonnell@gmail.com>  
Bug reports: https://github.com/RobertMyles/tidyrss/issues

0 errors | 0 warnings | 0 notes

## tidytext (0.1.2)
Maintainer: Julia Silge <julia.silge@gmail.com>  
Bug reports: http://github.com/juliasilge/tidytext/issues

1 error  | 1 warning  | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  7: withVisible(function_list[[k]](value)) at /tmp/RtmpiX8wKT/R.INSTALL339d2ac32b99/magrittr/R/freduce.R:12
  8: function_list[[k]](value)
  9: count(., document, wt = gamma)
  10: tally(x, wt = !(!wt), sort = sort) at /home/muelleki/git/R/dplyr/R/count-tally.R:113
  11: summarise(x, `:=`(!(!n_name), !(!n))) at /home/muelleki/git/R/dplyr/R/count-tally.R:81
  12: summarise.tbl_df(x, `:=`(!(!n_name), !(!n))) at /home/muelleki/git/R/dplyr/R/manip.r:135
  13: summarise_impl(.data, dots) at /home/muelleki/git/R/dplyr/R/tbl-df.r:94
  
  testthat results ================================================================
  OK: 132 SKIPPED: 0 FAILED: 2
  1. Error: can tidy beta matrix (@test-lda-tidiers.R#26) 
  2. Error: can tidy gamma matrix (@test-lda-tidiers.R#48) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union
... 8 lines ...

Attaching package: 'NLP'

The following object is masked from 'package:ggplot2':

    annotate

Quitting from lines 49-57 (tidying_casting.Rmd) 
Error: processing vignette 'tidying_casting.Rmd' failed with diagnostics:
invalid 'type' (closure) of argument
Execution halted
```

## tidyverse (1.1.1)
Maintainer: Hadley Wickham <hadley@rstudio.com>  
Bug reports: https://github.com/tidyverse/tidyverse/issues

0 errors | 0 warnings | 0 notes

## tigger (0.2.8)
Maintainer: Daniel Gadala-Maria <daniel.gadala-maria@yale.edu>  
Bug reports: https://bitbucket.org/kleinstein/tigger/issues

1 error  | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘tigger-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: getPopularMutationCount
> ### Title: Find Frequent Sequences' Mutation Counts
> ### Aliases: getPopularMutationCount
> 
> ### ** Examples
> 
> data(sample_db, germline_ighv)
> getPopularMutationCount(sample_db, germline_ighv)
Error: Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Loading required package: ggplot2

Attaching package: 'dplyr'

The following objects are masked from 'package:stats':

    filter, lag

The following objects are masked from 'package:base':

    intersect, setdiff, setequal, union

Quitting from lines 111-116 (Tigger-Vignette.Rmd) 
Error: processing vignette 'Tigger-Vignette.Rmd' failed with diagnostics:
Cannot convert objects of type `closure` to `tidy_quote`
Execution halted

```

## tigris (0.3.3)
Maintainer: Kyle Walker <kyle.walker@tcu.edu>

1 error  | 0 warnings | 0 notes

```
checking whether package ‘tigris’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/tigris.Rcheck/00install.out’ for details.
```

## timelineS (0.1.1)
Maintainer: Dahee Lee <dhlee99@gmail.com>  
Bug reports: https://github.com/daheelee/timelineS/issues

0 errors | 0 warnings | 0 notes

## titanic (0.1.0)
Maintainer: Paul Hendricks <paul.hendricks.2013@owu.edu>  
Bug reports: https://github.com/paulhendricks/titanic/issues

0 errors | 0 warnings | 0 notes

## tmap (1.8-1)
Maintainer: Martijn Tennekes <mtennekes@gmail.com>  
Bug reports: https://github.com/mtennekes/tmap/issues

1 error  | 0 warnings | 0 notes

```
checking package dependencies ... ERROR
Packages required but not available: ‘tmaptools’ ‘mapview’

See section ‘The DESCRIPTION file’ in the ‘Writing R Extensions’
manual.
```

## Tmisc (0.1.12)
Maintainer: Stephen Turner <vustephen@gmail.com>

0 errors | 0 warnings | 0 notes

## traits (0.2.0)
Maintainer: Scott Chamberlain <myrmecocystus@gmail.com>  
Bug reports: http://www.github.com/ropensci/traits/issues

0 errors | 0 warnings | 0 notes

## treeplyr (0.1.2)
Maintainer: Josef Uyeda <josef.uyeda@gmail.com>  
Bug reports: https://github.com/uyedaj/treeplyr/issues

1 error  | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘treeplyr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: filter_.treedata
> ### Title: Function for filtering rows from an object of class 'treedata'
> ### Aliases: filter.grouped_treedata filter.treedata
> ###   filter_.grouped_treedata filter_.treedata
> 
> ### ** Examples
> 
> data(anolis)
> td <- make.treedata(anolis$phy, anolis$dat, name_column=1)
> tdfilter <- filter(td, island=="Cuba", SVL > 3.5)
Error in UseMethod("filter") : 
  no applicable method for 'filter' applied to an object of class "c('treedata', 'list')"
Calls: filter
Execution halted
```

## trelliscope (0.9.7)
Maintainer: Ryan Hafen <rhafen@gmail.com>  
Bug reports: https://github.com/delta-rho/trelliscope/issues

0 errors | 0 warnings | 0 notes

## trelloR (0.1.0)
Maintainer: Jakub Chromec <chromec.jakub@gmail.com>  
Bug reports: https://github.com/jchrom/trelloR/issues

0 errors | 0 warnings | 0 notes

## turfR (0.8-7)
Maintainer: Jack Horne <jack@jackhorne.net>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘turfR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: turf
> ### Title: TURF Analysis for R
> ### Aliases: turf
> 
> ### ** Examples
> 
>   ##Example 1
>   ##Evaluate all combinations of 3, 4, 5 and 6 items drawn from 10 items.
>   data(turf_ex_data)
>   ex1 <- turf(turf_ex_data, 10, 3:6)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: turf -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking R code for possible problems ... NOTE
turf: no visible global function definition for ‘read.table’
turf: no visible global function definition for ‘flush.console’
turf.combos: no visible global function definition for ‘combn’
Undefined global functions or variables:
  combn flush.console read.table
Consider adding
  importFrom("utils", "combn", "flush.console", "read.table")
to your NAMESPACE file.
```

## uaparserjs (0.1.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/uaparserjs/issues

0 errors | 0 warnings | 0 notes

## ubeR (0.1.4)
Maintainer: Andrew Collier <andrew@exegetic.biz>  
Bug reports: https://github.com/DataWookie/ubeR/issues

0 errors | 0 warnings | 0 notes

## unpivotr (0.1.0)
Maintainer: Duncan Garmonsway <nacnudus@gmail.com>  
Bug reports: https://github.com/nacnudus/unpivotr/issues

2 errors | 1 warning  | 0 notes

```
checking examples ... ERROR
Running examples in ‘unpivotr-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: join_header
> ### Title: Join a bag of data cells to some header, by proximity in a given
> ###   direction, e.g. NNW searches up and up-left from a data cell to find
> ###   a header cell.
> ### Aliases: join_header N E S W NNW NNE ENE ESE SSE SSW WSW WNW ABOVE LEFT
... 119 lines ...
5   2   5  0 - 6
6   2   6 7 - 10

> # From each data cell, search for the nearest one of each of the headers
> datacells %>%
+   NNW(col_headers$`1`) %>%
+   N(col_headers$`2`) %>%
+   WNW(row_headers$`1`) %>%
+   W(row_headers$`2`)
Error: Join column not found in lhs: col
Execution halted

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  
  testthat results ================================================================
  OK: 87 SKIPPED: 0 FAILED: 9
  1. Error: Compass directions N, NNW, W, and WNW work (@test-join_header.R#128) 
  2. Error: Compass directions NNE and WSW work (@test-join_header.R#155) 
  3. Error: Compass directions S, SSE , E and ESE work (@test-join_header.R#178) 
  4. Error: Compass directions SSW and ENE work (@test-join_header.R#205) 
  5. Error: Compass directions ABOVE and LEFT work (@test-join_header.R#228) 
  6. Error: Compass directions BELOW and RIGHT work (@test-join_header.R#251) 
  7. Error: Compass directions ABOVE and LEFT work with boundaries (@test-join_header.R#286) 
  8. Error: Compass directions BELOW and RIGHT work with boundaries (@test-join_header.R#332) 
  9. Error: join_header() works (@test-join_header.R#361) 
  
  Error: testthat unit tests failed
  Execution halted

checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 56-61 (compass-directions.Rmd) 
Error: processing vignette 'compass-directions.Rmd' failed with diagnostics:
cannot coerce type 'closure' to vector of type 'integer'
Execution halted

```

## unvotes (0.1.0)
Maintainer: David Robinson <admiral.david@gmail.com>  
Bug reports: http://github.com/dgrtwo/unvotes/issues

0 errors | 0 warnings | 0 notes

## useful (1.2.1)
Maintainer: Jared P. Lander <packages@jaredlander.com>

2 errors | 0 warnings | 0 notes

```
checking examples ... ERROR
Running examples in ‘useful-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: constant
> ### Title: constant
> ### Aliases: constant
> 
> ### ** Examples
> 
> constant(4)(1:10)
[1] 4
> 
> theDF <- data.frame(A=1:10, B=1:10, C=1:10)
> theDF[c(1, 4, 6), c(1)] <- NA
> theDF[c(3, 4, 8), c(3)] <- NA
> simple.impute(theDF, constant(4))
Warning: 'mutate_each_' is deprecated.
Use 'mutate_all' instead.
See help("Deprecated")
Error: All select() inputs must resolve to integer column positions. The following do not:

checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  12: function_list[[k]](value)
  13: mutate_each_(., funs(simple.impute(., fun = fun)), vars = names(x))
  14: apply_vars(funs, vars, tbl) at /home/muelleki/git/R/dplyr/R/colwise.R:280
  15: select_vars(tbl_vars(tbl), !(!(!vars)), exclude = group_vars(tbl)) at /home/muelleki/git/R/dplyr/R/colwise.R:191
  16: abort(glue("All select() inputs must resolve to integer column positions. \\\n       The following do not:\n       {labels}", 
         labels = paste("* ", labels, collapse = "\n"))) at /home/muelleki/git/R/dplyr/R/select-vars.R:76
  
  testthat results ================================================================
  OK: 330 SKIPPED: 0 FAILED: 3
  1. Failure: cart2pol maps cartesian to polar correctly (@test-coordinates.R#131) 
  2. Failure: cart2pol maps cartesian to polar correctly and returns degrees (@test-coordinates.R#150) 
  3. Error: simple.impute.data.frame works as expected (@test-simple-impute.r#68) 
  
  Error: testthat unit tests failed
  Execution halted
```

## vaersNDvax (1.0.4)
Maintainer: Irucka Embry <iembry@ecoccs.com>  
Bug reports: https://gitlab.com/iembry/vaersND/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘vaers’ ‘vaersND’
```

## vaersvax (1.0.4)
Maintainer: Irucka Embry <iembry@ecoccs.com>  
Bug reports: https://gitlab.com/iembry/vaers/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Packages suggested but not available for checking: ‘vaers’ ‘vaersND’
```

## valr (0.1.1)
Maintainer: Jay Hesselberth <jay.hesselberth@gmail.com>  
Bug reports: https://github.com/jayhesselberth/valr/issues

1 error  | 0 warnings | 0 notes

```
checking whether package ‘valr’ can be installed ... ERROR
Installation failed.
See ‘/home/muelleki/git/R/dplyr/revdep/checks/valr.Rcheck/00install.out’ for details.
```

## vcfR (1.4.0)
Maintainer: Brian J. Knaus <briank.lists@gmail.com>

1 error  | 0 warnings | 1 note 

```
checking examples ... ERROR
Running examples in ‘vcfR-Ex.R’ failed
The error most likely occurred in:

> base::assign(".ptime", proc.time(), pos = "CheckExEnv")
> ### Name: Convert to tidy data frames
> ### Title: Convert vcfR objects to tidy data frames
> ### Aliases: 'Convert to tidy data frames' extract_gt_tidy
> ###   extract_info_tidy vcfR2tidy vcf_field_names
> 
> ### ** Examples
> 
> # load the data
> data("vcfR_test")
> vcf <- vcfR_test
> 
> 
> # extract all the INFO and FORMAT fields into a list of tidy
> # data frames: fix, gt, and meta. Here we don't coerce columns
> # to integer or numeric types...
> Z <- vcfR2tidy(vcf)
Error in cbind_all(x) : cannot convert object to a data frame
Calls: vcfR2tidy ... eval -> cbind -> cbind -> bind_cols -> cbind_all -> .Call
Execution halted

checking installed package size ... NOTE
  installed size is  9.7Mb
  sub-directories of 1Mb or more:
    doc    3.0Mb
    libs   5.7Mb
```

## vdmR (0.2.3)
Maintainer: Tomokazu Fujino <fujino@fwu.ac.jp>

0 errors | 0 warnings | 0 notes

## VIM (4.6.0)
Maintainer: Matthias Templ <matthias.templ@gmail.com>

0 errors | 0 warnings | 1 note 

```
checking Rd cross-references ... NOTE
Package unavailable to check Rd xrefs: ‘StatDA’
```

## vkR (0.1)
Maintainer: Dmitriy Sorokin <dementiy@yandex.ru>  
Bug reports: https://github.com/Dementiy/vkR/issues

0 errors | 0 warnings | 0 notes

## vqtl (1.0)
Maintainer: Robert Corty <rcorty@gmail.com>

0 errors | 0 warnings | 0 notes

## vtreat (0.5.30)
Maintainer: John Mount <jmount@win-vector.com>  
Bug reports: https://github.com/WinVector/vtreat/issues

0 errors | 0 warnings | 0 notes

## VWPre (0.9.5)
Maintainer: Vincent Porretta <vincentporretta@gmail.com>

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Calculating proportions...
There are 320 data points with less than 20 samples per bin.
These can be examined and/or removed using the column 'NSamples'.
These occur at time bin(s): 1180
Subsequent Empirical Logit calculations may be influenced by the number of samples (depending on the number of observations requested).
Sampling rate(s) present in the data are: 50 Hz.

... 8 lines ...
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Warning: Removed 56 rows containing missing values (geom_point).
Warning: Removed 56 rows containing missing values (geom_path).
Warning: Removed 56 rows containing missing values (geom_errorbar).
Quitting from lines 81-87 (SR_Plotting.Rmd) 
Error: processing vignette 'SR_Plotting.Rmd' failed with diagnostics:
object 'talker' not found
Execution halted
```

## wakefield (0.3.0)
Maintainer: Tyler Rinker <tyler.rinker@gmail.com>  
Bug reports: https://github.com/trinker/wakefield/issues

0 errors | 0 warnings | 0 notes

## wallace (0.6.3)
Maintainer: Jamie M. Kass <jkass@gradcenter.cuny.edu>

0 errors | 0 warnings | 0 notes

## wand (0.2.0)
Maintainer: Bob Rudis <bob@rud.is>  
Bug reports: https://github.com/hrbrmstr/wand/issues

0 errors | 0 warnings | 0 notes

## wec (0.4)
Maintainer: Rense Nieuwenhuis <rense.nieuwenhuis@sofi.su.se>

0 errors | 0 warnings | 0 notes

## wfindr (0.1.0)
Maintainer: Iaroslav Domin <ya.domin@gmail.com>  
Bug reports: https://github.com/idmn/wfindr/issues

0 errors | 0 warnings | 0 notes

## WHO (0.2)
Maintainer: Eric Persson <expersso5@gmail.com>  
Bug reports: https://www.github.com/expersso/WHO/issues

1 error  | 0 warnings | 0 notes

```
checking tests ... ERROR
  Running ‘testthat.R’
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
  > test_check("WHO")
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  not compatible with STRSXP
  1: get_codes(TRUE) at testthat/tests.R:18
  2: dplyr::bind_rows(df_list)
  3: bind_rows_(x, .id) at /home/muelleki/git/R/dplyr/R/bind.r:86
  
  testthat results ================================================================
  OK: 1485 SKIPPED: 0 FAILED: 1
  1. Error: get_codes(TRUE) returns a data frame with positive length and more
            than three columns (@tests.R#18) 
  
  Error: testthat unit tests failed
  Execution halted
```

## WikidataQueryServiceR (0.1.0)
Maintainer: Mikhail Popov <mikhail@wikimedia.org>  
Bug reports: https://github.com/bearloga/WikidataQueryServiceR/issues

0 errors | 0 warnings | 0 notes

## wikipediatrend (1.1.10)
Maintainer: Peter Meissner <retep.meissner@gmail.com>  
Bug reports: https://github.com/petermeissner/wikipediatrend/issues

0 errors | 0 warnings | 1 note 

```
checking package dependencies ... NOTE
Packages suggested but not available for checking:
  ‘AnomalyDetection’ ‘BreakoutDetection’
```

## wordbankr (0.2.0)
Maintainer: Mika Braginsky <mika.br@gmail.com>  
Bug reports: http://github.com/langcog/wordbankr/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 27-31 (wordbankr.Rmd) 
Error: processing vignette 'wordbankr.Rmd' failed with diagnostics:
argument is missing, with no default
Execution halted

```

## worldmet (0.7.5)
Maintainer: David Carslaw <david.carslaw@york.ac.uk>  
Bug reports: http://github.com/davidcarslaw/worldmet/issues

0 errors | 0 warnings | 0 notes

## wrangle (0.3)
Maintainer: Tim Bergsma <bergsmat@gmail.com>

0 errors | 0 warnings | 0 notes

## wrswoR.benchmark (0.1-1)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/krlmlr/wrswoR.benchmark/issues

0 errors | 0 warnings | 0 notes

## wrswoR (1.0-1)
Maintainer: Kirill Müller <krlmlr+r@mailbox.org>  
Bug reports: https://github.com/krlmlr/wrswoR/issues

0 errors | 1 warning  | 0 notes

```
checking re-building of vignette outputs ... WARNING
Error in re-building vignettes:
  ...
Warning in engine$weave(file, quiet = quiet, encoding = enc) :
  Pandoc (>= 1.12.3) and/or pandoc-citeproc not available. Falling back to R Markdown v1.
Quitting from lines 622-635 (wrswoR.Rmd) 
Error: processing vignette 'wrswoR.Rmd' failed with diagnostics:

TeX was unable to calculate metrics for the following string
or character:

	36

Common reasons for failure include:
  * The string contains a character which is special to LaTeX unless
    escaped properly, such as % or $.
  * The string makes use of LaTeX commands provided by a package and
    the tikzDevice was not told to load the package.

The contents of the LaTeX log of the aborted run have been printed above,
it may contain additional details as to why the metric calculation failed.
Execution halted

```

## WRTDStidal (1.0.1)
Maintainer: Marcus W. Beck <mbafs2012@gmail.com>  
Bug reports: https://github.com/fawda123/wtreg_for_estuaries/issues

0 errors | 0 warnings | 0 notes

## WufooR (0.6.1)
Maintainer: John Malc <cincenko@outlook.com>  
Bug reports: http://github.com/dmpe/wufoor/issues

0 errors | 0 warnings | 0 notes

## yorkr (0.0.7)
Maintainer: Tinniam V Ganesh <tvganesh.85@gmail.com>

0 errors | 0 warnings | 0 notes

## ZeligChoice (0.9-4)
Maintainer: Christopher Gandrud <zelig.zee@gmail.com>  
Bug reports: https://github.com/IQSS/Zelig/issues

0 errors | 0 warnings | 0 notes

## ZeligEI (0.1-0)
Maintainer: James Honaker <zelig.zee@gmail.com>

0 errors | 0 warnings | 0 notes

## Zelig (5.0-17)
Maintainer: Christopher Gandrud <zelig.zee@gmail.com>  
Bug reports: https://github.com/IQSS/Zelig/issues

1 error  | 0 warnings | 1 note 

```
checking tests ... ERROR
  Running ‘testthat.R’ [58s/60s]
Running the tests in ‘tests/testthat.R’ failed.
Last 13 lines of output:
            mean         sd       50%      2.5%     97.5%
  [1,] 0.6990619 0.09467077 0.7081549 0.5028567 0.8603348
  pv
          0    1
  [1,] 0.31 0.69
  fd
             mean        sd        50%      2.5%       97.5%
  [1,] -0.2377065 0.1148547 -0.2380952 -0.466211 -0.01659926
  testthat results ================================================================
  OK: 195 SKIPPED: 1 FAILED: 2
  1. Error: REQUIRE TEST Bayes Diagnostics for factors (@test-bayesdiagnostics.R#19) 
  2. Error: REQUIRE TEST for to_zelig_mi (@test-utils.R#28) 
  
  Error: testthat unit tests failed
  Execution halted

checking installed package size ... NOTE
  installed size is 12.1Mb
  sub-directories of 1Mb or more:
    R  11.0Mb
```

## ztype (0.1.0)
Maintainer: Vincent Guyader <vincent@thinkr.fr>

0 errors | 0 warnings | 0 notes

